{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "__metaBEAT analysis of Illumina seq output for pollen transported by moths__"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The first step will be to __trim/clean our raw Illumina data__."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Prepare a text file specifying the samples to be processed including the format and location of the reads.\n",
    "\n",
    "The below command expects the Illumina data to be present in 2 fastq files (forward and reverse reads) per sample in a directory ./raw_data/. It expects the files to be named 'plateID_L001', followed by 'R1' or 'R2' to identify the forward/reverse read file respectively. \n",
    "\n",
    "We need a query map which lists these files along with the primer combinations for each well in each file, and the length of the primer sequence (including heterogeneity spacers, but NOT including any tags) to be trimmed off."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SA177\tfastq\traw_data/Moth1_S1_L001_R1_001.fastq.gz\traw_data/Moth1_S1_L001_R2_001.fastq.gz\tTCGCCTTA\tTAGATCGC\t20\t22\r",
      "\r\n",
      "SB140\tfastq\traw_data/Moth1_S1_L001_R1_001.fastq.gz\traw_data/Moth1_S1_L001_R2_001.fastq.gz\tAGCGTAGC\tCTCTCTAT\t20\t23\r",
      "\r\n",
      "T19\tfastq\traw_data/Moth1_S1_L001_R1_001.fastq.gz\traw_data/Moth1_S1_L001_R2_001.fastq.gz\tTCGCCTTA\tTATCCTCT\t20\t24\r",
      "\r\n",
      "Lg45\tfastq\traw_data/Moth1_S1_L001_R1_001.fastq.gz\traw_data/Moth1_S1_L001_R2_001.fastq.gz\tAGCGTAGC\tAGAGTAGA\t20\t25\r",
      "\r\n",
      "Q69\tfastq\traw_data/Moth1_S1_L001_R1_001.fastq.gz\traw_data/Moth1_S1_L001_R2_001.fastq.gz\tTCGCCTTA\tGTAAGGAG\t20\t26\r",
      "\r\n",
      "S72\tfastq\traw_data/Moth1_S1_L001_R1_001.fastq.gz\traw_data/Moth1_S1_L001_R2_001.fastq.gz\tAGCGTAGC\tACTGCATA\t20\t27\r",
      "\r\n",
      "SC66\tfastq\traw_data/Moth1_S1_L001_R1_001.fastq.gz\traw_data/Moth1_S1_L001_R2_001.fastq.gz\tTCGCCTTA\tAAGGAGTA\t20\t28\r",
      "\r\n",
      "S31\tfastq\traw_data/Moth1_S1_L001_R1_001.fastq.gz\traw_data/Moth1_S1_L001_R2_001.fastq.gz\tAGCGTAGC\tCTAAGCCT\t20\t29\r",
      "\r\n",
      "SC74\tfastq\traw_data/Moth1_S1_L001_R1_001.fastq.gz\traw_data/Moth1_S1_L001_R2_001.fastq.gz\tTCGCCTTA\tGCAGCGTA\t20\t22\r",
      "\r\n",
      "SA68\tfastq\traw_data/Moth1_S1_L001_R1_001.fastq.gz\traw_data/Moth1_S1_L001_R2_001.fastq.gz\tAGCGTAGC\tCTGCGCAT\t20\t28\r",
      "\r\n"
     ]
    }
   ],
   "source": [
    "!head Querymap_global.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Moth1_S1_L001_R1_001.fastq.gz  Moth3_S3_L001_R1_001.fastq.gz\r\n",
      "Moth1_S1_L001_R2_001.fastq.gz  Moth3_S3_L001_R2_001.fastq.gz\r\n",
      "Moth2_S2_L001_R1_001.fastq.gz  Moth4_S4_L001_R1_001.fastq.gz\r\n",
      "Moth2_S2_L001_R2_001.fastq.gz  Moth4_S4_L001_R2_001.fastq.gz\r\n"
     ]
    }
   ],
   "source": [
    "ls raw_data/"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "usage: metaBEAT.py [-h] [-Q <FILE>] [-B <FILE>] [--g_queries <FILE>] [-v] [-s]\n",
      "                   [-f] [-p] [-k] [-t] [-b] [-m <string>] [-n <INT>] [-E] [-e]\n",
      "                   [--read_stats_off] [--PCR_primer <FILE>]\n",
      "                   [--trim_adapter <FILE>] [--trim_qual <INT>] [--phred <INT>]\n",
      "                   [--trim_window <INT>] [--read_crop <INT>]\n",
      "                   [--trim_minlength <INT>] [--merge] [--product_length <INT>]\n",
      "                   [--merged_only] [--forward_only] [--length_filter <INT>]\n",
      "                   [--length_deviation <FLOAT>] [-R <FILE>] [--gb_out <FILE>]\n",
      "                   [--rec_check] [--gi_to_taxid <FILE>] [--cluster]\n",
      "                   [--clust_match <FLOAT>] [--clust_cov <INT>]\n",
      "                   [--blast_db <PATH>] [--blast_xml <PATH>]\n",
      "                   [--min_ident <FLOAT>] [--min_ali_length <FLOAT>]\n",
      "                   [--min_bit <INT>] [--refpkg <DIR>] [--jplace <FILE>]\n",
      "                   [--kraken_db <DIR>] [--rm_kraken_db] [-o OUTPUT_PREFIX]\n",
      "                   [--metadata METADATA] [--mock_meta_data]\n",
      "                   [-@ <email-address>] [--version]\n",
      "\n",
      "metaBEAT - metaBarcoding and Environmental DNA Analyses tool version:\n",
      "v.0.97.4-global\n",
      "\n",
      "optional arguments:\n",
      "  -h, --help            show this help message and exit\n",
      "  -Q <FILE>, --querylist <FILE>\n",
      "                        file containing a list of query files\n",
      "  -B <FILE>, --BIOM_input <FILE>\n",
      "                        OTU table in BIOM format\n",
      "  --g_queries <FILE>    fasta file containing query sequences (in combination\n",
      "                        with '-B' sequence headers are expected to match)\n",
      "  -v, --verbose         turn verbose output on\n",
      "  -s, --seqinfo         write out seq_info.csv file\n",
      "  -f, --fasta           write out ref.fasta file\n",
      "  -p, --pplace          perform phylogenetic placement\n",
      "  -k, --kraken          perform phylogenetic placement\n",
      "  -t, --taxids          write out taxid.txt file\n",
      "  -b, --blast           compile local blast db and blast queries\n",
      "  -m <string>, --marker <string>\n",
      "                        marker ID (default: marker)\n",
      "  -n <INT>, --n_threads <INT>\n",
      "                        Number of threads (default: 1)\n",
      "  -E, --extract_centroid_reads\n",
      "                        extract centroid reads to files\n",
      "  -e, --extract_all_reads\n",
      "                        extract reads to files\n",
      "  --read_stats_off      ommit writing read stats to file\n",
      "  --version             show program's version number and exit\n",
      "\n",
      "Query preprocessing:\n",
      "  The parameters in this group affect how the query sequences are processed\n",
      "\n",
      "  --PCR_primer <FILE>   PCR primers (provided in fasta file) to be clipped\n",
      "                        from reads\n",
      "  --trim_adapter <FILE>\n",
      "                        trim adapters provided in file\n",
      "  --trim_qual <INT>     minimum phred quality score (default: 30)\n",
      "  --phred <INT>         phred quality score offset - 33 or 64 (default: 33)\n",
      "  --trim_window <INT>   sliding window size (default: 5) for trimming; if\n",
      "                        average quality drops below the specified minimum\n",
      "                        quality all subsequent bases are removed from the\n",
      "                        reads\n",
      "  --read_crop <INT>     Crop reads to this length if they are longer than that\n",
      "                        (default: off)\n",
      "  --trim_minlength <INT>\n",
      "                        minimum length of reads to be retained after trimming\n",
      "                        (default: 50)\n",
      "  --merge               attempt to merge paired-end reads\n",
      "  --product_length <INT>\n",
      "                        estimated length of PCR product (specifying this\n",
      "                        option increases merging efficiency)\n",
      "  --merged_only         only process successfully merged read-pairs\n",
      "  --forward_only        only process sequences that contain forward reads\n",
      "                        (i.e. unmerged forward reads and merged reads)\n",
      "  --length_filter <INT>\n",
      "                        only process reads, which are within +/- 10 percent of\n",
      "                        this length\n",
      "  --length_deviation <FLOAT>\n",
      "                        allowed deviation (in percent) from length specified\n",
      "                        by --length_filter (default=0.1)\n",
      "\n",
      "Reference:\n",
      "  The parameters in this group affect the reference to be used in the\n",
      "  analyses\n",
      "\n",
      "  -R <FILE>, --REFlist <FILE>\n",
      "                        file containing a list of files to be used as\n",
      "                        reference sequences\n",
      "  --gb_out <FILE>       output the corrected gb file\n",
      "  --rec_check           check records to be used as reference\n",
      "  --gi_to_taxid <FILE>  comma delimited file containing 'gi accession,taxid'\n",
      "                        for a list of taxa\n",
      "\n",
      "Query clustering options:\n",
      "  The parameters in this group affect read clustering\n",
      "\n",
      "  --cluster             perform clustering of query sequences using vsearch\n",
      "  --clust_match <FLOAT>\n",
      "                        identity threshold for clustering in percent (default:\n",
      "                        1)\n",
      "  --clust_cov <INT>     minimum number of records in cluster (default: 1)\n",
      "\n",
      "BLAST search:\n",
      "  The parameters in this group affect BLAST search and BLAST based taxonomic\n",
      "  assignment\n",
      "\n",
      "  --blast_db <PATH>     path to precompiled blast database\n",
      "  --blast_xml <PATH>    path to Blast result in xml format\n",
      "  --min_ident <FLOAT>   minimum identity threshold in percent (default: 0.80)\n",
      "  --min_ali_length <FLOAT>\n",
      "                        minimum alignment length in percent of total query\n",
      "                        length (default: 0.95)\n",
      "  --min_bit <INT>       minimum bitscore (default: 80)\n",
      "\n",
      "Phylogenetic placement:\n",
      "  The parameters in this group affect phylogenetic placement\n",
      "\n",
      "  --refpkg <DIR>        PATH to refpkg for pplacer\n",
      "  --jplace <FILE>       phylogenetic placement result from prefious pplacer\n",
      "                        run in *.jplace format\n",
      "\n",
      "Kraken:\n",
      "  The parameters in this group affect taxonomic assignment using Kraken\n",
      "\n",
      "  --kraken_db <DIR>     PATH to a Kraken database\n",
      "  --rm_kraken_db        Remove Kraken database after successful completion\n",
      "\n",
      "BIOM OUTPUT:\n",
      "  The arguments in this groups affect the output in BIOM format\n",
      "\n",
      "  -o OUTPUT_PREFIX, --output_prefix OUTPUT_PREFIX\n",
      "                        prefix for BIOM output files (default='metaBEAT')\n",
      "  --metadata METADATA   comma delimited file containing metadata (optional)\n",
      "  --mock_meta_data      add mock metadata to the samples in the BIOM output\n",
      "\n",
      "Entrez identification:\n",
      "  metaBEAT is querying the NCBI Entrez databases, please provide an email\n",
      "  address for identification\n",
      "\n",
      "  -@ <email-address>, --email <email-address>\n",
      "                        provide your email address for identification to NCBI\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "metaBEAT_global.py -h"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "metaBEAT - metaBarcoding and Environmental DNA Analyses tool\n",
      "version: v.0.97.4-global\n",
      "\n",
      "\n",
      "Wed Oct 19 16:43:52 2016\n",
      "\n",
      "/usr/bin/metaBEAT_global.py -Q Querymap_global.txt --trim_qual 30 --trim_minlength 90 --merge --product_length 350 --merged_only -R REFmap.txt --cluster --clust_match 1 --clust_cov 5 --min_ident 0.90 -o MothPollenRestart -@ callumjmacgregor@gmail.com\n",
      "\n",
      "\n",
      "metaBEAT may be querying NCBI's Entrez databases to fetch/verify taxonomic ids. Entrez User requirements state that you need to identify yourself by providing an email address so that NCBI can contact you in case there is a problem.\n",
      "\n",
      "You have specified: 'callumjmacgregor@gmail.com'\n",
      "\n",
      "taxonomy.db found at /usr/bin/taxonomy.db\n",
      "\n",
      "Parsing querylist file\n",
      "\n",
      "Number of samples to process: 317\n",
      "Sequence input format: defaultdict(<type 'int'>, {'fastq': 336})\n",
      "Barcodes for demultiplexing provided for 336 samples\n",
      "Cropping instructions provided for 336 samples\n",
      "\n",
      "\n",
      "######## PROCESSING REFERENCE DATA ########\n",
      "\n",
      "\n",
      "processing reference/Delosperma.gb (containing 58 records)\n",
      "\n",
      "total number of valid records: 58\n",
      "\n",
      "\n",
      "processing reference/Crassulaceae.gb (containing 330 records)\n",
      "\n",
      "total number of valid records: 388\n",
      "\n",
      "\n",
      "processing reference/yorks_flora.gb (containing 9405 records)\n",
      "\n",
      "total number of valid records: 9793\n",
      "\n",
      "\n",
      "processing reference/Dracaena.gb (containing 37 records)\n",
      "\n",
      "total number of valid records: 9830\n",
      "\n",
      "\n",
      "Wed Oct 19 16:43:59 2016\n",
      "\n",
      "\n",
      "### DEMULTIPLEXING ###\n",
      "\n",
      "assessing basic characteristics\n",
      "data comes as paired end - ok\n",
      "process_shortreads -1 /home/working/raw_data/Moth1_S1_L001_R1_001.fastq.gz -2 /home/working/raw_data/Moth1_S1_L001_R2_001.fastq.gz -i gzfastq -o . -b barcodes --inline_inline -y fastq\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Reads trimmed shorter than 31 nucleotides will be discarded.\n",
      "Found 1 paired input file(s).\n",
      "Searching for single and paired-end, inlined barcodes.\n",
      "Loaded 96 barcodes (8bp / 8bp).\n",
      "Processing file 1 of 1 [Moth1_S1_L001_R1_001.fastq.gz]\n",
      "  Reading data from\n",
      "    /home/working/raw_data/Moth1_S1_L001_R1_001.fastq.gz and\n",
      "    /home/working/raw_data/Moth1_S1_L001_R2_001.fastq.gz\n",
      "  Processing short read 10000       \r",
      "  Processing short read 20000       \r",
      "  Processing short read 30000       \r",
      "  Processing short read 40000       \r",
      "  Processing short read 50000       \r",
      "  Processing short read 60000       \r",
      "  Processing short read 70000       \r",
      "  Processing short read 80000       \r",
      "  Processing short read 90000       \r",
      "  Processing short read 100000       \r",
      "  Processing short read 110000       \r",
      "  Processing short read 120000       \r",
      "  Processing short read 130000       \r",
      "  Processing short read 140000       \r",
      "  Processing short read 150000       \r",
      "  Processing short read 160000       \r",
      "  Processing short read 170000       \r",
      "  Processing short read 180000       \r",
      "  Processing short read 190000       \r",
      "  Processing short read 200000       \r",
      "  Processing short read 210000       \r",
      "  Processing short read 220000       \r",
      "  Processing short read 230000       \r",
      "  Processing short read 240000       \r",
      "  Processing short read 250000       \r",
      "  Processing short read 260000       \r",
      "  Processing short read 270000       \r",
      "  Processing short read 280000       \r",
      "  Processing short read 290000       \r",
      "  Processing short read 300000       \r",
      "  Processing short read 310000       \r",
      "  Processing short read 320000       \r",
      "  Processing short read 330000       \r",
      "  Processing short read 340000       \r",
      "  Processing short read 350000       \r",
      "  Processing short read 360000       \r",
      "  Processing short read 370000       \r",
      "  Processing short read 380000       \r",
      "  Processing short read 390000       \r",
      "  Processing short read 400000       \r",
      "  Processing short read 410000       \r",
      "  Processing short read 420000       \r",
      "  Processing short read 430000       \r",
      "  Processing short read 440000       \r",
      "  Processing short read 450000       \r",
      "  Processing short read 460000       \r",
      "  Processing short read 470000       \r",
      "  Processing short read 480000       \r",
      "  Processing short read 490000       \r",
      "  Processing short read 500000       \r",
      "  Processing short read 510000       \r",
      "  Processing short read 520000       \r",
      "  Processing short read 530000       \r",
      "  Processing short read 540000       \r",
      "  Processing short read 550000       \r",
      "  Processing short read 560000       \r",
      "  Processing short read 570000       \r",
      "  Processing short read 580000       \r",
      "  Processing short read 590000       \r",
      "  Processing short read 600000       \r",
      "  Processing short read 610000       \r",
      "  Processing short read 620000       \r",
      "  Processing short read 630000       \r",
      "  Processing short read 640000       \r",
      "  Processing short read 650000       \r",
      "  Processing short read 660000       \r",
      "  Processing short read 670000       \r",
      "  Processing short read 680000       \r",
      "  Processing short read 690000       \r",
      "  Processing short read 700000       \r",
      "  Processing short read 710000       \r",
      "  Processing short read 720000       \r",
      "  Processing short read 730000       \r",
      "  Processing short read 740000       \r",
      "  Processing short read 750000       \r",
      "  Processing short read 760000       \r",
      "  Processing short read 770000       \r",
      "  Processing short read 780000       \r",
      "  Processing short read 790000       \r",
      "  Processing short read 800000       \r",
      "  Processing short read 810000       \r",
      "  Processing short read 820000       \r",
      "  Processing short read 830000       \r",
      "  Processing short read 840000       \r",
      "  Processing short read 850000       \r",
      "  Processing short read 860000       \r",
      "  Processing short read 870000       \r",
      "  Processing short read 880000       \r",
      "  Processing short read 890000       \r",
      "  Processing short read 900000       \r",
      "  Processing short read 910000       \r",
      "  Processing short read 920000       \r",
      "  Processing short read 930000       \r",
      "  Processing short read 940000       \r",
      "  Processing short read 950000       \r",
      "  Processing short read 960000       \r",
      "  Processing short read 970000       \r",
      "  Processing short read 980000       \r",
      "  Processing short read 990000       \r",
      "  Processing short read 1000000       \r",
      "  Processing short read 1010000       \r",
      "  Processing short read 1020000       \r",
      "  Processing short read 1030000       \r",
      "  Processing short read 1040000       \r",
      "  Processing short read 1050000       \r",
      "  Processing short read 1060000       \r",
      "  Processing short read 1070000       \r",
      "  Processing short read 1080000       \r",
      "  Processing short read 1090000       \r",
      "  Processing short read 1100000       \r",
      "  Processing short read 1110000       \r",
      "  Processing short read 1120000       \r",
      "  Processing short read 1130000       \r",
      "  Processing short read 1140000       \r",
      "  Processing short read 1150000       \r",
      "  Processing short read 1160000       \r",
      "  Processing short read 1170000       \r",
      "  Processing short read 1180000       \r",
      "  Processing short read 1190000       \r",
      "  Processing short read 1200000       \r",
      "  Processing short read 1210000       \r",
      "  Processing short read 1220000       \r",
      "  Processing short read 1230000       \r",
      "  Processing short read 1240000       \r",
      "  Processing short read 1250000       \r",
      "  Processing short read 1260000       \r",
      "  Processing short read 1270000       \r",
      "  Processing short read 1280000       \r",
      "  Processing short read 1290000       \r",
      "  Processing short read 1300000       \r",
      "  Processing short read 1310000       \r",
      "  Processing short read 1320000       \r",
      "  Processing short read 1330000       \r",
      "  Processing short read 1340000       \r",
      "  Processing short read 1350000       \r",
      "  Processing short read 1360000       \r",
      "  Processing short read 1370000       \r",
      "  Processing short read 1380000       \r",
      "  Processing short read 1390000       \r",
      "  Processing short read 1400000       \r",
      "  Processing short read 1410000       \r",
      "  Processing short read 1420000       \r",
      "  Processing short read 1430000       \r",
      "  Processing short read 1440000       \r",
      "  Processing short read 1450000       \r",
      "  Processing short read 1460000       \r",
      "  Processing short read 1470000       \r",
      "  Processing short read 1480000       \r",
      "  Processing short read 1490000       \r",
      "  Processing short read 1500000       \r",
      "  Processing short read 1510000       \r",
      "  Processing short read 1520000       \r",
      "  Processing short read 1530000       \r",
      "  Processing short read 1540000       \r",
      "  Processing short read 1550000       \r",
      "  Processing short read 1560000       \r",
      "  Processing short read 1570000       \r",
      "  Processing short read 1580000       \r",
      "  Processing short read 1590000       \r",
      "  Processing short read 1600000       \r",
      "  Processing short read 1610000       \r",
      "  Processing short read 1620000       \r",
      "  Processing short read 1630000       \r",
      "  Processing short read 1640000       \r",
      "  Processing short read 1650000       \r",
      "  Processing short read 1660000       \r",
      "  Processing short read 1670000       \r",
      "  Processing short read 1680000       \r",
      "  Processing short read 1690000       \r",
      "  Processing short read 1700000       \r",
      "  Processing short read 1710000       \r",
      "  Processing short read 1720000       \r",
      "  Processing short read 1730000       \r",
      "  Processing short read 1740000       \r",
      "  Processing short read 1750000       \r",
      "  Processing short read 1760000       \r",
      "  Processing short read 1770000       \r",
      "  Processing short read 1780000       \r",
      "  Processing short read 1790000       \r",
      "  Processing short read 1800000       \r",
      "  Processing short read 1810000       \r",
      "  Processing short read 1820000       \r",
      "  Processing short read 1830000       \r",
      "  Processing short read 1840000       \r",
      "  Processing short read 1850000       \r",
      "  Processing short read 1860000       \r",
      "  Processing short read 1870000       \r",
      "  Processing short read 1880000       \r",
      "  Processing short read 1890000       \r",
      "  Processing short read 1900000       \r",
      "  Processing short read 1910000       \r",
      "  Processing short read 1920000       \r",
      "  Processing short read 1930000       \r",
      "  Processing short read 1940000       \r",
      "  Processing short read 1950000       \r",
      "  Processing short read 1960000       \r",
      "  Processing short read 1970000       \r",
      "  Processing short read 1980000       \r",
      "  Processing short read 1990000       \r",
      "  Processing short read 2000000       \r",
      "  Processing short read 2010000       \r",
      "  Processing short read 2020000       \r",
      "  Processing short read 2030000       \r",
      "  Processing short read 2040000       \r",
      "  Processing short read 2050000       \r",
      "  Processing short read 2060000       \r",
      "  Processing short read 2070000       \r",
      "  Processing short read 2080000       \r",
      "  Processing short read 2090000       \r",
      "  Processing short read 2100000       \r",
      "  Processing short read 2110000       \r",
      "  Processing short read 2120000       \r",
      "  Processing short read 2130000       \r",
      "  Processing short read 2140000       \r",
      "  Processing short read 2150000       \r",
      "  Processing short read 2160000       \r",
      "  Processing short read 2170000       \r",
      "  Processing short read 2180000       \r",
      "  Processing short read 2190000       \r",
      "  Processing short read 2200000       \r",
      "  Processing short read 2210000       \r",
      "  Processing short read 2220000       \r",
      "  Processing short read 2230000       \r",
      "  Processing short read 2240000       \r",
      "  Processing short read 2250000       \r",
      "  Processing short read 2260000       \r",
      "  Processing short read 2270000       \r",
      "  Processing short read 2280000       \r",
      "  Processing short read 2290000       \r",
      "  Processing short read 2300000       \r",
      "  Processing short read 2310000       \r",
      "  Processing short read 2320000       \r",
      "  Processing short read 2330000       \r",
      "  Processing short read 2340000       \r",
      "  Processing short read 2350000       \r",
      "  Processing short read 2360000       \r",
      "  Processing short read 2370000       \r",
      "  Processing short read 2380000       \r",
      "  Processing short read 2390000       \r",
      "  Processing short read 2400000       \r",
      "  Processing short read 2410000       \r",
      "  Processing short read 2420000       \r",
      "  Processing short read 2430000       \r",
      "  Processing short read 2440000       \r",
      "  Processing short read 2450000       \r",
      "  Processing short read 2460000       \r",
      "  Processing short read 2470000       \r",
      "  Processing short read 2480000       \r",
      "  Processing short read 2490000       \r",
      "  Processing short read 2500000       \r",
      "  Processing short read 2510000       \r",
      "  Processing short read 2520000       \r",
      "  Processing short read 2530000       \r",
      "  Processing short read 2540000       \r",
      "  Processing short read 2550000       \r",
      "  Processing short read 2560000       \r",
      "  Processing short read 2570000       \r",
      "  Processing short read 2580000       \r",
      "  Processing short read 2590000       \r",
      "  Processing short read 2600000       \r",
      "  Processing short read 2610000       \r",
      "  Processing short read 2620000       \r",
      "  Processing short read 2630000       \r",
      "  Processing short read 2640000       \r",
      "  Processing short read 2650000       \r",
      "  Processing short read 2660000       \r",
      "  Processing short read 2670000       \r",
      "  Processing short read 2680000       \r",
      "  Processing short read 2690000       \r",
      "  Processing short read 2700000       \r",
      "  Processing short read 2710000       \r",
      "  Processing short read 2720000       \r",
      "  Processing short read 2730000       \r",
      "  Processing short read 2740000       \r",
      "  Processing short read 2750000       \r",
      "  Processing short read 2760000       \r",
      "  Processing short read 2770000       \r",
      "  Processing short read 2780000       \r",
      "  Processing short read 2790000       \r",
      "  Processing short read 2800000       \r",
      "  Processing short read 2810000       \r",
      "  Processing short read 2820000       \r",
      "  Processing short read 2830000       \r",
      "  Processing short read 2840000       \r",
      "  Processing short read 2850000       \r",
      "  Processing short read 2860000       \r",
      "  Processing short read 2870000       \r",
      "  Processing short read 2880000       \r",
      "  Processing short read 2890000       \r",
      "  Processing short read 2900000       \r",
      "  Processing short read 2910000       \r",
      "  Processing short read 2920000       \r",
      "  Processing short read 2930000       \r",
      "  Processing short read 2940000       \r",
      "  Processing short read 2950000       \r",
      "  Processing short read 2960000       \r",
      "  Processing short read 2970000       \r",
      "  Processing short read 2980000       \r",
      "  5976182 total reads; -1259396 ambiguous barcodes; +0 recovered; -0 low quality reads; 4716786 retained reads.\n",
      "    0 trimmed reads; 0 orphaned paired-ends.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: './process_shortreads.log'\n",
      "\n",
      "5976182 total sequences;\n",
      "  1259396 ambiguous barcode drops;\n",
      "  0 low quality read drops;\n",
      "  0 trimmed reads;\n",
      "  0 orphaned paired-end reads;\n",
      "4716786 retained reads.\n",
      "\n",
      "assessing basic characteristics\n",
      "data comes as paired end - ok\n",
      "process_shortreads -1 /home/working/raw_data/Moth4_S4_L001_R1_001.fastq.gz -2 /home/working/raw_data/Moth4_S4_L001_R2_001.fastq.gz -i gzfastq -o . -b barcodes --inline_inline -y fastq\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Reads trimmed shorter than 31 nucleotides will be discarded.\n",
      "Found 1 paired input file(s).\n",
      "Searching for single and paired-end, inlined barcodes.\n",
      "Loaded 48 barcodes (8bp / 8bp).\n",
      "Processing file 1 of 1 [Moth4_S4_L001_R1_001.fastq.gz]\n",
      "  Reading data from\n",
      "    /home/working/raw_data/Moth4_S4_L001_R1_001.fastq.gz and\n",
      "    /home/working/raw_data/Moth4_S4_L001_R2_001.fastq.gz\n",
      "  Processing short read 10000       \r",
      "  Processing short read 20000       \r",
      "  Processing short read 30000       \r",
      "  Processing short read 40000       \r",
      "  Processing short read 50000       \r",
      "  Processing short read 60000       \r",
      "  Processing short read 70000       \r",
      "  Processing short read 80000       \r",
      "  Processing short read 90000       \r",
      "  Processing short read 100000       \r",
      "  Processing short read 110000       \r",
      "  Processing short read 120000       \r",
      "  Processing short read 130000       \r",
      "  Processing short read 140000       \r",
      "  Processing short read 150000       \r",
      "  Processing short read 160000       \r",
      "  Processing short read 170000       \r",
      "  Processing short read 180000       \r",
      "  Processing short read 190000       \r",
      "  Processing short read 200000       \r",
      "  Processing short read 210000       \r",
      "  Processing short read 220000       \r",
      "  Processing short read 230000       \r",
      "  Processing short read 240000       \r",
      "  Processing short read 250000       \r",
      "  Processing short read 260000       \r",
      "  Processing short read 270000       \r",
      "  Processing short read 280000       \r",
      "  Processing short read 290000       \r",
      "  Processing short read 300000       \r",
      "  Processing short read 310000       \r",
      "  Processing short read 320000       \r",
      "  Processing short read 330000       \r",
      "  Processing short read 340000       \r",
      "  Processing short read 350000       \r",
      "  Processing short read 360000       \r",
      "  Processing short read 370000       \r",
      "  Processing short read 380000       \r",
      "  Processing short read 390000       \r",
      "  Processing short read 400000       \r",
      "  Processing short read 410000       \r",
      "  Processing short read 420000       \r",
      "  Processing short read 430000       \r",
      "  Processing short read 440000       \r",
      "  Processing short read 450000       \r",
      "  Processing short read 460000       \r",
      "  Processing short read 470000       \r",
      "  Processing short read 480000       \r",
      "  Processing short read 490000       \r",
      "  Processing short read 500000       \r",
      "  Processing short read 510000       \r",
      "  Processing short read 520000       \r",
      "  Processing short read 530000       \r",
      "  Processing short read 540000       \r",
      "  Processing short read 550000       \r",
      "  Processing short read 560000       \r",
      "  Processing short read 570000       \r",
      "  Processing short read 580000       \r",
      "  Processing short read 590000       \r",
      "  Processing short read 600000       \r",
      "  Processing short read 610000       \r",
      "  Processing short read 620000       \r",
      "  Processing short read 630000       \r",
      "  Processing short read 640000       \r",
      "  Processing short read 650000       \r",
      "  Processing short read 660000       \r",
      "  Processing short read 670000       \r",
      "  Processing short read 680000       \r",
      "  Processing short read 690000       \r",
      "  Processing short read 700000       \r",
      "  Processing short read 710000       \r",
      "  Processing short read 720000       \r",
      "  Processing short read 730000       \r",
      "  Processing short read 740000       \r",
      "  Processing short read 750000       \r",
      "  Processing short read 760000       \r",
      "  Processing short read 770000       \r",
      "  Processing short read 780000       \r",
      "  Processing short read 790000       \r",
      "  Processing short read 800000       \r",
      "  Processing short read 810000       \r",
      "  Processing short read 820000       \r",
      "  Processing short read 830000       \r",
      "  Processing short read 840000       \r",
      "  Processing short read 850000       \r",
      "  Processing short read 860000       \r",
      "  Processing short read 870000       \r",
      "  Processing short read 880000       \r",
      "  Processing short read 890000       \r",
      "  Processing short read 900000       \r",
      "  Processing short read 910000       \r",
      "  Processing short read 920000       \r",
      "  Processing short read 930000       \r",
      "  Processing short read 940000       \r",
      "  Processing short read 950000       \r",
      "  Processing short read 960000       \r",
      "  Processing short read 970000       \r",
      "  Processing short read 980000       \r",
      "  Processing short read 990000       \r",
      "  Processing short read 1000000       \r",
      "  Processing short read 1010000       \r",
      "  Processing short read 1020000       \r",
      "  Processing short read 1030000       \r",
      "  Processing short read 1040000       \r",
      "  Processing short read 1050000       \r",
      "  Processing short read 1060000       \r",
      "  Processing short read 1070000       \r",
      "  Processing short read 1080000       \r",
      "  Processing short read 1090000       \r",
      "  Processing short read 1100000       \r",
      "  Processing short read 1110000       \r",
      "  Processing short read 1120000       \r",
      "  Processing short read 1130000       \r",
      "  Processing short read 1140000       \r",
      "  Processing short read 1150000       \r",
      "  Processing short read 1160000       \r",
      "  Processing short read 1170000       \r",
      "  Processing short read 1180000       \r",
      "  Processing short read 1190000       \r",
      "  Processing short read 1200000       \r",
      "  Processing short read 1210000       \r",
      "  Processing short read 1220000       \r",
      "  Processing short read 1230000       \r",
      "  Processing short read 1240000       \r",
      "  Processing short read 1250000       \r",
      "  Processing short read 1260000       \r",
      "  Processing short read 1270000       \r",
      "  Processing short read 1280000       \r",
      "  Processing short read 1290000       \r",
      "  Processing short read 1300000       \r",
      "  Processing short read 1310000       \r",
      "  Processing short read 1320000       \r",
      "  Processing short read 1330000       \r",
      "  Processing short read 1340000       \r",
      "  Processing short read 1350000       \r",
      "  Processing short read 1360000       \r",
      "  Processing short read 1370000       \r",
      "  Processing short read 1380000       \r",
      "  Processing short read 1390000       \r",
      "  Processing short read 1400000       \r",
      "  Processing short read 1410000       \r",
      "  Processing short read 1420000       \r",
      "  Processing short read 1430000       \r",
      "  Processing short read 1440000       \r",
      "  Processing short read 1450000       \r",
      "  Processing short read 1460000       \r",
      "  Processing short read 1470000       \r",
      "  Processing short read 1480000       \r",
      "  Processing short read 1490000       \r",
      "  Processing short read 1500000       \r",
      "  Processing short read 1510000       \r",
      "  Processing short read 1520000       \r",
      "  Processing short read 1530000       \r",
      "  Processing short read 1540000       \r",
      "  Processing short read 1550000       \r",
      "  Processing short read 1560000       \r",
      "  Processing short read 1570000       \r",
      "  Processing short read 1580000       \r",
      "  Processing short read 1590000       \r",
      "  Processing short read 1600000       \r",
      "  Processing short read 1610000       \r",
      "  Processing short read 1620000       \r",
      "  Processing short read 1630000       \r",
      "  Processing short read 1640000       \r",
      "  Processing short read 1650000       \r",
      "  Processing short read 1660000       \r",
      "  Processing short read 1670000       \r",
      "  Processing short read 1680000       \r",
      "  Processing short read 1690000       \r",
      "  Processing short read 1700000       \r",
      "  Processing short read 1710000       \r",
      "  Processing short read 1720000       \r",
      "  Processing short read 1730000       \r",
      "  Processing short read 1740000       \r",
      "  Processing short read 1750000       \r",
      "  Processing short read 1760000       \r",
      "  Processing short read 1770000       \r",
      "  Processing short read 1780000       \r",
      "  Processing short read 1790000       \r",
      "  Processing short read 1800000       \r",
      "  Processing short read 1810000       \r",
      "  Processing short read 1820000       \r",
      "  Processing short read 1830000       \r",
      "  Processing short read 1840000       \r",
      "  Processing short read 1850000       \r",
      "  Processing short read 1860000       \r",
      "  Processing short read 1870000       \r",
      "  Processing short read 1880000       \r",
      "  Processing short read 1890000       \r",
      "  Processing short read 1900000       \r",
      "  Processing short read 1910000       \r",
      "  Processing short read 1920000       \r",
      "  Processing short read 1930000       \r",
      "  Processing short read 1940000       \r",
      "  Processing short read 1950000       \r",
      "  Processing short read 1960000       \r",
      "  Processing short read 1970000       \r",
      "  Processing short read 1980000       \r",
      "  Processing short read 1990000       \r",
      "  Processing short read 2000000       \r",
      "  Processing short read 2010000       \r",
      "  Processing short read 2020000       \r",
      "  Processing short read 2030000       \r",
      "  Processing short read 2040000       \r",
      "  Processing short read 2050000       \r",
      "  Processing short read 2060000       \r",
      "  Processing short read 2070000       \r",
      "  Processing short read 2080000       \r",
      "  Processing short read 2090000       \r",
      "  Processing short read 2100000       \r",
      "  Processing short read 2110000       \r",
      "  Processing short read 2120000       \r",
      "  Processing short read 2130000       \r",
      "  Processing short read 2140000       \r",
      "  Processing short read 2150000       \r",
      "  Processing short read 2160000       \r",
      "  Processing short read 2170000       \r",
      "  Processing short read 2180000       \r",
      "  Processing short read 2190000       \r",
      "  Processing short read 2200000       \r",
      "  Processing short read 2210000       \r",
      "  Processing short read 2220000       \r",
      "  Processing short read 2230000       \r",
      "  Processing short read 2240000       \r",
      "  Processing short read 2250000       \r",
      "  Processing short read 2260000       \r",
      "  Processing short read 2270000       \r",
      "  Processing short read 2280000       \r",
      "  Processing short read 2290000       \r",
      "  Processing short read 2300000       \r",
      "  Processing short read 2310000       \r",
      "  Processing short read 2320000       \r",
      "  Processing short read 2330000       \r",
      "  Processing short read 2340000       \r",
      "  Processing short read 2350000       \r",
      "  Processing short read 2360000       \r",
      "  Processing short read 2370000       \r",
      "  Processing short read 2380000       \r",
      "  Processing short read 2390000       \r",
      "  Processing short read 2400000       \r",
      "  Processing short read 2410000       \r",
      "  Processing short read 2420000       \r",
      "  Processing short read 2430000       \r",
      "  Processing short read 2440000       \r",
      "  Processing short read 2450000       \r",
      "  Processing short read 2460000       \r",
      "  Processing short read 2470000       \r",
      "  Processing short read 2480000       \r",
      "  Processing short read 2490000       \r",
      "  Processing short read 2500000       \r",
      "  Processing short read 2510000       \r",
      "  Processing short read 2520000       \r",
      "  Processing short read 2530000       \r",
      "  Processing short read 2540000       \r",
      "  Processing short read 2550000       \r",
      "  Processing short read 2560000       \r",
      "  Processing short read 2570000       \r",
      "  Processing short read 2580000       \r",
      "  Processing short read 2590000       \r",
      "  Processing short read 2600000       \r",
      "  Processing short read 2610000       \r",
      "  Processing short read 2620000       \r",
      "  Processing short read 2630000       \r",
      "  Processing short read 2640000       \r",
      "  Processing short read 2650000       \r",
      "  Processing short read 2660000       \r",
      "  Processing short read 2670000       \r",
      "  Processing short read 2680000       \r",
      "  Processing short read 2690000       \r",
      "  Processing short read 2700000       \r",
      "  Processing short read 2710000       \r",
      "  Processing short read 2720000       \r",
      "  Processing short read 2730000       \r",
      "  Processing short read 2740000       \r",
      "  Processing short read 2750000       \r",
      "  Processing short read 2760000       \r",
      "  Processing short read 2770000       \r",
      "  Processing short read 2780000       \r",
      "  Processing short read 2790000       \r",
      "  Processing short read 2800000       \r",
      "  Processing short read 2810000       \r",
      "  Processing short read 2820000       \r",
      "  Processing short read 2830000       \r",
      "  Processing short read 2840000       \r",
      "  Processing short read 2850000       \r",
      "  Processing short read 2860000       \r",
      "  Processing short read 2870000       \r",
      "  Processing short read 2880000       \r",
      "  Processing short read 2890000       \r",
      "  Processing short read 2900000       \r",
      "  Processing short read 2910000       \r",
      "  Processing short read 2920000       \r",
      "  Processing short read 2930000       \r",
      "  Processing short read 2940000       \r",
      "  Processing short read 2950000       \r",
      "  Processing short read 2960000       \r",
      "  Processing short read 2970000       \r",
      "  Processing short read 2980000       \r",
      "  Processing short read 2990000       \r",
      "  Processing short read 3000000       \r",
      "  Processing short read 3010000       \r",
      "  Processing short read 3020000       \r",
      "  Processing short read 3030000       \r",
      "  Processing short read 3040000       \r",
      "  Processing short read 3050000       \r",
      "  Processing short read 3060000       \r",
      "  Processing short read 3070000       \r",
      "  Processing short read 3080000       \r",
      "  Processing short read 3090000       \r",
      "  Processing short read 3100000       \r",
      "  Processing short read 3110000       \r",
      "  Processing short read 3120000       \r",
      "  Processing short read 3130000       \r",
      "  Processing short read 3140000       \r",
      "  Processing short read 3150000       \r",
      "  Processing short read 3160000       \r",
      "  Processing short read 3170000       \r",
      "  Processing short read 3180000       \r",
      "  Processing short read 3190000       \r",
      "  Processing short read 3200000       \r",
      "  Processing short read 3210000       \r",
      "  Processing short read 3220000       \r",
      "  Processing short read 3230000       \r",
      "  Processing short read 3240000       \r",
      "  Processing short read 3250000       \r",
      "  Processing short read 3260000       \r",
      "  Processing short read 3270000       \r",
      "  Processing short read 3280000       \r",
      "  Processing short read 3290000       \r",
      "  Processing short read 3300000       \r",
      "  Processing short read 3310000       \r",
      "  Processing short read 3320000       \r",
      "  Processing short read 3330000       \r",
      "  Processing short read 3340000       \r",
      "  Processing short read 3350000       \r",
      "  Processing short read 3360000       \r",
      "  Processing short read 3370000       \r",
      "  Processing short read 3380000       \r",
      "  Processing short read 3390000       \r",
      "  Processing short read 3400000       \r",
      "  Processing short read 3410000       \r",
      "  Processing short read 3420000       \r",
      "  Processing short read 3430000       \r",
      "  Processing short read 3440000       \r",
      "  Processing short read 3450000       \r",
      "  Processing short read 3460000       \r",
      "  Processing short read 3470000       \r",
      "  Processing short read 3480000       \r",
      "  Processing short read 3490000       \r",
      "  Processing short read 3500000       \r",
      "  Processing short read 3510000       \r",
      "  Processing short read 3520000       \r",
      "  Processing short read 3530000       \r",
      "  Processing short read 3540000       \r",
      "  Processing short read 3550000       \r",
      "  Processing short read 3560000       \r",
      "  Processing short read 3570000       \r",
      "  Processing short read 3580000       \r",
      "  Processing short read 3590000       \r",
      "  Processing short read 3600000       \r",
      "  Processing short read 3610000       \r",
      "  Processing short read 3620000       \r",
      "  Processing short read 3630000       \r",
      "  Processing short read 3640000       \r",
      "  Processing short read 3650000       \r",
      "  Processing short read 3660000       \r",
      "  Processing short read 3670000       \r",
      "  Processing short read 3680000       \r",
      "  Processing short read 3690000       \r",
      "  Processing short read 3700000       \r",
      "  Processing short read 3710000       \r",
      "  Processing short read 3720000       \r",
      "  Processing short read 3730000       \r",
      "  Processing short read 3740000       \r",
      "  Processing short read 3750000       \r",
      "  Processing short read 3760000       \r",
      "  Processing short read 3770000       \r",
      "  Processing short read 3780000       \r",
      "  Processing short read 3790000       \r",
      "  Processing short read 3800000       \r",
      "  Processing short read 3810000       \r",
      "  Processing short read 3820000       \r",
      "  Processing short read 3830000       \r",
      "  Processing short read 3840000       \r",
      "  Processing short read 3850000       \r",
      "  Processing short read 3860000       \r",
      "  Processing short read 3870000       \r",
      "  Processing short read 3880000       \r",
      "  Processing short read 3890000       \r",
      "  Processing short read 3900000       \r",
      "  Processing short read 3910000       \r",
      "  Processing short read 3920000       \r",
      "  Processing short read 3930000       \r",
      "  Processing short read 3940000       \r",
      "  Processing short read 3950000       \r",
      "  Processing short read 3960000       \r",
      "  Processing short read 3970000       \r",
      "  Processing short read 3980000       \r",
      "  Processing short read 3990000       \r",
      "  Processing short read 4000000       \r",
      "  Processing short read 4010000       \r",
      "  Processing short read 4020000       \r",
      "  Processing short read 4030000       \r",
      "  Processing short read 4040000       \r",
      "  Processing short read 4050000       \r",
      "  Processing short read 4060000       \r",
      "  Processing short read 4070000       \r",
      "  Processing short read 4080000       \r",
      "  Processing short read 4090000       \r",
      "  Processing short read 4100000       \r",
      "  Processing short read 4110000       \r",
      "  Processing short read 4120000       \r",
      "  Processing short read 4130000       \r",
      "  Processing short read 4140000       \r",
      "  Processing short read 4150000       \r",
      "  Processing short read 4160000       \r",
      "  Processing short read 4170000       \r",
      "  Processing short read 4180000       \r",
      "  Processing short read 4190000       \r",
      "  Processing short read 4200000       \r",
      "  Processing short read 4210000       \r",
      "  Processing short read 4220000       \r",
      "  Processing short read 4230000       \r",
      "  8464282 total reads; -1099580 ambiguous barcodes; +0 recovered; -0 low quality reads; 7364702 retained reads.\n",
      "    0 trimmed reads; 0 orphaned paired-ends.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: './process_shortreads.log'\n",
      "\n",
      "8464282 total sequences;\n",
      "  1099580 ambiguous barcode drops;\n",
      "  0 low quality read drops;\n",
      "  0 trimmed reads;\n",
      "  0 orphaned paired-end reads;\n",
      "7364702 retained reads.\n",
      "\n",
      "assessing basic characteristics\n",
      "data comes as paired end - ok\n",
      "process_shortreads -1 /home/working/raw_data/Moth3_S3_L001_R1_001.fastq.gz -2 /home/working/raw_data/Moth3_S3_L001_R2_001.fastq.gz -i gzfastq -o . -b barcodes --inline_inline -y fastq\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Reads trimmed shorter than 31 nucleotides will be discarded.\n",
      "Found 1 paired input file(s).\n",
      "Searching for single and paired-end, inlined barcodes.\n",
      "Loaded 96 barcodes (8bp / 8bp).\n",
      "Processing file 1 of 1 [Moth3_S3_L001_R1_001.fastq.gz]\n",
      "  Reading data from\n",
      "    /home/working/raw_data/Moth3_S3_L001_R1_001.fastq.gz and\n",
      "    /home/working/raw_data/Moth3_S3_L001_R2_001.fastq.gz\n",
      "  Processing short read 10000       \r",
      "  Processing short read 20000       \r",
      "  Processing short read 30000       \r",
      "  Processing short read 40000       \r",
      "  Processing short read 50000       \r",
      "  Processing short read 60000       \r",
      "  Processing short read 70000       \r",
      "  Processing short read 80000       \r",
      "  Processing short read 90000       \r",
      "  Processing short read 100000       \r",
      "  Processing short read 110000       \r",
      "  Processing short read 120000       \r",
      "  Processing short read 130000       \r",
      "  Processing short read 140000       \r",
      "  Processing short read 150000       \r",
      "  Processing short read 160000       \r",
      "  Processing short read 170000       \r",
      "  Processing short read 180000       \r",
      "  Processing short read 190000       \r",
      "  Processing short read 200000       \r",
      "  Processing short read 210000       \r",
      "  Processing short read 220000       \r",
      "  Processing short read 230000       \r",
      "  Processing short read 240000       \r",
      "  Processing short read 250000       \r",
      "  Processing short read 260000       \r",
      "  Processing short read 270000       \r",
      "  Processing short read 280000       \r",
      "  Processing short read 290000       \r",
      "  Processing short read 300000       \r",
      "  Processing short read 310000       \r",
      "  Processing short read 320000       \r",
      "  Processing short read 330000       \r",
      "  Processing short read 340000       \r",
      "  Processing short read 350000       \r",
      "  Processing short read 360000       \r",
      "  Processing short read 370000       \r",
      "  Processing short read 380000       \r",
      "  Processing short read 390000       \r",
      "  Processing short read 400000       \r",
      "  Processing short read 410000       \r",
      "  Processing short read 420000       \r",
      "  Processing short read 430000       \r",
      "  Processing short read 440000       \r",
      "  Processing short read 450000       \r",
      "  Processing short read 460000       \r",
      "  Processing short read 470000       \r",
      "  Processing short read 480000       \r",
      "  Processing short read 490000       \r",
      "  Processing short read 500000       \r",
      "  Processing short read 510000       \r",
      "  Processing short read 520000       \r",
      "  Processing short read 530000       \r",
      "  Processing short read 540000       \r",
      "  Processing short read 550000       \r",
      "  Processing short read 560000       \r",
      "  Processing short read 570000       \r",
      "  Processing short read 580000       \r",
      "  Processing short read 590000       \r",
      "  Processing short read 600000       \r",
      "  Processing short read 610000       \r",
      "  Processing short read 620000       \r",
      "  Processing short read 630000       \r",
      "  Processing short read 640000       \r",
      "  Processing short read 650000       \r",
      "  Processing short read 660000       \r",
      "  Processing short read 670000       \r",
      "  Processing short read 680000       \r",
      "  Processing short read 690000       \r",
      "  Processing short read 700000       \r",
      "  Processing short read 710000       \r",
      "  Processing short read 720000       \r",
      "  Processing short read 730000       \r",
      "  Processing short read 740000       \r",
      "  Processing short read 750000       \r",
      "  Processing short read 760000       \r",
      "  Processing short read 770000       \r",
      "  Processing short read 780000       \r",
      "  Processing short read 790000       \r",
      "  Processing short read 800000       \r",
      "  Processing short read 810000       \r",
      "  Processing short read 820000       \r",
      "  Processing short read 830000       \r",
      "  Processing short read 840000       \r",
      "  Processing short read 850000       \r",
      "  Processing short read 860000       \r",
      "  Processing short read 870000       \r",
      "  Processing short read 880000       \r",
      "  Processing short read 890000       \r",
      "  Processing short read 900000       \r",
      "  Processing short read 910000       \r",
      "  Processing short read 920000       \r",
      "  Processing short read 930000       \r",
      "  Processing short read 940000       \r",
      "  Processing short read 950000       \r",
      "  Processing short read 960000       \r",
      "  Processing short read 970000       \r",
      "  Processing short read 980000       \r",
      "  Processing short read 990000       \r",
      "  Processing short read 1000000       \r",
      "  Processing short read 1010000       \r",
      "  Processing short read 1020000       \r",
      "  Processing short read 1030000       \r",
      "  Processing short read 1040000       \r",
      "  Processing short read 1050000       \r",
      "  Processing short read 1060000       \r",
      "  Processing short read 1070000       \r",
      "  Processing short read 1080000       \r",
      "  Processing short read 1090000       \r",
      "  Processing short read 1100000       \r",
      "  Processing short read 1110000       \r",
      "  Processing short read 1120000       \r",
      "  Processing short read 1130000       \r",
      "  Processing short read 1140000       \r",
      "  Processing short read 1150000       \r",
      "  Processing short read 1160000       \r",
      "  Processing short read 1170000       \r",
      "  Processing short read 1180000       \r",
      "  Processing short read 1190000       \r",
      "  Processing short read 1200000       \r",
      "  Processing short read 1210000       \r",
      "  Processing short read 1220000       \r",
      "  Processing short read 1230000       \r",
      "  Processing short read 1240000       \r",
      "  Processing short read 1250000       \r",
      "  Processing short read 1260000       \r",
      "  Processing short read 1270000       \r",
      "  Processing short read 1280000       \r",
      "  Processing short read 1290000       \r",
      "  Processing short read 1300000       \r",
      "  Processing short read 1310000       \r",
      "  Processing short read 1320000       \r",
      "  Processing short read 1330000       \r",
      "  Processing short read 1340000       \r",
      "  Processing short read 1350000       \r",
      "  Processing short read 1360000       \r",
      "  Processing short read 1370000       \r",
      "  Processing short read 1380000       \r",
      "  Processing short read 1390000       \r",
      "  Processing short read 1400000       \r",
      "  Processing short read 1410000       \r",
      "  Processing short read 1420000       \r",
      "  Processing short read 1430000       \r",
      "  Processing short read 1440000       \r",
      "  Processing short read 1450000       \r",
      "  Processing short read 1460000       \r",
      "  Processing short read 1470000       \r",
      "  Processing short read 1480000       \r",
      "  Processing short read 1490000       \r",
      "  Processing short read 1500000       \r",
      "  Processing short read 1510000       \r",
      "  Processing short read 1520000       \r",
      "  Processing short read 1530000       \r",
      "  Processing short read 1540000       \r",
      "  Processing short read 1550000       \r",
      "  Processing short read 1560000       \r",
      "  Processing short read 1570000       \r",
      "  Processing short read 1580000       \r",
      "  Processing short read 1590000       \r",
      "  Processing short read 1600000       \r",
      "  Processing short read 1610000       \r",
      "  Processing short read 1620000       \r",
      "  Processing short read 1630000       \r",
      "  Processing short read 1640000       \r",
      "  Processing short read 1650000       \r",
      "  Processing short read 1660000       \r",
      "  Processing short read 1670000       \r",
      "  Processing short read 1680000       \r",
      "  Processing short read 1690000       \r",
      "  Processing short read 1700000       \r",
      "  Processing short read 1710000       \r",
      "  Processing short read 1720000       \r",
      "  Processing short read 1730000       \r",
      "  Processing short read 1740000       \r",
      "  Processing short read 1750000       \r",
      "  Processing short read 1760000       \r",
      "  Processing short read 1770000       \r",
      "  Processing short read 1780000       \r",
      "  Processing short read 1790000       \r",
      "  Processing short read 1800000       \r",
      "  Processing short read 1810000       \r",
      "  Processing short read 1820000       \r",
      "  Processing short read 1830000       \r",
      "  Processing short read 1840000       \r",
      "  Processing short read 1850000       \r",
      "  Processing short read 1860000       \r",
      "  Processing short read 1870000       \r",
      "  Processing short read 1880000       \r",
      "  Processing short read 1890000       \r",
      "  Processing short read 1900000       \r",
      "  Processing short read 1910000       \r",
      "  Processing short read 1920000       \r",
      "  Processing short read 1930000       \r",
      "  Processing short read 1940000       \r",
      "  Processing short read 1950000       \r",
      "  Processing short read 1960000       \r",
      "  Processing short read 1970000       \r",
      "  Processing short read 1980000       \r",
      "  Processing short read 1990000       \r",
      "  Processing short read 2000000       \r",
      "  Processing short read 2010000       \r",
      "  Processing short read 2020000       \r",
      "  Processing short read 2030000       \r",
      "  Processing short read 2040000       \r",
      "  Processing short read 2050000       \r",
      "  Processing short read 2060000       \r",
      "  Processing short read 2070000       \r",
      "  Processing short read 2080000       \r",
      "  Processing short read 2090000       \r",
      "  Processing short read 2100000       \r",
      "  Processing short read 2110000       \r",
      "  Processing short read 2120000       \r",
      "  Processing short read 2130000       \r",
      "  Processing short read 2140000       \r",
      "  Processing short read 2150000       \r",
      "  Processing short read 2160000       \r",
      "  Processing short read 2170000       \r",
      "  Processing short read 2180000       \r",
      "  Processing short read 2190000       \r",
      "  Processing short read 2200000       \r",
      "  Processing short read 2210000       \r",
      "  Processing short read 2220000       \r",
      "  Processing short read 2230000       \r",
      "  Processing short read 2240000       \r",
      "  Processing short read 2250000       \r",
      "  Processing short read 2260000       \r",
      "  Processing short read 2270000       \r",
      "  Processing short read 2280000       \r",
      "  Processing short read 2290000       \r",
      "  Processing short read 2300000       \r",
      "  Processing short read 2310000       \r",
      "  Processing short read 2320000       \r",
      "  Processing short read 2330000       \r",
      "  Processing short read 2340000       \r",
      "  Processing short read 2350000       \r",
      "  Processing short read 2360000       \r",
      "  Processing short read 2370000       \r",
      "  Processing short read 2380000       \r",
      "  Processing short read 2390000       \r",
      "  Processing short read 2400000       \r",
      "  Processing short read 2410000       \r",
      "  Processing short read 2420000       \r",
      "  Processing short read 2430000       \r",
      "  Processing short read 2440000       \r",
      "  Processing short read 2450000       \r",
      "  Processing short read 2460000       \r",
      "  Processing short read 2470000       \r",
      "  Processing short read 2480000       \r",
      "  Processing short read 2490000       \r",
      "  Processing short read 2500000       \r",
      "  Processing short read 2510000       \r",
      "  Processing short read 2520000       \r",
      "  Processing short read 2530000       \r",
      "  Processing short read 2540000       \r",
      "  Processing short read 2550000       \r",
      "  Processing short read 2560000       \r",
      "  Processing short read 2570000       \r",
      "  Processing short read 2580000       \r",
      "  Processing short read 2590000       \r",
      "  Processing short read 2600000       \r",
      "  Processing short read 2610000       \r",
      "  Processing short read 2620000       \r",
      "  Processing short read 2630000       \r",
      "  Processing short read 2640000       \r",
      "  Processing short read 2650000       \r",
      "  Processing short read 2660000       \r",
      "  Processing short read 2670000       \r",
      "  Processing short read 2680000       \r",
      "  Processing short read 2690000       \r",
      "  Processing short read 2700000       \r",
      "  Processing short read 2710000       \r",
      "  Processing short read 2720000       \r",
      "  Processing short read 2730000       \r",
      "  Processing short read 2740000       \r",
      "  Processing short read 2750000       \r",
      "  Processing short read 2760000       \r",
      "  Processing short read 2770000       \r",
      "  Processing short read 2780000       \r",
      "  Processing short read 2790000       \r",
      "  Processing short read 2800000       \r",
      "  Processing short read 2810000       \r",
      "  Processing short read 2820000       \r",
      "  Processing short read 2830000       \r",
      "  Processing short read 2840000       \r",
      "  Processing short read 2850000       \r",
      "  Processing short read 2860000       \r",
      "  Processing short read 2870000       \r",
      "  Processing short read 2880000       \r",
      "  Processing short read 2890000       \r",
      "  Processing short read 2900000       \r",
      "  Processing short read 2910000       \r",
      "  Processing short read 2920000       \r",
      "  Processing short read 2930000       \r",
      "  Processing short read 2940000       \r",
      "  Processing short read 2950000       \r",
      "  Processing short read 2960000       \r",
      "  Processing short read 2970000       \r",
      "  Processing short read 2980000       \r",
      "  Processing short read 2990000       \r",
      "  Processing short read 3000000       \r",
      "  Processing short read 3010000       \r",
      "  Processing short read 3020000       \r",
      "  Processing short read 3030000       \r",
      "  Processing short read 3040000       \r",
      "  Processing short read 3050000       \r",
      "  Processing short read 3060000       \r",
      "  Processing short read 3070000       \r",
      "  Processing short read 3080000       \r",
      "  Processing short read 3090000       \r",
      "  Processing short read 3100000       \r",
      "  Processing short read 3110000       \r",
      "  Processing short read 3120000       \r",
      "  Processing short read 3130000       \r",
      "  Processing short read 3140000       \r",
      "  Processing short read 3150000       \r",
      "  Processing short read 3160000       \r",
      "  Processing short read 3170000       \r",
      "  Processing short read 3180000       \r",
      "  Processing short read 3190000       \r",
      "  Processing short read 3200000       \r",
      "  Processing short read 3210000       \r",
      "  Processing short read 3220000       \r",
      "  Processing short read 3230000       \r",
      "  Processing short read 3240000       \r",
      "  Processing short read 3250000       \r",
      "  Processing short read 3260000       \r",
      "  Processing short read 3270000       \r",
      "  Processing short read 3280000       \r",
      "  Processing short read 3290000       \r",
      "  Processing short read 3300000       \r",
      "  Processing short read 3310000       \r",
      "  Processing short read 3320000       \r",
      "  Processing short read 3330000       \r",
      "  Processing short read 3340000       \r",
      "  Processing short read 3350000       \r",
      "  Processing short read 3360000       \r",
      "  Processing short read 3370000       \r",
      "  6757328 total reads; -1226048 ambiguous barcodes; +0 recovered; -0 low quality reads; 5531280 retained reads.\n",
      "    0 trimmed reads; 0 orphaned paired-ends.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: './process_shortreads.log'\n",
      "\n",
      "6757328 total sequences;\n",
      "  1226048 ambiguous barcode drops;\n",
      "  0 low quality read drops;\n",
      "  0 trimmed reads;\n",
      "  0 orphaned paired-end reads;\n",
      "5531280 retained reads.\n",
      "\n",
      "assessing basic characteristics\n",
      "data comes as paired end - ok\n",
      "process_shortreads -1 /home/working/raw_data/Moth2_S2_L001_R1_001.fastq.gz -2 /home/working/raw_data/Moth2_S2_L001_R2_001.fastq.gz -i gzfastq -o . -b barcodes --inline_inline -y fastq\n",
      "Using Phred+33 encoding for quality scores.\n",
      "Reads trimmed shorter than 31 nucleotides will be discarded.\n",
      "Found 1 paired input file(s).\n",
      "Searching for single and paired-end, inlined barcodes.\n",
      "Loaded 95 barcodes (8bp / 8bp).\n",
      "Processing file 1 of 1 [Moth2_S2_L001_R1_001.fastq.gz]\n",
      "  Reading data from\n",
      "    /home/working/raw_data/Moth2_S2_L001_R1_001.fastq.gz and\n",
      "    /home/working/raw_data/Moth2_S2_L001_R2_001.fastq.gz\n",
      "  Processing short read 10000       \r",
      "  Processing short read 20000       \r",
      "  Processing short read 30000       \r",
      "  Processing short read 40000       \r",
      "  Processing short read 50000       \r",
      "  Processing short read 60000       \r",
      "  Processing short read 70000       \r",
      "  Processing short read 80000       \r",
      "  Processing short read 90000       \r",
      "  Processing short read 100000       \r",
      "  Processing short read 110000       \r",
      "  Processing short read 120000       \r",
      "  Processing short read 130000       \r",
      "  Processing short read 140000       \r",
      "  Processing short read 150000       \r",
      "  Processing short read 160000       \r",
      "  Processing short read 170000       \r",
      "  Processing short read 180000       \r",
      "  Processing short read 190000       \r",
      "  Processing short read 200000       \r",
      "  Processing short read 210000       \r",
      "  Processing short read 220000       \r",
      "  Processing short read 230000       \r",
      "  Processing short read 240000       \r",
      "  Processing short read 250000       \r",
      "  Processing short read 260000       \r",
      "  Processing short read 270000       \r",
      "  Processing short read 280000       \r",
      "  Processing short read 290000       \r",
      "  Processing short read 300000       \r",
      "  Processing short read 310000       \r",
      "  Processing short read 320000       \r",
      "  Processing short read 330000       \r",
      "  Processing short read 340000       \r",
      "  Processing short read 350000       \r",
      "  Processing short read 360000       \r",
      "  Processing short read 370000       \r",
      "  Processing short read 380000       \r",
      "  Processing short read 390000       \r",
      "  Processing short read 400000       \r",
      "  Processing short read 410000       \r",
      "  Processing short read 420000       \r",
      "  Processing short read 430000       \r",
      "  Processing short read 440000       \r",
      "  Processing short read 450000       \r",
      "  Processing short read 460000       \r",
      "  Processing short read 470000       \r",
      "  Processing short read 480000       \r",
      "  Processing short read 490000       \r",
      "  Processing short read 500000       \r",
      "  Processing short read 510000       \r",
      "  Processing short read 520000       \r",
      "  Processing short read 530000       \r",
      "  Processing short read 540000       \r",
      "  Processing short read 550000       \r",
      "  Processing short read 560000       \r",
      "  Processing short read 570000       \r",
      "  Processing short read 580000       \r",
      "  Processing short read 590000       \r",
      "  Processing short read 600000       \r",
      "  Processing short read 610000       \r",
      "  Processing short read 620000       \r",
      "  Processing short read 630000       \r",
      "  Processing short read 640000       \r",
      "  Processing short read 650000       \r",
      "  Processing short read 660000       \r",
      "  Processing short read 670000       \r",
      "  Processing short read 680000       \r",
      "  Processing short read 690000       \r",
      "  Processing short read 700000       \r",
      "  Processing short read 710000       \r",
      "  Processing short read 720000       \r",
      "  Processing short read 730000       \r",
      "  Processing short read 740000       \r",
      "  Processing short read 750000       \r",
      "  Processing short read 760000       \r",
      "  Processing short read 770000       \r",
      "  Processing short read 780000       \r",
      "  Processing short read 790000       \r",
      "  Processing short read 800000       \r",
      "  Processing short read 810000       \r",
      "  Processing short read 820000       \r",
      "  Processing short read 830000       \r",
      "  Processing short read 840000       \r",
      "  Processing short read 850000       \r",
      "  Processing short read 860000       \r",
      "  Processing short read 870000       \r",
      "  Processing short read 880000       \r",
      "  Processing short read 890000       \r",
      "  Processing short read 900000       \r",
      "  Processing short read 910000       \r",
      "  Processing short read 920000       \r",
      "  Processing short read 930000       \r",
      "  Processing short read 940000       \r",
      "  Processing short read 950000       \r",
      "  Processing short read 960000       \r",
      "  Processing short read 970000       \r",
      "  Processing short read 980000       \r",
      "  Processing short read 990000       \r",
      "  Processing short read 1000000       \r",
      "  Processing short read 1010000       \r",
      "  Processing short read 1020000       \r",
      "  Processing short read 1030000       \r",
      "  Processing short read 1040000       \r",
      "  Processing short read 1050000       \r",
      "  Processing short read 1060000       \r",
      "  Processing short read 1070000       \r",
      "  Processing short read 1080000       \r",
      "  Processing short read 1090000       \r",
      "  Processing short read 1100000       \r",
      "  Processing short read 1110000       \r",
      "  Processing short read 1120000       \r",
      "  Processing short read 1130000       \r",
      "  Processing short read 1140000       \r",
      "  Processing short read 1150000       \r",
      "  Processing short read 1160000       \r",
      "  Processing short read 1170000       \r",
      "  Processing short read 1180000       \r",
      "  Processing short read 1190000       \r",
      "  Processing short read 1200000       \r",
      "  Processing short read 1210000       \r",
      "  Processing short read 1220000       \r",
      "  Processing short read 1230000       \r",
      "  Processing short read 1240000       \r",
      "  Processing short read 1250000       \r",
      "  Processing short read 1260000       \r",
      "  Processing short read 1270000       \r",
      "  Processing short read 1280000       \r",
      "  Processing short read 1290000       \r",
      "  Processing short read 1300000       \r",
      "  Processing short read 1310000       \r",
      "  Processing short read 1320000       \r",
      "  Processing short read 1330000       \r",
      "  Processing short read 1340000       \r",
      "  Processing short read 1350000       \r",
      "  Processing short read 1360000       \r",
      "  Processing short read 1370000       \r",
      "  Processing short read 1380000       \r",
      "  Processing short read 1390000       \r",
      "  Processing short read 1400000       \r",
      "  Processing short read 1410000       \r",
      "  Processing short read 1420000       \r",
      "  Processing short read 1430000       \r",
      "  Processing short read 1440000       \r",
      "  Processing short read 1450000       \r",
      "  Processing short read 1460000       \r",
      "  Processing short read 1470000       \r",
      "  Processing short read 1480000       \r",
      "  Processing short read 1490000       \r",
      "  Processing short read 1500000       \r",
      "  Processing short read 1510000       \r",
      "  Processing short read 1520000       \r",
      "  Processing short read 1530000       \r",
      "  Processing short read 1540000       \r",
      "  Processing short read 1550000       \r",
      "  Processing short read 1560000       \r",
      "  Processing short read 1570000       \r",
      "  Processing short read 1580000       \r",
      "  Processing short read 1590000       \r",
      "  Processing short read 1600000       \r",
      "  Processing short read 1610000       \r",
      "  Processing short read 1620000       \r",
      "  Processing short read 1630000       \r",
      "  Processing short read 1640000       \r",
      "  Processing short read 1650000       \r",
      "  Processing short read 1660000       \r",
      "  Processing short read 1670000       \r",
      "  Processing short read 1680000       \r",
      "  Processing short read 1690000       \r",
      "  Processing short read 1700000       \r",
      "  Processing short read 1710000       \r",
      "  Processing short read 1720000       \r",
      "  Processing short read 1730000       \r",
      "  Processing short read 1740000       \r",
      "  Processing short read 1750000       \r",
      "  Processing short read 1760000       \r",
      "  Processing short read 1770000       \r",
      "  Processing short read 1780000       \r",
      "  Processing short read 1790000       \r",
      "  Processing short read 1800000       \r",
      "  Processing short read 1810000       \r",
      "  Processing short read 1820000       \r",
      "  Processing short read 1830000       \r",
      "  Processing short read 1840000       \r",
      "  Processing short read 1850000       \r",
      "  Processing short read 1860000       \r",
      "  Processing short read 1870000       \r",
      "  Processing short read 1880000       \r",
      "  Processing short read 1890000       \r",
      "  Processing short read 1900000       \r",
      "  Processing short read 1910000       \r",
      "  Processing short read 1920000       \r",
      "  Processing short read 1930000       \r",
      "  Processing short read 1940000       \r",
      "  Processing short read 1950000       \r",
      "  Processing short read 1960000       \r",
      "  Processing short read 1970000       \r",
      "  Processing short read 1980000       \r",
      "  Processing short read 1990000       \r",
      "  Processing short read 2000000       \r",
      "  Processing short read 2010000       \r",
      "  Processing short read 2020000       \r",
      "  Processing short read 2030000       \r",
      "  Processing short read 2040000       \r",
      "  Processing short read 2050000       \r",
      "  Processing short read 2060000       \r",
      "  Processing short read 2070000       \r",
      "  Processing short read 2080000       \r",
      "  Processing short read 2090000       \r",
      "  4193542 total reads; -826944 ambiguous barcodes; +0 recovered; -0 low quality reads; 3366598 retained reads.\n",
      "    0 trimmed reads; 0 orphaned paired-ends.\n",
      "Closing files, flushing buffers...\n",
      "Outputing details to log: './process_shortreads.log'\n",
      "\n",
      "4193542 total sequences;\n",
      "  826944 ambiguous barcode drops;\n",
      "  0 low quality read drops;\n",
      "  0 trimmed reads;\n",
      "  0 orphaned paired-end reads;\n",
      "3366598 retained reads.\n",
      "\n",
      "\n",
      "##### processing query ID: Lg27 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg27_1.fastq /home/working/demultiplexed/Lg27_2.fastq Lg27_forward.paired.fastq.gz Lg27_forward.singletons.fastq.gz Lg27_reverse.paired.fastq.gz Lg27_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg27_1.fastq /home/working/demultiplexed/Lg27_2.fastq Lg27_forward.paired.fastq.gz Lg27_forward.singletons.fastq.gz Lg27_reverse.paired.fastq.gz Lg27_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:45:41 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/Lg27/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 26514 Both Surviving: 6740 (25.42%) Forward Only Surviving: 3298 (12.44%) Reverse Only Surviving: 209 (0.79%) Dropped: 16267 (61.35%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: Lg27_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: Lg27_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: Lg27_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: Lg27_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash Lg27_forward.paired.headclipped.fastq.gz Lg27_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg27 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     Lg27_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     Lg27_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./Lg27.extendedFrags.fastq.gz\n",
      "[FLASH]     ./Lg27.notCombined_1.fastq.gz\n",
      "[FLASH]     ./Lg27.notCombined_2.fastq.gz\n",
      "[FLASH]     ./Lg27.hist\n",
      "[FLASH]     ./Lg27.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 6740 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      6740\n",
      "[FLASH]     Combined pairs:   5348\n",
      "[FLASH]     Uncombined pairs: 1392\n",
      "[FLASH]     Percent combined: 79.35%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.147 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "5348 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/Lg27/Lg27_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg27_centroids.fasta --uc Lg27.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 5348 sequences and identified 2955 clusters (clustering threshold 1.00) - 72 clusters (representing 2021 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: Lg37 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg37_1.fastq /home/working/demultiplexed/Lg37_2.fastq Lg37_forward.paired.fastq.gz Lg37_forward.singletons.fastq.gz Lg37_reverse.paired.fastq.gz Lg37_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg37_1.fastq /home/working/demultiplexed/Lg37_2.fastq Lg37_forward.paired.fastq.gz Lg37_forward.singletons.fastq.gz Lg37_reverse.paired.fastq.gz Lg37_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:45:53 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/Lg37/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 31577 Both Surviving: 247 (0.78%) Forward Only Surviving: 489 (1.55%) Reverse Only Surviving: 197 (0.62%) Dropped: 30644 (97.05%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: Lg37_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: Lg37_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: Lg37_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: Lg37_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash Lg37_forward.paired.headclipped.fastq.gz Lg37_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg37 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     Lg37_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     Lg37_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./Lg37.extendedFrags.fastq.gz\n",
      "[FLASH]     ./Lg37.notCombined_1.fastq.gz\n",
      "[FLASH]     ./Lg37.notCombined_2.fastq.gz\n",
      "[FLASH]     ./Lg37.hist\n",
      "[FLASH]     ./Lg37.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 247 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      247\n",
      "[FLASH]     Combined pairs:   28\n",
      "[FLASH]     Uncombined pairs: 219\n",
      "[FLASH]     Percent combined: 11.34%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.003 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "28 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/Lg37/Lg37_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg37_centroids.fasta --uc Lg37.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 28 sequences and identified 26 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: Lg45 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg45_1.fastq /home/working/demultiplexed/Lg45_2.fastq Lg45_forward.paired.fastq.gz Lg45_forward.singletons.fastq.gz Lg45_reverse.paired.fastq.gz Lg45_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg45_1.fastq /home/working/demultiplexed/Lg45_2.fastq Lg45_forward.paired.fastq.gz Lg45_forward.singletons.fastq.gz Lg45_reverse.paired.fastq.gz Lg45_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:45:55 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/Lg45/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 9421 Both Surviving: 2219 (23.55%) Forward Only Surviving: 590 (6.26%) Reverse Only Surviving: 112 (1.19%) Dropped: 6500 (68.99%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: Lg45_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: Lg45_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: Lg45_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: Lg45_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash Lg45_forward.paired.headclipped.fastq.gz Lg45_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg45 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     Lg45_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     Lg45_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./Lg45.extendedFrags.fastq.gz\n",
      "[FLASH]     ./Lg45.notCombined_1.fastq.gz\n",
      "[FLASH]     ./Lg45.notCombined_2.fastq.gz\n",
      "[FLASH]     ./Lg45.hist\n",
      "[FLASH]     ./Lg45.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2219 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2219\n",
      "[FLASH]     Combined pairs:   2093\n",
      "[FLASH]     Uncombined pairs: 126\n",
      "[FLASH]     Percent combined: 94.32%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.059 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2093 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/Lg45/Lg45_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg45_centroids.fasta --uc Lg45.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2093 sequences and identified 1056 clusters (clustering threshold 1.00) - 21 clusters (representing 859 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: Lg53 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg53_1.fastq /home/working/demultiplexed/Lg53_2.fastq Lg53_forward.paired.fastq.gz Lg53_forward.singletons.fastq.gz Lg53_reverse.paired.fastq.gz Lg53_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg53_1.fastq /home/working/demultiplexed/Lg53_2.fastq Lg53_forward.paired.fastq.gz Lg53_forward.singletons.fastq.gz Lg53_reverse.paired.fastq.gz Lg53_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:46:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/Lg53/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 22402 Both Surviving: 5675 (25.33%) Forward Only Surviving: 1586 (7.08%) Reverse Only Surviving: 283 (1.26%) Dropped: 14858 (66.32%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: Lg53_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: Lg53_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: Lg53_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: Lg53_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash Lg53_forward.paired.headclipped.fastq.gz Lg53_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg53 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     Lg53_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     Lg53_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./Lg53.extendedFrags.fastq.gz\n",
      "[FLASH]     ./Lg53.notCombined_1.fastq.gz\n",
      "[FLASH]     ./Lg53.notCombined_2.fastq.gz\n",
      "[FLASH]     ./Lg53.hist\n",
      "[FLASH]     ./Lg53.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5675 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5675\n",
      "[FLASH]     Combined pairs:   877\n",
      "[FLASH]     Uncombined pairs: 4798\n",
      "[FLASH]     Percent combined: 15.45%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.032 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "877 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/Lg53/Lg53_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg53_centroids.fasta --uc Lg53.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 877 sequences and identified 582 clusters (clustering threshold 1.00) - 15 clusters (representing 245 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: Lg57 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg57_1.fastq /home/working/demultiplexed/Lg57_2.fastq Lg57_forward.paired.fastq.gz Lg57_forward.singletons.fastq.gz Lg57_reverse.paired.fastq.gz Lg57_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg57_1.fastq /home/working/demultiplexed/Lg57_2.fastq Lg57_forward.paired.fastq.gz Lg57_forward.singletons.fastq.gz Lg57_reverse.paired.fastq.gz Lg57_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:46:06 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/Lg57/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 15465 Both Surviving: 4506 (29.14%) Forward Only Surviving: 1245 (8.05%) Reverse Only Surviving: 171 (1.11%) Dropped: 9543 (61.71%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: Lg57_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: Lg57_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: Lg57_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: Lg57_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash Lg57_forward.paired.headclipped.fastq.gz Lg57_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg57 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     Lg57_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     Lg57_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./Lg57.extendedFrags.fastq.gz\n",
      "[FLASH]     ./Lg57.notCombined_1.fastq.gz\n",
      "[FLASH]     ./Lg57.notCombined_2.fastq.gz\n",
      "[FLASH]     ./Lg57.hist\n",
      "[FLASH]     ./Lg57.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 4506 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      4506\n",
      "[FLASH]     Combined pairs:   399\n",
      "[FLASH]     Uncombined pairs: 4107\n",
      "[FLASH]     Percent combined: 8.85%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.030 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "399 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/Lg57/Lg57_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg57_centroids.fasta --uc Lg57.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 399 sequences and identified 318 clusters (clustering threshold 1.00) - 7 clusters (representing 56 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: Lg65 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg65_1.fastq /home/working/demultiplexed/Lg65_2.fastq Lg65_forward.paired.fastq.gz Lg65_forward.singletons.fastq.gz Lg65_reverse.paired.fastq.gz Lg65_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg65_1.fastq /home/working/demultiplexed/Lg65_2.fastq Lg65_forward.paired.fastq.gz Lg65_forward.singletons.fastq.gz Lg65_reverse.paired.fastq.gz Lg65_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:46:10 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/Lg65/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 37917 Both Surviving: 804 (2.12%) Forward Only Surviving: 550 (1.45%) Reverse Only Surviving: 304 (0.80%) Dropped: 36259 (95.63%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: Lg65_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: Lg65_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: Lg65_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: Lg65_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash Lg65_forward.paired.headclipped.fastq.gz Lg65_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg65 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     Lg65_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     Lg65_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./Lg65.extendedFrags.fastq.gz\n",
      "[FLASH]     ./Lg65.notCombined_1.fastq.gz\n",
      "[FLASH]     ./Lg65.notCombined_2.fastq.gz\n",
      "[FLASH]     ./Lg65.hist\n",
      "[FLASH]     ./Lg65.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 804 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      804\n",
      "[FLASH]     Combined pairs:   84\n",
      "[FLASH]     Uncombined pairs: 720\n",
      "[FLASH]     Percent combined: 10.45%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.006 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "84 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/Lg65/Lg65_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg65_centroids.fasta --uc Lg65.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 84 sequences and identified 61 clusters (clustering threshold 1.00) - 1 clusters (representing 22 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: Lg9 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg9_1.fastq /home/working/demultiplexed/Lg9_2.fastq Lg9_forward.paired.fastq.gz Lg9_forward.singletons.fastq.gz Lg9_reverse.paired.fastq.gz Lg9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Lg9_1.fastq /home/working/demultiplexed/Lg9_2.fastq Lg9_forward.paired.fastq.gz Lg9_forward.singletons.fastq.gz Lg9_reverse.paired.fastq.gz Lg9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:46:13 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/Lg9/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 19281 Both Surviving: 90 (0.47%) Forward Only Surviving: 335 (1.74%) Reverse Only Surviving: 199 (1.03%) Dropped: 18657 (96.76%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: Lg9_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: Lg9_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: Lg9_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: Lg9_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash Lg9_forward.paired.headclipped.fastq.gz Lg9_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Lg9 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     Lg9_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     Lg9_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./Lg9.extendedFrags.fastq.gz\n",
      "[FLASH]     ./Lg9.notCombined_1.fastq.gz\n",
      "[FLASH]     ./Lg9.notCombined_2.fastq.gz\n",
      "[FLASH]     ./Lg9.hist\n",
      "[FLASH]     ./Lg9.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 90 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      90\n",
      "[FLASH]     Combined pairs:   67\n",
      "[FLASH]     Uncombined pairs: 23\n",
      "[FLASH]     Percent combined: 74.44%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.003 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "67 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/Lg9/Lg9_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Lg9_centroids.fasta --uc Lg9.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 67 sequences and identified 52 clusters (clustering threshold 1.00) - 2 clusters (representing 13 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: NEG #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/NEG_1.fastq /home/working/demultiplexed/NEG_2.fastq NEG_forward.paired.fastq.gz NEG_forward.singletons.fastq.gz NEG_reverse.paired.fastq.gz NEG_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/NEG_1.fastq /home/working/demultiplexed/NEG_2.fastq NEG_forward.paired.fastq.gz NEG_forward.singletons.fastq.gz NEG_reverse.paired.fastq.gz NEG_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:46:15 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/NEG/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 3505 Both Surviving: 402 (11.47%) Forward Only Surviving: 167 (4.76%) Reverse Only Surviving: 54 (1.54%) Dropped: 2882 (82.23%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: NEG_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: NEG_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: NEG_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: NEG_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash NEG_forward.paired.headclipped.fastq.gz NEG_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o NEG -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     NEG_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     NEG_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./NEG.extendedFrags.fastq.gz\n",
      "[FLASH]     ./NEG.notCombined_1.fastq.gz\n",
      "[FLASH]     ./NEG.notCombined_2.fastq.gz\n",
      "[FLASH]     ./NEG.hist\n",
      "[FLASH]     ./NEG.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 402 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      402\n",
      "[FLASH]     Combined pairs:   139\n",
      "[FLASH]     Uncombined pairs: 263\n",
      "[FLASH]     Percent combined: 34.58%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.005 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "139 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/NEG/NEG_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids NEG_centroids.fasta --uc NEG.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 139 sequences and identified 115 clusters (clustering threshold 1.00) - 2 clusters (representing 15 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: O84 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/O84_1.fastq /home/working/demultiplexed/O84_2.fastq O84_forward.paired.fastq.gz O84_forward.singletons.fastq.gz O84_reverse.paired.fastq.gz O84_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/O84_1.fastq /home/working/demultiplexed/O84_2.fastq O84_forward.paired.fastq.gz O84_forward.singletons.fastq.gz O84_reverse.paired.fastq.gz O84_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:46:18 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/O84/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 36398 Both Surviving: 15690 (43.11%) Forward Only Surviving: 5948 (16.34%) Reverse Only Surviving: 326 (0.90%) Dropped: 14434 (39.66%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: O84_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: O84_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: O84_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: O84_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash O84_forward.paired.headclipped.fastq.gz O84_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o O84 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     O84_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     O84_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./O84.extendedFrags.fastq.gz\n",
      "[FLASH]     ./O84.notCombined_1.fastq.gz\n",
      "[FLASH]     ./O84.notCombined_2.fastq.gz\n",
      "[FLASH]     ./O84.hist\n",
      "[FLASH]     ./O84.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 15690 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      15690\n",
      "[FLASH]     Combined pairs:   13492\n",
      "[FLASH]     Uncombined pairs: 2198\n",
      "[FLASH]     Percent combined: 85.99%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.355 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "13492 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/O84/O84_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids O84_centroids.fasta --uc O84.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 13492 sequences and identified 4050 clusters (clustering threshold 1.00) - 312 clusters (representing 9371 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: O96 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/O96_1.fastq /home/working/demultiplexed/O96_2.fastq O96_forward.paired.fastq.gz O96_forward.singletons.fastq.gz O96_reverse.paired.fastq.gz O96_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/O96_1.fastq /home/working/demultiplexed/O96_2.fastq O96_forward.paired.fastq.gz O96_forward.singletons.fastq.gz O96_reverse.paired.fastq.gz O96_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:46:42 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/O96/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 14802 Both Surviving: 5346 (36.12%) Forward Only Surviving: 1579 (10.67%) Reverse Only Surviving: 210 (1.42%) Dropped: 7667 (51.80%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: O96_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: O96_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: O96_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: O96_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash O96_forward.paired.headclipped.fastq.gz O96_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o O96 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     O96_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     O96_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./O96.extendedFrags.fastq.gz\n",
      "[FLASH]     ./O96.notCombined_1.fastq.gz\n",
      "[FLASH]     ./O96.notCombined_2.fastq.gz\n",
      "[FLASH]     ./O96.hist\n",
      "[FLASH]     ./O96.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5346 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5346\n",
      "[FLASH]     Combined pairs:   1914\n",
      "[FLASH]     Uncombined pairs: 3432\n",
      "[FLASH]     Percent combined: 35.80%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.058 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1914 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/O96/O96_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids O96_centroids.fasta --uc O96.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1914 sequences and identified 1228 clusters (clustering threshold 1.00) - 28 clusters (representing 583 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: P18 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P18_1.fastq /home/working/demultiplexed/P18_2.fastq P18_forward.paired.fastq.gz P18_forward.singletons.fastq.gz P18_reverse.paired.fastq.gz P18_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P18_1.fastq /home/working/demultiplexed/P18_2.fastq P18_forward.paired.fastq.gz P18_forward.singletons.fastq.gz P18_reverse.paired.fastq.gz P18_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:46:48 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/P18/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 25495 Both Surviving: 5531 (21.69%) Forward Only Surviving: 1954 (7.66%) Reverse Only Surviving: 326 (1.28%) Dropped: 17684 (69.36%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: P18_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: P18_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: P18_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: P18_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash P18_forward.paired.headclipped.fastq.gz P18_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o P18 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     P18_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     P18_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./P18.extendedFrags.fastq.gz\n",
      "[FLASH]     ./P18.notCombined_1.fastq.gz\n",
      "[FLASH]     ./P18.notCombined_2.fastq.gz\n",
      "[FLASH]     ./P18.hist\n",
      "[FLASH]     ./P18.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5531 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5531\n",
      "[FLASH]     Combined pairs:   3010\n",
      "[FLASH]     Uncombined pairs: 2521\n",
      "[FLASH]     Percent combined: 54.42%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.083 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "3010 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/P18/P18_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids P18_centroids.fasta --uc P18.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 3010 sequences and identified 1116 clusters (clustering threshold 1.00) - 67 clusters (representing 1723 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: P54 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P54_1.fastq /home/working/demultiplexed/P54_2.fastq P54_forward.paired.fastq.gz P54_forward.singletons.fastq.gz P54_reverse.paired.fastq.gz P54_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P54_1.fastq /home/working/demultiplexed/P54_2.fastq P54_forward.paired.fastq.gz P54_forward.singletons.fastq.gz P54_reverse.paired.fastq.gz P54_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:46:57 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/P54/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 50485 Both Surviving: 15012 (29.74%) Forward Only Surviving: 3807 (7.54%) Reverse Only Surviving: 1539 (3.05%) Dropped: 30127 (59.68%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: P54_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: P54_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: P54_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: P54_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash P54_forward.paired.headclipped.fastq.gz P54_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o P54 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     P54_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     P54_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./P54.extendedFrags.fastq.gz\n",
      "[FLASH]     ./P54.notCombined_1.fastq.gz\n",
      "[FLASH]     ./P54.notCombined_2.fastq.gz\n",
      "[FLASH]     ./P54.hist\n",
      "[FLASH]     ./P54.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 15012 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      15012\n",
      "[FLASH]     Combined pairs:   5359\n",
      "[FLASH]     Uncombined pairs: 9653\n",
      "[FLASH]     Percent combined: 35.70%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.172 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "5359 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/P54/P54_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids P54_centroids.fasta --uc P54.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 5359 sequences and identified 2153 clusters (clustering threshold 1.00) - 86 clusters (representing 2811 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: P59 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P59_1.fastq /home/working/demultiplexed/P59_2.fastq P59_forward.paired.fastq.gz P59_forward.singletons.fastq.gz P59_reverse.paired.fastq.gz P59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P59_1.fastq /home/working/demultiplexed/P59_2.fastq P59_forward.paired.fastq.gz P59_forward.singletons.fastq.gz P59_reverse.paired.fastq.gz P59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:47:13 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/P59/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 10337 Both Surviving: 1710 (16.54%) Forward Only Surviving: 420 (4.06%) Reverse Only Surviving: 272 (2.63%) Dropped: 7935 (76.76%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: P59_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: P59_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: P59_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: P59_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash P59_forward.paired.headclipped.fastq.gz P59_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o P59 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     P59_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     P59_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./P59.extendedFrags.fastq.gz\n",
      "[FLASH]     ./P59.notCombined_1.fastq.gz\n",
      "[FLASH]     ./P59.notCombined_2.fastq.gz\n",
      "[FLASH]     ./P59.hist\n",
      "[FLASH]     ./P59.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1710 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1710\n",
      "[FLASH]     Combined pairs:   29\n",
      "[FLASH]     Uncombined pairs: 1681\n",
      "[FLASH]     Percent combined: 1.70%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.013 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "29 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/P59/P59_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids P59_centroids.fasta --uc P59.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 29 sequences and identified 28 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: P60 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P60_1.fastq /home/working/demultiplexed/P60_2.fastq P60_forward.paired.fastq.gz P60_forward.singletons.fastq.gz P60_reverse.paired.fastq.gz P60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P60_1.fastq /home/working/demultiplexed/P60_2.fastq P60_forward.paired.fastq.gz P60_forward.singletons.fastq.gz P60_reverse.paired.fastq.gz P60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:47:16 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/P60/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 26955 Both Surviving: 10127 (37.57%) Forward Only Surviving: 3468 (12.87%) Reverse Only Surviving: 309 (1.15%) Dropped: 13051 (48.42%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: P60_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: P60_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: P60_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: P60_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash P60_forward.paired.headclipped.fastq.gz P60_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o P60 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     P60_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     P60_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./P60.extendedFrags.fastq.gz\n",
      "[FLASH]     ./P60.notCombined_1.fastq.gz\n",
      "[FLASH]     ./P60.notCombined_2.fastq.gz\n",
      "[FLASH]     ./P60.hist\n",
      "[FLASH]     ./P60.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 10127 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      10127\n",
      "[FLASH]     Combined pairs:   5608\n",
      "[FLASH]     Uncombined pairs: 4519\n",
      "[FLASH]     Percent combined: 55.38%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.155 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "5608 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/P60/P60_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids P60_centroids.fasta --uc P60.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 5608 sequences and identified 3054 clusters (clustering threshold 1.00) - 92 clusters (representing 2206 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: P73 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P73_1.fastq /home/working/demultiplexed/P73_2.fastq P73_forward.paired.fastq.gz P73_forward.singletons.fastq.gz P73_reverse.paired.fastq.gz P73_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P73_1.fastq /home/working/demultiplexed/P73_2.fastq P73_forward.paired.fastq.gz P73_forward.singletons.fastq.gz P73_reverse.paired.fastq.gz P73_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:47:28 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/P73/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 22 bases off reads in file: P73_forward.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: P73_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: P73_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: P73_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash P73_forward.paired.headclipped.fastq.gz P73_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o P73 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     P73_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     P73_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./P73.extendedFrags.fastq.gz\n",
      "[FLASH]     ./P73.notCombined_1.fastq.gz\n",
      "[FLASH]     ./P73.notCombined_2.fastq.gz\n",
      "[FLASH]     ./P73.hist\n",
      "[FLASH]     ./P73.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      0\n",
      "[FLASH]     Combined pairs:   0\n",
      "[FLASH]     Uncombined pairs: 0\n",
      "[FLASH]     Percent combined: 0.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "0 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/P73/P73_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids P73_centroids.fasta --uc P73.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: P98 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P98_1.fastq /home/working/demultiplexed/P98_2.fastq P98_forward.paired.fastq.gz P98_forward.singletons.fastq.gz P98_reverse.paired.fastq.gz P98_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/P98_1.fastq /home/working/demultiplexed/P98_2.fastq P98_forward.paired.fastq.gz P98_forward.singletons.fastq.gz P98_reverse.paired.fastq.gz P98_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:47:29 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/P98/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 4 Both Surviving: 1 (25.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 3 (75.00%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 22 bases off reads in file: P98_forward.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: P98_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: P98_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: P98_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash P98_forward.paired.headclipped.fastq.gz P98_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o P98 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     P98_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     P98_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./P98.extendedFrags.fastq.gz\n",
      "[FLASH]     ./P98.notCombined_1.fastq.gz\n",
      "[FLASH]     ./P98.notCombined_2.fastq.gz\n",
      "[FLASH]     ./P98.hist\n",
      "[FLASH]     ./P98.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1\n",
      "[FLASH]     Combined pairs:   1\n",
      "[FLASH]     Uncombined pairs: 0\n",
      "[FLASH]     Percent combined: 100.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/P98/P98_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids P98_centroids.fasta --uc P98.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1 sequences and identified 1 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: POS1 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/POS1_1.fastq /home/working/demultiplexed/POS1_2.fastq POS1_forward.paired.fastq.gz POS1_forward.singletons.fastq.gz POS1_reverse.paired.fastq.gz POS1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/POS1_1.fastq /home/working/demultiplexed/POS1_2.fastq POS1_forward.paired.fastq.gz POS1_forward.singletons.fastq.gz POS1_reverse.paired.fastq.gz POS1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:47:30 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/POS1/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 28492 Both Surviving: 10615 (37.26%) Forward Only Surviving: 3913 (13.73%) Reverse Only Surviving: 296 (1.04%) Dropped: 13668 (47.97%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: POS1_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: POS1_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: POS1_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: POS1_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash POS1_forward.paired.headclipped.fastq.gz POS1_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS1 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     POS1_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     POS1_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./POS1.extendedFrags.fastq.gz\n",
      "[FLASH]     ./POS1.notCombined_1.fastq.gz\n",
      "[FLASH]     ./POS1.notCombined_2.fastq.gz\n",
      "[FLASH]     ./POS1.hist\n",
      "[FLASH]     ./POS1.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 10615 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      10615\n",
      "[FLASH]     Combined pairs:   6159\n",
      "[FLASH]     Uncombined pairs: 4456\n",
      "[FLASH]     Percent combined: 58.02%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.171 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "6159 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/POS1/POS1_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS1_centroids.fasta --uc POS1.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 6159 sequences and identified 4116 clusters (clustering threshold 1.00) - 88 clusters (representing 1672 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: POS2 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/POS2_1.fastq /home/working/demultiplexed/POS2_2.fastq POS2_forward.paired.fastq.gz POS2_forward.singletons.fastq.gz POS2_reverse.paired.fastq.gz POS2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/POS2_1.fastq /home/working/demultiplexed/POS2_2.fastq POS2_forward.paired.fastq.gz POS2_forward.singletons.fastq.gz POS2_reverse.paired.fastq.gz POS2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:47:44 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/POS2/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 11207 Both Surviving: 1086 (9.69%) Forward Only Surviving: 317 (2.83%) Reverse Only Surviving: 109 (0.97%) Dropped: 9695 (86.51%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: POS2_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: POS2_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: POS2_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: POS2_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash POS2_forward.paired.headclipped.fastq.gz POS2_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS2 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     POS2_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     POS2_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./POS2.extendedFrags.fastq.gz\n",
      "[FLASH]     ./POS2.notCombined_1.fastq.gz\n",
      "[FLASH]     ./POS2.notCombined_2.fastq.gz\n",
      "[FLASH]     ./POS2.hist\n",
      "[FLASH]     ./POS2.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1086 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1086\n",
      "[FLASH]     Combined pairs:   356\n",
      "[FLASH]     Uncombined pairs: 730\n",
      "[FLASH]     Percent combined: 32.78%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.011 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "356 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/POS2/POS2_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS2_centroids.fasta --uc POS2.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 356 sequences and identified 221 clusters (clustering threshold 1.00) - 3 clusters (representing 121 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: POS3 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/POS3_1.fastq /home/working/demultiplexed/POS3_2.fastq POS3_forward.paired.fastq.gz POS3_forward.singletons.fastq.gz POS3_reverse.paired.fastq.gz POS3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/POS3_1.fastq /home/working/demultiplexed/POS3_2.fastq POS3_forward.paired.fastq.gz POS3_forward.singletons.fastq.gz POS3_reverse.paired.fastq.gz POS3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:47:46 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/POS3/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 10258 Both Surviving: 3469 (33.82%) Forward Only Surviving: 838 (8.17%) Reverse Only Surviving: 138 (1.35%) Dropped: 5813 (56.67%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: POS3_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: POS3_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: POS3_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: POS3_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash POS3_forward.paired.headclipped.fastq.gz POS3_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS3 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     POS3_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     POS3_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./POS3.extendedFrags.fastq.gz\n",
      "[FLASH]     ./POS3.notCombined_1.fastq.gz\n",
      "[FLASH]     ./POS3.notCombined_2.fastq.gz\n",
      "[FLASH]     ./POS3.hist\n",
      "[FLASH]     ./POS3.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3469 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3469\n",
      "[FLASH]     Combined pairs:   2854\n",
      "[FLASH]     Uncombined pairs: 615\n",
      "[FLASH]     Percent combined: 82.27%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.076 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2854 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/POS3/POS3_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS3_centroids.fasta --uc POS3.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2854 sequences and identified 1775 clusters (clustering threshold 1.00) - 27 clusters (representing 794 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: POS4 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/POS4_1.fastq /home/working/demultiplexed/POS4_2.fastq POS4_forward.paired.fastq.gz POS4_forward.singletons.fastq.gz POS4_reverse.paired.fastq.gz POS4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/POS4_1.fastq /home/working/demultiplexed/POS4_2.fastq POS4_forward.paired.fastq.gz POS4_forward.singletons.fastq.gz POS4_reverse.paired.fastq.gz POS4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:47:53 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/POS4/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 11301 Both Surviving: 1999 (17.69%) Forward Only Surviving: 548 (4.85%) Reverse Only Surviving: 109 (0.96%) Dropped: 8645 (76.50%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: POS4_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: POS4_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: POS4_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: POS4_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash POS4_forward.paired.headclipped.fastq.gz POS4_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o POS4 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     POS4_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     POS4_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./POS4.extendedFrags.fastq.gz\n",
      "[FLASH]     ./POS4.notCombined_1.fastq.gz\n",
      "[FLASH]     ./POS4.notCombined_2.fastq.gz\n",
      "[FLASH]     ./POS4.hist\n",
      "[FLASH]     ./POS4.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1999 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1999\n",
      "[FLASH]     Combined pairs:   1192\n",
      "[FLASH]     Uncombined pairs: 807\n",
      "[FLASH]     Percent combined: 59.63%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.032 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1192 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/POS4/POS4_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids POS4_centroids.fasta --uc POS4.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1192 sequences and identified 786 clusters (clustering threshold 1.00) - 13 clusters (representing 296 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: PROC #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/PROC_1.fastq /home/working/demultiplexed/PROC_2.fastq PROC_forward.paired.fastq.gz PROC_forward.singletons.fastq.gz PROC_reverse.paired.fastq.gz PROC_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/PROC_1.fastq /home/working/demultiplexed/PROC_2.fastq PROC_forward.paired.fastq.gz PROC_forward.singletons.fastq.gz PROC_reverse.paired.fastq.gz PROC_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:47:57 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/PROC/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 16777 Both Surviving: 3210 (19.13%) Forward Only Surviving: 860 (5.13%) Reverse Only Surviving: 167 (1.00%) Dropped: 12540 (74.75%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: PROC_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: PROC_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: PROC_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: PROC_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash PROC_forward.paired.headclipped.fastq.gz PROC_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o PROC -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     PROC_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     PROC_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./PROC.extendedFrags.fastq.gz\n",
      "[FLASH]     ./PROC.notCombined_1.fastq.gz\n",
      "[FLASH]     ./PROC.notCombined_2.fastq.gz\n",
      "[FLASH]     ./PROC.hist\n",
      "[FLASH]     ./PROC.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3210 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3210\n",
      "[FLASH]     Combined pairs:   155\n",
      "[FLASH]     Uncombined pairs: 3055\n",
      "[FLASH]     Percent combined: 4.83%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.021 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "155 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/PROC/PROC_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids PROC_centroids.fasta --uc PROC.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 155 sequences and identified 116 clusters (clustering threshold 1.00) - 4 clusters (representing 38 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: Q10 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q10_1.fastq /home/working/demultiplexed/Q10_2.fastq Q10_forward.paired.fastq.gz Q10_forward.singletons.fastq.gz Q10_reverse.paired.fastq.gz Q10_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q10_1.fastq /home/working/demultiplexed/Q10_2.fastq Q10_forward.paired.fastq.gz Q10_forward.singletons.fastq.gz Q10_reverse.paired.fastq.gz Q10_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:48:00 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/Q10/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 23215 Both Surviving: 1090 (4.70%) Forward Only Surviving: 694 (2.99%) Reverse Only Surviving: 300 (1.29%) Dropped: 21131 (91.02%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: Q10_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: Q10_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: Q10_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: Q10_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash Q10_forward.paired.headclipped.fastq.gz Q10_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Q10 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     Q10_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     Q10_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./Q10.extendedFrags.fastq.gz\n",
      "[FLASH]     ./Q10.notCombined_1.fastq.gz\n",
      "[FLASH]     ./Q10.notCombined_2.fastq.gz\n",
      "[FLASH]     ./Q10.hist\n",
      "[FLASH]     ./Q10.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1090 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1090\n",
      "[FLASH]     Combined pairs:   251\n",
      "[FLASH]     Uncombined pairs: 839\n",
      "[FLASH]     Percent combined: 23.03%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.012 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "251 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/Q10/Q10_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Q10_centroids.fasta --uc Q10.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 251 sequences and identified 203 clusters (clustering threshold 1.00) - 4 clusters (representing 38 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: Q30 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q30_1.fastq /home/working/demultiplexed/Q30_2.fastq Q30_forward.paired.fastq.gz Q30_forward.singletons.fastq.gz Q30_reverse.paired.fastq.gz Q30_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q30_1.fastq /home/working/demultiplexed/Q30_2.fastq Q30_forward.paired.fastq.gz Q30_forward.singletons.fastq.gz Q30_reverse.paired.fastq.gz Q30_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:48:02 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/Q30/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 2 Both Surviving: 2 (100.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 0 (0.00%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 22 bases off reads in file: Q30_forward.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: Q30_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: Q30_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: Q30_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash Q30_forward.paired.headclipped.fastq.gz Q30_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Q30 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     Q30_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     Q30_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./Q30.extendedFrags.fastq.gz\n",
      "[FLASH]     ./Q30.notCombined_1.fastq.gz\n",
      "[FLASH]     ./Q30.notCombined_2.fastq.gz\n",
      "[FLASH]     ./Q30.hist\n",
      "[FLASH]     ./Q30.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2\n",
      "[FLASH]     Combined pairs:   2\n",
      "[FLASH]     Uncombined pairs: 0\n",
      "[FLASH]     Percent combined: 100.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/Q30/Q30_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Q30_centroids.fasta --uc Q30.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2 sequences and identified 2 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: Q43 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q43_1.fastq /home/working/demultiplexed/Q43_2.fastq Q43_forward.paired.fastq.gz Q43_forward.singletons.fastq.gz Q43_reverse.paired.fastq.gz Q43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q43_1.fastq /home/working/demultiplexed/Q43_2.fastq Q43_forward.paired.fastq.gz Q43_forward.singletons.fastq.gz Q43_reverse.paired.fastq.gz Q43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:48:02 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/Q43/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 1 Both Surviving: 0 (0.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 1 (100.00%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 22 bases off reads in file: Q43_forward.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: Q43_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: Q43_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: Q43_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash Q43_forward.paired.headclipped.fastq.gz Q43_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Q43 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     Q43_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     Q43_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./Q43.extendedFrags.fastq.gz\n",
      "[FLASH]     ./Q43.notCombined_1.fastq.gz\n",
      "[FLASH]     ./Q43.notCombined_2.fastq.gz\n",
      "[FLASH]     ./Q43.hist\n",
      "[FLASH]     ./Q43.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      0\n",
      "[FLASH]     Combined pairs:   0\n",
      "[FLASH]     Uncombined pairs: 0\n",
      "[FLASH]     Percent combined: 0.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "0 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/Q43/Q43_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Q43_centroids.fasta --uc Q43.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: Q64 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q64_1.fastq /home/working/demultiplexed/Q64_2.fastq Q64_forward.paired.fastq.gz Q64_forward.singletons.fastq.gz Q64_reverse.paired.fastq.gz Q64_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q64_1.fastq /home/working/demultiplexed/Q64_2.fastq Q64_forward.paired.fastq.gz Q64_forward.singletons.fastq.gz Q64_reverse.paired.fastq.gz Q64_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:48:03 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/Q64/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 2 Both Surviving: 1 (50.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 1 (50.00%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 22 bases off reads in file: Q64_forward.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: Q64_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: Q64_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: Q64_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash Q64_forward.paired.headclipped.fastq.gz Q64_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Q64 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     Q64_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     Q64_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./Q64.extendedFrags.fastq.gz\n",
      "[FLASH]     ./Q64.notCombined_1.fastq.gz\n",
      "[FLASH]     ./Q64.notCombined_2.fastq.gz\n",
      "[FLASH]     ./Q64.hist\n",
      "[FLASH]     ./Q64.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1\n",
      "[FLASH]     Combined pairs:   1\n",
      "[FLASH]     Uncombined pairs: 0\n",
      "[FLASH]     Percent combined: 100.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/Q64/Q64_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Q64_centroids.fasta --uc Q64.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1 sequences and identified 1 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: Q69 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q69_1.fastq /home/working/demultiplexed/Q69_2.fastq Q69_forward.paired.fastq.gz Q69_forward.singletons.fastq.gz Q69_reverse.paired.fastq.gz Q69_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q69_1.fastq /home/working/demultiplexed/Q69_2.fastq Q69_forward.paired.fastq.gz Q69_forward.singletons.fastq.gz Q69_reverse.paired.fastq.gz Q69_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:48:03 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/Q69/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 4825 Both Surviving: 51 (1.06%) Forward Only Surviving: 97 (2.01%) Reverse Only Surviving: 34 (0.70%) Dropped: 4643 (96.23%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: Q69_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: Q69_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: Q69_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: Q69_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash Q69_forward.paired.headclipped.fastq.gz Q69_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Q69 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     Q69_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     Q69_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./Q69.extendedFrags.fastq.gz\n",
      "[FLASH]     ./Q69.notCombined_1.fastq.gz\n",
      "[FLASH]     ./Q69.notCombined_2.fastq.gz\n",
      "[FLASH]     ./Q69.hist\n",
      "[FLASH]     ./Q69.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 51 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      51\n",
      "[FLASH]     Combined pairs:   21\n",
      "[FLASH]     Uncombined pairs: 30\n",
      "[FLASH]     Percent combined: 41.18%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "21 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/Q69/Q69_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Q69_centroids.fasta --uc Q69.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 21 sequences and identified 20 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: Q83 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q83_1.fastq /home/working/demultiplexed/Q83_2.fastq Q83_forward.paired.fastq.gz Q83_forward.singletons.fastq.gz Q83_reverse.paired.fastq.gz Q83_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/Q83_1.fastq /home/working/demultiplexed/Q83_2.fastq Q83_forward.paired.fastq.gz Q83_forward.singletons.fastq.gz Q83_reverse.paired.fastq.gz Q83_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:48:05 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/Q83/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 29958 Both Surviving: 5419 (18.09%) Forward Only Surviving: 1242 (4.15%) Reverse Only Surviving: 466 (1.56%) Dropped: 22831 (76.21%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: Q83_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: Q83_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: Q83_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: Q83_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash Q83_forward.paired.headclipped.fastq.gz Q83_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o Q83 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     Q83_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     Q83_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./Q83.extendedFrags.fastq.gz\n",
      "[FLASH]     ./Q83.notCombined_1.fastq.gz\n",
      "[FLASH]     ./Q83.notCombined_2.fastq.gz\n",
      "[FLASH]     ./Q83.hist\n",
      "[FLASH]     ./Q83.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5419 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5419\n",
      "[FLASH]     Combined pairs:   1324\n",
      "[FLASH]     Uncombined pairs: 4095\n",
      "[FLASH]     Percent combined: 24.43%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.054 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1324 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/Q83/Q83_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids Q83_centroids.fasta --uc Q83.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1324 sequences and identified 822 clusters (clustering threshold 1.00) - 17 clusters (representing 405 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R1 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R1_1.fastq /home/working/demultiplexed/R1_2.fastq R1_forward.paired.fastq.gz R1_forward.singletons.fastq.gz R1_reverse.paired.fastq.gz R1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R1_1.fastq /home/working/demultiplexed/R1_2.fastq R1_forward.paired.fastq.gz R1_forward.singletons.fastq.gz R1_reverse.paired.fastq.gz R1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:48:11 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R1/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 16656 Both Surviving: 5127 (30.78%) Forward Only Surviving: 1237 (7.43%) Reverse Only Surviving: 238 (1.43%) Dropped: 10054 (60.36%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: R1_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: R1_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: R1_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: R1_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R1_forward.paired.headclipped.fastq.gz R1_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R1 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R1_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R1_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R1.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R1.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R1.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R1.hist\n",
      "[FLASH]     ./R1.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5127 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5127\n",
      "[FLASH]     Combined pairs:   2511\n",
      "[FLASH]     Uncombined pairs: 2616\n",
      "[FLASH]     Percent combined: 48.98%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.070 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2511 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R1/R1_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R1_centroids.fasta --uc R1.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2511 sequences and identified 872 clusters (clustering threshold 1.00) - 44 clusters (representing 1438 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R10 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R10_1.fastq /home/working/demultiplexed/R10_2.fastq R10_forward.paired.fastq.gz R10_forward.singletons.fastq.gz R10_reverse.paired.fastq.gz R10_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R10_1.fastq /home/working/demultiplexed/R10_2.fastq R10_forward.paired.fastq.gz R10_forward.singletons.fastq.gz R10_reverse.paired.fastq.gz R10_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:48:18 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R10/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 22 bases off reads in file: R10_forward.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: R10_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: R10_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: R10_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R10_forward.paired.headclipped.fastq.gz R10_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R10 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R10_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R10_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R10.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R10.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R10.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R10.hist\n",
      "[FLASH]     ./R10.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      0\n",
      "[FLASH]     Combined pairs:   0\n",
      "[FLASH]     Uncombined pairs: 0\n",
      "[FLASH]     Percent combined: 0.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "0 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R10/R10_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R10_centroids.fasta --uc R10.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R102 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R102_1.fastq /home/working/demultiplexed/R102_2.fastq R102_forward.paired.fastq.gz R102_forward.singletons.fastq.gz R102_reverse.paired.fastq.gz R102_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R102_1.fastq /home/working/demultiplexed/R102_2.fastq R102_forward.paired.fastq.gz R102_forward.singletons.fastq.gz R102_reverse.paired.fastq.gz R102_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:48:21 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R102/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 62773 Both Surviving: 13515 (21.53%) Forward Only Surviving: 3949 (6.29%) Reverse Only Surviving: 1852 (2.95%) Dropped: 43457 (69.23%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: R102_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R102_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R102_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R102_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R102_forward.paired.headclipped.fastq.gz R102_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R102 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R102_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R102_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R102.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R102.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R102.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R102.hist\n",
      "[FLASH]     ./R102.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 13515 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      13515\n",
      "[FLASH]     Combined pairs:   765\n",
      "[FLASH]     Uncombined pairs: 12750\n",
      "[FLASH]     Percent combined: 5.66%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.092 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "765 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R102/R102_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R102_centroids.fasta --uc R102.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 765 sequences and identified 582 clusters (clustering threshold 1.00) - 14 clusters (representing 145 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R13 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R13_1.fastq /home/working/demultiplexed/R13_2.fastq R13_forward.paired.fastq.gz R13_forward.singletons.fastq.gz R13_reverse.paired.fastq.gz R13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R13_1.fastq /home/working/demultiplexed/R13_2.fastq R13_forward.paired.fastq.gz R13_forward.singletons.fastq.gz R13_reverse.paired.fastq.gz R13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:48:30 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R13/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 19033 Both Surviving: 4660 (24.48%) Forward Only Surviving: 1222 (6.42%) Reverse Only Surviving: 224 (1.18%) Dropped: 12927 (67.92%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: R13_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: R13_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R13_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R13_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R13_forward.paired.headclipped.fastq.gz R13_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R13 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R13_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R13_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R13.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R13.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R13.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R13.hist\n",
      "[FLASH]     ./R13.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 4660 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      4660\n",
      "[FLASH]     Combined pairs:   629\n",
      "[FLASH]     Uncombined pairs: 4031\n",
      "[FLASH]     Percent combined: 13.50%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.036 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "629 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R13/R13_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R13_centroids.fasta --uc R13.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 629 sequences and identified 406 clusters (clustering threshold 1.00) - 12 clusters (representing 203 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R15 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R15_1.fastq /home/working/demultiplexed/R15_2.fastq R15_forward.paired.fastq.gz R15_forward.singletons.fastq.gz R15_reverse.paired.fastq.gz R15_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R15_1.fastq /home/working/demultiplexed/R15_2.fastq R15_forward.paired.fastq.gz R15_forward.singletons.fastq.gz R15_reverse.paired.fastq.gz R15_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:48:37 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R15/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 65631 Both Surviving: 3298 (5.03%) Forward Only Surviving: 2188 (3.33%) Reverse Only Surviving: 804 (1.23%) Dropped: 59341 (90.42%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: R15_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: R15_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: R15_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: R15_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R15_forward.paired.headclipped.fastq.gz R15_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R15 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R15_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R15_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R15.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R15.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R15.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R15.hist\n",
      "[FLASH]     ./R15.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3298 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3298\n",
      "[FLASH]     Combined pairs:   2086\n",
      "[FLASH]     Uncombined pairs: 1212\n",
      "[FLASH]     Percent combined: 63.25%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.061 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2086 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R15/R15_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R15_centroids.fasta --uc R15.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2086 sequences and identified 1204 clusters (clustering threshold 1.00) - 32 clusters (representing 807 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R16 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R16_1.fastq /home/working/demultiplexed/R16_2.fastq R16_forward.paired.fastq.gz R16_forward.singletons.fastq.gz R16_reverse.paired.fastq.gz R16_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R16_1.fastq /home/working/demultiplexed/R16_2.fastq R16_forward.paired.fastq.gz R16_forward.singletons.fastq.gz R16_reverse.paired.fastq.gz R16_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:48:45 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R16/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 25112 Both Surviving: 8135 (32.39%) Forward Only Surviving: 3246 (12.93%) Reverse Only Surviving: 283 (1.13%) Dropped: 13448 (53.55%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: R16_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: R16_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: R16_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: R16_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R16_forward.paired.headclipped.fastq.gz R16_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R16 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R16_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R16_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R16.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R16.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R16.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R16.hist\n",
      "[FLASH]     ./R16.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 8135 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      8135\n",
      "[FLASH]     Combined pairs:   4002\n",
      "[FLASH]     Uncombined pairs: 4133\n",
      "[FLASH]     Percent combined: 49.19%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.114 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "4002 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R16/R16_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R16_centroids.fasta --uc R16.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 4002 sequences and identified 2239 clusters (clustering threshold 1.00) - 59 clusters (representing 1497 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R20 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R20_1.fastq /home/working/demultiplexed/R20_2.fastq R20_forward.paired.fastq.gz R20_forward.singletons.fastq.gz R20_reverse.paired.fastq.gz R20_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R20_1.fastq /home/working/demultiplexed/R20_2.fastq R20_forward.paired.fastq.gz R20_forward.singletons.fastq.gz R20_reverse.paired.fastq.gz R20_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:48:56 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R20/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 41196 Both Surviving: 10311 (25.03%) Forward Only Surviving: 6801 (16.51%) Reverse Only Surviving: 389 (0.94%) Dropped: 23695 (57.52%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: R20_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: R20_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: R20_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: R20_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R20_forward.paired.headclipped.fastq.gz R20_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R20 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R20_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R20_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R20.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R20.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R20.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R20.hist\n",
      "[FLASH]     ./R20.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 10311 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      10311\n",
      "[FLASH]     Combined pairs:   8629\n",
      "[FLASH]     Uncombined pairs: 1682\n",
      "[FLASH]     Percent combined: 83.69%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.230 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "8629 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R20/R20_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R20_centroids.fasta --uc R20.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 8629 sequences and identified 3449 clusters (clustering threshold 1.00) - 186 clusters (representing 4843 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R22 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R22_1.fastq /home/working/demultiplexed/R22_2.fastq R22_forward.paired.fastq.gz R22_forward.singletons.fastq.gz R22_reverse.paired.fastq.gz R22_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R22_1.fastq /home/working/demultiplexed/R22_2.fastq R22_forward.paired.fastq.gz R22_forward.singletons.fastq.gz R22_reverse.paired.fastq.gz R22_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:49:14 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R22/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 40727 Both Surviving: 5266 (12.93%) Forward Only Surviving: 1531 (3.76%) Reverse Only Surviving: 618 (1.52%) Dropped: 33312 (81.79%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: R22_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: R22_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: R22_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: R22_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R22_forward.paired.headclipped.fastq.gz R22_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R22 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R22_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R22_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R22.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R22.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R22.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R22.hist\n",
      "[FLASH]     ./R22.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5266 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5266\n",
      "[FLASH]     Combined pairs:   1400\n",
      "[FLASH]     Uncombined pairs: 3866\n",
      "[FLASH]     Percent combined: 26.59%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.047 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1400 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R22/R22_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R22_centroids.fasta --uc R22.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1400 sequences and identified 870 clusters (clustering threshold 1.00) - 21 clusters (representing 458 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R23 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R23_1.fastq /home/working/demultiplexed/R23_2.fastq R23_forward.paired.fastq.gz R23_forward.singletons.fastq.gz R23_reverse.paired.fastq.gz R23_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R23_1.fastq /home/working/demultiplexed/R23_2.fastq R23_forward.paired.fastq.gz R23_forward.singletons.fastq.gz R23_reverse.paired.fastq.gz R23_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:49:21 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R23/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 27238 Both Surviving: 5046 (18.53%) Forward Only Surviving: 994 (3.65%) Reverse Only Surviving: 810 (2.97%) Dropped: 20388 (74.85%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: R23_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R23_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: R23_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: R23_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R23_forward.paired.headclipped.fastq.gz R23_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R23 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R23_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R23_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R23.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R23.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R23.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R23.hist\n",
      "[FLASH]     ./R23.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5046 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5046\n",
      "[FLASH]     Combined pairs:   642\n",
      "[FLASH]     Uncombined pairs: 4404\n",
      "[FLASH]     Percent combined: 12.72%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.036 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "642 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R23/R23_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R23_centroids.fasta --uc R23.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 642 sequences and identified 364 clusters (clustering threshold 1.00) - 13 clusters (representing 256 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R27 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R27_1.fastq /home/working/demultiplexed/R27_2.fastq R27_forward.paired.fastq.gz R27_forward.singletons.fastq.gz R27_reverse.paired.fastq.gz R27_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R27_1.fastq /home/working/demultiplexed/R27_2.fastq R27_forward.paired.fastq.gz R27_forward.singletons.fastq.gz R27_reverse.paired.fastq.gz R27_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:49:28 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R27/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 41488 Both Surviving: 9286 (22.38%) Forward Only Surviving: 3270 (7.88%) Reverse Only Surviving: 522 (1.26%) Dropped: 28410 (68.48%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: R27_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: R27_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R27_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R27_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R27_forward.paired.headclipped.fastq.gz R27_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R27 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R27_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R27_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R27.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R27.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R27.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R27.hist\n",
      "[FLASH]     ./R27.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 9286 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      9286\n",
      "[FLASH]     Combined pairs:   7926\n",
      "[FLASH]     Uncombined pairs: 1360\n",
      "[FLASH]     Percent combined: 85.35%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.209 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "7926 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R27/R27_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R27_centroids.fasta --uc R27.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 7926 sequences and identified 2335 clusters (clustering threshold 1.00) - 172 clusters (representing 5330 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R29 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R29_1.fastq /home/working/demultiplexed/R29_2.fastq R29_forward.paired.fastq.gz R29_forward.singletons.fastq.gz R29_reverse.paired.fastq.gz R29_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R29_1.fastq /home/working/demultiplexed/R29_2.fastq R29_forward.paired.fastq.gz R29_forward.singletons.fastq.gz R29_reverse.paired.fastq.gz R29_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:49:43 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R29/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 33797 Both Surviving: 8652 (25.60%) Forward Only Surviving: 2104 (6.23%) Reverse Only Surviving: 517 (1.53%) Dropped: 22524 (66.64%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: R29_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: R29_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: R29_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: R29_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R29_forward.paired.headclipped.fastq.gz R29_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R29 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R29_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R29_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R29.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R29.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R29.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R29.hist\n",
      "[FLASH]     ./R29.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 8652 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      8652\n",
      "[FLASH]     Combined pairs:   2616\n",
      "[FLASH]     Uncombined pairs: 6036\n",
      "[FLASH]     Percent combined: 30.24%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.085 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2616 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R29/R29_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R29_centroids.fasta --uc R29.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2616 sequences and identified 1564 clusters (clustering threshold 1.00) - 49 clusters (representing 816 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R30 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R30_1.fastq /home/working/demultiplexed/R30_2.fastq R30_forward.paired.fastq.gz R30_forward.singletons.fastq.gz R30_reverse.paired.fastq.gz R30_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R30_1.fastq /home/working/demultiplexed/R30_2.fastq R30_forward.paired.fastq.gz R30_forward.singletons.fastq.gz R30_reverse.paired.fastq.gz R30_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:49:52 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R30/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 16649 Both Surviving: 2332 (14.01%) Forward Only Surviving: 703 (4.22%) Reverse Only Surviving: 184 (1.11%) Dropped: 13430 (80.67%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: R30_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: R30_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: R30_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: R30_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R30_forward.paired.headclipped.fastq.gz R30_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R30 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R30_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R30_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R30.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R30.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R30.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R30.hist\n",
      "[FLASH]     ./R30.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2332 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2332\n",
      "[FLASH]     Combined pairs:   531\n",
      "[FLASH]     Uncombined pairs: 1801\n",
      "[FLASH]     Percent combined: 22.77%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.029 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "531 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R30/R30_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R30_centroids.fasta --uc R30.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 531 sequences and identified 277 clusters (clustering threshold 1.00) - 5 clusters (representing 225 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R31 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R31_1.fastq /home/working/demultiplexed/R31_2.fastq R31_forward.paired.fastq.gz R31_forward.singletons.fastq.gz R31_reverse.paired.fastq.gz R31_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R31_1.fastq /home/working/demultiplexed/R31_2.fastq R31_forward.paired.fastq.gz R31_forward.singletons.fastq.gz R31_reverse.paired.fastq.gz R31_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:49:56 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R31/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 26755 Both Surviving: 5140 (19.21%) Forward Only Surviving: 1230 (4.60%) Reverse Only Surviving: 349 (1.30%) Dropped: 20036 (74.89%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: R31_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: R31_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: R31_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: R31_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R31_forward.paired.headclipped.fastq.gz R31_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R31 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R31_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R31_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R31.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R31.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R31.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R31.hist\n",
      "[FLASH]     ./R31.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5140 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5140\n",
      "[FLASH]     Combined pairs:   592\n",
      "[FLASH]     Uncombined pairs: 4548\n",
      "[FLASH]     Percent combined: 11.52%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.044 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "592 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R31/R31_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R31_centroids.fasta --uc R31.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 592 sequences and identified 414 clusters (clustering threshold 1.00) - 10 clusters (representing 132 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R32 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R32_1.fastq /home/working/demultiplexed/R32_2.fastq R32_forward.paired.fastq.gz R32_forward.singletons.fastq.gz R32_reverse.paired.fastq.gz R32_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R32_1.fastq /home/working/demultiplexed/R32_2.fastq R32_forward.paired.fastq.gz R32_forward.singletons.fastq.gz R32_reverse.paired.fastq.gz R32_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:50:02 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R32/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 32624 Both Surviving: 1639 (5.02%) Forward Only Surviving: 1353 (4.15%) Reverse Only Surviving: 415 (1.27%) Dropped: 29217 (89.56%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: R32_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: R32_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R32_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R32_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R32_forward.paired.headclipped.fastq.gz R32_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R32 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R32_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R32_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R32.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R32.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R32.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R32.hist\n",
      "[FLASH]     ./R32.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1639 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1639\n",
      "[FLASH]     Combined pairs:   328\n",
      "[FLASH]     Uncombined pairs: 1311\n",
      "[FLASH]     Percent combined: 20.01%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.024 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "328 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R32/R32_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R32_centroids.fasta --uc R32.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 328 sequences and identified 263 clusters (clustering threshold 1.00) - 5 clusters (representing 44 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R34 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R34_1.fastq /home/working/demultiplexed/R34_2.fastq R34_forward.paired.fastq.gz R34_forward.singletons.fastq.gz R34_reverse.paired.fastq.gz R34_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R34_1.fastq /home/working/demultiplexed/R34_2.fastq R34_forward.paired.fastq.gz R34_forward.singletons.fastq.gz R34_reverse.paired.fastq.gz R34_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:50:06 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R34/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 9121 Both Surviving: 2265 (24.83%) Forward Only Surviving: 636 (6.97%) Reverse Only Surviving: 93 (1.02%) Dropped: 6127 (67.17%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: R34_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: R34_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: R34_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: R34_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R34_forward.paired.headclipped.fastq.gz R34_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R34 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R34_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R34_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R34.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R34.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R34.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R34.hist\n",
      "[FLASH]     ./R34.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2265 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2265\n",
      "[FLASH]     Combined pairs:   74\n",
      "[FLASH]     Uncombined pairs: 2191\n",
      "[FLASH]     Percent combined: 3.27%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.022 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "74 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R34/R34_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R34_centroids.fasta --uc R34.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 74 sequences and identified 66 clusters (clustering threshold 1.00) - 1 clusters (representing 7 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R36 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R36_1.fastq /home/working/demultiplexed/R36_2.fastq R36_forward.paired.fastq.gz R36_forward.singletons.fastq.gz R36_reverse.paired.fastq.gz R36_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R36_1.fastq /home/working/demultiplexed/R36_2.fastq R36_forward.paired.fastq.gz R36_forward.singletons.fastq.gz R36_reverse.paired.fastq.gz R36_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:50:08 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R36/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 12398 Both Surviving: 2888 (23.29%) Forward Only Surviving: 1053 (8.49%) Reverse Only Surviving: 127 (1.02%) Dropped: 8330 (67.19%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: R36_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: R36_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R36_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R36_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R36_forward.paired.headclipped.fastq.gz R36_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R36 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R36_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R36_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R36.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R36.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R36.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R36.hist\n",
      "[FLASH]     ./R36.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2888 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2888\n",
      "[FLASH]     Combined pairs:   2562\n",
      "[FLASH]     Uncombined pairs: 326\n",
      "[FLASH]     Percent combined: 88.71%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.074 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2562 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R36/R36_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R36_centroids.fasta --uc R36.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2562 sequences and identified 1528 clusters (clustering threshold 1.00) - 16 clusters (representing 849 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R39 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R39_1.fastq /home/working/demultiplexed/R39_2.fastq R39_forward.paired.fastq.gz R39_forward.singletons.fastq.gz R39_reverse.paired.fastq.gz R39_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R39_1.fastq /home/working/demultiplexed/R39_2.fastq R39_forward.paired.fastq.gz R39_forward.singletons.fastq.gz R39_reverse.paired.fastq.gz R39_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:50:15 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R39/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 30931 Both Surviving: 8970 (29.00%) Forward Only Surviving: 3918 (12.67%) Reverse Only Surviving: 373 (1.21%) Dropped: 17670 (57.13%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: R39_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: R39_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: R39_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: R39_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R39_forward.paired.headclipped.fastq.gz R39_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R39 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R39_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R39_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R39.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R39.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R39.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R39.hist\n",
      "[FLASH]     ./R39.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 8970 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      8970\n",
      "[FLASH]     Combined pairs:   3486\n",
      "[FLASH]     Uncombined pairs: 5484\n",
      "[FLASH]     Percent combined: 38.86%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.116 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "3486 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R39/R39_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R39_centroids.fasta --uc R39.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 3486 sequences and identified 1707 clusters (clustering threshold 1.00) - 62 clusters (representing 1535 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R4 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R4_1.fastq /home/working/demultiplexed/R4_2.fastq R4_forward.paired.fastq.gz R4_forward.singletons.fastq.gz R4_reverse.paired.fastq.gz R4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R4_1.fastq /home/working/demultiplexed/R4_2.fastq R4_forward.paired.fastq.gz R4_forward.singletons.fastq.gz R4_reverse.paired.fastq.gz R4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:50:26 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R4/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 12106 Both Surviving: 3362 (27.77%) Forward Only Surviving: 786 (6.49%) Reverse Only Surviving: 225 (1.86%) Dropped: 7733 (63.88%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: R4_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: R4_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: R4_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: R4_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R4_forward.paired.headclipped.fastq.gz R4_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R4 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R4_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R4_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R4.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R4.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R4.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R4.hist\n",
      "[FLASH]     ./R4.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3362 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3362\n",
      "[FLASH]     Combined pairs:   1701\n",
      "[FLASH]     Uncombined pairs: 1661\n",
      "[FLASH]     Percent combined: 50.59%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.071 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1701 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R4/R4_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R4_centroids.fasta --uc R4.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1701 sequences and identified 1031 clusters (clustering threshold 1.00) - 31 clusters (representing 573 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R40 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R40_1.fastq /home/working/demultiplexed/R40_2.fastq R40_forward.paired.fastq.gz R40_forward.singletons.fastq.gz R40_reverse.paired.fastq.gz R40_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R40_1.fastq /home/working/demultiplexed/R40_2.fastq R40_forward.paired.fastq.gz R40_forward.singletons.fastq.gz R40_reverse.paired.fastq.gz R40_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:50:31 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R40/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 10491 Both Surviving: 1666 (15.88%) Forward Only Surviving: 412 (3.93%) Reverse Only Surviving: 153 (1.46%) Dropped: 8260 (78.73%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: R40_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: R40_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R40_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R40_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R40_forward.paired.headclipped.fastq.gz R40_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R40 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R40_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R40_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R40.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R40.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R40.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R40.hist\n",
      "[FLASH]     ./R40.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1666 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1666\n",
      "[FLASH]     Combined pairs:   349\n",
      "[FLASH]     Uncombined pairs: 1317\n",
      "[FLASH]     Percent combined: 20.95%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.027 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "349 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R40/R40_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R40_centroids.fasta --uc R40.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 349 sequences and identified 246 clusters (clustering threshold 1.00) - 3 clusters (representing 91 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R43 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R43_1.fastq /home/working/demultiplexed/R43_2.fastq R43_forward.paired.fastq.gz R43_forward.singletons.fastq.gz R43_reverse.paired.fastq.gz R43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R43_1.fastq /home/working/demultiplexed/R43_2.fastq R43_forward.paired.fastq.gz R43_forward.singletons.fastq.gz R43_reverse.paired.fastq.gz R43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:50:39 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R43/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 102059 Both Surviving: 26955 (26.41%) Forward Only Surviving: 7384 (7.24%) Reverse Only Surviving: 1624 (1.59%) Dropped: 66096 (64.76%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: R43_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: R43_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: R43_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: R43_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R43_forward.paired.headclipped.fastq.gz R43_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R43 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R43_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R43_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R43.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R43.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R43.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R43.hist\n",
      "[FLASH]     ./R43.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 25000 read pairs\n",
      "[FLASH] Processed 26955 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      26955\n",
      "[FLASH]     Combined pairs:   5230\n",
      "[FLASH]     Uncombined pairs: 21725\n",
      "[FLASH]     Percent combined: 19.40%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.253 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "5230 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R43/R43_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R43_centroids.fasta --uc R43.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 5230 sequences and identified 2691 clusters (clustering threshold 1.00) - 84 clusters (representing 2203 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R44 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R44_1.fastq /home/working/demultiplexed/R44_2.fastq R44_forward.paired.fastq.gz R44_forward.singletons.fastq.gz R44_reverse.paired.fastq.gz R44_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R44_1.fastq /home/working/demultiplexed/R44_2.fastq R44_forward.paired.fastq.gz R44_forward.singletons.fastq.gz R44_reverse.paired.fastq.gz R44_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:51:02 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R44/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 59807 Both Surviving: 3915 (6.55%) Forward Only Surviving: 1319 (2.21%) Reverse Only Surviving: 818 (1.37%) Dropped: 53755 (89.88%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: R44_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: R44_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: R44_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: R44_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R44_forward.paired.headclipped.fastq.gz R44_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R44 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R44_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R44_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R44.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R44.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R44.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R44.hist\n",
      "[FLASH]     ./R44.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3915 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3915\n",
      "[FLASH]     Combined pairs:   1308\n",
      "[FLASH]     Uncombined pairs: 2607\n",
      "[FLASH]     Percent combined: 33.41%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.040 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1308 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R44/R44_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R44_centroids.fasta --uc R44.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1308 sequences and identified 566 clusters (clustering threshold 1.00) - 13 clusters (representing 617 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R45 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R45_1.fastq /home/working/demultiplexed/R45_2.fastq R45_forward.paired.fastq.gz R45_forward.singletons.fastq.gz R45_reverse.paired.fastq.gz R45_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R45_1.fastq /home/working/demultiplexed/R45_2.fastq R45_forward.paired.fastq.gz R45_forward.singletons.fastq.gz R45_reverse.paired.fastq.gz R45_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:51:11 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R45/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 52759 Both Surviving: 17663 (33.48%) Forward Only Surviving: 10402 (19.72%) Reverse Only Surviving: 418 (0.79%) Dropped: 24276 (46.01%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: R45_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: R45_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: R45_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: R45_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R45_forward.paired.headclipped.fastq.gz R45_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R45 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R45_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R45_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R45.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R45.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R45.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R45.hist\n",
      "[FLASH]     ./R45.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 17663 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      17663\n",
      "[FLASH]     Combined pairs:   14829\n",
      "[FLASH]     Uncombined pairs: 2834\n",
      "[FLASH]     Percent combined: 83.96%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.418 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "14829 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R45/R45_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R45_centroids.fasta --uc R45.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 14829 sequences and identified 6262 clusters (clustering threshold 1.00) - 316 clusters (representing 8149 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R48 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R48_1.fastq /home/working/demultiplexed/R48_2.fastq R48_forward.paired.fastq.gz R48_forward.singletons.fastq.gz R48_reverse.paired.fastq.gz R48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R48_1.fastq /home/working/demultiplexed/R48_2.fastq R48_forward.paired.fastq.gz R48_forward.singletons.fastq.gz R48_reverse.paired.fastq.gz R48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:51:41 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R48/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 20979 Both Surviving: 5392 (25.70%) Forward Only Surviving: 1126 (5.37%) Reverse Only Surviving: 316 (1.51%) Dropped: 14145 (67.42%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: R48_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: R48_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: R48_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: R48_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R48_forward.paired.headclipped.fastq.gz R48_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R48 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R48_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R48_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R48.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R48.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R48.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R48.hist\n",
      "[FLASH]     ./R48.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5392 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5392\n",
      "[FLASH]     Combined pairs:   2042\n",
      "[FLASH]     Uncombined pairs: 3350\n",
      "[FLASH]     Percent combined: 37.87%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.066 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2042 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R48/R48_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R48_centroids.fasta --uc R48.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2042 sequences and identified 1303 clusters (clustering threshold 1.00) - 26 clusters (representing 655 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R49 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R49_1.fastq /home/working/demultiplexed/R49_2.fastq R49_forward.paired.fastq.gz R49_forward.singletons.fastq.gz R49_reverse.paired.fastq.gz R49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R49_1.fastq /home/working/demultiplexed/R49_2.fastq R49_forward.paired.fastq.gz R49_forward.singletons.fastq.gz R49_reverse.paired.fastq.gz R49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:51:47 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R49/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 9114 Both Surviving: 1505 (16.51%) Forward Only Surviving: 341 (3.74%) Reverse Only Surviving: 295 (3.24%) Dropped: 6973 (76.51%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: R49_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R49_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R49_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R49_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R49_forward.paired.headclipped.fastq.gz R49_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R49 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R49_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R49_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R49.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R49.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R49.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R49.hist\n",
      "[FLASH]     ./R49.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1505 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1505\n",
      "[FLASH]     Combined pairs:   358\n",
      "[FLASH]     Uncombined pairs: 1147\n",
      "[FLASH]     Percent combined: 23.79%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.015 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "358 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R49/R49_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R49_centroids.fasta --uc R49.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 358 sequences and identified 195 clusters (clustering threshold 1.00) - 6 clusters (representing 156 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R59 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R59_1.fastq /home/working/demultiplexed/R59_2.fastq R59_forward.paired.fastq.gz R59_forward.singletons.fastq.gz R59_reverse.paired.fastq.gz R59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R59_1.fastq /home/working/demultiplexed/R59_2.fastq R59_forward.paired.fastq.gz R59_forward.singletons.fastq.gz R59_reverse.paired.fastq.gz R59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:51:50 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R59/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 35926 Both Surviving: 3298 (9.18%) Forward Only Surviving: 970 (2.70%) Reverse Only Surviving: 529 (1.47%) Dropped: 31129 (86.65%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: R59_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: R59_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: R59_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: R59_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R59_forward.paired.headclipped.fastq.gz R59_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R59 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R59_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R59_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R59.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R59.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R59.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R59.hist\n",
      "[FLASH]     ./R59.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3298 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3298\n",
      "[FLASH]     Combined pairs:   129\n",
      "[FLASH]     Uncombined pairs: 3169\n",
      "[FLASH]     Percent combined: 3.91%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.024 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "129 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R59/R59_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R59_centroids.fasta --uc R59.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 129 sequences and identified 87 clusters (clustering threshold 1.00) - 3 clusters (representing 34 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R60 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R60_1.fastq /home/working/demultiplexed/R60_2.fastq R60_forward.paired.fastq.gz R60_forward.singletons.fastq.gz R60_reverse.paired.fastq.gz R60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R60_1.fastq /home/working/demultiplexed/R60_2.fastq R60_forward.paired.fastq.gz R60_forward.singletons.fastq.gz R60_reverse.paired.fastq.gz R60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:51:55 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R60/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 39486 Both Surviving: 5392 (13.66%) Forward Only Surviving: 1158 (2.93%) Reverse Only Surviving: 443 (1.12%) Dropped: 32493 (82.29%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: R60_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: R60_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R60_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R60_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R60_forward.paired.headclipped.fastq.gz R60_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R60 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R60_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R60_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R60.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R60.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R60.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R60.hist\n",
      "[FLASH]     ./R60.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5392 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5392\n",
      "[FLASH]     Combined pairs:   47\n",
      "[FLASH]     Uncombined pairs: 5345\n",
      "[FLASH]     Percent combined: 0.87%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.049 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "47 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R60/R60_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R60_centroids.fasta --uc R60.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 47 sequences and identified 42 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R67 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R67_1.fastq /home/working/demultiplexed/R67_2.fastq R67_forward.paired.fastq.gz R67_forward.singletons.fastq.gz R67_reverse.paired.fastq.gz R67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R67_1.fastq /home/working/demultiplexed/R67_2.fastq R67_forward.paired.fastq.gz R67_forward.singletons.fastq.gz R67_reverse.paired.fastq.gz R67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:52:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R67/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 27457 Both Surviving: 6301 (22.95%) Forward Only Surviving: 4040 (14.71%) Reverse Only Surviving: 307 (1.12%) Dropped: 16809 (61.22%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: R67_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: R67_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: R67_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: R67_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R67_forward.paired.headclipped.fastq.gz R67_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R67 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R67_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R67_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R67.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R67.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R67.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R67.hist\n",
      "[FLASH]     ./R67.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 6301 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      6301\n",
      "[FLASH]     Combined pairs:   4584\n",
      "[FLASH]     Uncombined pairs: 1717\n",
      "[FLASH]     Percent combined: 72.75%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.127 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "4584 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R67/R67_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R67_centroids.fasta --uc R67.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 4584 sequences and identified 2358 clusters (clustering threshold 1.00) - 60 clusters (representing 1875 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R68 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R68_1.fastq /home/working/demultiplexed/R68_2.fastq R68_forward.paired.fastq.gz R68_forward.singletons.fastq.gz R68_reverse.paired.fastq.gz R68_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R68_1.fastq /home/working/demultiplexed/R68_2.fastq R68_forward.paired.fastq.gz R68_forward.singletons.fastq.gz R68_reverse.paired.fastq.gz R68_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:52:11 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R68/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 12701 Both Surviving: 2577 (20.29%) Forward Only Surviving: 1071 (8.43%) Reverse Only Surviving: 201 (1.58%) Dropped: 8852 (69.70%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: R68_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: R68_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: R68_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: R68_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R68_forward.paired.headclipped.fastq.gz R68_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R68 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R68_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R68_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R68.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R68.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R68.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R68.hist\n",
      "[FLASH]     ./R68.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2577 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2577\n",
      "[FLASH]     Combined pairs:   2078\n",
      "[FLASH]     Uncombined pairs: 499\n",
      "[FLASH]     Percent combined: 80.64%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.057 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2078 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R68/R68_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R68_centroids.fasta --uc R68.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2078 sequences and identified 912 clusters (clustering threshold 1.00) - 34 clusters (representing 1014 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R71 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R71_1.fastq /home/working/demultiplexed/R71_2.fastq R71_forward.paired.fastq.gz R71_forward.singletons.fastq.gz R71_reverse.paired.fastq.gz R71_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R71_1.fastq /home/working/demultiplexed/R71_2.fastq R71_forward.paired.fastq.gz R71_forward.singletons.fastq.gz R71_reverse.paired.fastq.gz R71_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:52:17 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R71/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 35268 Both Surviving: 4575 (12.97%) Forward Only Surviving: 1404 (3.98%) Reverse Only Surviving: 472 (1.34%) Dropped: 28817 (81.71%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: R71_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: R71_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: R71_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: R71_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R71_forward.paired.headclipped.fastq.gz R71_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R71 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R71_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R71_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R71.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R71.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R71.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R71.hist\n",
      "[FLASH]     ./R71.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 4575 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      4575\n",
      "[FLASH]     Combined pairs:   262\n",
      "[FLASH]     Uncombined pairs: 4313\n",
      "[FLASH]     Percent combined: 5.73%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.036 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "262 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R71/R71_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R71_centroids.fasta --uc R71.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 262 sequences and identified 189 clusters (clustering threshold 1.00) - 7 clusters (representing 55 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R74 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R74_1.fastq /home/working/demultiplexed/R74_2.fastq R74_forward.paired.fastq.gz R74_forward.singletons.fastq.gz R74_reverse.paired.fastq.gz R74_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R74_1.fastq /home/working/demultiplexed/R74_2.fastq R74_forward.paired.fastq.gz R74_forward.singletons.fastq.gz R74_reverse.paired.fastq.gz R74_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:52:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R74/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 37447 Both Surviving: 185 (0.49%) Forward Only Surviving: 449 (1.20%) Reverse Only Surviving: 251 (0.67%) Dropped: 36562 (97.64%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: R74_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: R74_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R74_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R74_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R74_forward.paired.headclipped.fastq.gz R74_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R74 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R74_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R74_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R74.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R74.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R74.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R74.hist\n",
      "[FLASH]     ./R74.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 185 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      185\n",
      "[FLASH]     Combined pairs:   128\n",
      "[FLASH]     Uncombined pairs: 57\n",
      "[FLASH]     Percent combined: 69.19%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.004 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "128 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R74/R74_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R74_centroids.fasta --uc R74.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 128 sequences and identified 73 clusters (clustering threshold 1.00) - 3 clusters (representing 52 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R8 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R8_1.fastq /home/working/demultiplexed/R8_2.fastq R8_forward.paired.fastq.gz R8_forward.singletons.fastq.gz R8_reverse.paired.fastq.gz R8_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R8_1.fastq /home/working/demultiplexed/R8_2.fastq R8_forward.paired.fastq.gz R8_forward.singletons.fastq.gz R8_reverse.paired.fastq.gz R8_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:52:26 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R8/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 29705 Both Surviving: 2428 (8.17%) Forward Only Surviving: 762 (2.57%) Reverse Only Surviving: 299 (1.01%) Dropped: 26216 (88.25%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: R8_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: R8_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: R8_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: R8_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R8_forward.paired.headclipped.fastq.gz R8_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R8 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R8_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R8_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R8.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R8.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R8.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R8.hist\n",
      "[FLASH]     ./R8.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2428 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2428\n",
      "[FLASH]     Combined pairs:   2143\n",
      "[FLASH]     Uncombined pairs: 285\n",
      "[FLASH]     Percent combined: 88.26%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.062 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2143 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R8/R8_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R8_centroids.fasta --uc R8.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2143 sequences and identified 1180 clusters (clustering threshold 1.00) - 27 clusters (representing 850 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R85 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R85_1.fastq /home/working/demultiplexed/R85_2.fastq R85_forward.paired.fastq.gz R85_forward.singletons.fastq.gz R85_reverse.paired.fastq.gz R85_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R85_1.fastq /home/working/demultiplexed/R85_2.fastq R85_forward.paired.fastq.gz R85_forward.singletons.fastq.gz R85_reverse.paired.fastq.gz R85_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:52:32 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R85/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 18834 Both Surviving: 3948 (20.96%) Forward Only Surviving: 1269 (6.74%) Reverse Only Surviving: 242 (1.28%) Dropped: 13375 (71.02%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: R85_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: R85_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: R85_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: R85_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R85_forward.paired.headclipped.fastq.gz R85_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R85 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R85_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R85_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R85.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R85.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R85.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R85.hist\n",
      "[FLASH]     ./R85.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3948 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3948\n",
      "[FLASH]     Combined pairs:   1219\n",
      "[FLASH]     Uncombined pairs: 2729\n",
      "[FLASH]     Percent combined: 30.88%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.043 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1219 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R85/R85_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R85_centroids.fasta --uc R85.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1219 sequences and identified 733 clusters (clustering threshold 1.00) - 16 clusters (representing 439 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R9 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R9_1.fastq /home/working/demultiplexed/R9_2.fastq R9_forward.paired.fastq.gz R9_forward.singletons.fastq.gz R9_reverse.paired.fastq.gz R9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R9_1.fastq /home/working/demultiplexed/R9_2.fastq R9_forward.paired.fastq.gz R9_forward.singletons.fastq.gz R9_reverse.paired.fastq.gz R9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:52:37 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R9/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 22136 Both Surviving: 5623 (25.40%) Forward Only Surviving: 1911 (8.63%) Reverse Only Surviving: 301 (1.36%) Dropped: 14301 (64.61%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: R9_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: R9_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: R9_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: R9_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R9_forward.paired.headclipped.fastq.gz R9_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R9 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R9_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R9_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R9.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R9.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R9.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R9.hist\n",
      "[FLASH]     ./R9.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5623 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5623\n",
      "[FLASH]     Combined pairs:   251\n",
      "[FLASH]     Uncombined pairs: 5372\n",
      "[FLASH]     Percent combined: 4.46%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.041 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "251 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R9/R9_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R9_centroids.fasta --uc R9.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 251 sequences and identified 219 clusters (clustering threshold 1.00) - 2 clusters (representing 16 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R90 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R90_1.fastq /home/working/demultiplexed/R90_2.fastq R90_forward.paired.fastq.gz R90_forward.singletons.fastq.gz R90_reverse.paired.fastq.gz R90_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R90_1.fastq /home/working/demultiplexed/R90_2.fastq R90_forward.paired.fastq.gz R90_forward.singletons.fastq.gz R90_reverse.paired.fastq.gz R90_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:52:42 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R90/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 15190 Both Surviving: 328 (2.16%) Forward Only Surviving: 380 (2.50%) Reverse Only Surviving: 160 (1.05%) Dropped: 14322 (94.29%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: R90_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: R90_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: R90_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: R90_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R90_forward.paired.headclipped.fastq.gz R90_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R90 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R90_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R90_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R90.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R90.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R90.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R90.hist\n",
      "[FLASH]     ./R90.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 328 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      328\n",
      "[FLASH]     Combined pairs:   231\n",
      "[FLASH]     Uncombined pairs: 97\n",
      "[FLASH]     Percent combined: 70.43%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.007 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "231 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R90/R90_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R90_centroids.fasta --uc R90.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 231 sequences and identified 178 clusters (clustering threshold 1.00) - 3 clusters (representing 45 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R95 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R95_1.fastq /home/working/demultiplexed/R95_2.fastq R95_forward.paired.fastq.gz R95_forward.singletons.fastq.gz R95_reverse.paired.fastq.gz R95_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R95_1.fastq /home/working/demultiplexed/R95_2.fastq R95_forward.paired.fastq.gz R95_forward.singletons.fastq.gz R95_reverse.paired.fastq.gz R95_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:52:44 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R95/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 15768 Both Surviving: 300 (1.90%) Forward Only Surviving: 279 (1.77%) Reverse Only Surviving: 149 (0.94%) Dropped: 15040 (95.38%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: R95_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: R95_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: R95_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: R95_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R95_forward.paired.headclipped.fastq.gz R95_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R95 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R95_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R95_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R95.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R95.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R95.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R95.hist\n",
      "[FLASH]     ./R95.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 300 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      300\n",
      "[FLASH]     Combined pairs:   60\n",
      "[FLASH]     Uncombined pairs: 240\n",
      "[FLASH]     Percent combined: 20.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.005 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "60 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R95/R95_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R95_centroids.fasta --uc R95.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 60 sequences and identified 53 clusters (clustering threshold 1.00) - 1 clusters (representing 8 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R98 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R98_1.fastq /home/working/demultiplexed/R98_2.fastq R98_forward.paired.fastq.gz R98_forward.singletons.fastq.gz R98_reverse.paired.fastq.gz R98_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R98_1.fastq /home/working/demultiplexed/R98_2.fastq R98_forward.paired.fastq.gz R98_forward.singletons.fastq.gz R98_reverse.paired.fastq.gz R98_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:52:48 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R98/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 47670 Both Surviving: 8088 (16.97%) Forward Only Surviving: 2258 (4.74%) Reverse Only Surviving: 560 (1.17%) Dropped: 36764 (77.12%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: R98_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: R98_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R98_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: R98_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R98_forward.paired.headclipped.fastq.gz R98_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R98 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R98_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R98_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R98.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R98.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R98.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R98.hist\n",
      "[FLASH]     ./R98.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 8088 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      8088\n",
      "[FLASH]     Combined pairs:   3682\n",
      "[FLASH]     Uncombined pairs: 4406\n",
      "[FLASH]     Percent combined: 45.52%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.116 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "3682 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R98/R98_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R98_centroids.fasta --uc R98.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 3682 sequences and identified 1948 clusters (clustering threshold 1.00) - 49 clusters (representing 1523 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: R99 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R99_1.fastq /home/working/demultiplexed/R99_2.fastq R99_forward.paired.fastq.gz R99_forward.singletons.fastq.gz R99_reverse.paired.fastq.gz R99_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/R99_1.fastq /home/working/demultiplexed/R99_2.fastq R99_forward.paired.fastq.gz R99_forward.singletons.fastq.gz R99_reverse.paired.fastq.gz R99_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:53:02 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/R99/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 82372 Both Surviving: 1529 (1.86%) Forward Only Surviving: 2684 (3.26%) Reverse Only Surviving: 882 (1.07%) Dropped: 77277 (93.81%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: R99_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: R99_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: R99_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: R99_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash R99_forward.paired.headclipped.fastq.gz R99_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o R99 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     R99_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     R99_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./R99.extendedFrags.fastq.gz\n",
      "[FLASH]     ./R99.notCombined_1.fastq.gz\n",
      "[FLASH]     ./R99.notCombined_2.fastq.gz\n",
      "[FLASH]     ./R99.hist\n",
      "[FLASH]     ./R99.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1529 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1529\n",
      "[FLASH]     Combined pairs:   736\n",
      "[FLASH]     Uncombined pairs: 793\n",
      "[FLASH]     Percent combined: 48.14%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.020 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "736 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/R99/R99_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids R99_centroids.fasta --uc R99.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 736 sequences and identified 559 clusters (clustering threshold 1.00) - 10 clusters (representing 117 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S100 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S100_1.fastq /home/working/demultiplexed/S100_2.fastq S100_forward.paired.fastq.gz S100_forward.singletons.fastq.gz S100_reverse.paired.fastq.gz S100_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S100_1.fastq /home/working/demultiplexed/S100_2.fastq S100_forward.paired.fastq.gz S100_forward.singletons.fastq.gz S100_reverse.paired.fastq.gz S100_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:53:08 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S100/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 15016 Both Surviving: 1579 (10.52%) Forward Only Surviving: 676 (4.50%) Reverse Only Surviving: 115 (0.77%) Dropped: 12646 (84.22%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: S100_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: S100_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: S100_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: S100_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S100_forward.paired.headclipped.fastq.gz S100_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S100 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S100_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S100_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S100.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S100.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S100.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S100.hist\n",
      "[FLASH]     ./S100.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1579 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1579\n",
      "[FLASH]     Combined pairs:   1345\n",
      "[FLASH]     Uncombined pairs: 234\n",
      "[FLASH]     Percent combined: 85.18%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.037 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1345 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S100/S100_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S100_centroids.fasta --uc S100.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1345 sequences and identified 747 clusters (clustering threshold 1.00) - 22 clusters (representing 525 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S101 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S101_1.fastq /home/working/demultiplexed/S101_2.fastq S101_forward.paired.fastq.gz S101_forward.singletons.fastq.gz S101_reverse.paired.fastq.gz S101_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S101_1.fastq /home/working/demultiplexed/S101_2.fastq S101_forward.paired.fastq.gz S101_forward.singletons.fastq.gz S101_reverse.paired.fastq.gz S101_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:53:12 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S101/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 9613 Both Surviving: 328 (3.41%) Forward Only Surviving: 199 (2.07%) Reverse Only Surviving: 243 (2.53%) Dropped: 8843 (91.99%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: S101_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: S101_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S101_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S101_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S101_forward.paired.headclipped.fastq.gz S101_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S101 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S101_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S101_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S101.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S101.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S101.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S101.hist\n",
      "[FLASH]     ./S101.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 328 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      328\n",
      "[FLASH]     Combined pairs:   59\n",
      "[FLASH]     Uncombined pairs: 269\n",
      "[FLASH]     Percent combined: 17.99%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.005 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "59 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S101/S101_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S101_centroids.fasta --uc S101.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 59 sequences and identified 50 clusters (clustering threshold 1.00) - 1 clusters (representing 10 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S102 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S102_1.fastq /home/working/demultiplexed/S102_2.fastq S102_forward.paired.fastq.gz S102_forward.singletons.fastq.gz S102_reverse.paired.fastq.gz S102_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S102_1.fastq /home/working/demultiplexed/S102_2.fastq S102_forward.paired.fastq.gz S102_forward.singletons.fastq.gz S102_reverse.paired.fastq.gz S102_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:53:14 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S102/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 17129 Both Surviving: 3440 (20.08%) Forward Only Surviving: 1158 (6.76%) Reverse Only Surviving: 149 (0.87%) Dropped: 12382 (72.29%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: S102_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: S102_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: S102_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: S102_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S102_forward.paired.headclipped.fastq.gz S102_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S102 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S102_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S102_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S102.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S102.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S102.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S102.hist\n",
      "[FLASH]     ./S102.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3440 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3440\n",
      "[FLASH]     Combined pairs:   1551\n",
      "[FLASH]     Uncombined pairs: 1889\n",
      "[FLASH]     Percent combined: 45.09%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.051 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1551 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S102/S102_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S102_centroids.fasta --uc S102.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1551 sequences and identified 816 clusters (clustering threshold 1.00) - 14 clusters (representing 650 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S11 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S11_1.fastq /home/working/demultiplexed/S11_2.fastq S11_forward.paired.fastq.gz S11_forward.singletons.fastq.gz S11_reverse.paired.fastq.gz S11_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S11_1.fastq /home/working/demultiplexed/S11_2.fastq S11_forward.paired.fastq.gz S11_forward.singletons.fastq.gz S11_reverse.paired.fastq.gz S11_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:53:20 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S11/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 41543 Both Surviving: 671 (1.62%) Forward Only Surviving: 1052 (2.53%) Reverse Only Surviving: 567 (1.36%) Dropped: 39253 (94.49%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: S11_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: S11_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S11_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S11_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S11_forward.paired.headclipped.fastq.gz S11_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S11 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S11_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S11_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S11.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S11.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S11.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S11.hist\n",
      "[FLASH]     ./S11.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 671 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      671\n",
      "[FLASH]     Combined pairs:   345\n",
      "[FLASH]     Uncombined pairs: 326\n",
      "[FLASH]     Percent combined: 51.42%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.010 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "345 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S11/S11_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S11_centroids.fasta --uc S11.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 345 sequences and identified 185 clusters (clustering threshold 1.00) - 5 clusters (representing 141 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S13 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S13_1.fastq /home/working/demultiplexed/S13_2.fastq S13_forward.paired.fastq.gz S13_forward.singletons.fastq.gz S13_reverse.paired.fastq.gz S13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S13_1.fastq /home/working/demultiplexed/S13_2.fastq S13_forward.paired.fastq.gz S13_forward.singletons.fastq.gz S13_reverse.paired.fastq.gz S13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:53:24 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S13/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 21977 Both Surviving: 3585 (16.31%) Forward Only Surviving: 869 (3.95%) Reverse Only Surviving: 356 (1.62%) Dropped: 17167 (78.11%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: S13_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S13_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: S13_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: S13_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S13_forward.paired.headclipped.fastq.gz S13_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S13 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S13_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S13_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S13.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S13.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S13.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S13.hist\n",
      "[FLASH]     ./S13.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3585 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3585\n",
      "[FLASH]     Combined pairs:   904\n",
      "[FLASH]     Uncombined pairs: 2681\n",
      "[FLASH]     Percent combined: 25.22%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.033 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "904 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S13/S13_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S13_centroids.fasta --uc S13.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 904 sequences and identified 511 clusters (clustering threshold 1.00) - 9 clusters (representing 347 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S18 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S18_1.fastq /home/working/demultiplexed/S18_2.fastq S18_forward.paired.fastq.gz S18_forward.singletons.fastq.gz S18_reverse.paired.fastq.gz S18_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S18_1.fastq /home/working/demultiplexed/S18_2.fastq S18_forward.paired.fastq.gz S18_forward.singletons.fastq.gz S18_reverse.paired.fastq.gz S18_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:53:30 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S18/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 45130 Both Surviving: 6734 (14.92%) Forward Only Surviving: 1728 (3.83%) Reverse Only Surviving: 829 (1.84%) Dropped: 35839 (79.41%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: S18_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S18_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: S18_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: S18_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S18_forward.paired.headclipped.fastq.gz S18_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S18 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S18_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S18_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S18.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S18.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S18.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S18.hist\n",
      "[FLASH]     ./S18.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 6734 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      6734\n",
      "[FLASH]     Combined pairs:   71\n",
      "[FLASH]     Uncombined pairs: 6663\n",
      "[FLASH]     Percent combined: 1.05%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.049 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "71 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S18/S18_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S18_centroids.fasta --uc S18.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 71 sequences and identified 61 clusters (clustering threshold 1.00) - 1 clusters (representing 9 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S19 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S19_1.fastq /home/working/demultiplexed/S19_2.fastq S19_forward.paired.fastq.gz S19_forward.singletons.fastq.gz S19_reverse.paired.fastq.gz S19_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S19_1.fastq /home/working/demultiplexed/S19_2.fastq S19_forward.paired.fastq.gz S19_forward.singletons.fastq.gz S19_reverse.paired.fastq.gz S19_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:53:35 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S19/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 14524 Both Surviving: 3946 (27.17%) Forward Only Surviving: 1547 (10.65%) Reverse Only Surviving: 301 (2.07%) Dropped: 8730 (60.11%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: S19_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: S19_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S19_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S19_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S19_forward.paired.headclipped.fastq.gz S19_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S19 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S19_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S19_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S19.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S19.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S19.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S19.hist\n",
      "[FLASH]     ./S19.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3946 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3946\n",
      "[FLASH]     Combined pairs:   2350\n",
      "[FLASH]     Uncombined pairs: 1596\n",
      "[FLASH]     Percent combined: 59.55%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.070 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2350 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S19/S19_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S19_centroids.fasta --uc S19.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2350 sequences and identified 1203 clusters (clustering threshold 1.00) - 28 clusters (representing 980 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S24 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S24_1.fastq /home/working/demultiplexed/S24_2.fastq S24_forward.paired.fastq.gz S24_forward.singletons.fastq.gz S24_reverse.paired.fastq.gz S24_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S24_1.fastq /home/working/demultiplexed/S24_2.fastq S24_forward.paired.fastq.gz S24_forward.singletons.fastq.gz S24_reverse.paired.fastq.gz S24_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:53:41 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S24/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 9162 Both Surviving: 875 (9.55%) Forward Only Surviving: 690 (7.53%) Reverse Only Surviving: 70 (0.76%) Dropped: 7527 (82.15%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: S24_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: S24_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S24_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S24_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S24_forward.paired.headclipped.fastq.gz S24_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S24 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S24_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S24_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S24.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S24.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S24.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S24.hist\n",
      "[FLASH]     ./S24.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 875 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      875\n",
      "[FLASH]     Combined pairs:   288\n",
      "[FLASH]     Uncombined pairs: 587\n",
      "[FLASH]     Percent combined: 32.91%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.010 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "288 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S24/S24_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S24_centroids.fasta --uc S24.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 288 sequences and identified 240 clusters (clustering threshold 1.00) - 2 clusters (representing 24 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S30 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S30_1.fastq /home/working/demultiplexed/S30_2.fastq S30_forward.paired.fastq.gz S30_forward.singletons.fastq.gz S30_reverse.paired.fastq.gz S30_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S30_1.fastq /home/working/demultiplexed/S30_2.fastq S30_forward.paired.fastq.gz S30_forward.singletons.fastq.gz S30_reverse.paired.fastq.gz S30_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:53:43 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S30/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 11465 Both Surviving: 2208 (19.26%) Forward Only Surviving: 583 (5.09%) Reverse Only Surviving: 164 (1.43%) Dropped: 8510 (74.23%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: S30_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S30_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S30_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S30_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S30_forward.paired.headclipped.fastq.gz S30_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S30 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S30_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S30_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S30.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S30.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S30.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S30.hist\n",
      "[FLASH]     ./S30.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2208 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2208\n",
      "[FLASH]     Combined pairs:   32\n",
      "[FLASH]     Uncombined pairs: 2176\n",
      "[FLASH]     Percent combined: 1.45%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.019 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "32 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S30/S30_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S30_centroids.fasta --uc S30.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 32 sequences and identified 29 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S31 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S31_1.fastq /home/working/demultiplexed/S31_2.fastq S31_forward.paired.fastq.gz S31_forward.singletons.fastq.gz S31_reverse.paired.fastq.gz S31_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S31_1.fastq /home/working/demultiplexed/S31_2.fastq S31_forward.paired.fastq.gz S31_forward.singletons.fastq.gz S31_reverse.paired.fastq.gz S31_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:53:45 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S31/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 8667 Both Surviving: 111 (1.28%) Forward Only Surviving: 129 (1.49%) Reverse Only Surviving: 100 (1.15%) Dropped: 8327 (96.08%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: S31_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: S31_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: S31_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: S31_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S31_forward.paired.headclipped.fastq.gz S31_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S31 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S31_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S31_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S31.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S31.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S31.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S31.hist\n",
      "[FLASH]     ./S31.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 111 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      111\n",
      "[FLASH]     Combined pairs:   34\n",
      "[FLASH]     Uncombined pairs: 77\n",
      "[FLASH]     Percent combined: 30.63%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.002 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "34 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S31/S31_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S31_centroids.fasta --uc S31.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 34 sequences and identified 33 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S33 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S33_1.fastq /home/working/demultiplexed/S33_2.fastq S33_forward.paired.fastq.gz S33_forward.singletons.fastq.gz S33_reverse.paired.fastq.gz S33_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S33_1.fastq /home/working/demultiplexed/S33_2.fastq S33_forward.paired.fastq.gz S33_forward.singletons.fastq.gz S33_reverse.paired.fastq.gz S33_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:53:47 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S33/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 22355 Both Surviving: 761 (3.40%) Forward Only Surviving: 773 (3.46%) Reverse Only Surviving: 235 (1.05%) Dropped: 20586 (92.09%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: S33_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: S33_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: S33_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: S33_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S33_forward.paired.headclipped.fastq.gz S33_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S33 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S33_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S33_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S33.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S33.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S33.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S33.hist\n",
      "[FLASH]     ./S33.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 761 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      761\n",
      "[FLASH]     Combined pairs:   432\n",
      "[FLASH]     Uncombined pairs: 329\n",
      "[FLASH]     Percent combined: 56.77%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.013 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "432 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S33/S33_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S33_centroids.fasta --uc S33.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 432 sequences and identified 320 clusters (clustering threshold 1.00) - 3 clusters (representing 89 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S35 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S35_1.fastq /home/working/demultiplexed/S35_2.fastq S35_forward.paired.fastq.gz S35_forward.singletons.fastq.gz S35_reverse.paired.fastq.gz S35_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S35_1.fastq /home/working/demultiplexed/S35_2.fastq S35_forward.paired.fastq.gz S35_forward.singletons.fastq.gz S35_reverse.paired.fastq.gz S35_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:53:53 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S35/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 77091 Both Surviving: 5196 (6.74%) Forward Only Surviving: 3552 (4.61%) Reverse Only Surviving: 988 (1.28%) Dropped: 67355 (87.37%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: S35_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: S35_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: S35_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: S35_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S35_forward.paired.headclipped.fastq.gz S35_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S35 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S35_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S35_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S35.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S35.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S35.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S35.hist\n",
      "[FLASH]     ./S35.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5196 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5196\n",
      "[FLASH]     Combined pairs:   2485\n",
      "[FLASH]     Uncombined pairs: 2711\n",
      "[FLASH]     Percent combined: 47.83%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.075 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2485 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S35/S35_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S35_centroids.fasta --uc S35.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2485 sequences and identified 1438 clusters (clustering threshold 1.00) - 40 clusters (representing 905 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S39 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S39_1.fastq /home/working/demultiplexed/S39_2.fastq S39_forward.paired.fastq.gz S39_forward.singletons.fastq.gz S39_reverse.paired.fastq.gz S39_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S39_1.fastq /home/working/demultiplexed/S39_2.fastq S39_forward.paired.fastq.gz S39_forward.singletons.fastq.gz S39_reverse.paired.fastq.gz S39_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:54:02 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S39/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 16769 Both Surviving: 3591 (21.41%) Forward Only Surviving: 1075 (6.41%) Reverse Only Surviving: 215 (1.28%) Dropped: 11888 (70.89%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: S39_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S39_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S39_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S39_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S39_forward.paired.headclipped.fastq.gz S39_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S39 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S39_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S39_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S39.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S39.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S39.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S39.hist\n",
      "[FLASH]     ./S39.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3591 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3591\n",
      "[FLASH]     Combined pairs:   482\n",
      "[FLASH]     Uncombined pairs: 3109\n",
      "[FLASH]     Percent combined: 13.42%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.028 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "482 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S39/S39_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S39_centroids.fasta --uc S39.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 482 sequences and identified 345 clusters (clustering threshold 1.00) - 12 clusters (representing 128 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S43 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S43_1.fastq /home/working/demultiplexed/S43_2.fastq S43_forward.paired.fastq.gz S43_forward.singletons.fastq.gz S43_reverse.paired.fastq.gz S43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S43_1.fastq /home/working/demultiplexed/S43_2.fastq S43_forward.paired.fastq.gz S43_forward.singletons.fastq.gz S43_reverse.paired.fastq.gz S43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:54:06 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S43/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 9106 Both Surviving: 1630 (17.90%) Forward Only Surviving: 825 (9.06%) Reverse Only Surviving: 113 (1.24%) Dropped: 6538 (71.80%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: S43_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S43_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S43_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S43_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S43_forward.paired.headclipped.fastq.gz S43_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S43 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S43_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S43_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S43.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S43.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S43.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S43.hist\n",
      "[FLASH]     ./S43.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1630 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1630\n",
      "[FLASH]     Combined pairs:   1138\n",
      "[FLASH]     Uncombined pairs: 492\n",
      "[FLASH]     Percent combined: 69.82%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.032 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1138 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S43/S43_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S43_centroids.fasta --uc S43.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1138 sequences and identified 567 clusters (clustering threshold 1.00) - 8 clusters (representing 479 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S50 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S50_1.fastq /home/working/demultiplexed/S50_2.fastq S50_forward.paired.fastq.gz S50_forward.singletons.fastq.gz S50_reverse.paired.fastq.gz S50_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S50_1.fastq /home/working/demultiplexed/S50_2.fastq S50_forward.paired.fastq.gz S50_forward.singletons.fastq.gz S50_reverse.paired.fastq.gz S50_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:54:10 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S50/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 23841 Both Surviving: 5537 (23.22%) Forward Only Surviving: 1563 (6.56%) Reverse Only Surviving: 325 (1.36%) Dropped: 16416 (68.86%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: S50_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S50_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: S50_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: S50_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S50_forward.paired.headclipped.fastq.gz S50_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S50 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S50_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S50_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S50.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S50.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S50.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S50.hist\n",
      "[FLASH]     ./S50.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5537 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5537\n",
      "[FLASH]     Combined pairs:   2823\n",
      "[FLASH]     Uncombined pairs: 2714\n",
      "[FLASH]     Percent combined: 50.98%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.080 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2823 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S50/S50_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S50_centroids.fasta --uc S50.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2823 sequences and identified 1523 clusters (clustering threshold 1.00) - 37 clusters (representing 1105 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S56 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S56_1.fastq /home/working/demultiplexed/S56_2.fastq S56_forward.paired.fastq.gz S56_forward.singletons.fastq.gz S56_reverse.paired.fastq.gz S56_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S56_1.fastq /home/working/demultiplexed/S56_2.fastq S56_forward.paired.fastq.gz S56_forward.singletons.fastq.gz S56_reverse.paired.fastq.gz S56_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:54:16 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S56/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 22 bases off reads in file: S56_forward.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: S56_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S56_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S56_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S56_forward.paired.headclipped.fastq.gz S56_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S56 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S56_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S56_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S56.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S56.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S56.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S56.hist\n",
      "[FLASH]     ./S56.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      0\n",
      "[FLASH]     Combined pairs:   0\n",
      "[FLASH]     Uncombined pairs: 0\n",
      "[FLASH]     Percent combined: 0.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "0 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S56/S56_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S56_centroids.fasta --uc S56.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S59 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S59_1.fastq /home/working/demultiplexed/S59_2.fastq S59_forward.paired.fastq.gz S59_forward.singletons.fastq.gz S59_reverse.paired.fastq.gz S59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S59_1.fastq /home/working/demultiplexed/S59_2.fastq S59_forward.paired.fastq.gz S59_forward.singletons.fastq.gz S59_reverse.paired.fastq.gz S59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:54:17 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S59/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 18121 Both Surviving: 3365 (18.57%) Forward Only Surviving: 1089 (6.01%) Reverse Only Surviving: 238 (1.31%) Dropped: 13429 (74.11%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: S59_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S59_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: S59_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: S59_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S59_forward.paired.headclipped.fastq.gz S59_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S59 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S59_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S59_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S59.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S59.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S59.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S59.hist\n",
      "[FLASH]     ./S59.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3365 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3365\n",
      "[FLASH]     Combined pairs:   312\n",
      "[FLASH]     Uncombined pairs: 3053\n",
      "[FLASH]     Percent combined: 9.27%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.024 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "312 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S59/S59_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S59_centroids.fasta --uc S59.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 312 sequences and identified 237 clusters (clustering threshold 1.00) - 4 clusters (representing 65 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S62 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S62_1.fastq /home/working/demultiplexed/S62_2.fastq S62_forward.paired.fastq.gz S62_forward.singletons.fastq.gz S62_reverse.paired.fastq.gz S62_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S62_1.fastq /home/working/demultiplexed/S62_2.fastq S62_forward.paired.fastq.gz S62_forward.singletons.fastq.gz S62_reverse.paired.fastq.gz S62_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:54:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S62/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 39931 Both Surviving: 4343 (10.88%) Forward Only Surviving: 1632 (4.09%) Reverse Only Surviving: 405 (1.01%) Dropped: 33551 (84.02%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: S62_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: S62_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S62_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S62_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S62_forward.paired.headclipped.fastq.gz S62_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S62 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S62_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S62_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S62.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S62.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S62.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S62.hist\n",
      "[FLASH]     ./S62.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 4343 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      4343\n",
      "[FLASH]     Combined pairs:   1611\n",
      "[FLASH]     Uncombined pairs: 2732\n",
      "[FLASH]     Percent combined: 37.09%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.050 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1611 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S62/S62_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S62_centroids.fasta --uc S62.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1611 sequences and identified 697 clusters (clustering threshold 1.00) - 26 clusters (representing 796 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S63 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S63_1.fastq /home/working/demultiplexed/S63_2.fastq S63_forward.paired.fastq.gz S63_forward.singletons.fastq.gz S63_reverse.paired.fastq.gz S63_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S63_1.fastq /home/working/demultiplexed/S63_2.fastq S63_forward.paired.fastq.gz S63_forward.singletons.fastq.gz S63_reverse.paired.fastq.gz S63_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:54:27 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S63/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 6702 Both Surviving: 610 (9.10%) Forward Only Surviving: 253 (3.77%) Reverse Only Surviving: 80 (1.19%) Dropped: 5759 (85.93%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: S63_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: S63_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S63_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S63_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S63_forward.paired.headclipped.fastq.gz S63_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S63 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S63_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S63_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S63.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S63.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S63.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S63.hist\n",
      "[FLASH]     ./S63.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 610 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      610\n",
      "[FLASH]     Combined pairs:   540\n",
      "[FLASH]     Uncombined pairs: 70\n",
      "[FLASH]     Percent combined: 88.52%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.016 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "540 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S63/S63_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S63_centroids.fasta --uc S63.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 540 sequences and identified 341 clusters (clustering threshold 1.00) - 6 clusters (representing 176 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S65 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S65_1.fastq /home/working/demultiplexed/S65_2.fastq S65_forward.paired.fastq.gz S65_forward.singletons.fastq.gz S65_reverse.paired.fastq.gz S65_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S65_1.fastq /home/working/demultiplexed/S65_2.fastq S65_forward.paired.fastq.gz S65_forward.singletons.fastq.gz S65_reverse.paired.fastq.gz S65_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:54:29 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S65/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 7804 Both Surviving: 2569 (32.92%) Forward Only Surviving: 1071 (13.72%) Reverse Only Surviving: 88 (1.13%) Dropped: 4076 (52.23%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: S65_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: S65_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: S65_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: S65_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S65_forward.paired.headclipped.fastq.gz S65_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S65 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S65_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S65_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S65.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S65.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S65.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S65.hist\n",
      "[FLASH]     ./S65.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2569 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2569\n",
      "[FLASH]     Combined pairs:   2216\n",
      "[FLASH]     Uncombined pairs: 353\n",
      "[FLASH]     Percent combined: 86.26%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.060 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2216 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S65/S65_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S65_centroids.fasta --uc S65.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2216 sequences and identified 1018 clusters (clustering threshold 1.00) - 46 clusters (representing 1026 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S67 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S67_1.fastq /home/working/demultiplexed/S67_2.fastq S67_forward.paired.fastq.gz S67_forward.singletons.fastq.gz S67_reverse.paired.fastq.gz S67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S67_1.fastq /home/working/demultiplexed/S67_2.fastq S67_forward.paired.fastq.gz S67_forward.singletons.fastq.gz S67_reverse.paired.fastq.gz S67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:54:34 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S67/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 18116 Both Surviving: 4019 (22.18%) Forward Only Surviving: 945 (5.22%) Reverse Only Surviving: 327 (1.81%) Dropped: 12825 (70.79%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: S67_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S67_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S67_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S67_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S67_forward.paired.headclipped.fastq.gz S67_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S67 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S67_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S67_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S67.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S67.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S67.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S67.hist\n",
      "[FLASH]     ./S67.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 4019 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      4019\n",
      "[FLASH]     Combined pairs:   224\n",
      "[FLASH]     Uncombined pairs: 3795\n",
      "[FLASH]     Percent combined: 5.57%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.026 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "224 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S67/S67_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S67_centroids.fasta --uc S67.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 224 sequences and identified 174 clusters (clustering threshold 1.00) - 3 clusters (representing 37 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S7 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S7_1.fastq /home/working/demultiplexed/S7_2.fastq S7_forward.paired.fastq.gz S7_forward.singletons.fastq.gz S7_reverse.paired.fastq.gz S7_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S7_1.fastq /home/working/demultiplexed/S7_2.fastq S7_forward.paired.fastq.gz S7_forward.singletons.fastq.gz S7_reverse.paired.fastq.gz S7_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:54:39 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S7/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 32963 Both Surviving: 4645 (14.09%) Forward Only Surviving: 1505 (4.57%) Reverse Only Surviving: 245 (0.74%) Dropped: 26568 (80.60%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: S7_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: S7_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: S7_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: S7_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S7_forward.paired.headclipped.fastq.gz S7_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S7 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S7_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S7_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S7.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S7.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S7.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S7.hist\n",
      "[FLASH]     ./S7.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 4645 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      4645\n",
      "[FLASH]     Combined pairs:   4316\n",
      "[FLASH]     Uncombined pairs: 329\n",
      "[FLASH]     Percent combined: 92.92%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.115 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "4316 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S7/S7_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S7_centroids.fasta --uc S7.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 4316 sequences and identified 2427 clusters (clustering threshold 1.00) - 46 clusters (representing 1620 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S72 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S72_1.fastq /home/working/demultiplexed/S72_2.fastq S72_forward.paired.fastq.gz S72_forward.singletons.fastq.gz S72_reverse.paired.fastq.gz S72_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S72_1.fastq /home/working/demultiplexed/S72_2.fastq S72_forward.paired.fastq.gz S72_forward.singletons.fastq.gz S72_reverse.paired.fastq.gz S72_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:54:49 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S72/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 38851 Both Surviving: 115 (0.30%) Forward Only Surviving: 569 (1.46%) Reverse Only Surviving: 398 (1.02%) Dropped: 37769 (97.22%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: S72_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: S72_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: S72_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: S72_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S72_forward.paired.headclipped.fastq.gz S72_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S72 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S72_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S72_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S72.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S72.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S72.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S72.hist\n",
      "[FLASH]     ./S72.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 115 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      115\n",
      "[FLASH]     Combined pairs:   80\n",
      "[FLASH]     Uncombined pairs: 35\n",
      "[FLASH]     Percent combined: 69.57%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.004 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "80 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S72/S72_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S72_centroids.fasta --uc S72.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 80 sequences and identified 71 clusters (clustering threshold 1.00) - 1 clusters (representing 6 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S77 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S77_1.fastq /home/working/demultiplexed/S77_2.fastq S77_forward.paired.fastq.gz S77_forward.singletons.fastq.gz S77_reverse.paired.fastq.gz S77_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S77_1.fastq /home/working/demultiplexed/S77_2.fastq S77_forward.paired.fastq.gz S77_forward.singletons.fastq.gz S77_reverse.paired.fastq.gz S77_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:54:52 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S77/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 14430 Both Surviving: 1819 (12.61%) Forward Only Surviving: 613 (4.25%) Reverse Only Surviving: 134 (0.93%) Dropped: 11864 (82.22%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: S77_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: S77_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S77_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S77_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S77_forward.paired.headclipped.fastq.gz S77_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S77 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S77_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S77_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S77.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S77.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S77.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S77.hist\n",
      "[FLASH]     ./S77.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1819 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1819\n",
      "[FLASH]     Combined pairs:   115\n",
      "[FLASH]     Uncombined pairs: 1704\n",
      "[FLASH]     Percent combined: 6.32%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.012 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "115 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S77/S77_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S77_centroids.fasta --uc S77.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 115 sequences and identified 101 clusters (clustering threshold 1.00) - 1 clusters (representing 10 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S79 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S79_1.fastq /home/working/demultiplexed/S79_2.fastq S79_forward.paired.fastq.gz S79_forward.singletons.fastq.gz S79_reverse.paired.fastq.gz S79_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S79_1.fastq /home/working/demultiplexed/S79_2.fastq S79_forward.paired.fastq.gz S79_forward.singletons.fastq.gz S79_reverse.paired.fastq.gz S79_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:55:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S79/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 143319 Both Surviving: 50529 (35.26%) Forward Only Surviving: 11833 (8.26%) Reverse Only Surviving: 2377 (1.66%) Dropped: 78580 (54.83%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: S79_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: S79_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: S79_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: S79_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S79_forward.paired.headclipped.fastq.gz S79_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S79 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S79_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S79_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S79.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S79.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S79.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S79.hist\n",
      "[FLASH]     ./S79.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 25000 read pairs\n",
      "[FLASH] Processed 50000 read pairs\n",
      "[FLASH] Processed 50529 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      50529\n",
      "[FLASH]     Combined pairs:   32082\n",
      "[FLASH]     Uncombined pairs: 18447\n",
      "[FLASH]     Percent combined: 63.49%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.861 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "32082 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S79/S79_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S79_centroids.fasta --uc S79.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 32082 sequences and identified 9396 clusters (clustering threshold 1.00) - 592 clusters (representing 21271 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S8 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S8_1.fastq /home/working/demultiplexed/S8_2.fastq S8_forward.paired.fastq.gz S8_forward.singletons.fastq.gz S8_reverse.paired.fastq.gz S8_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S8_1.fastq /home/working/demultiplexed/S8_2.fastq S8_forward.paired.fastq.gz S8_forward.singletons.fastq.gz S8_reverse.paired.fastq.gz S8_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:56:07 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S8/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 26392 Both Surviving: 9081 (34.41%) Forward Only Surviving: 3915 (14.83%) Reverse Only Surviving: 236 (0.89%) Dropped: 13160 (49.86%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: S8_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S8_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S8_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S8_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S8_forward.paired.headclipped.fastq.gz S8_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S8 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S8_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S8_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S8.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S8.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S8.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S8.hist\n",
      "[FLASH]     ./S8.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 9081 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      9081\n",
      "[FLASH]     Combined pairs:   5726\n",
      "[FLASH]     Uncombined pairs: 3355\n",
      "[FLASH]     Percent combined: 63.05%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.195 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "5726 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S8/S8_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S8_centroids.fasta --uc S8.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 5726 sequences and identified 2705 clusters (clustering threshold 1.00) - 81 clusters (representing 2588 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S80 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S80_1.fastq /home/working/demultiplexed/S80_2.fastq S80_forward.paired.fastq.gz S80_forward.singletons.fastq.gz S80_reverse.paired.fastq.gz S80_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S80_1.fastq /home/working/demultiplexed/S80_2.fastq S80_forward.paired.fastq.gz S80_forward.singletons.fastq.gz S80_reverse.paired.fastq.gz S80_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:56:19 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S80/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 10756 Both Surviving: 1785 (16.60%) Forward Only Surviving: 535 (4.97%) Reverse Only Surviving: 148 (1.38%) Dropped: 8288 (77.05%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: S80_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S80_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S80_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S80_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S80_forward.paired.headclipped.fastq.gz S80_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S80 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S80_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S80_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S80.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S80.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S80.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S80.hist\n",
      "[FLASH]     ./S80.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1785 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1785\n",
      "[FLASH]     Combined pairs:   362\n",
      "[FLASH]     Uncombined pairs: 1423\n",
      "[FLASH]     Percent combined: 20.28%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.022 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "362 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S80/S80_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S80_centroids.fasta --uc S80.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 362 sequences and identified 234 clusters (clustering threshold 1.00) - 6 clusters (representing 121 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S81 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S81_1.fastq /home/working/demultiplexed/S81_2.fastq S81_forward.paired.fastq.gz S81_forward.singletons.fastq.gz S81_reverse.paired.fastq.gz S81_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S81_1.fastq /home/working/demultiplexed/S81_2.fastq S81_forward.paired.fastq.gz S81_forward.singletons.fastq.gz S81_reverse.paired.fastq.gz S81_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:56:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S81/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 20291 Both Surviving: 373 (1.84%) Forward Only Surviving: 524 (2.58%) Reverse Only Surviving: 177 (0.87%) Dropped: 19217 (94.71%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: S81_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: S81_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S81_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S81_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S81_forward.paired.headclipped.fastq.gz S81_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S81 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S81_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S81_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S81.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S81.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S81.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S81.hist\n",
      "[FLASH]     ./S81.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 373 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      373\n",
      "[FLASH]     Combined pairs:   240\n",
      "[FLASH]     Uncombined pairs: 133\n",
      "[FLASH]     Percent combined: 64.34%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.008 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "240 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S81/S81_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S81_centroids.fasta --uc S81.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 240 sequences and identified 190 clusters (clustering threshold 1.00) - 3 clusters (representing 43 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S83 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S83_1.fastq /home/working/demultiplexed/S83_2.fastq S83_forward.paired.fastq.gz S83_forward.singletons.fastq.gz S83_reverse.paired.fastq.gz S83_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S83_1.fastq /home/working/demultiplexed/S83_2.fastq S83_forward.paired.fastq.gz S83_forward.singletons.fastq.gz S83_reverse.paired.fastq.gz S83_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:56:26 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S83/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 40119 Both Surviving: 17744 (44.23%) Forward Only Surviving: 8944 (22.29%) Reverse Only Surviving: 329 (0.82%) Dropped: 13102 (32.66%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: S83_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: S83_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S83_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S83_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S83_forward.paired.headclipped.fastq.gz S83_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S83 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S83_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S83_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S83.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S83.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S83.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S83.hist\n",
      "[FLASH]     ./S83.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 17744 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      17744\n",
      "[FLASH]     Combined pairs:   13157\n",
      "[FLASH]     Uncombined pairs: 4587\n",
      "[FLASH]     Percent combined: 74.15%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.365 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "13157 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S83/S83_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S83_centroids.fasta --uc S83.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 13157 sequences and identified 3794 clusters (clustering threshold 1.00) - 314 clusters (representing 9363 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S9 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S9_1.fastq /home/working/demultiplexed/S9_2.fastq S9_forward.paired.fastq.gz S9_forward.singletons.fastq.gz S9_reverse.paired.fastq.gz S9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S9_1.fastq /home/working/demultiplexed/S9_2.fastq S9_forward.paired.fastq.gz S9_forward.singletons.fastq.gz S9_reverse.paired.fastq.gz S9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:56:53 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S9/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 41954 Both Surviving: 13198 (31.46%) Forward Only Surviving: 4239 (10.10%) Reverse Only Surviving: 607 (1.45%) Dropped: 23910 (56.99%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: S9_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: S9_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: S9_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: S9_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S9_forward.paired.headclipped.fastq.gz S9_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S9 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S9_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S9_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S9.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S9.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S9.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S9.hist\n",
      "[FLASH]     ./S9.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 13198 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      13198\n",
      "[FLASH]     Combined pairs:   8987\n",
      "[FLASH]     Uncombined pairs: 4211\n",
      "[FLASH]     Percent combined: 68.09%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.414 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "8987 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S9/S9_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S9_centroids.fasta --uc S9.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 8987 sequences and identified 2196 clusters (clustering threshold 1.00) - 235 clusters (representing 6715 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S90 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S90_1.fastq /home/working/demultiplexed/S90_2.fastq S90_forward.paired.fastq.gz S90_forward.singletons.fastq.gz S90_reverse.paired.fastq.gz S90_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S90_1.fastq /home/working/demultiplexed/S90_2.fastq S90_forward.paired.fastq.gz S90_forward.singletons.fastq.gz S90_reverse.paired.fastq.gz S90_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:57:14 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S90/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 18970 Both Surviving: 5970 (31.47%) Forward Only Surviving: 1659 (8.75%) Reverse Only Surviving: 249 (1.31%) Dropped: 11092 (58.47%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: S90_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S90_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S90_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S90_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S90_forward.paired.headclipped.fastq.gz S90_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S90 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S90_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S90_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S90.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S90.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S90.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S90.hist\n",
      "[FLASH]     ./S90.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5970 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5970\n",
      "[FLASH]     Combined pairs:   3361\n",
      "[FLASH]     Uncombined pairs: 2609\n",
      "[FLASH]     Percent combined: 56.30%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.129 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "3361 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S90/S90_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S90_centroids.fasta --uc S90.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 3361 sequences and identified 2119 clusters (clustering threshold 1.00) - 41 clusters (representing 1066 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S92 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S92_1.fastq /home/working/demultiplexed/S92_2.fastq S92_forward.paired.fastq.gz S92_forward.singletons.fastq.gz S92_reverse.paired.fastq.gz S92_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S92_1.fastq /home/working/demultiplexed/S92_2.fastq S92_forward.paired.fastq.gz S92_forward.singletons.fastq.gz S92_reverse.paired.fastq.gz S92_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:57:26 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S92/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 58656 Both Surviving: 3312 (5.65%) Forward Only Surviving: 2833 (4.83%) Reverse Only Surviving: 755 (1.29%) Dropped: 51756 (88.24%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: S92_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: S92_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S92_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S92_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S92_forward.paired.headclipped.fastq.gz S92_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S92 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S92_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S92_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S92.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S92.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S92.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S92.hist\n",
      "[FLASH]     ./S92.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3312 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3312\n",
      "[FLASH]     Combined pairs:   805\n",
      "[FLASH]     Uncombined pairs: 2507\n",
      "[FLASH]     Percent combined: 24.31%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.051 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "805 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S92/S92_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S92_centroids.fasta --uc S92.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 805 sequences and identified 575 clusters (clustering threshold 1.00) - 13 clusters (representing 167 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S96 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S96_1.fastq /home/working/demultiplexed/S96_2.fastq S96_forward.paired.fastq.gz S96_forward.singletons.fastq.gz S96_reverse.paired.fastq.gz S96_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S96_1.fastq /home/working/demultiplexed/S96_2.fastq S96_forward.paired.fastq.gz S96_forward.singletons.fastq.gz S96_reverse.paired.fastq.gz S96_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:57:33 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S96/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 17331 Both Surviving: 2217 (12.79%) Forward Only Surviving: 807 (4.66%) Reverse Only Surviving: 207 (1.19%) Dropped: 14100 (81.36%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: S96_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: S96_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: S96_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: S96_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S96_forward.paired.headclipped.fastq.gz S96_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S96 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S96_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S96_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S96.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S96.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S96.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S96.hist\n",
      "[FLASH]     ./S96.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2217 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2217\n",
      "[FLASH]     Combined pairs:   2021\n",
      "[FLASH]     Uncombined pairs: 196\n",
      "[FLASH]     Percent combined: 91.16%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.056 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2021 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S96/S96_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S96_centroids.fasta --uc S96.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2021 sequences and identified 1156 clusters (clustering threshold 1.00) - 26 clusters (representing 799 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: S97 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S97_1.fastq /home/working/demultiplexed/S97_2.fastq S97_forward.paired.fastq.gz S97_forward.singletons.fastq.gz S97_reverse.paired.fastq.gz S97_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/S97_1.fastq /home/working/demultiplexed/S97_2.fastq S97_forward.paired.fastq.gz S97_forward.singletons.fastq.gz S97_reverse.paired.fastq.gz S97_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:57:39 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/S97/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 19611 Both Surviving: 5305 (27.05%) Forward Only Surviving: 1691 (8.62%) Reverse Only Surviving: 253 (1.29%) Dropped: 12362 (63.04%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: S97_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: S97_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S97_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: S97_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash S97_forward.paired.headclipped.fastq.gz S97_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o S97 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     S97_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     S97_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./S97.extendedFrags.fastq.gz\n",
      "[FLASH]     ./S97.notCombined_1.fastq.gz\n",
      "[FLASH]     ./S97.notCombined_2.fastq.gz\n",
      "[FLASH]     ./S97.hist\n",
      "[FLASH]     ./S97.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5305 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5305\n",
      "[FLASH]     Combined pairs:   276\n",
      "[FLASH]     Uncombined pairs: 5029\n",
      "[FLASH]     Percent combined: 5.20%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.052 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "276 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/S97/S97_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids S97_centroids.fasta --uc S97.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 276 sequences and identified 211 clusters (clustering threshold 1.00) - 5 clusters (representing 37 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA1 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA1_1.fastq /home/working/demultiplexed/SA1_2.fastq SA1_forward.paired.fastq.gz SA1_forward.singletons.fastq.gz SA1_reverse.paired.fastq.gz SA1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA1_1.fastq /home/working/demultiplexed/SA1_2.fastq SA1_forward.paired.fastq.gz SA1_forward.singletons.fastq.gz SA1_reverse.paired.fastq.gz SA1_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:57:43 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA1/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 8479 Both Surviving: 396 (4.67%) Forward Only Surviving: 277 (3.27%) Reverse Only Surviving: 151 (1.78%) Dropped: 7655 (90.28%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SA1_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA1_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA1_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA1_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA1_forward.paired.headclipped.fastq.gz SA1_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA1 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA1_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA1_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA1.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA1.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA1.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA1.hist\n",
      "[FLASH]     ./SA1.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 396 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      396\n",
      "[FLASH]     Combined pairs:   329\n",
      "[FLASH]     Uncombined pairs: 67\n",
      "[FLASH]     Percent combined: 83.08%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.011 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "329 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA1/SA1_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA1_centroids.fasta --uc SA1.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 329 sequences and identified 217 clusters (clustering threshold 1.00) - 10 clusters (representing 103 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA104 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA104_1.fastq /home/working/demultiplexed/SA104_2.fastq SA104_forward.paired.fastq.gz SA104_forward.singletons.fastq.gz SA104_reverse.paired.fastq.gz SA104_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA104_1.fastq /home/working/demultiplexed/SA104_2.fastq SA104_forward.paired.fastq.gz SA104_forward.singletons.fastq.gz SA104_reverse.paired.fastq.gz SA104_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:57:45 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA104/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 24043 Both Surviving: 3279 (13.64%) Forward Only Surviving: 915 (3.81%) Reverse Only Surviving: 431 (1.79%) Dropped: 19418 (80.76%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SA104_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA104_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA104_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA104_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA104_forward.paired.headclipped.fastq.gz SA104_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA104 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA104_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA104_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA104.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA104.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA104.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA104.hist\n",
      "[FLASH]     ./SA104.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3279 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3279\n",
      "[FLASH]     Combined pairs:   82\n",
      "[FLASH]     Uncombined pairs: 3197\n",
      "[FLASH]     Percent combined: 2.50%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.041 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "82 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA104/SA104_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA104_centroids.fasta --uc SA104.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 82 sequences and identified 62 clusters (clustering threshold 1.00) - 2 clusters (representing 10 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA105 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA105_1.fastq /home/working/demultiplexed/SA105_2.fastq SA105_forward.paired.fastq.gz SA105_forward.singletons.fastq.gz SA105_reverse.paired.fastq.gz SA105_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA105_1.fastq /home/working/demultiplexed/SA105_2.fastq SA105_forward.paired.fastq.gz SA105_forward.singletons.fastq.gz SA105_reverse.paired.fastq.gz SA105_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:57:49 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA105/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 23190 Both Surviving: 114 (0.49%) Forward Only Surviving: 242 (1.04%) Reverse Only Surviving: 165 (0.71%) Dropped: 22669 (97.75%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SA105_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA105_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA105_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA105_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA105_forward.paired.headclipped.fastq.gz SA105_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA105 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA105_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA105_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA105.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA105.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA105.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA105.hist\n",
      "[FLASH]     ./SA105.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 114 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      114\n",
      "[FLASH]     Combined pairs:   85\n",
      "[FLASH]     Uncombined pairs: 29\n",
      "[FLASH]     Percent combined: 74.56%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.005 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "85 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA105/SA105_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA105_centroids.fasta --uc SA105.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 85 sequences and identified 59 clusters (clustering threshold 1.00) - 2 clusters (representing 24 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA109 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA109_1.fastq /home/working/demultiplexed/SA109_2.fastq SA109_forward.paired.fastq.gz SA109_forward.singletons.fastq.gz SA109_reverse.paired.fastq.gz SA109_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA109_1.fastq /home/working/demultiplexed/SA109_2.fastq SA109_forward.paired.fastq.gz SA109_forward.singletons.fastq.gz SA109_reverse.paired.fastq.gz SA109_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:57:51 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA109/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 6 Both Surviving: 1 (16.67%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 5 (83.33%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 22 bases off reads in file: SA109_forward.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA109_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA109_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA109_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA109_forward.paired.headclipped.fastq.gz SA109_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA109 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA109_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA109_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA109.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA109.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA109.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA109.hist\n",
      "[FLASH]     ./SA109.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1\n",
      "[FLASH]     Combined pairs:   1\n",
      "[FLASH]     Uncombined pairs: 0\n",
      "[FLASH]     Percent combined: 100.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA109/SA109_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA109_centroids.fasta --uc SA109.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1 sequences and identified 1 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA111 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA111_1.fastq /home/working/demultiplexed/SA111_2.fastq SA111_forward.paired.fastq.gz SA111_forward.singletons.fastq.gz SA111_reverse.paired.fastq.gz SA111_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA111_1.fastq /home/working/demultiplexed/SA111_2.fastq SA111_forward.paired.fastq.gz SA111_forward.singletons.fastq.gz SA111_reverse.paired.fastq.gz SA111_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:57:52 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA111/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 16872 Both Surviving: 3412 (20.22%) Forward Only Surviving: 1618 (9.59%) Reverse Only Surviving: 186 (1.10%) Dropped: 11656 (69.08%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SA111_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SA111_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA111_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA111_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA111_forward.paired.headclipped.fastq.gz SA111_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA111 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA111_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA111_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA111.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA111.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA111.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA111.hist\n",
      "[FLASH]     ./SA111.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3412 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3412\n",
      "[FLASH]     Combined pairs:   412\n",
      "[FLASH]     Uncombined pairs: 3000\n",
      "[FLASH]     Percent combined: 12.08%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.032 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "412 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA111/SA111_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA111_centroids.fasta --uc SA111.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 412 sequences and identified 319 clusters (clustering threshold 1.00) - 8 clusters (representing 56 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA112 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA112_1.fastq /home/working/demultiplexed/SA112_2.fastq SA112_forward.paired.fastq.gz SA112_forward.singletons.fastq.gz SA112_reverse.paired.fastq.gz SA112_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA112_1.fastq /home/working/demultiplexed/SA112_2.fastq SA112_forward.paired.fastq.gz SA112_forward.singletons.fastq.gz SA112_reverse.paired.fastq.gz SA112_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:57:59 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA112/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 86230 Both Surviving: 26880 (31.17%) Forward Only Surviving: 5576 (6.47%) Reverse Only Surviving: 3113 (3.61%) Dropped: 50661 (58.75%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: SA112_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA112_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA112_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA112_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA112_forward.paired.headclipped.fastq.gz SA112_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA112 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA112_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA112_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA112.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA112.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA112.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA112.hist\n",
      "[FLASH]     ./SA112.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 25000 read pairs\n",
      "[FLASH] Processed 26880 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      26880\n",
      "[FLASH]     Combined pairs:   5185\n",
      "[FLASH]     Uncombined pairs: 21695\n",
      "[FLASH]     Percent combined: 19.29%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.245 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "5185 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA112/SA112_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA112_centroids.fasta --uc SA112.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 5185 sequences and identified 2926 clusters (clustering threshold 1.00) - 76 clusters (representing 1962 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA113 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA113_1.fastq /home/working/demultiplexed/SA113_2.fastq SA113_forward.paired.fastq.gz SA113_forward.singletons.fastq.gz SA113_reverse.paired.fastq.gz SA113_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA113_1.fastq /home/working/demultiplexed/SA113_2.fastq SA113_forward.paired.fastq.gz SA113_forward.singletons.fastq.gz SA113_reverse.paired.fastq.gz SA113_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:58:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA113/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 31186 Both Surviving: 6006 (19.26%) Forward Only Surviving: 1650 (5.29%) Reverse Only Surviving: 427 (1.37%) Dropped: 23103 (74.08%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA113_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA113_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA113_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA113_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA113_forward.paired.headclipped.fastq.gz SA113_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA113 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA113_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA113_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA113.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA113.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA113.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA113.hist\n",
      "[FLASH]     ./SA113.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 6006 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      6006\n",
      "[FLASH]     Combined pairs:   500\n",
      "[FLASH]     Uncombined pairs: 5506\n",
      "[FLASH]     Percent combined: 8.33%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.050 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "500 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA113/SA113_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA113_centroids.fasta --uc SA113.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 500 sequences and identified 338 clusters (clustering threshold 1.00) - 13 clusters (representing 149 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA114 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA114_1.fastq /home/working/demultiplexed/SA114_2.fastq SA114_forward.paired.fastq.gz SA114_forward.singletons.fastq.gz SA114_reverse.paired.fastq.gz SA114_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA114_1.fastq /home/working/demultiplexed/SA114_2.fastq SA114_forward.paired.fastq.gz SA114_forward.singletons.fastq.gz SA114_reverse.paired.fastq.gz SA114_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:58:33 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA114/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 133965 Both Surviving: 33203 (24.78%) Forward Only Surviving: 12322 (9.20%) Reverse Only Surviving: 1774 (1.32%) Dropped: 86666 (64.69%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: SA114_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SA114_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SA114_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SA114_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA114_forward.paired.headclipped.fastq.gz SA114_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA114 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA114_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA114_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA114.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA114.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA114.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA114.hist\n",
      "[FLASH]     ./SA114.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 25000 read pairs\n",
      "[FLASH] Processed 33203 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      33203\n",
      "[FLASH]     Combined pairs:   28702\n",
      "[FLASH]     Uncombined pairs: 4501\n",
      "[FLASH]     Percent combined: 86.44%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.749 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "28702 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA114/SA114_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA114_centroids.fasta --uc SA114.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 28702 sequences and identified 11395 clusters (clustering threshold 1.00) - 476 clusters (representing 16481 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA115 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA115_1.fastq /home/working/demultiplexed/SA115_2.fastq SA115_forward.paired.fastq.gz SA115_forward.singletons.fastq.gz SA115_reverse.paired.fastq.gz SA115_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA115_1.fastq /home/working/demultiplexed/SA115_2.fastq SA115_forward.paired.fastq.gz SA115_forward.singletons.fastq.gz SA115_reverse.paired.fastq.gz SA115_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:59:30 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA115/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 21323 Both Surviving: 323 (1.51%) Forward Only Surviving: 345 (1.62%) Reverse Only Surviving: 232 (1.09%) Dropped: 20423 (95.78%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: SA115_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: SA115_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA115_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA115_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA115_forward.paired.headclipped.fastq.gz SA115_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA115 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA115_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA115_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA115.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA115.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA115.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA115.hist\n",
      "[FLASH]     ./SA115.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 323 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      323\n",
      "[FLASH]     Combined pairs:   79\n",
      "[FLASH]     Uncombined pairs: 244\n",
      "[FLASH]     Percent combined: 24.46%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.004 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "79 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA115/SA115_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA115_centroids.fasta --uc SA115.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 79 sequences and identified 61 clusters (clustering threshold 1.00) - 2 clusters (representing 15 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA116 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA116_1.fastq /home/working/demultiplexed/SA116_2.fastq SA116_forward.paired.fastq.gz SA116_forward.singletons.fastq.gz SA116_reverse.paired.fastq.gz SA116_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA116_1.fastq /home/working/demultiplexed/SA116_2.fastq SA116_forward.paired.fastq.gz SA116_forward.singletons.fastq.gz SA116_reverse.paired.fastq.gz SA116_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:59:33 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA116/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 31818 Both Surviving: 2035 (6.40%) Forward Only Surviving: 881 (2.77%) Reverse Only Surviving: 589 (1.85%) Dropped: 28313 (88.98%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SA116_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA116_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA116_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA116_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA116_forward.paired.headclipped.fastq.gz SA116_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA116 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA116_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA116_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA116.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA116.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA116.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA116.hist\n",
      "[FLASH]     ./SA116.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2035 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2035\n",
      "[FLASH]     Combined pairs:   13\n",
      "[FLASH]     Uncombined pairs: 2022\n",
      "[FLASH]     Percent combined: 0.64%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.020 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "13 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA116/SA116_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA116_centroids.fasta --uc SA116.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 13 sequences and identified 13 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA117 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA117_1.fastq /home/working/demultiplexed/SA117_2.fastq SA117_forward.paired.fastq.gz SA117_forward.singletons.fastq.gz SA117_reverse.paired.fastq.gz SA117_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA117_1.fastq /home/working/demultiplexed/SA117_2.fastq SA117_forward.paired.fastq.gz SA117_forward.singletons.fastq.gz SA117_reverse.paired.fastq.gz SA117_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:59:37 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA117/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 37714 Both Surviving: 7464 (19.79%) Forward Only Surviving: 1783 (4.73%) Reverse Only Surviving: 467 (1.24%) Dropped: 28000 (74.24%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SA117_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SA117_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA117_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA117_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA117_forward.paired.headclipped.fastq.gz SA117_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA117 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA117_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA117_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA117.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA117.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA117.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA117.hist\n",
      "[FLASH]     ./SA117.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 7464 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      7464\n",
      "[FLASH]     Combined pairs:   1516\n",
      "[FLASH]     Uncombined pairs: 5948\n",
      "[FLASH]     Percent combined: 20.31%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.059 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1516 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA117/SA117_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA117_centroids.fasta --uc SA117.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1516 sequences and identified 932 clusters (clustering threshold 1.00) - 19 clusters (representing 523 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA119 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA119_1.fastq /home/working/demultiplexed/SA119_2.fastq SA119_forward.paired.fastq.gz SA119_forward.singletons.fastq.gz SA119_reverse.paired.fastq.gz SA119_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA119_1.fastq /home/working/demultiplexed/SA119_2.fastq SA119_forward.paired.fastq.gz SA119_forward.singletons.fastq.gz SA119_reverse.paired.fastq.gz SA119_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:59:46 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA119/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 76452 Both Surviving: 12333 (16.13%) Forward Only Surviving: 3713 (4.86%) Reverse Only Surviving: 2374 (3.11%) Dropped: 58032 (75.91%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: SA119_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA119_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA119_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA119_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA119_forward.paired.headclipped.fastq.gz SA119_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA119 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA119_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA119_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA119.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA119.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA119.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA119.hist\n",
      "[FLASH]     ./SA119.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 12333 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      12333\n",
      "[FLASH]     Combined pairs:   548\n",
      "[FLASH]     Uncombined pairs: 11785\n",
      "[FLASH]     Percent combined: 4.44%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.087 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "548 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA119/SA119_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA119_centroids.fasta --uc SA119.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 548 sequences and identified 429 clusters (clustering threshold 1.00) - 3 clusters (representing 86 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA120 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA120_1.fastq /home/working/demultiplexed/SA120_2.fastq SA120_forward.paired.fastq.gz SA120_forward.singletons.fastq.gz SA120_reverse.paired.fastq.gz SA120_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA120_1.fastq /home/working/demultiplexed/SA120_2.fastq SA120_forward.paired.fastq.gz SA120_forward.singletons.fastq.gz SA120_reverse.paired.fastq.gz SA120_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 4:59:56 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA120/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 26355 Both Surviving: 872 (3.31%) Forward Only Surviving: 734 (2.79%) Reverse Only Surviving: 223 (0.85%) Dropped: 24526 (93.06%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SA120_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SA120_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA120_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA120_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA120_forward.paired.headclipped.fastq.gz SA120_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA120 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA120_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA120_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA120.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA120.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA120.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA120.hist\n",
      "[FLASH]     ./SA120.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 872 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      872\n",
      "[FLASH]     Combined pairs:   669\n",
      "[FLASH]     Uncombined pairs: 203\n",
      "[FLASH]     Percent combined: 76.72%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.018 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "669 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA120/SA120_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA120_centroids.fasta --uc SA120.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 669 sequences and identified 419 clusters (clustering threshold 1.00) - 15 clusters (representing 234 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA121 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA121_1.fastq /home/working/demultiplexed/SA121_2.fastq SA121_forward.paired.fastq.gz SA121_forward.singletons.fastq.gz SA121_reverse.paired.fastq.gz SA121_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA121_1.fastq /home/working/demultiplexed/SA121_2.fastq SA121_forward.paired.fastq.gz SA121_forward.singletons.fastq.gz SA121_reverse.paired.fastq.gz SA121_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:00:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA121/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 50866 Both Surviving: 10952 (21.53%) Forward Only Surviving: 3441 (6.76%) Reverse Only Surviving: 729 (1.43%) Dropped: 35744 (70.27%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: SA121_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SA121_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA121_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA121_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA121_forward.paired.headclipped.fastq.gz SA121_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA121 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA121_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA121_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA121.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA121.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA121.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA121.hist\n",
      "[FLASH]     ./SA121.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 10952 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      10952\n",
      "[FLASH]     Combined pairs:   1060\n",
      "[FLASH]     Uncombined pairs: 9892\n",
      "[FLASH]     Percent combined: 9.68%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.070 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1060 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA121/SA121_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA121_centroids.fasta --uc SA121.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1060 sequences and identified 715 clusters (clustering threshold 1.00) - 17 clusters (representing 283 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA122 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA122_1.fastq /home/working/demultiplexed/SA122_2.fastq SA122_forward.paired.fastq.gz SA122_forward.singletons.fastq.gz SA122_reverse.paired.fastq.gz SA122_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA122_1.fastq /home/working/demultiplexed/SA122_2.fastq SA122_forward.paired.fastq.gz SA122_forward.singletons.fastq.gz SA122_reverse.paired.fastq.gz SA122_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:00:09 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA122/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 13391 Both Surviving: 4648 (34.71%) Forward Only Surviving: 940 (7.02%) Reverse Only Surviving: 201 (1.50%) Dropped: 7602 (56.77%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: SA122_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SA122_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA122_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA122_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA122_forward.paired.headclipped.fastq.gz SA122_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA122 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA122_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA122_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA122.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA122.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA122.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA122.hist\n",
      "[FLASH]     ./SA122.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 4648 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      4648\n",
      "[FLASH]     Combined pairs:   2464\n",
      "[FLASH]     Uncombined pairs: 2184\n",
      "[FLASH]     Percent combined: 53.01%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.074 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2464 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA122/SA122_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA122_centroids.fasta --uc SA122.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2464 sequences and identified 1373 clusters (clustering threshold 1.00) - 56 clusters (representing 941 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA124 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA124_1.fastq /home/working/demultiplexed/SA124_2.fastq SA124_forward.paired.fastq.gz SA124_forward.singletons.fastq.gz SA124_reverse.paired.fastq.gz SA124_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA124_1.fastq /home/working/demultiplexed/SA124_2.fastq SA124_forward.paired.fastq.gz SA124_forward.singletons.fastq.gz SA124_reverse.paired.fastq.gz SA124_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:00:14 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA124/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 22 bases off reads in file: SA124_forward.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA124_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA124_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA124_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA124_forward.paired.headclipped.fastq.gz SA124_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA124 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA124_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA124_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA124.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA124.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA124.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA124.hist\n",
      "[FLASH]     ./SA124.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      0\n",
      "[FLASH]     Combined pairs:   0\n",
      "[FLASH]     Uncombined pairs: 0\n",
      "[FLASH]     Percent combined: 0.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "0 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA124/SA124_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA124_centroids.fasta --uc SA124.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA125 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA125_1.fastq /home/working/demultiplexed/SA125_2.fastq SA125_forward.paired.fastq.gz SA125_forward.singletons.fastq.gz SA125_reverse.paired.fastq.gz SA125_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA125_1.fastq /home/working/demultiplexed/SA125_2.fastq SA125_forward.paired.fastq.gz SA125_forward.singletons.fastq.gz SA125_reverse.paired.fastq.gz SA125_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:00:19 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA125/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 84537 Both Surviving: 26226 (31.02%) Forward Only Surviving: 8061 (9.54%) Reverse Only Surviving: 1377 (1.63%) Dropped: 48873 (57.81%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA125_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA125_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA125_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA125_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA125_forward.paired.headclipped.fastq.gz SA125_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA125 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA125_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA125_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA125.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA125.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA125.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA125.hist\n",
      "[FLASH]     ./SA125.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 25000 read pairs\n",
      "[FLASH] Processed 26226 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      26226\n",
      "[FLASH]     Combined pairs:   6724\n",
      "[FLASH]     Uncombined pairs: 19502\n",
      "[FLASH]     Percent combined: 25.64%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.252 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "6724 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA125/SA125_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA125_centroids.fasta --uc SA125.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 6724 sequences and identified 2696 clusters (clustering threshold 1.00) - 109 clusters (representing 3473 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA126 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA126_1.fastq /home/working/demultiplexed/SA126_2.fastq SA126_forward.paired.fastq.gz SA126_forward.singletons.fastq.gz SA126_reverse.paired.fastq.gz SA126_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA126_1.fastq /home/working/demultiplexed/SA126_2.fastq SA126_forward.paired.fastq.gz SA126_forward.singletons.fastq.gz SA126_reverse.paired.fastq.gz SA126_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:00:41 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA126/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 43041 Both Surviving: 12938 (30.06%) Forward Only Surviving: 3091 (7.18%) Reverse Only Surviving: 878 (2.04%) Dropped: 26134 (60.72%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: SA126_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SA126_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA126_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA126_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA126_forward.paired.headclipped.fastq.gz SA126_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA126 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA126_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA126_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA126.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA126.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA126.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA126.hist\n",
      "[FLASH]     ./SA126.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 12938 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      12938\n",
      "[FLASH]     Combined pairs:   568\n",
      "[FLASH]     Uncombined pairs: 12370\n",
      "[FLASH]     Percent combined: 4.39%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.074 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "568 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA126/SA126_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA126_centroids.fasta --uc SA126.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 568 sequences and identified 417 clusters (clustering threshold 1.00) - 7 clusters (representing 105 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA128 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA128_1.fastq /home/working/demultiplexed/SA128_2.fastq SA128_forward.paired.fastq.gz SA128_forward.singletons.fastq.gz SA128_reverse.paired.fastq.gz SA128_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA128_1.fastq /home/working/demultiplexed/SA128_2.fastq SA128_forward.paired.fastq.gz SA128_forward.singletons.fastq.gz SA128_reverse.paired.fastq.gz SA128_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:00:49 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA128/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 10561 Both Surviving: 3860 (36.55%) Forward Only Surviving: 809 (7.66%) Reverse Only Surviving: 188 (1.78%) Dropped: 5704 (54.01%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SA128_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA128_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA128_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA128_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA128_forward.paired.headclipped.fastq.gz SA128_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA128 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA128_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA128_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA128.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA128.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA128.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA128.hist\n",
      "[FLASH]     ./SA128.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3860 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3860\n",
      "[FLASH]     Combined pairs:   2075\n",
      "[FLASH]     Uncombined pairs: 1785\n",
      "[FLASH]     Percent combined: 53.76%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.060 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2075 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA128/SA128_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA128_centroids.fasta --uc SA128.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2075 sequences and identified 1191 clusters (clustering threshold 1.00) - 23 clusters (representing 769 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA13 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA13_1.fastq /home/working/demultiplexed/SA13_2.fastq SA13_forward.paired.fastq.gz SA13_forward.singletons.fastq.gz SA13_reverse.paired.fastq.gz SA13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA13_1.fastq /home/working/demultiplexed/SA13_2.fastq SA13_forward.paired.fastq.gz SA13_forward.singletons.fastq.gz SA13_reverse.paired.fastq.gz SA13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:00:55 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA13/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 31253 Both Surviving: 10666 (34.13%) Forward Only Surviving: 2268 (7.26%) Reverse Only Surviving: 484 (1.55%) Dropped: 17835 (57.07%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SA13_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA13_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA13_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA13_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA13_forward.paired.headclipped.fastq.gz SA13_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA13 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA13_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA13_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA13.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA13.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA13.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA13.hist\n",
      "[FLASH]     ./SA13.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 10666 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      10666\n",
      "[FLASH]     Combined pairs:   6139\n",
      "[FLASH]     Uncombined pairs: 4527\n",
      "[FLASH]     Percent combined: 57.56%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.172 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "6139 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA13/SA13_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA13_centroids.fasta --uc SA13.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 6139 sequences and identified 3472 clusters (clustering threshold 1.00) - 98 clusters (representing 2365 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA130 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA130_1.fastq /home/working/demultiplexed/SA130_2.fastq SA130_forward.paired.fastq.gz SA130_forward.singletons.fastq.gz SA130_reverse.paired.fastq.gz SA130_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA130_1.fastq /home/working/demultiplexed/SA130_2.fastq SA130_forward.paired.fastq.gz SA130_forward.singletons.fastq.gz SA130_reverse.paired.fastq.gz SA130_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:01:08 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA130/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 20540 Both Surviving: 3652 (17.78%) Forward Only Surviving: 800 (3.89%) Reverse Only Surviving: 646 (3.15%) Dropped: 15442 (75.18%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: SA130_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA130_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA130_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA130_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA130_forward.paired.headclipped.fastq.gz SA130_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA130 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA130_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA130_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA130.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA130.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA130.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA130.hist\n",
      "[FLASH]     ./SA130.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3652 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3652\n",
      "[FLASH]     Combined pairs:   514\n",
      "[FLASH]     Uncombined pairs: 3138\n",
      "[FLASH]     Percent combined: 14.07%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.025 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "514 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA130/SA130_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA130_centroids.fasta --uc SA130.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 514 sequences and identified 284 clusters (clustering threshold 1.00) - 7 clusters (representing 218 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA132 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA132_1.fastq /home/working/demultiplexed/SA132_2.fastq SA132_forward.paired.fastq.gz SA132_forward.singletons.fastq.gz SA132_reverse.paired.fastq.gz SA132_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA132_1.fastq /home/working/demultiplexed/SA132_2.fastq SA132_forward.paired.fastq.gz SA132_forward.singletons.fastq.gz SA132_reverse.paired.fastq.gz SA132_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:01:12 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA132/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 5677 Both Surviving: 1186 (20.89%) Forward Only Surviving: 263 (4.63%) Reverse Only Surviving: 195 (3.43%) Dropped: 4033 (71.04%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: SA132_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA132_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA132_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA132_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA132_forward.paired.headclipped.fastq.gz SA132_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA132 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA132_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA132_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA132.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA132.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA132.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA132.hist\n",
      "[FLASH]     ./SA132.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1186 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1186\n",
      "[FLASH]     Combined pairs:   455\n",
      "[FLASH]     Uncombined pairs: 731\n",
      "[FLASH]     Percent combined: 38.36%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.014 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "455 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA132/SA132_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA132_centroids.fasta --uc SA132.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 455 sequences and identified 274 clusters (clustering threshold 1.00) - 7 clusters (representing 159 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA138 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA138_1.fastq /home/working/demultiplexed/SA138_2.fastq SA138_forward.paired.fastq.gz SA138_forward.singletons.fastq.gz SA138_reverse.paired.fastq.gz SA138_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA138_1.fastq /home/working/demultiplexed/SA138_2.fastq SA138_forward.paired.fastq.gz SA138_forward.singletons.fastq.gz SA138_reverse.paired.fastq.gz SA138_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:01:14 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA138/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 16117 Both Surviving: 648 (4.02%) Forward Only Surviving: 393 (2.44%) Reverse Only Surviving: 272 (1.69%) Dropped: 14804 (91.85%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SA138_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA138_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA138_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA138_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA138_forward.paired.headclipped.fastq.gz SA138_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA138 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA138_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA138_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA138.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA138.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA138.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA138.hist\n",
      "[FLASH]     ./SA138.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 648 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      648\n",
      "[FLASH]     Combined pairs:   332\n",
      "[FLASH]     Uncombined pairs: 316\n",
      "[FLASH]     Percent combined: 51.23%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.010 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "332 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA138/SA138_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA138_centroids.fasta --uc SA138.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 332 sequences and identified 196 clusters (clustering threshold 1.00) - 7 clusters (representing 135 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA140 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA140_1.fastq /home/working/demultiplexed/SA140_2.fastq SA140_forward.paired.fastq.gz SA140_forward.singletons.fastq.gz SA140_reverse.paired.fastq.gz SA140_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA140_1.fastq /home/working/demultiplexed/SA140_2.fastq SA140_forward.paired.fastq.gz SA140_forward.singletons.fastq.gz SA140_reverse.paired.fastq.gz SA140_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:01:20 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA140/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 81551 Both Surviving: 8759 (10.74%) Forward Only Surviving: 2778 (3.41%) Reverse Only Surviving: 1121 (1.37%) Dropped: 68893 (84.48%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SA140_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SA140_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA140_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA140_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA140_forward.paired.headclipped.fastq.gz SA140_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA140 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA140_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA140_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA140.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA140.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA140.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA140.hist\n",
      "[FLASH]     ./SA140.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 8759 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      8759\n",
      "[FLASH]     Combined pairs:   242\n",
      "[FLASH]     Uncombined pairs: 8517\n",
      "[FLASH]     Percent combined: 2.76%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.050 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "242 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA140/SA140_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA140_centroids.fasta --uc SA140.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 242 sequences and identified 192 clusters (clustering threshold 1.00) - 3 clusters (representing 38 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA144 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA144_1.fastq /home/working/demultiplexed/SA144_2.fastq SA144_forward.paired.fastq.gz SA144_forward.singletons.fastq.gz SA144_reverse.paired.fastq.gz SA144_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA144_1.fastq /home/working/demultiplexed/SA144_2.fastq SA144_forward.paired.fastq.gz SA144_forward.singletons.fastq.gz SA144_reverse.paired.fastq.gz SA144_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:01:27 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA144/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 28135 Both Surviving: 7735 (27.49%) Forward Only Surviving: 2906 (10.33%) Reverse Only Surviving: 328 (1.17%) Dropped: 17166 (61.01%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SA144_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SA144_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA144_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA144_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA144_forward.paired.headclipped.fastq.gz SA144_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA144 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA144_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA144_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA144.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA144.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA144.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA144.hist\n",
      "[FLASH]     ./SA144.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 7735 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      7735\n",
      "[FLASH]     Combined pairs:   6523\n",
      "[FLASH]     Uncombined pairs: 1212\n",
      "[FLASH]     Percent combined: 84.33%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.181 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "6523 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA144/SA144_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA144_centroids.fasta --uc SA144.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 6523 sequences and identified 3140 clusters (clustering threshold 1.00) - 131 clusters (representing 3022 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA147 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA147_1.fastq /home/working/demultiplexed/SA147_2.fastq SA147_forward.paired.fastq.gz SA147_forward.singletons.fastq.gz SA147_reverse.paired.fastq.gz SA147_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA147_1.fastq /home/working/demultiplexed/SA147_2.fastq SA147_forward.paired.fastq.gz SA147_forward.singletons.fastq.gz SA147_reverse.paired.fastq.gz SA147_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:01:42 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA147/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 52843 Both Surviving: 9208 (17.43%) Forward Only Surviving: 2868 (5.43%) Reverse Only Surviving: 630 (1.19%) Dropped: 40137 (75.96%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA147_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA147_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA147_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA147_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA147_forward.paired.headclipped.fastq.gz SA147_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA147 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA147_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA147_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA147.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA147.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA147.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA147.hist\n",
      "[FLASH]     ./SA147.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 9208 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      9208\n",
      "[FLASH]     Combined pairs:   3170\n",
      "[FLASH]     Uncombined pairs: 6038\n",
      "[FLASH]     Percent combined: 34.43%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.108 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "3170 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA147/SA147_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA147_centroids.fasta --uc SA147.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 3170 sequences and identified 1383 clusters (clustering threshold 1.00) - 67 clusters (representing 1555 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA15 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA15_1.fastq /home/working/demultiplexed/SA15_2.fastq SA15_forward.paired.fastq.gz SA15_forward.singletons.fastq.gz SA15_reverse.paired.fastq.gz SA15_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA15_1.fastq /home/working/demultiplexed/SA15_2.fastq SA15_forward.paired.fastq.gz SA15_forward.singletons.fastq.gz SA15_reverse.paired.fastq.gz SA15_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:01:55 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA15/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 86082 Both Surviving: 26672 (30.98%) Forward Only Surviving: 6078 (7.06%) Reverse Only Surviving: 1725 (2.00%) Dropped: 51607 (59.95%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA15_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA15_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA15_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA15_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA15_forward.paired.headclipped.fastq.gz SA15_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA15 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA15_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA15_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA15.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA15.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA15.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA15.hist\n",
      "[FLASH]     ./SA15.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 25000 read pairs\n",
      "[FLASH] Processed 26672 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      26672\n",
      "[FLASH]     Combined pairs:   889\n",
      "[FLASH]     Uncombined pairs: 25783\n",
      "[FLASH]     Percent combined: 3.33%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.153 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "889 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA15/SA15_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA15_centroids.fasta --uc SA15.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 889 sequences and identified 586 clusters (clustering threshold 1.00) - 20 clusters (representing 252 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA153 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA153_1.fastq /home/working/demultiplexed/SA153_2.fastq SA153_forward.paired.fastq.gz SA153_forward.singletons.fastq.gz SA153_reverse.paired.fastq.gz SA153_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA153_1.fastq /home/working/demultiplexed/SA153_2.fastq SA153_forward.paired.fastq.gz SA153_forward.singletons.fastq.gz SA153_reverse.paired.fastq.gz SA153_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:02:10 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA153/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 54014 Both Surviving: 11939 (22.10%) Forward Only Surviving: 2234 (4.14%) Reverse Only Surviving: 992 (1.84%) Dropped: 38849 (71.92%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SA153_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA153_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA153_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA153_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA153_forward.paired.headclipped.fastq.gz SA153_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA153 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA153_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA153_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA153.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA153.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA153.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA153.hist\n",
      "[FLASH]     ./SA153.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 11939 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      11939\n",
      "[FLASH]     Combined pairs:   1679\n",
      "[FLASH]     Uncombined pairs: 10260\n",
      "[FLASH]     Percent combined: 14.06%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.072 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1679 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA153/SA153_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA153_centroids.fasta --uc SA153.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1679 sequences and identified 1046 clusters (clustering threshold 1.00) - 29 clusters (representing 564 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA155 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA155_1.fastq /home/working/demultiplexed/SA155_2.fastq SA155_forward.paired.fastq.gz SA155_forward.singletons.fastq.gz SA155_reverse.paired.fastq.gz SA155_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA155_1.fastq /home/working/demultiplexed/SA155_2.fastq SA155_forward.paired.fastq.gz SA155_forward.singletons.fastq.gz SA155_reverse.paired.fastq.gz SA155_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:02:20 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA155/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 31128 Both Surviving: 752 (2.42%) Forward Only Surviving: 717 (2.30%) Reverse Only Surviving: 480 (1.54%) Dropped: 29179 (93.74%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SA155_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA155_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA155_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA155_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA155_forward.paired.headclipped.fastq.gz SA155_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA155 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA155_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA155_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA155.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA155.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA155.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA155.hist\n",
      "[FLASH]     ./SA155.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 752 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      752\n",
      "[FLASH]     Combined pairs:   561\n",
      "[FLASH]     Uncombined pairs: 191\n",
      "[FLASH]     Percent combined: 74.60%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.023 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "561 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA155/SA155_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA155_centroids.fasta --uc SA155.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 561 sequences and identified 321 clusters (clustering threshold 1.00) - 8 clusters (representing 201 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA156 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA156_1.fastq /home/working/demultiplexed/SA156_2.fastq SA156_forward.paired.fastq.gz SA156_forward.singletons.fastq.gz SA156_reverse.paired.fastq.gz SA156_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA156_1.fastq /home/working/demultiplexed/SA156_2.fastq SA156_forward.paired.fastq.gz SA156_forward.singletons.fastq.gz SA156_reverse.paired.fastq.gz SA156_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:02:25 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA156/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 53417 Both Surviving: 2365 (4.43%) Forward Only Surviving: 1836 (3.44%) Reverse Only Surviving: 713 (1.33%) Dropped: 48503 (90.80%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SA156_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SA156_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA156_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA156_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA156_forward.paired.headclipped.fastq.gz SA156_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA156 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA156_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA156_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA156.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA156.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA156.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA156.hist\n",
      "[FLASH]     ./SA156.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2365 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2365\n",
      "[FLASH]     Combined pairs:   312\n",
      "[FLASH]     Uncombined pairs: 2053\n",
      "[FLASH]     Percent combined: 13.19%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.021 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "312 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA156/SA156_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA156_centroids.fasta --uc SA156.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 312 sequences and identified 258 clusters (clustering threshold 1.00) - 3 clusters (representing 21 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA157 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA157_1.fastq /home/working/demultiplexed/SA157_2.fastq SA157_forward.paired.fastq.gz SA157_forward.singletons.fastq.gz SA157_reverse.paired.fastq.gz SA157_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA157_1.fastq /home/working/demultiplexed/SA157_2.fastq SA157_forward.paired.fastq.gz SA157_forward.singletons.fastq.gz SA157_reverse.paired.fastq.gz SA157_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:02:31 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA157/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 37950 Both Surviving: 6388 (16.83%) Forward Only Surviving: 1776 (4.68%) Reverse Only Surviving: 666 (1.75%) Dropped: 29120 (76.73%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SA157_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA157_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA157_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA157_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA157_forward.paired.headclipped.fastq.gz SA157_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA157 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA157_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA157_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA157.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA157.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA157.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA157.hist\n",
      "[FLASH]     ./SA157.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 6388 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      6388\n",
      "[FLASH]     Combined pairs:   41\n",
      "[FLASH]     Uncombined pairs: 6347\n",
      "[FLASH]     Percent combined: 0.64%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.040 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "41 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA157/SA157_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA157_centroids.fasta --uc SA157.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 41 sequences and identified 35 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA160 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA160_1.fastq /home/working/demultiplexed/SA160_2.fastq SA160_forward.paired.fastq.gz SA160_forward.singletons.fastq.gz SA160_reverse.paired.fastq.gz SA160_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA160_1.fastq /home/working/demultiplexed/SA160_2.fastq SA160_forward.paired.fastq.gz SA160_forward.singletons.fastq.gz SA160_reverse.paired.fastq.gz SA160_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:02:36 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA160/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 39762 Both Surviving: 6875 (17.29%) Forward Only Surviving: 1725 (4.34%) Reverse Only Surviving: 618 (1.55%) Dropped: 30544 (76.82%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SA160_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SA160_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA160_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA160_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA160_forward.paired.headclipped.fastq.gz SA160_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA160 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA160_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA160_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA160.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA160.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA160.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA160.hist\n",
      "[FLASH]     ./SA160.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 6875 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      6875\n",
      "[FLASH]     Combined pairs:   758\n",
      "[FLASH]     Uncombined pairs: 6117\n",
      "[FLASH]     Percent combined: 11.03%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.041 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "758 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA160/SA160_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA160_centroids.fasta --uc SA160.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 758 sequences and identified 367 clusters (clustering threshold 1.00) - 14 clusters (representing 367 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA162 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA162_1.fastq /home/working/demultiplexed/SA162_2.fastq SA162_forward.paired.fastq.gz SA162_forward.singletons.fastq.gz SA162_reverse.paired.fastq.gz SA162_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA162_1.fastq /home/working/demultiplexed/SA162_2.fastq SA162_forward.paired.fastq.gz SA162_forward.singletons.fastq.gz SA162_reverse.paired.fastq.gz SA162_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:02:43 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA162/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 37273 Both Surviving: 11447 (30.71%) Forward Only Surviving: 3157 (8.47%) Reverse Only Surviving: 439 (1.18%) Dropped: 22230 (59.64%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SA162_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA162_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA162_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA162_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA162_forward.paired.headclipped.fastq.gz SA162_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA162 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA162_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA162_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA162.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA162.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA162.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA162.hist\n",
      "[FLASH]     ./SA162.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 11447 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      11447\n",
      "[FLASH]     Combined pairs:   5798\n",
      "[FLASH]     Uncombined pairs: 5649\n",
      "[FLASH]     Percent combined: 50.65%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.164 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "5798 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA162/SA162_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA162_centroids.fasta --uc SA162.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 5798 sequences and identified 2921 clusters (clustering threshold 1.00) - 79 clusters (representing 2543 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA164 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA164_1.fastq /home/working/demultiplexed/SA164_2.fastq SA164_forward.paired.fastq.gz SA164_forward.singletons.fastq.gz SA164_reverse.paired.fastq.gz SA164_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA164_1.fastq /home/working/demultiplexed/SA164_2.fastq SA164_forward.paired.fastq.gz SA164_forward.singletons.fastq.gz SA164_reverse.paired.fastq.gz SA164_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:02:56 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA164/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 22 bases off reads in file: SA164_forward.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA164_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA164_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA164_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA164_forward.paired.headclipped.fastq.gz SA164_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA164 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA164_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA164_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA164.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA164.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA164.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA164.hist\n",
      "[FLASH]     ./SA164.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      0\n",
      "[FLASH]     Combined pairs:   0\n",
      "[FLASH]     Uncombined pairs: 0\n",
      "[FLASH]     Percent combined: 0.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "0 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA164/SA164_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA164_centroids.fasta --uc SA164.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA165 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA165_1.fastq /home/working/demultiplexed/SA165_2.fastq SA165_forward.paired.fastq.gz SA165_forward.singletons.fastq.gz SA165_reverse.paired.fastq.gz SA165_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA165_1.fastq /home/working/demultiplexed/SA165_2.fastq SA165_forward.paired.fastq.gz SA165_forward.singletons.fastq.gz SA165_reverse.paired.fastq.gz SA165_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:02:58 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA165/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 31845 Both Surviving: 9829 (30.87%) Forward Only Surviving: 2198 (6.90%) Reverse Only Surviving: 609 (1.91%) Dropped: 19209 (60.32%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SA165_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA165_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA165_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA165_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA165_forward.paired.headclipped.fastq.gz SA165_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA165 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA165_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA165_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA165.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA165.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA165.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA165.hist\n",
      "[FLASH]     ./SA165.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 9829 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      9829\n",
      "[FLASH]     Combined pairs:   4629\n",
      "[FLASH]     Uncombined pairs: 5200\n",
      "[FLASH]     Percent combined: 47.10%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.144 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "4629 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA165/SA165_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA165_centroids.fasta --uc SA165.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 4629 sequences and identified 2550 clusters (clustering threshold 1.00) - 59 clusters (representing 1820 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA166 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA166_1.fastq /home/working/demultiplexed/SA166_2.fastq SA166_forward.paired.fastq.gz SA166_forward.singletons.fastq.gz SA166_reverse.paired.fastq.gz SA166_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA166_1.fastq /home/working/demultiplexed/SA166_2.fastq SA166_forward.paired.fastq.gz SA166_forward.singletons.fastq.gz SA166_reverse.paired.fastq.gz SA166_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:03:09 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA166/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 22760 Both Surviving: 9143 (40.17%) Forward Only Surviving: 1993 (8.76%) Reverse Only Surviving: 322 (1.41%) Dropped: 11302 (49.66%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA166_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA166_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA166_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA166_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA166_forward.paired.headclipped.fastq.gz SA166_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA166 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA166_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA166_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA166.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA166.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA166.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA166.hist\n",
      "[FLASH]     ./SA166.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 9143 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      9143\n",
      "[FLASH]     Combined pairs:   4628\n",
      "[FLASH]     Uncombined pairs: 4515\n",
      "[FLASH]     Percent combined: 50.62%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.140 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "4628 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA166/SA166_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA166_centroids.fasta --uc SA166.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 4628 sequences and identified 2605 clusters (clustering threshold 1.00) - 110 clusters (representing 1757 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA171 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA171_1.fastq /home/working/demultiplexed/SA171_2.fastq SA171_forward.paired.fastq.gz SA171_forward.singletons.fastq.gz SA171_reverse.paired.fastq.gz SA171_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA171_1.fastq /home/working/demultiplexed/SA171_2.fastq SA171_forward.paired.fastq.gz SA171_forward.singletons.fastq.gz SA171_reverse.paired.fastq.gz SA171_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:03:19 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA171/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 7014 Both Surviving: 2316 (33.02%) Forward Only Surviving: 562 (8.01%) Reverse Only Surviving: 193 (2.75%) Dropped: 3943 (56.22%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: SA171_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA171_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA171_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA171_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA171_forward.paired.headclipped.fastq.gz SA171_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA171 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA171_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA171_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA171.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA171.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA171.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA171.hist\n",
      "[FLASH]     ./SA171.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2316 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2316\n",
      "[FLASH]     Combined pairs:   456\n",
      "[FLASH]     Uncombined pairs: 1860\n",
      "[FLASH]     Percent combined: 19.69%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.023 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "456 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA171/SA171_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA171_centroids.fasta --uc SA171.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 456 sequences and identified 336 clusters (clustering threshold 1.00) - 8 clusters (representing 110 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA172 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA172_1.fastq /home/working/demultiplexed/SA172_2.fastq SA172_forward.paired.fastq.gz SA172_forward.singletons.fastq.gz SA172_reverse.paired.fastq.gz SA172_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA172_1.fastq /home/working/demultiplexed/SA172_2.fastq SA172_forward.paired.fastq.gz SA172_forward.singletons.fastq.gz SA172_reverse.paired.fastq.gz SA172_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:03:23 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA172/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 29978 Both Surviving: 9181 (30.63%) Forward Only Surviving: 3718 (12.40%) Reverse Only Surviving: 329 (1.10%) Dropped: 16750 (55.87%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: SA172_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: SA172_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA172_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA172_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA172_forward.paired.headclipped.fastq.gz SA172_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA172 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA172_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA172_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA172.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA172.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA172.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA172.hist\n",
      "[FLASH]     ./SA172.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 9181 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      9181\n",
      "[FLASH]     Combined pairs:   4464\n",
      "[FLASH]     Uncombined pairs: 4717\n",
      "[FLASH]     Percent combined: 48.62%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.128 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "4464 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA172/SA172_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA172_centroids.fasta --uc SA172.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 4464 sequences and identified 1339 clusters (clustering threshold 1.00) - 127 clusters (representing 2965 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA174 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA174_1.fastq /home/working/demultiplexed/SA174_2.fastq SA174_forward.paired.fastq.gz SA174_forward.singletons.fastq.gz SA174_reverse.paired.fastq.gz SA174_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA174_1.fastq /home/working/demultiplexed/SA174_2.fastq SA174_forward.paired.fastq.gz SA174_forward.singletons.fastq.gz SA174_reverse.paired.fastq.gz SA174_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:03:33 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA174/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 24531 Both Surviving: 7385 (30.10%) Forward Only Surviving: 1759 (7.17%) Reverse Only Surviving: 279 (1.14%) Dropped: 15108 (61.59%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SA174_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA174_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA174_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA174_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA174_forward.paired.headclipped.fastq.gz SA174_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA174 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA174_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA174_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA174.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA174.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA174.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA174.hist\n",
      "[FLASH]     ./SA174.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 7385 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      7385\n",
      "[FLASH]     Combined pairs:   2705\n",
      "[FLASH]     Uncombined pairs: 4680\n",
      "[FLASH]     Percent combined: 36.63%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.082 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2705 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA174/SA174_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA174_centroids.fasta --uc SA174.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2705 sequences and identified 1744 clusters (clustering threshold 1.00) - 31 clusters (representing 808 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA177 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA177_1.fastq /home/working/demultiplexed/SA177_2.fastq SA177_forward.paired.fastq.gz SA177_forward.singletons.fastq.gz SA177_reverse.paired.fastq.gz SA177_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA177_1.fastq /home/working/demultiplexed/SA177_2.fastq SA177_forward.paired.fastq.gz SA177_forward.singletons.fastq.gz SA177_reverse.paired.fastq.gz SA177_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:03:41 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA177/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 18782 Both Surviving: 385 (2.05%) Forward Only Surviving: 385 (2.05%) Reverse Only Surviving: 127 (0.68%) Dropped: 17885 (95.22%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SA177_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SA177_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA177_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA177_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA177_forward.paired.headclipped.fastq.gz SA177_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA177 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA177_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA177_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA177.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA177.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA177.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA177.hist\n",
      "[FLASH]     ./SA177.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 385 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      385\n",
      "[FLASH]     Combined pairs:   258\n",
      "[FLASH]     Uncombined pairs: 127\n",
      "[FLASH]     Percent combined: 67.01%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.008 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "258 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA177/SA177_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA177_centroids.fasta --uc SA177.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 258 sequences and identified 176 clusters (clustering threshold 1.00) - 3 clusters (representing 63 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA180 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA180_1.fastq /home/working/demultiplexed/SA180_2.fastq SA180_forward.paired.fastq.gz SA180_forward.singletons.fastq.gz SA180_reverse.paired.fastq.gz SA180_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA180_1.fastq /home/working/demultiplexed/SA180_2.fastq SA180_forward.paired.fastq.gz SA180_forward.singletons.fastq.gz SA180_reverse.paired.fastq.gz SA180_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:03:43 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA180/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 15643 Both Surviving: 717 (4.58%) Forward Only Surviving: 310 (1.98%) Reverse Only Surviving: 175 (1.12%) Dropped: 14441 (92.32%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SA180_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA180_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA180_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA180_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA180_forward.paired.headclipped.fastq.gz SA180_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA180 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA180_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA180_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA180.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA180.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA180.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA180.hist\n",
      "[FLASH]     ./SA180.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 717 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      717\n",
      "[FLASH]     Combined pairs:   223\n",
      "[FLASH]     Uncombined pairs: 494\n",
      "[FLASH]     Percent combined: 31.10%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.007 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "223 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA180/SA180_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA180_centroids.fasta --uc SA180.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 223 sequences and identified 142 clusters (clustering threshold 1.00) - 2 clusters (representing 70 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA182 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA182_1.fastq /home/working/demultiplexed/SA182_2.fastq SA182_forward.paired.fastq.gz SA182_forward.singletons.fastq.gz SA182_reverse.paired.fastq.gz SA182_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA182_1.fastq /home/working/demultiplexed/SA182_2.fastq SA182_forward.paired.fastq.gz SA182_forward.singletons.fastq.gz SA182_reverse.paired.fastq.gz SA182_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:03:45 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA182/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 14801 Both Surviving: 2276 (15.38%) Forward Only Surviving: 487 (3.29%) Reverse Only Surviving: 441 (2.98%) Dropped: 11597 (78.35%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: SA182_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA182_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA182_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA182_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA182_forward.paired.headclipped.fastq.gz SA182_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA182 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA182_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA182_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA182.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA182.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA182.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA182.hist\n",
      "[FLASH]     ./SA182.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2276 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2276\n",
      "[FLASH]     Combined pairs:   1194\n",
      "[FLASH]     Uncombined pairs: 1082\n",
      "[FLASH]     Percent combined: 52.46%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.033 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1194 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA182/SA182_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA182_centroids.fasta --uc SA182.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1194 sequences and identified 669 clusters (clustering threshold 1.00) - 16 clusters (representing 465 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA189 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA189_1.fastq /home/working/demultiplexed/SA189_2.fastq SA189_forward.paired.fastq.gz SA189_forward.singletons.fastq.gz SA189_reverse.paired.fastq.gz SA189_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA189_1.fastq /home/working/demultiplexed/SA189_2.fastq SA189_forward.paired.fastq.gz SA189_forward.singletons.fastq.gz SA189_reverse.paired.fastq.gz SA189_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:03:51 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA189/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 40281 Both Surviving: 6626 (16.45%) Forward Only Surviving: 1736 (4.31%) Reverse Only Surviving: 589 (1.46%) Dropped: 31330 (77.78%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: SA189_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: SA189_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA189_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA189_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA189_forward.paired.headclipped.fastq.gz SA189_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA189 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA189_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA189_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA189.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA189.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA189.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA189.hist\n",
      "[FLASH]     ./SA189.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 6626 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      6626\n",
      "[FLASH]     Combined pairs:   1042\n",
      "[FLASH]     Uncombined pairs: 5584\n",
      "[FLASH]     Percent combined: 15.73%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.042 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1042 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA189/SA189_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA189_centroids.fasta --uc SA189.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1042 sequences and identified 562 clusters (clustering threshold 1.00) - 14 clusters (representing 431 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA19 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA19_1.fastq /home/working/demultiplexed/SA19_2.fastq SA19_forward.paired.fastq.gz SA19_forward.singletons.fastq.gz SA19_reverse.paired.fastq.gz SA19_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA19_1.fastq /home/working/demultiplexed/SA19_2.fastq SA19_forward.paired.fastq.gz SA19_forward.singletons.fastq.gz SA19_reverse.paired.fastq.gz SA19_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:04:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA19/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 97463 Both Surviving: 21686 (22.25%) Forward Only Surviving: 6419 (6.59%) Reverse Only Surviving: 1547 (1.59%) Dropped: 67811 (69.58%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA19_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA19_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA19_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA19_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA19_forward.paired.headclipped.fastq.gz SA19_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA19 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA19_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA19_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA19.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA19.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA19.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA19.hist\n",
      "[FLASH]     ./SA19.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 21686 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      21686\n",
      "[FLASH]     Combined pairs:   1454\n",
      "[FLASH]     Uncombined pairs: 20232\n",
      "[FLASH]     Percent combined: 6.70%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.124 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1454 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA19/SA19_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA19_centroids.fasta --uc SA19.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1454 sequences and identified 925 clusters (clustering threshold 1.00) - 33 clusters (representing 447 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA190 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA190_1.fastq /home/working/demultiplexed/SA190_2.fastq SA190_forward.paired.fastq.gz SA190_forward.singletons.fastq.gz SA190_reverse.paired.fastq.gz SA190_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA190_1.fastq /home/working/demultiplexed/SA190_2.fastq SA190_forward.paired.fastq.gz SA190_forward.singletons.fastq.gz SA190_reverse.paired.fastq.gz SA190_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:04:15 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA190/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 57953 Both Surviving: 708 (1.22%) Forward Only Surviving: 1425 (2.46%) Reverse Only Surviving: 656 (1.13%) Dropped: 55164 (95.19%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SA190_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SA190_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA190_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA190_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA190_forward.paired.headclipped.fastq.gz SA190_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA190 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA190_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA190_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA190.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA190.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA190.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA190.hist\n",
      "[FLASH]     ./SA190.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 708 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      708\n",
      "[FLASH]     Combined pairs:   189\n",
      "[FLASH]     Uncombined pairs: 519\n",
      "[FLASH]     Percent combined: 26.69%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.010 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "189 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA190/SA190_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA190_centroids.fasta --uc SA190.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 189 sequences and identified 153 clusters (clustering threshold 1.00) - 1 clusters (representing 8 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA196 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA196_1.fastq /home/working/demultiplexed/SA196_2.fastq SA196_forward.paired.fastq.gz SA196_forward.singletons.fastq.gz SA196_reverse.paired.fastq.gz SA196_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA196_1.fastq /home/working/demultiplexed/SA196_2.fastq SA196_forward.paired.fastq.gz SA196_forward.singletons.fastq.gz SA196_reverse.paired.fastq.gz SA196_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:04:20 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA196/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 30293 Both Surviving: 431 (1.42%) Forward Only Surviving: 604 (1.99%) Reverse Only Surviving: 436 (1.44%) Dropped: 28822 (95.14%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: SA196_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: SA196_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA196_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA196_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA196_forward.paired.headclipped.fastq.gz SA196_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA196 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA196_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA196_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA196.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA196.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA196.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA196.hist\n",
      "[FLASH]     ./SA196.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 431 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      431\n",
      "[FLASH]     Combined pairs:   304\n",
      "[FLASH]     Uncombined pairs: 127\n",
      "[FLASH]     Percent combined: 70.53%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.016 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "304 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA196/SA196_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA196_centroids.fasta --uc SA196.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 304 sequences and identified 156 clusters (clustering threshold 1.00) - 6 clusters (representing 143 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA198 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA198_1.fastq /home/working/demultiplexed/SA198_2.fastq SA198_forward.paired.fastq.gz SA198_forward.singletons.fastq.gz SA198_reverse.paired.fastq.gz SA198_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA198_1.fastq /home/working/demultiplexed/SA198_2.fastq SA198_forward.paired.fastq.gz SA198_forward.singletons.fastq.gz SA198_reverse.paired.fastq.gz SA198_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:04:23 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA198/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 14709 Both Surviving: 3771 (25.64%) Forward Only Surviving: 1102 (7.49%) Reverse Only Surviving: 201 (1.37%) Dropped: 9635 (65.50%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA198_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA198_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA198_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA198_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA198_forward.paired.headclipped.fastq.gz SA198_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA198 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA198_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA198_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA198.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA198.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA198.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA198.hist\n",
      "[FLASH]     ./SA198.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3771 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3771\n",
      "[FLASH]     Combined pairs:   660\n",
      "[FLASH]     Uncombined pairs: 3111\n",
      "[FLASH]     Percent combined: 17.50%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.027 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "660 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA198/SA198_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA198_centroids.fasta --uc SA198.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 660 sequences and identified 452 clusters (clustering threshold 1.00) - 12 clusters (representing 189 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA199 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA199_1.fastq /home/working/demultiplexed/SA199_2.fastq SA199_forward.paired.fastq.gz SA199_forward.singletons.fastq.gz SA199_reverse.paired.fastq.gz SA199_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA199_1.fastq /home/working/demultiplexed/SA199_2.fastq SA199_forward.paired.fastq.gz SA199_forward.singletons.fastq.gz SA199_reverse.paired.fastq.gz SA199_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:04:26 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA199/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 1 Both Surviving: 0 (0.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 1 (100.00%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 22 bases off reads in file: SA199_forward.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA199_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA199_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA199_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA199_forward.paired.headclipped.fastq.gz SA199_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA199 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA199_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA199_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA199.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA199.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA199.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA199.hist\n",
      "[FLASH]     ./SA199.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      0\n",
      "[FLASH]     Combined pairs:   0\n",
      "[FLASH]     Uncombined pairs: 0\n",
      "[FLASH]     Percent combined: 0.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "0 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA199/SA199_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA199_centroids.fasta --uc SA199.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA2 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA2_1.fastq /home/working/demultiplexed/SA2_2.fastq SA2_forward.paired.fastq.gz SA2_forward.singletons.fastq.gz SA2_reverse.paired.fastq.gz SA2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA2_1.fastq /home/working/demultiplexed/SA2_2.fastq SA2_forward.paired.fastq.gz SA2_forward.singletons.fastq.gz SA2_reverse.paired.fastq.gz SA2_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:04:32 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA2/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 96310 Both Surviving: 30413 (31.58%) Forward Only Surviving: 7502 (7.79%) Reverse Only Surviving: 1434 (1.49%) Dropped: 56961 (59.14%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA2_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA2_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA2_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA2_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA2_forward.paired.headclipped.fastq.gz SA2_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA2 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA2_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA2_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA2.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA2.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA2.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA2.hist\n",
      "[FLASH]     ./SA2.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 25000 read pairs\n",
      "[FLASH] Processed 30413 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      30413\n",
      "[FLASH]     Combined pairs:   18029\n",
      "[FLASH]     Uncombined pairs: 12384\n",
      "[FLASH]     Percent combined: 59.28%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.478 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "18029 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA2/SA2_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA2_centroids.fasta --uc SA2.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 18029 sequences and identified 5296 clusters (clustering threshold 1.00) - 434 clusters (representing 12156 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA202 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA202_1.fastq /home/working/demultiplexed/SA202_2.fastq SA202_forward.paired.fastq.gz SA202_forward.singletons.fastq.gz SA202_reverse.paired.fastq.gz SA202_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA202_1.fastq /home/working/demultiplexed/SA202_2.fastq SA202_forward.paired.fastq.gz SA202_forward.singletons.fastq.gz SA202_reverse.paired.fastq.gz SA202_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:05:12 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA202/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 102107 Both Surviving: 26482 (25.94%) Forward Only Surviving: 7417 (7.26%) Reverse Only Surviving: 1638 (1.60%) Dropped: 66570 (65.20%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA202_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA202_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA202_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA202_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA202_forward.paired.headclipped.fastq.gz SA202_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA202 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA202_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA202_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA202.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA202.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA202.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA202.hist\n",
      "[FLASH]     ./SA202.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 25000 read pairs\n",
      "[FLASH] Processed 26482 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      26482\n",
      "[FLASH]     Combined pairs:   2971\n",
      "[FLASH]     Uncombined pairs: 23511\n",
      "[FLASH]     Percent combined: 11.22%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.207 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2971 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA202/SA202_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA202_centroids.fasta --uc SA202.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2971 sequences and identified 1577 clusters (clustering threshold 1.00) - 41 clusters (representing 1228 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA21 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA21_1.fastq /home/working/demultiplexed/SA21_2.fastq SA21_forward.paired.fastq.gz SA21_forward.singletons.fastq.gz SA21_reverse.paired.fastq.gz SA21_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA21_1.fastq /home/working/demultiplexed/SA21_2.fastq SA21_forward.paired.fastq.gz SA21_forward.singletons.fastq.gz SA21_reverse.paired.fastq.gz SA21_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:05:31 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA21/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 37070 Both Surviving: 5135 (13.85%) Forward Only Surviving: 1558 (4.20%) Reverse Only Surviving: 623 (1.68%) Dropped: 29754 (80.26%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SA21_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA21_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA21_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA21_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA21_forward.paired.headclipped.fastq.gz SA21_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA21 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA21_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA21_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA21.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA21.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA21.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA21.hist\n",
      "[FLASH]     ./SA21.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5135 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5135\n",
      "[FLASH]     Combined pairs:   56\n",
      "[FLASH]     Uncombined pairs: 5079\n",
      "[FLASH]     Percent combined: 1.09%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.049 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "56 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA21/SA21_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA21_centroids.fasta --uc SA21.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 56 sequences and identified 49 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA28 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA28_1.fastq /home/working/demultiplexed/SA28_2.fastq SA28_forward.paired.fastq.gz SA28_forward.singletons.fastq.gz SA28_reverse.paired.fastq.gz SA28_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA28_1.fastq /home/working/demultiplexed/SA28_2.fastq SA28_forward.paired.fastq.gz SA28_forward.singletons.fastq.gz SA28_reverse.paired.fastq.gz SA28_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:05:38 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA28/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 55630 Both Surviving: 13372 (24.04%) Forward Only Surviving: 4577 (8.23%) Reverse Only Surviving: 769 (1.38%) Dropped: 36912 (66.35%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA28_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA28_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA28_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA28_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA28_forward.paired.headclipped.fastq.gz SA28_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA28 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA28_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA28_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA28.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA28.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA28.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA28.hist\n",
      "[FLASH]     ./SA28.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 13372 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      13372\n",
      "[FLASH]     Combined pairs:   2894\n",
      "[FLASH]     Uncombined pairs: 10478\n",
      "[FLASH]     Percent combined: 21.64%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.123 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2894 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA28/SA28_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA28_centroids.fasta --uc SA28.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2894 sequences and identified 1162 clusters (clustering threshold 1.00) - 33 clusters (representing 1432 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA31 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA31_1.fastq /home/working/demultiplexed/SA31_2.fastq SA31_forward.paired.fastq.gz SA31_forward.singletons.fastq.gz SA31_reverse.paired.fastq.gz SA31_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA31_1.fastq /home/working/demultiplexed/SA31_2.fastq SA31_forward.paired.fastq.gz SA31_forward.singletons.fastq.gz SA31_reverse.paired.fastq.gz SA31_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:05:50 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA31/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 33983 Both Surviving: 11512 (33.88%) Forward Only Surviving: 2671 (7.86%) Reverse Only Surviving: 509 (1.50%) Dropped: 19291 (56.77%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA31_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA31_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA31_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA31_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA31_forward.paired.headclipped.fastq.gz SA31_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA31 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA31_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA31_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA31.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA31.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA31.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA31.hist\n",
      "[FLASH]     ./SA31.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 11512 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      11512\n",
      "[FLASH]     Combined pairs:   3713\n",
      "[FLASH]     Uncombined pairs: 7799\n",
      "[FLASH]     Percent combined: 32.25%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.160 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "3713 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA31/SA31_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA31_centroids.fasta --uc SA31.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 3713 sequences and identified 1957 clusters (clustering threshold 1.00) - 100 clusters (representing 1543 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA32 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA32_1.fastq /home/working/demultiplexed/SA32_2.fastq SA32_forward.paired.fastq.gz SA32_forward.singletons.fastq.gz SA32_reverse.paired.fastq.gz SA32_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA32_1.fastq /home/working/demultiplexed/SA32_2.fastq SA32_forward.paired.fastq.gz SA32_forward.singletons.fastq.gz SA32_reverse.paired.fastq.gz SA32_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:06:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA32/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 21785 Both Surviving: 2978 (13.67%) Forward Only Surviving: 887 (4.07%) Reverse Only Surviving: 276 (1.27%) Dropped: 17644 (80.99%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: SA32_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: SA32_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA32_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA32_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA32_forward.paired.headclipped.fastq.gz SA32_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA32 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA32_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA32_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA32.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA32.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA32.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA32.hist\n",
      "[FLASH]     ./SA32.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2978 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2978\n",
      "[FLASH]     Combined pairs:   1249\n",
      "[FLASH]     Uncombined pairs: 1729\n",
      "[FLASH]     Percent combined: 41.94%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.056 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1249 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA32/SA32_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA32_centroids.fasta --uc SA32.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1249 sequences and identified 682 clusters (clustering threshold 1.00) - 21 clusters (representing 527 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA33 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA33_1.fastq /home/working/demultiplexed/SA33_2.fastq SA33_forward.paired.fastq.gz SA33_forward.singletons.fastq.gz SA33_reverse.paired.fastq.gz SA33_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA33_1.fastq /home/working/demultiplexed/SA33_2.fastq SA33_forward.paired.fastq.gz SA33_forward.singletons.fastq.gz SA33_reverse.paired.fastq.gz SA33_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:06:11 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA33/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 98419 Both Surviving: 3584 (3.64%) Forward Only Surviving: 3007 (3.06%) Reverse Only Surviving: 995 (1.01%) Dropped: 90833 (92.29%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SA33_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SA33_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA33_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA33_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA33_forward.paired.headclipped.fastq.gz SA33_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA33 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA33_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA33_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA33.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA33.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA33.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA33.hist\n",
      "[FLASH]     ./SA33.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3584 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3584\n",
      "[FLASH]     Combined pairs:   2545\n",
      "[FLASH]     Uncombined pairs: 1039\n",
      "[FLASH]     Percent combined: 71.01%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.091 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2545 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA33/SA33_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA33_centroids.fasta --uc SA33.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2545 sequences and identified 1255 clusters (clustering threshold 1.00) - 41 clusters (representing 1163 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA36 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA36_1.fastq /home/working/demultiplexed/SA36_2.fastq SA36_forward.paired.fastq.gz SA36_forward.singletons.fastq.gz SA36_reverse.paired.fastq.gz SA36_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA36_1.fastq /home/working/demultiplexed/SA36_2.fastq SA36_forward.paired.fastq.gz SA36_forward.singletons.fastq.gz SA36_reverse.paired.fastq.gz SA36_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:06:24 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA36/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 51138 Both Surviving: 7411 (14.49%) Forward Only Surviving: 1797 (3.51%) Reverse Only Surviving: 663 (1.30%) Dropped: 41267 (80.70%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA36_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA36_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SA36_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SA36_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA36_forward.paired.headclipped.fastq.gz SA36_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA36 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA36_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA36_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA36.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA36.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA36.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA36.hist\n",
      "[FLASH]     ./SA36.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 7411 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      7411\n",
      "[FLASH]     Combined pairs:   200\n",
      "[FLASH]     Uncombined pairs: 7211\n",
      "[FLASH]     Percent combined: 2.70%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.087 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "200 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA36/SA36_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA36_centroids.fasta --uc SA36.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 200 sequences and identified 135 clusters (clustering threshold 1.00) - 6 clusters (representing 43 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA43 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA43_1.fastq /home/working/demultiplexed/SA43_2.fastq SA43_forward.paired.fastq.gz SA43_forward.singletons.fastq.gz SA43_reverse.paired.fastq.gz SA43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA43_1.fastq /home/working/demultiplexed/SA43_2.fastq SA43_forward.paired.fastq.gz SA43_forward.singletons.fastq.gz SA43_reverse.paired.fastq.gz SA43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:06:32 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA43/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 22712 Both Surviving: 5221 (22.99%) Forward Only Surviving: 1302 (5.73%) Reverse Only Surviving: 488 (2.15%) Dropped: 15701 (69.13%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SA43_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA43_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA43_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA43_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA43_forward.paired.headclipped.fastq.gz SA43_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA43 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA43_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA43_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA43.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA43.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA43.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA43.hist\n",
      "[FLASH]     ./SA43.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5221 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5221\n",
      "[FLASH]     Combined pairs:   57\n",
      "[FLASH]     Uncombined pairs: 5164\n",
      "[FLASH]     Percent combined: 1.09%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.088 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "57 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA43/SA43_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA43_centroids.fasta --uc SA43.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 57 sequences and identified 48 clusters (clustering threshold 1.00) - 1 clusters (representing 6 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA46 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA46_1.fastq /home/working/demultiplexed/SA46_2.fastq SA46_forward.paired.fastq.gz SA46_forward.singletons.fastq.gz SA46_reverse.paired.fastq.gz SA46_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA46_1.fastq /home/working/demultiplexed/SA46_2.fastq SA46_forward.paired.fastq.gz SA46_forward.singletons.fastq.gz SA46_reverse.paired.fastq.gz SA46_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:06:39 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA46/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 62158 Both Surviving: 14590 (23.47%) Forward Only Surviving: 4249 (6.84%) Reverse Only Surviving: 926 (1.49%) Dropped: 42393 (68.20%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA46_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA46_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA46_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA46_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA46_forward.paired.headclipped.fastq.gz SA46_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA46 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA46_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA46_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA46.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA46.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA46.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA46.hist\n",
      "[FLASH]     ./SA46.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 14590 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      14590\n",
      "[FLASH]     Combined pairs:   1007\n",
      "[FLASH]     Uncombined pairs: 13583\n",
      "[FLASH]     Percent combined: 6.90%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.284 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1007 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA46/SA46_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA46_centroids.fasta --uc SA46.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1007 sequences and identified 562 clusters (clustering threshold 1.00) - 13 clusters (representing 376 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA47 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA47_1.fastq /home/working/demultiplexed/SA47_2.fastq SA47_forward.paired.fastq.gz SA47_forward.singletons.fastq.gz SA47_reverse.paired.fastq.gz SA47_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA47_1.fastq /home/working/demultiplexed/SA47_2.fastq SA47_forward.paired.fastq.gz SA47_forward.singletons.fastq.gz SA47_reverse.paired.fastq.gz SA47_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:06:51 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA47/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 17517 Both Surviving: 7452 (42.54%) Forward Only Surviving: 3387 (19.34%) Reverse Only Surviving: 204 (1.16%) Dropped: 6474 (36.96%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SA47_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA47_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA47_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA47_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA47_forward.paired.headclipped.fastq.gz SA47_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA47 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA47_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA47_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA47.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA47.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA47.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA47.hist\n",
      "[FLASH]     ./SA47.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 7452 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      7452\n",
      "[FLASH]     Combined pairs:   5420\n",
      "[FLASH]     Uncombined pairs: 2032\n",
      "[FLASH]     Percent combined: 72.73%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.267 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "5420 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA47/SA47_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA47_centroids.fasta --uc SA47.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 5420 sequences and identified 1636 clusters (clustering threshold 1.00) - 162 clusters (representing 3625 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA48 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA48_1.fastq /home/working/demultiplexed/SA48_2.fastq SA48_forward.paired.fastq.gz SA48_forward.singletons.fastq.gz SA48_reverse.paired.fastq.gz SA48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA48_1.fastq /home/working/demultiplexed/SA48_2.fastq SA48_forward.paired.fastq.gz SA48_forward.singletons.fastq.gz SA48_reverse.paired.fastq.gz SA48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:07:05 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA48/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 11761 Both Surviving: 145 (1.23%) Forward Only Surviving: 144 (1.22%) Reverse Only Surviving: 68 (0.58%) Dropped: 11404 (96.96%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA48_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA48_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA48_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA48_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA48_forward.paired.headclipped.fastq.gz SA48_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA48 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA48_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA48_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA48.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA48.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA48.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA48.hist\n",
      "[FLASH]     ./SA48.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 145 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      145\n",
      "[FLASH]     Combined pairs:   12\n",
      "[FLASH]     Uncombined pairs: 133\n",
      "[FLASH]     Percent combined: 8.28%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.003 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "12 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA48/SA48_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA48_centroids.fasta --uc SA48.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 12 sequences and identified 11 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA49 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA49_1.fastq /home/working/demultiplexed/SA49_2.fastq SA49_forward.paired.fastq.gz SA49_forward.singletons.fastq.gz SA49_reverse.paired.fastq.gz SA49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA49_1.fastq /home/working/demultiplexed/SA49_2.fastq SA49_forward.paired.fastq.gz SA49_forward.singletons.fastq.gz SA49_reverse.paired.fastq.gz SA49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:07:08 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA49/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 23373 Both Surviving: 3250 (13.90%) Forward Only Surviving: 989 (4.23%) Reverse Only Surviving: 238 (1.02%) Dropped: 18896 (80.85%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SA49_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA49_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA49_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA49_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA49_forward.paired.headclipped.fastq.gz SA49_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA49 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA49_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA49_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA49.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA49.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA49.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA49.hist\n",
      "[FLASH]     ./SA49.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3250 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3250\n",
      "[FLASH]     Combined pairs:   2976\n",
      "[FLASH]     Uncombined pairs: 274\n",
      "[FLASH]     Percent combined: 91.57%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.107 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2976 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA49/SA49_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA49_centroids.fasta --uc SA49.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2976 sequences and identified 1540 clusters (clustering threshold 1.00) - 77 clusters (representing 1271 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA5 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA5_1.fastq /home/working/demultiplexed/SA5_2.fastq SA5_forward.paired.fastq.gz SA5_forward.singletons.fastq.gz SA5_reverse.paired.fastq.gz SA5_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA5_1.fastq /home/working/demultiplexed/SA5_2.fastq SA5_forward.paired.fastq.gz SA5_forward.singletons.fastq.gz SA5_reverse.paired.fastq.gz SA5_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:07:19 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA5/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 39236 Both Surviving: 902 (2.30%) Forward Only Surviving: 560 (1.43%) Reverse Only Surviving: 619 (1.58%) Dropped: 37155 (94.70%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SA5_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA5_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA5_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA5_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA5_forward.paired.headclipped.fastq.gz SA5_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA5 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA5_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA5_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA5.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA5.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA5.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA5.hist\n",
      "[FLASH]     ./SA5.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 902 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      902\n",
      "[FLASH]     Combined pairs:   137\n",
      "[FLASH]     Uncombined pairs: 765\n",
      "[FLASH]     Percent combined: 15.19%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.013 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "137 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA5/SA5_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA5_centroids.fasta --uc SA5.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 137 sequences and identified 80 clusters (clustering threshold 1.00) - 3 clusters (representing 53 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA50 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA50_1.fastq /home/working/demultiplexed/SA50_2.fastq SA50_forward.paired.fastq.gz SA50_forward.singletons.fastq.gz SA50_reverse.paired.fastq.gz SA50_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA50_1.fastq /home/working/demultiplexed/SA50_2.fastq SA50_forward.paired.fastq.gz SA50_forward.singletons.fastq.gz SA50_reverse.paired.fastq.gz SA50_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:07:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA50/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 8879 Both Surviving: 3139 (35.35%) Forward Only Surviving: 843 (9.49%) Reverse Only Surviving: 139 (1.57%) Dropped: 4758 (53.59%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA50_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA50_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA50_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA50_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA50_forward.paired.headclipped.fastq.gz SA50_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA50 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA50_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA50_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA50.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA50.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA50.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA50.hist\n",
      "[FLASH]     ./SA50.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3139 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3139\n",
      "[FLASH]     Combined pairs:   918\n",
      "[FLASH]     Uncombined pairs: 2221\n",
      "[FLASH]     Percent combined: 29.24%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.062 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "918 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA50/SA50_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA50_centroids.fasta --uc SA50.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 918 sequences and identified 566 clusters (clustering threshold 1.00) - 11 clusters (representing 313 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA51 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA51_1.fastq /home/working/demultiplexed/SA51_2.fastq SA51_forward.paired.fastq.gz SA51_forward.singletons.fastq.gz SA51_reverse.paired.fastq.gz SA51_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA51_1.fastq /home/working/demultiplexed/SA51_2.fastq SA51_forward.paired.fastq.gz SA51_forward.singletons.fastq.gz SA51_reverse.paired.fastq.gz SA51_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:07:28 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA51/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 24102 Both Surviving: 5856 (24.30%) Forward Only Surviving: 1209 (5.02%) Reverse Only Surviving: 454 (1.88%) Dropped: 16583 (68.80%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SA51_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA51_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA51_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA51_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA51_forward.paired.headclipped.fastq.gz SA51_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA51 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA51_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA51_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA51.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA51.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA51.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA51.hist\n",
      "[FLASH]     ./SA51.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5856 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5856\n",
      "[FLASH]     Combined pairs:   1967\n",
      "[FLASH]     Uncombined pairs: 3889\n",
      "[FLASH]     Percent combined: 33.59%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.119 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1967 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA51/SA51_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA51_centroids.fasta --uc SA51.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1967 sequences and identified 858 clusters (clustering threshold 1.00) - 27 clusters (representing 976 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA53 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA53_1.fastq /home/working/demultiplexed/SA53_2.fastq SA53_forward.paired.fastq.gz SA53_forward.singletons.fastq.gz SA53_reverse.paired.fastq.gz SA53_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA53_1.fastq /home/working/demultiplexed/SA53_2.fastq SA53_forward.paired.fastq.gz SA53_forward.singletons.fastq.gz SA53_reverse.paired.fastq.gz SA53_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:07:41 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA53/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 73318 Both Surviving: 20776 (28.34%) Forward Only Surviving: 5766 (7.86%) Reverse Only Surviving: 1405 (1.92%) Dropped: 45371 (61.88%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SA53_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SA53_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA53_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA53_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA53_forward.paired.headclipped.fastq.gz SA53_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA53 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA53_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA53_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA53.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA53.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA53.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA53.hist\n",
      "[FLASH]     ./SA53.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 20776 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      20776\n",
      "[FLASH]     Combined pairs:   9111\n",
      "[FLASH]     Uncombined pairs: 11665\n",
      "[FLASH]     Percent combined: 43.85%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.546 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "9111 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA53/SA53_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA53_centroids.fasta --uc SA53.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 9111 sequences and identified 3677 clusters (clustering threshold 1.00) - 97 clusters (representing 4680 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA55 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA55_1.fastq /home/working/demultiplexed/SA55_2.fastq SA55_forward.paired.fastq.gz SA55_forward.singletons.fastq.gz SA55_reverse.paired.fastq.gz SA55_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA55_1.fastq /home/working/demultiplexed/SA55_2.fastq SA55_forward.paired.fastq.gz SA55_forward.singletons.fastq.gz SA55_reverse.paired.fastq.gz SA55_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:08:14 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA55/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 49553 Both Surviving: 11984 (24.18%) Forward Only Surviving: 2637 (5.32%) Reverse Only Surviving: 826 (1.67%) Dropped: 34106 (68.83%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SA55_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA55_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA55_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA55_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA55_forward.paired.headclipped.fastq.gz SA55_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA55 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA55_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA55_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA55.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA55.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA55.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA55.hist\n",
      "[FLASH]     ./SA55.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 11984 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      11984\n",
      "[FLASH]     Combined pairs:   3330\n",
      "[FLASH]     Uncombined pairs: 8654\n",
      "[FLASH]     Percent combined: 27.79%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.158 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "3330 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA55/SA55_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA55_centroids.fasta --uc SA55.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 3330 sequences and identified 1674 clusters (clustering threshold 1.00) - 47 clusters (representing 1432 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA58 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA58_1.fastq /home/working/demultiplexed/SA58_2.fastq SA58_forward.paired.fastq.gz SA58_forward.singletons.fastq.gz SA58_reverse.paired.fastq.gz SA58_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA58_1.fastq /home/working/demultiplexed/SA58_2.fastq SA58_forward.paired.fastq.gz SA58_forward.singletons.fastq.gz SA58_reverse.paired.fastq.gz SA58_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:08:28 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA58/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 43499 Both Surviving: 10937 (25.14%) Forward Only Surviving: 2426 (5.58%) Reverse Only Surviving: 671 (1.54%) Dropped: 29465 (67.74%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SA58_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SA58_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA58_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA58_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA58_forward.paired.headclipped.fastq.gz SA58_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA58 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA58_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA58_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA58.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA58.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA58.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA58.hist\n",
      "[FLASH]     ./SA58.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 10937 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      10937\n",
      "[FLASH]     Combined pairs:   2439\n",
      "[FLASH]     Uncombined pairs: 8498\n",
      "[FLASH]     Percent combined: 22.30%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.119 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2439 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA58/SA58_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA58_centroids.fasta --uc SA58.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2439 sequences and identified 1500 clusters (clustering threshold 1.00) - 29 clusters (representing 816 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA62 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA62_1.fastq /home/working/demultiplexed/SA62_2.fastq SA62_forward.paired.fastq.gz SA62_forward.singletons.fastq.gz SA62_reverse.paired.fastq.gz SA62_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA62_1.fastq /home/working/demultiplexed/SA62_2.fastq SA62_forward.paired.fastq.gz SA62_forward.singletons.fastq.gz SA62_reverse.paired.fastq.gz SA62_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:08:40 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA62/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 39926 Both Surviving: 104 (0.26%) Forward Only Surviving: 753 (1.89%) Reverse Only Surviving: 484 (1.21%) Dropped: 38585 (96.64%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SA62_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA62_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA62_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA62_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA62_forward.paired.headclipped.fastq.gz SA62_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA62 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA62_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA62_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA62.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA62.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA62.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA62.hist\n",
      "[FLASH]     ./SA62.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 104 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      104\n",
      "[FLASH]     Combined pairs:   45\n",
      "[FLASH]     Uncombined pairs: 59\n",
      "[FLASH]     Percent combined: 43.27%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.004 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "45 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA62/SA62_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA62_centroids.fasta --uc SA62.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 45 sequences and identified 37 clusters (clustering threshold 1.00) - 1 clusters (representing 5 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA64 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA64_1.fastq /home/working/demultiplexed/SA64_2.fastq SA64_forward.paired.fastq.gz SA64_forward.singletons.fastq.gz SA64_reverse.paired.fastq.gz SA64_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA64_1.fastq /home/working/demultiplexed/SA64_2.fastq SA64_forward.paired.fastq.gz SA64_forward.singletons.fastq.gz SA64_reverse.paired.fastq.gz SA64_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:08:44 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA64/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 28581 Both Surviving: 5231 (18.30%) Forward Only Surviving: 1867 (6.53%) Reverse Only Surviving: 380 (1.33%) Dropped: 21103 (73.84%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA64_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA64_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA64_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA64_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA64_forward.paired.headclipped.fastq.gz SA64_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA64 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA64_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA64_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA64.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA64.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA64.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA64.hist\n",
      "[FLASH]     ./SA64.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5231 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5231\n",
      "[FLASH]     Combined pairs:   366\n",
      "[FLASH]     Uncombined pairs: 4865\n",
      "[FLASH]     Percent combined: 7.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.057 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "366 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA64/SA64_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA64_centroids.fasta --uc SA64.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 366 sequences and identified 262 clusters (clustering threshold 1.00) - 8 clusters (representing 78 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA67 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA67_1.fastq /home/working/demultiplexed/SA67_2.fastq SA67_forward.paired.fastq.gz SA67_forward.singletons.fastq.gz SA67_reverse.paired.fastq.gz SA67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA67_1.fastq /home/working/demultiplexed/SA67_2.fastq SA67_forward.paired.fastq.gz SA67_forward.singletons.fastq.gz SA67_reverse.paired.fastq.gz SA67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:08:49 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA67/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 29766 Both Surviving: 11022 (37.03%) Forward Only Surviving: 3974 (13.35%) Reverse Only Surviving: 376 (1.26%) Dropped: 14394 (48.36%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SA67_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA67_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA67_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA67_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA67_forward.paired.headclipped.fastq.gz SA67_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA67 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA67_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA67_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA67.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA67.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA67.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA67.hist\n",
      "[FLASH]     ./SA67.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 11022 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      11022\n",
      "[FLASH]     Combined pairs:   4029\n",
      "[FLASH]     Uncombined pairs: 6993\n",
      "[FLASH]     Percent combined: 36.55%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.159 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "4029 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA67/SA67_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA67_centroids.fasta --uc SA67.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 4029 sequences and identified 2678 clusters (clustering threshold 1.00) - 77 clusters (representing 1169 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA68 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA68_1.fastq /home/working/demultiplexed/SA68_2.fastq SA68_forward.paired.fastq.gz SA68_forward.singletons.fastq.gz SA68_reverse.paired.fastq.gz SA68_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA68_1.fastq /home/working/demultiplexed/SA68_2.fastq SA68_forward.paired.fastq.gz SA68_forward.singletons.fastq.gz SA68_reverse.paired.fastq.gz SA68_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:09:03 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA68/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 36383 Both Surviving: 2264 (6.22%) Forward Only Surviving: 770 (2.12%) Reverse Only Surviving: 425 (1.17%) Dropped: 32924 (90.49%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SA68_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SA68_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA68_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA68_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA68_forward.paired.headclipped.fastq.gz SA68_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA68 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA68_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA68_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA68.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA68.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA68.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA68.hist\n",
      "[FLASH]     ./SA68.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2264 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2264\n",
      "[FLASH]     Combined pairs:   1962\n",
      "[FLASH]     Uncombined pairs: 302\n",
      "[FLASH]     Percent combined: 86.66%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.056 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1962 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA68/SA68_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA68_centroids.fasta --uc SA68.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1962 sequences and identified 891 clusters (clustering threshold 1.00) - 32 clusters (representing 959 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA7 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA7_1.fastq /home/working/demultiplexed/SA7_2.fastq SA7_forward.paired.fastq.gz SA7_forward.singletons.fastq.gz SA7_reverse.paired.fastq.gz SA7_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA7_1.fastq /home/working/demultiplexed/SA7_2.fastq SA7_forward.paired.fastq.gz SA7_forward.singletons.fastq.gz SA7_reverse.paired.fastq.gz SA7_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:09:10 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA7/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 30674 Both Surviving: 2927 (9.54%) Forward Only Surviving: 1329 (4.33%) Reverse Only Surviving: 307 (1.00%) Dropped: 26111 (85.12%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SA7_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SA7_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA7_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA7_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA7_forward.paired.headclipped.fastq.gz SA7_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA7 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA7_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA7_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA7.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA7.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA7.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA7.hist\n",
      "[FLASH]     ./SA7.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2927 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2927\n",
      "[FLASH]     Combined pairs:   2420\n",
      "[FLASH]     Uncombined pairs: 507\n",
      "[FLASH]     Percent combined: 82.68%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.091 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2420 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA7/SA7_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA7_centroids.fasta --uc SA7.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2420 sequences and identified 1319 clusters (clustering threshold 1.00) - 49 clusters (representing 954 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA70 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA70_1.fastq /home/working/demultiplexed/SA70_2.fastq SA70_forward.paired.fastq.gz SA70_forward.singletons.fastq.gz SA70_reverse.paired.fastq.gz SA70_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA70_1.fastq /home/working/demultiplexed/SA70_2.fastq SA70_forward.paired.fastq.gz SA70_forward.singletons.fastq.gz SA70_reverse.paired.fastq.gz SA70_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:09:18 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA70/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 48118 Both Surviving: 417 (0.87%) Forward Only Surviving: 633 (1.32%) Reverse Only Surviving: 620 (1.29%) Dropped: 46448 (96.53%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SA70_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA70_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA70_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA70_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA70_forward.paired.headclipped.fastq.gz SA70_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA70 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA70_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA70_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA70.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA70.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA70.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA70.hist\n",
      "[FLASH]     ./SA70.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 417 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      417\n",
      "[FLASH]     Combined pairs:   267\n",
      "[FLASH]     Uncombined pairs: 150\n",
      "[FLASH]     Percent combined: 64.03%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.009 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "267 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA70/SA70_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA70_centroids.fasta --uc SA70.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 267 sequences and identified 163 clusters (clustering threshold 1.00) - 7 clusters (representing 97 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA73 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA73_1.fastq /home/working/demultiplexed/SA73_2.fastq SA73_forward.paired.fastq.gz SA73_forward.singletons.fastq.gz SA73_reverse.paired.fastq.gz SA73_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA73_1.fastq /home/working/demultiplexed/SA73_2.fastq SA73_forward.paired.fastq.gz SA73_forward.singletons.fastq.gz SA73_reverse.paired.fastq.gz SA73_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:09:24 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA73/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 45415 Both Surviving: 9585 (21.11%) Forward Only Surviving: 2169 (4.78%) Reverse Only Surviving: 756 (1.66%) Dropped: 32905 (72.45%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SA73_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA73_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA73_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA73_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA73_forward.paired.headclipped.fastq.gz SA73_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA73 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA73_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA73_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA73.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA73.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA73.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA73.hist\n",
      "[FLASH]     ./SA73.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 9585 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      9585\n",
      "[FLASH]     Combined pairs:   2029\n",
      "[FLASH]     Uncombined pairs: 7556\n",
      "[FLASH]     Percent combined: 21.17%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.096 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2029 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA73/SA73_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA73_centroids.fasta --uc SA73.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2029 sequences and identified 1038 clusters (clustering threshold 1.00) - 37 clusters (representing 840 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA75 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA75_1.fastq /home/working/demultiplexed/SA75_2.fastq SA75_forward.paired.fastq.gz SA75_forward.singletons.fastq.gz SA75_reverse.paired.fastq.gz SA75_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA75_1.fastq /home/working/demultiplexed/SA75_2.fastq SA75_forward.paired.fastq.gz SA75_forward.singletons.fastq.gz SA75_reverse.paired.fastq.gz SA75_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:09:33 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA75/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 17374 Both Surviving: 1201 (6.91%) Forward Only Surviving: 573 (3.30%) Reverse Only Surviving: 227 (1.31%) Dropped: 15373 (88.48%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SA75_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA75_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA75_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA75_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA75_forward.paired.headclipped.fastq.gz SA75_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA75 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA75_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA75_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA75.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA75.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA75.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA75.hist\n",
      "[FLASH]     ./SA75.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1201 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1201\n",
      "[FLASH]     Combined pairs:   421\n",
      "[FLASH]     Uncombined pairs: 780\n",
      "[FLASH]     Percent combined: 35.05%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.021 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "421 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA75/SA75_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA75_centroids.fasta --uc SA75.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 421 sequences and identified 300 clusters (clustering threshold 1.00) - 6 clusters (representing 110 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA76 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA76_1.fastq /home/working/demultiplexed/SA76_2.fastq SA76_forward.paired.fastq.gz SA76_forward.singletons.fastq.gz SA76_reverse.paired.fastq.gz SA76_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA76_1.fastq /home/working/demultiplexed/SA76_2.fastq SA76_forward.paired.fastq.gz SA76_forward.singletons.fastq.gz SA76_reverse.paired.fastq.gz SA76_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:09:37 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA76/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 20250 Both Surviving: 7598 (37.52%) Forward Only Surviving: 3215 (15.88%) Reverse Only Surviving: 243 (1.20%) Dropped: 9194 (45.40%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: SA76_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: SA76_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA76_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA76_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA76_forward.paired.headclipped.fastq.gz SA76_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA76 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA76_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA76_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA76.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA76.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA76.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA76.hist\n",
      "[FLASH]     ./SA76.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 7598 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      7598\n",
      "[FLASH]     Combined pairs:   6303\n",
      "[FLASH]     Uncombined pairs: 1295\n",
      "[FLASH]     Percent combined: 82.96%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.176 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "6303 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA76/SA76_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA76_centroids.fasta --uc SA76.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 6303 sequences and identified 2397 clusters (clustering threshold 1.00) - 142 clusters (representing 3603 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA81 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA81_1.fastq /home/working/demultiplexed/SA81_2.fastq SA81_forward.paired.fastq.gz SA81_forward.singletons.fastq.gz SA81_reverse.paired.fastq.gz SA81_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA81_1.fastq /home/working/demultiplexed/SA81_2.fastq SA81_forward.paired.fastq.gz SA81_forward.singletons.fastq.gz SA81_reverse.paired.fastq.gz SA81_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:09:49 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA81/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 30631 Both Surviving: 585 (1.91%) Forward Only Surviving: 387 (1.26%) Reverse Only Surviving: 304 (0.99%) Dropped: 29355 (95.83%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SA81_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA81_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA81_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA81_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA81_forward.paired.headclipped.fastq.gz SA81_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA81 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA81_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA81_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA81.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA81.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA81.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA81.hist\n",
      "[FLASH]     ./SA81.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 585 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      585\n",
      "[FLASH]     Combined pairs:   47\n",
      "[FLASH]     Uncombined pairs: 538\n",
      "[FLASH]     Percent combined: 8.03%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.007 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "47 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA81/SA81_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA81_centroids.fasta --uc SA81.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 47 sequences and identified 42 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA86 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA86_1.fastq /home/working/demultiplexed/SA86_2.fastq SA86_forward.paired.fastq.gz SA86_forward.singletons.fastq.gz SA86_reverse.paired.fastq.gz SA86_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA86_1.fastq /home/working/demultiplexed/SA86_2.fastq SA86_forward.paired.fastq.gz SA86_forward.singletons.fastq.gz SA86_reverse.paired.fastq.gz SA86_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:09:54 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA86/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 60032 Both Surviving: 8569 (14.27%) Forward Only Surviving: 2738 (4.56%) Reverse Only Surviving: 656 (1.09%) Dropped: 48069 (80.07%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SA86_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA86_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA86_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA86_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA86_forward.paired.headclipped.fastq.gz SA86_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA86 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA86_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA86_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA86.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA86.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA86.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA86.hist\n",
      "[FLASH]     ./SA86.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 8569 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      8569\n",
      "[FLASH]     Combined pairs:   216\n",
      "[FLASH]     Uncombined pairs: 8353\n",
      "[FLASH]     Percent combined: 2.52%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.072 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "216 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA86/SA86_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA86_centroids.fasta --uc SA86.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 216 sequences and identified 181 clusters (clustering threshold 1.00) - 2 clusters (representing 28 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA87 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA87_1.fastq /home/working/demultiplexed/SA87_2.fastq SA87_forward.paired.fastq.gz SA87_forward.singletons.fastq.gz SA87_reverse.paired.fastq.gz SA87_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA87_1.fastq /home/working/demultiplexed/SA87_2.fastq SA87_forward.paired.fastq.gz SA87_forward.singletons.fastq.gz SA87_reverse.paired.fastq.gz SA87_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:10:02 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA87/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 31996 Both Surviving: 1905 (5.95%) Forward Only Surviving: 1110 (3.47%) Reverse Only Surviving: 484 (1.51%) Dropped: 28497 (89.06%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SA87_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA87_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA87_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA87_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA87_forward.paired.headclipped.fastq.gz SA87_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA87 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA87_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA87_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA87.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA87.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA87.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA87.hist\n",
      "[FLASH]     ./SA87.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1905 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1905\n",
      "[FLASH]     Combined pairs:   643\n",
      "[FLASH]     Uncombined pairs: 1262\n",
      "[FLASH]     Percent combined: 33.75%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.025 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "643 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA87/SA87_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA87_centroids.fasta --uc SA87.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 643 sequences and identified 404 clusters (clustering threshold 1.00) - 9 clusters (representing 198 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA88 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA88_1.fastq /home/working/demultiplexed/SA88_2.fastq SA88_forward.paired.fastq.gz SA88_forward.singletons.fastq.gz SA88_reverse.paired.fastq.gz SA88_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA88_1.fastq /home/working/demultiplexed/SA88_2.fastq SA88_forward.paired.fastq.gz SA88_forward.singletons.fastq.gz SA88_reverse.paired.fastq.gz SA88_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:10:11 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA88/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 95056 Both Surviving: 32779 (34.48%) Forward Only Surviving: 7861 (8.27%) Reverse Only Surviving: 2531 (2.66%) Dropped: 51885 (54.58%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SA88_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SA88_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA88_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA88_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA88_forward.paired.headclipped.fastq.gz SA88_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA88 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA88_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA88_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA88.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA88.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA88.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA88.hist\n",
      "[FLASH]     ./SA88.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 25000 read pairs\n",
      "[FLASH] Processed 32779 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      32779\n",
      "[FLASH]     Combined pairs:   1124\n",
      "[FLASH]     Uncombined pairs: 31655\n",
      "[FLASH]     Percent combined: 3.43%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.370 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1124 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA88/SA88_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA88_centroids.fasta --uc SA88.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1124 sequences and identified 882 clusters (clustering threshold 1.00) - 17 clusters (representing 150 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA89 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA89_1.fastq /home/working/demultiplexed/SA89_2.fastq SA89_forward.paired.fastq.gz SA89_forward.singletons.fastq.gz SA89_reverse.paired.fastq.gz SA89_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA89_1.fastq /home/working/demultiplexed/SA89_2.fastq SA89_forward.paired.fastq.gz SA89_forward.singletons.fastq.gz SA89_reverse.paired.fastq.gz SA89_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:10:30 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA89/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 20721 Both Surviving: 4774 (23.04%) Forward Only Surviving: 992 (4.79%) Reverse Only Surviving: 412 (1.99%) Dropped: 14543 (70.18%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA89_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA89_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA89_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA89_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA89_forward.paired.headclipped.fastq.gz SA89_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA89 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA89_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA89_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA89.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA89.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA89.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA89.hist\n",
      "[FLASH]     ./SA89.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 4774 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      4774\n",
      "[FLASH]     Combined pairs:   177\n",
      "[FLASH]     Uncombined pairs: 4597\n",
      "[FLASH]     Percent combined: 3.71%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.053 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "177 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA89/SA89_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA89_centroids.fasta --uc SA89.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 177 sequences and identified 130 clusters (clustering threshold 1.00) - 2 clusters (representing 38 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA9 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA9_1.fastq /home/working/demultiplexed/SA9_2.fastq SA9_forward.paired.fastq.gz SA9_forward.singletons.fastq.gz SA9_reverse.paired.fastq.gz SA9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA9_1.fastq /home/working/demultiplexed/SA9_2.fastq SA9_forward.paired.fastq.gz SA9_forward.singletons.fastq.gz SA9_reverse.paired.fastq.gz SA9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:10:35 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA9/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 27387 Both Surviving: 3248 (11.86%) Forward Only Surviving: 1038 (3.79%) Reverse Only Surviving: 271 (0.99%) Dropped: 22830 (83.36%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SA9_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SA9_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA9_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SA9_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA9_forward.paired.headclipped.fastq.gz SA9_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA9 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA9_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA9_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA9.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA9.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA9.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA9.hist\n",
      "[FLASH]     ./SA9.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3248 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3248\n",
      "[FLASH]     Combined pairs:   2832\n",
      "[FLASH]     Uncombined pairs: 416\n",
      "[FLASH]     Percent combined: 87.19%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.087 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2832 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA9/SA9_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA9_centroids.fasta --uc SA9.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2832 sequences and identified 1263 clusters (clustering threshold 1.00) - 32 clusters (representing 1368 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA91 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA91_1.fastq /home/working/demultiplexed/SA91_2.fastq SA91_forward.paired.fastq.gz SA91_forward.singletons.fastq.gz SA91_reverse.paired.fastq.gz SA91_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA91_1.fastq /home/working/demultiplexed/SA91_2.fastq SA91_forward.paired.fastq.gz SA91_forward.singletons.fastq.gz SA91_reverse.paired.fastq.gz SA91_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:10:45 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA91/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 58616 Both Surviving: 9782 (16.69%) Forward Only Surviving: 3409 (5.82%) Reverse Only Surviving: 846 (1.44%) Dropped: 44579 (76.05%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: SA91_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: SA91_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA91_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA91_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA91_forward.paired.headclipped.fastq.gz SA91_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA91 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA91_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA91_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA91.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA91.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA91.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA91.hist\n",
      "[FLASH]     ./SA91.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 9782 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      9782\n",
      "[FLASH]     Combined pairs:   1474\n",
      "[FLASH]     Uncombined pairs: 8308\n",
      "[FLASH]     Percent combined: 15.07%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.092 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1474 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA91/SA91_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA91_centroids.fasta --uc SA91.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1474 sequences and identified 745 clusters (clustering threshold 1.00) - 16 clusters (representing 648 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA93 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA93_1.fastq /home/working/demultiplexed/SA93_2.fastq SA93_forward.paired.fastq.gz SA93_forward.singletons.fastq.gz SA93_reverse.paired.fastq.gz SA93_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA93_1.fastq /home/working/demultiplexed/SA93_2.fastq SA93_forward.paired.fastq.gz SA93_forward.singletons.fastq.gz SA93_reverse.paired.fastq.gz SA93_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:10:53 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA93/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 20338 Both Surviving: 3625 (17.82%) Forward Only Surviving: 818 (4.02%) Reverse Only Surviving: 288 (1.42%) Dropped: 15607 (76.74%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA93_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA93_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA93_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA93_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA93_forward.paired.headclipped.fastq.gz SA93_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA93 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA93_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA93_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA93.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA93.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA93.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA93.hist\n",
      "[FLASH]     ./SA93.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3625 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3625\n",
      "[FLASH]     Combined pairs:   1692\n",
      "[FLASH]     Uncombined pairs: 1933\n",
      "[FLASH]     Percent combined: 46.68%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.059 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1692 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA93/SA93_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA93_centroids.fasta --uc SA93.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1692 sequences and identified 1004 clusters (clustering threshold 1.00) - 20 clusters (representing 607 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA94 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA94_1.fastq /home/working/demultiplexed/SA94_2.fastq SA94_forward.paired.fastq.gz SA94_forward.singletons.fastq.gz SA94_reverse.paired.fastq.gz SA94_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA94_1.fastq /home/working/demultiplexed/SA94_2.fastq SA94_forward.paired.fastq.gz SA94_forward.singletons.fastq.gz SA94_reverse.paired.fastq.gz SA94_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:10:59 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA94/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 25533 Both Surviving: 139 (0.54%) Forward Only Surviving: 402 (1.57%) Reverse Only Surviving: 269 (1.05%) Dropped: 24723 (96.83%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SA94_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SA94_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA94_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA94_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA94_forward.paired.headclipped.fastq.gz SA94_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA94 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA94_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA94_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA94.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA94.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA94.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA94.hist\n",
      "[FLASH]     ./SA94.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 139 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      139\n",
      "[FLASH]     Combined pairs:   72\n",
      "[FLASH]     Uncombined pairs: 67\n",
      "[FLASH]     Percent combined: 51.80%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.004 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "72 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA94/SA94_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA94_centroids.fasta --uc SA94.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 72 sequences and identified 44 clusters (clustering threshold 1.00) - 1 clusters (representing 26 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA95 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA95_1.fastq /home/working/demultiplexed/SA95_2.fastq SA95_forward.paired.fastq.gz SA95_forward.singletons.fastq.gz SA95_reverse.paired.fastq.gz SA95_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA95_1.fastq /home/working/demultiplexed/SA95_2.fastq SA95_forward.paired.fastq.gz SA95_forward.singletons.fastq.gz SA95_reverse.paired.fastq.gz SA95_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:11:02 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA95/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 11287 Both Surviving: 698 (6.18%) Forward Only Surviving: 258 (2.29%) Reverse Only Surviving: 124 (1.10%) Dropped: 10207 (90.43%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: SA95_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: SA95_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA95_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA95_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA95_forward.paired.headclipped.fastq.gz SA95_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA95 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA95_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA95_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA95.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA95.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA95.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA95.hist\n",
      "[FLASH]     ./SA95.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 698 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      698\n",
      "[FLASH]     Combined pairs:   526\n",
      "[FLASH]     Uncombined pairs: 172\n",
      "[FLASH]     Percent combined: 75.36%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.017 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "526 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA95/SA95_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA95_centroids.fasta --uc SA95.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 526 sequences and identified 290 clusters (clustering threshold 1.00) - 8 clusters (representing 216 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA96 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA96_1.fastq /home/working/demultiplexed/SA96_2.fastq SA96_forward.paired.fastq.gz SA96_forward.singletons.fastq.gz SA96_reverse.paired.fastq.gz SA96_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA96_1.fastq /home/working/demultiplexed/SA96_2.fastq SA96_forward.paired.fastq.gz SA96_forward.singletons.fastq.gz SA96_reverse.paired.fastq.gz SA96_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:11:04 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA96/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 15602 Both Surviving: 4680 (30.00%) Forward Only Surviving: 1124 (7.20%) Reverse Only Surviving: 244 (1.56%) Dropped: 9554 (61.24%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SA96_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SA96_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA96_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SA96_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA96_forward.paired.headclipped.fastq.gz SA96_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA96 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA96_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA96_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA96.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA96.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA96.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA96.hist\n",
      "[FLASH]     ./SA96.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 4680 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      4680\n",
      "[FLASH]     Combined pairs:   1353\n",
      "[FLASH]     Uncombined pairs: 3327\n",
      "[FLASH]     Percent combined: 28.91%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.049 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1353 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA96/SA96_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA96_centroids.fasta --uc SA96.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1353 sequences and identified 828 clusters (clustering threshold 1.00) - 22 clusters (representing 483 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA97 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA97_1.fastq /home/working/demultiplexed/SA97_2.fastq SA97_forward.paired.fastq.gz SA97_forward.singletons.fastq.gz SA97_reverse.paired.fastq.gz SA97_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA97_1.fastq /home/working/demultiplexed/SA97_2.fastq SA97_forward.paired.fastq.gz SA97_forward.singletons.fastq.gz SA97_reverse.paired.fastq.gz SA97_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:11:09 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA97/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 11026 Both Surviving: 2186 (19.83%) Forward Only Surviving: 409 (3.71%) Reverse Only Surviving: 184 (1.67%) Dropped: 8247 (74.80%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SA97_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SA97_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA97_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SA97_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA97_forward.paired.headclipped.fastq.gz SA97_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA97 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA97_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA97_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA97.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA97.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA97.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA97.hist\n",
      "[FLASH]     ./SA97.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2186 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2186\n",
      "[FLASH]     Combined pairs:   816\n",
      "[FLASH]     Uncombined pairs: 1370\n",
      "[FLASH]     Percent combined: 37.33%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.028 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "816 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA97/SA97_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA97_centroids.fasta --uc SA97.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 816 sequences and identified 476 clusters (clustering threshold 1.00) - 9 clusters (representing 299 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SA98 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA98_1.fastq /home/working/demultiplexed/SA98_2.fastq SA98_forward.paired.fastq.gz SA98_forward.singletons.fastq.gz SA98_reverse.paired.fastq.gz SA98_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SA98_1.fastq /home/working/demultiplexed/SA98_2.fastq SA98_forward.paired.fastq.gz SA98_forward.singletons.fastq.gz SA98_reverse.paired.fastq.gz SA98_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:11:14 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SA98/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 42857 Both Surviving: 5781 (13.49%) Forward Only Surviving: 2218 (5.18%) Reverse Only Surviving: 333 (0.78%) Dropped: 34525 (80.56%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SA98_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SA98_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA98_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SA98_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SA98_forward.paired.headclipped.fastq.gz SA98_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SA98 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SA98_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SA98_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SA98.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SA98.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SA98.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SA98.hist\n",
      "[FLASH]     ./SA98.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5781 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5781\n",
      "[FLASH]     Combined pairs:   5329\n",
      "[FLASH]     Uncombined pairs: 452\n",
      "[FLASH]     Percent combined: 92.18%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.139 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "5329 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SA98/SA98_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SA98_centroids.fasta --uc SA98.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 5329 sequences and identified 1260 clusters (clustering threshold 1.00) - 143 clusters (representing 3925 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB102 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB102_1.fastq /home/working/demultiplexed/SB102_2.fastq SB102_forward.paired.fastq.gz SB102_forward.singletons.fastq.gz SB102_reverse.paired.fastq.gz SB102_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB102_1.fastq /home/working/demultiplexed/SB102_2.fastq SB102_forward.paired.fastq.gz SB102_forward.singletons.fastq.gz SB102_reverse.paired.fastq.gz SB102_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:11:30 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB102/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 95589 Both Surviving: 18233 (19.07%) Forward Only Surviving: 5549 (5.81%) Reverse Only Surviving: 1427 (1.49%) Dropped: 70380 (73.63%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SB102_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB102_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB102_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB102_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB102_forward.paired.headclipped.fastq.gz SB102_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB102 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB102_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB102_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB102.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB102.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB102.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB102.hist\n",
      "[FLASH]     ./SB102.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 18233 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      18233\n",
      "[FLASH]     Combined pairs:   980\n",
      "[FLASH]     Uncombined pairs: 17253\n",
      "[FLASH]     Percent combined: 5.37%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.111 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "980 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB102/SB102_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB102_centroids.fasta --uc SB102.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 980 sequences and identified 623 clusters (clustering threshold 1.00) - 18 clusters (representing 291 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB104 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB104_1.fastq /home/working/demultiplexed/SB104_2.fastq SB104_forward.paired.fastq.gz SB104_forward.singletons.fastq.gz SB104_reverse.paired.fastq.gz SB104_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB104_1.fastq /home/working/demultiplexed/SB104_2.fastq SB104_forward.paired.fastq.gz SB104_forward.singletons.fastq.gz SB104_reverse.paired.fastq.gz SB104_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:11:42 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB104/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 21182 Both Surviving: 9019 (42.58%) Forward Only Surviving: 1940 (9.16%) Reverse Only Surviving: 659 (3.11%) Dropped: 9564 (45.15%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: SB104_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB104_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB104_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB104_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB104_forward.paired.headclipped.fastq.gz SB104_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB104 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB104_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB104_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB104.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB104.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB104.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB104.hist\n",
      "[FLASH]     ./SB104.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 9019 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      9019\n",
      "[FLASH]     Combined pairs:   5159\n",
      "[FLASH]     Uncombined pairs: 3860\n",
      "[FLASH]     Percent combined: 57.20%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.146 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "5159 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB104/SB104_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB104_centroids.fasta --uc SB104.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 5159 sequences and identified 3829 clusters (clustering threshold 1.00) - 65 clusters (representing 924 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB106 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB106_1.fastq /home/working/demultiplexed/SB106_2.fastq SB106_forward.paired.fastq.gz SB106_forward.singletons.fastq.gz SB106_reverse.paired.fastq.gz SB106_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB106_1.fastq /home/working/demultiplexed/SB106_2.fastq SB106_forward.paired.fastq.gz SB106_forward.singletons.fastq.gz SB106_reverse.paired.fastq.gz SB106_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:11:55 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB106/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 49076 Both Surviving: 8570 (17.46%) Forward Only Surviving: 2504 (5.10%) Reverse Only Surviving: 465 (0.95%) Dropped: 37537 (76.49%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SB106_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SB106_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB106_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB106_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB106_forward.paired.headclipped.fastq.gz SB106_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB106 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB106_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB106_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB106.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB106.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB106.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB106.hist\n",
      "[FLASH]     ./SB106.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 8570 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      8570\n",
      "[FLASH]     Combined pairs:   7840\n",
      "[FLASH]     Uncombined pairs: 730\n",
      "[FLASH]     Percent combined: 91.48%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.226 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "7840 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB106/SB106_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB106_centroids.fasta --uc SB106.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 7840 sequences and identified 3631 clusters (clustering threshold 1.00) - 111 clusters (representing 3692 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB111 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB111_1.fastq /home/working/demultiplexed/SB111_2.fastq SB111_forward.paired.fastq.gz SB111_forward.singletons.fastq.gz SB111_reverse.paired.fastq.gz SB111_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB111_1.fastq /home/working/demultiplexed/SB111_2.fastq SB111_forward.paired.fastq.gz SB111_forward.singletons.fastq.gz SB111_reverse.paired.fastq.gz SB111_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:12:11 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB111/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 31510 Both Surviving: 103 (0.33%) Forward Only Surviving: 529 (1.68%) Reverse Only Surviving: 306 (0.97%) Dropped: 30572 (97.02%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SB111_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SB111_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB111_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB111_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB111_forward.paired.headclipped.fastq.gz SB111_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB111 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB111_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB111_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB111.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB111.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB111.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB111.hist\n",
      "[FLASH]     ./SB111.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 103 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      103\n",
      "[FLASH]     Combined pairs:   27\n",
      "[FLASH]     Uncombined pairs: 76\n",
      "[FLASH]     Percent combined: 26.21%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.002 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "27 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB111/SB111_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB111_centroids.fasta --uc SB111.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 27 sequences and identified 26 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB116 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB116_1.fastq /home/working/demultiplexed/SB116_2.fastq SB116_forward.paired.fastq.gz SB116_forward.singletons.fastq.gz SB116_reverse.paired.fastq.gz SB116_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB116_1.fastq /home/working/demultiplexed/SB116_2.fastq SB116_forward.paired.fastq.gz SB116_forward.singletons.fastq.gz SB116_reverse.paired.fastq.gz SB116_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:12:13 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB116/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 12335 Both Surviving: 2168 (17.58%) Forward Only Surviving: 604 (4.90%) Reverse Only Surviving: 215 (1.74%) Dropped: 9348 (75.78%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SB116_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SB116_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB116_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB116_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB116_forward.paired.headclipped.fastq.gz SB116_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB116 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB116_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB116_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB116.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB116.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB116.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB116.hist\n",
      "[FLASH]     ./SB116.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2168 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2168\n",
      "[FLASH]     Combined pairs:   333\n",
      "[FLASH]     Uncombined pairs: 1835\n",
      "[FLASH]     Percent combined: 15.36%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.015 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "333 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB116/SB116_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB116_centroids.fasta --uc SB116.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 333 sequences and identified 189 clusters (clustering threshold 1.00) - 3 clusters (representing 124 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB119 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB119_1.fastq /home/working/demultiplexed/SB119_2.fastq SB119_forward.paired.fastq.gz SB119_forward.singletons.fastq.gz SB119_reverse.paired.fastq.gz SB119_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB119_1.fastq /home/working/demultiplexed/SB119_2.fastq SB119_forward.paired.fastq.gz SB119_forward.singletons.fastq.gz SB119_reverse.paired.fastq.gz SB119_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:12:17 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB119/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 27754 Both Surviving: 5093 (18.35%) Forward Only Surviving: 1118 (4.03%) Reverse Only Surviving: 486 (1.75%) Dropped: 21057 (75.87%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SB119_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB119_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB119_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB119_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB119_forward.paired.headclipped.fastq.gz SB119_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB119 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB119_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB119_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB119.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB119.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB119.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB119.hist\n",
      "[FLASH]     ./SB119.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5093 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5093\n",
      "[FLASH]     Combined pairs:   106\n",
      "[FLASH]     Uncombined pairs: 4987\n",
      "[FLASH]     Percent combined: 2.08%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.032 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "106 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB119/SB119_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB119_centroids.fasta --uc SB119.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 106 sequences and identified 77 clusters (clustering threshold 1.00) - 1 clusters (representing 27 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB120 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB120_1.fastq /home/working/demultiplexed/SB120_2.fastq SB120_forward.paired.fastq.gz SB120_forward.singletons.fastq.gz SB120_reverse.paired.fastq.gz SB120_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB120_1.fastq /home/working/demultiplexed/SB120_2.fastq SB120_forward.paired.fastq.gz SB120_forward.singletons.fastq.gz SB120_reverse.paired.fastq.gz SB120_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:12:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB120/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 60339 Both Surviving: 12705 (21.06%) Forward Only Surviving: 3855 (6.39%) Reverse Only Surviving: 963 (1.60%) Dropped: 42816 (70.96%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SB120_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB120_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB120_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB120_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB120_forward.paired.headclipped.fastq.gz SB120_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB120 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB120_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB120_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB120.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB120.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB120.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB120.hist\n",
      "[FLASH]     ./SB120.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 12705 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      12705\n",
      "[FLASH]     Combined pairs:   2445\n",
      "[FLASH]     Uncombined pairs: 10260\n",
      "[FLASH]     Percent combined: 19.24%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.095 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2445 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB120/SB120_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB120_centroids.fasta --uc SB120.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2445 sequences and identified 1432 clusters (clustering threshold 1.00) - 38 clusters (representing 850 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB127 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB127_1.fastq /home/working/demultiplexed/SB127_2.fastq SB127_forward.paired.fastq.gz SB127_forward.singletons.fastq.gz SB127_reverse.paired.fastq.gz SB127_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB127_1.fastq /home/working/demultiplexed/SB127_2.fastq SB127_forward.paired.fastq.gz SB127_forward.singletons.fastq.gz SB127_reverse.paired.fastq.gz SB127_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:12:34 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB127/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 38679 Both Surviving: 1764 (4.56%) Forward Only Surviving: 696 (1.80%) Reverse Only Surviving: 470 (1.22%) Dropped: 35749 (92.42%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SB127_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SB127_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB127_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB127_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB127_forward.paired.headclipped.fastq.gz SB127_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB127 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB127_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB127_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB127.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB127.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB127.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB127.hist\n",
      "[FLASH]     ./SB127.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1764 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1764\n",
      "[FLASH]     Combined pairs:   1660\n",
      "[FLASH]     Uncombined pairs: 104\n",
      "[FLASH]     Percent combined: 94.10%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.042 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1660 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB127/SB127_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB127_centroids.fasta --uc SB127.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1660 sequences and identified 765 clusters (clustering threshold 1.00) - 24 clusters (representing 749 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB128 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB128_1.fastq /home/working/demultiplexed/SB128_2.fastq SB128_forward.paired.fastq.gz SB128_forward.singletons.fastq.gz SB128_reverse.paired.fastq.gz SB128_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB128_1.fastq /home/working/demultiplexed/SB128_2.fastq SB128_forward.paired.fastq.gz SB128_forward.singletons.fastq.gz SB128_reverse.paired.fastq.gz SB128_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:12:41 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB128/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 54931 Both Surviving: 9843 (17.92%) Forward Only Surviving: 2614 (4.76%) Reverse Only Surviving: 903 (1.64%) Dropped: 41571 (75.68%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SB128_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB128_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB128_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB128_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB128_forward.paired.headclipped.fastq.gz SB128_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB128 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB128_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB128_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB128.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB128.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB128.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB128.hist\n",
      "[FLASH]     ./SB128.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 9843 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      9843\n",
      "[FLASH]     Combined pairs:   1131\n",
      "[FLASH]     Uncombined pairs: 8712\n",
      "[FLASH]     Percent combined: 11.49%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.057 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1131 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB128/SB128_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB128_centroids.fasta --uc SB128.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1131 sequences and identified 596 clusters (clustering threshold 1.00) - 23 clusters (representing 488 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB129 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB129_1.fastq /home/working/demultiplexed/SB129_2.fastq SB129_forward.paired.fastq.gz SB129_forward.singletons.fastq.gz SB129_reverse.paired.fastq.gz SB129_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB129_1.fastq /home/working/demultiplexed/SB129_2.fastq SB129_forward.paired.fastq.gz SB129_forward.singletons.fastq.gz SB129_reverse.paired.fastq.gz SB129_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:12:50 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB129/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 41461 Both Surviving: 12224 (29.48%) Forward Only Surviving: 2762 (6.66%) Reverse Only Surviving: 712 (1.72%) Dropped: 25763 (62.14%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SB129_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB129_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SB129_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SB129_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB129_forward.paired.headclipped.fastq.gz SB129_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB129 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB129_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB129_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB129.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB129.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB129.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB129.hist\n",
      "[FLASH]     ./SB129.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 12224 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      12224\n",
      "[FLASH]     Combined pairs:   4737\n",
      "[FLASH]     Uncombined pairs: 7487\n",
      "[FLASH]     Percent combined: 38.75%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.141 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "4737 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB129/SB129_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB129_centroids.fasta --uc SB129.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 4737 sequences and identified 2274 clusters (clustering threshold 1.00) - 71 clusters (representing 2123 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB13 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB13_1.fastq /home/working/demultiplexed/SB13_2.fastq SB13_forward.paired.fastq.gz SB13_forward.singletons.fastq.gz SB13_reverse.paired.fastq.gz SB13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB13_1.fastq /home/working/demultiplexed/SB13_2.fastq SB13_forward.paired.fastq.gz SB13_forward.singletons.fastq.gz SB13_reverse.paired.fastq.gz SB13_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:13:02 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB13/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 34681 Both Surviving: 6218 (17.93%) Forward Only Surviving: 1604 (4.63%) Reverse Only Surviving: 580 (1.67%) Dropped: 26279 (75.77%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SB13_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB13_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB13_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB13_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB13_forward.paired.headclipped.fastq.gz SB13_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB13 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB13_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB13_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB13.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB13.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB13.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB13.hist\n",
      "[FLASH]     ./SB13.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 6218 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      6218\n",
      "[FLASH]     Combined pairs:   40\n",
      "[FLASH]     Uncombined pairs: 6178\n",
      "[FLASH]     Percent combined: 0.64%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.040 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "40 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB13/SB13_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB13_centroids.fasta --uc SB13.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 40 sequences and identified 38 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB131 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB131_1.fastq /home/working/demultiplexed/SB131_2.fastq SB131_forward.paired.fastq.gz SB131_forward.singletons.fastq.gz SB131_reverse.paired.fastq.gz SB131_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB131_1.fastq /home/working/demultiplexed/SB131_2.fastq SB131_forward.paired.fastq.gz SB131_forward.singletons.fastq.gz SB131_reverse.paired.fastq.gz SB131_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:13:08 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB131/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 40610 Both Surviving: 3564 (8.78%) Forward Only Surviving: 1088 (2.68%) Reverse Only Surviving: 579 (1.43%) Dropped: 35379 (87.12%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SB131_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB131_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB131_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB131_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB131_forward.paired.headclipped.fastq.gz SB131_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB131 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB131_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB131_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB131.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB131.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB131.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB131.hist\n",
      "[FLASH]     ./SB131.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3564 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3564\n",
      "[FLASH]     Combined pairs:   95\n",
      "[FLASH]     Uncombined pairs: 3469\n",
      "[FLASH]     Percent combined: 2.67%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.022 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "95 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB131/SB131_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB131_centroids.fasta --uc SB131.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 95 sequences and identified 82 clusters (clustering threshold 1.00) - 1 clusters (representing 11 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB135 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB135_1.fastq /home/working/demultiplexed/SB135_2.fastq SB135_forward.paired.fastq.gz SB135_forward.singletons.fastq.gz SB135_reverse.paired.fastq.gz SB135_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB135_1.fastq /home/working/demultiplexed/SB135_2.fastq SB135_forward.paired.fastq.gz SB135_forward.singletons.fastq.gz SB135_reverse.paired.fastq.gz SB135_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:13:12 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB135/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 29244 Both Surviving: 2587 (8.85%) Forward Only Surviving: 787 (2.69%) Reverse Only Surviving: 356 (1.22%) Dropped: 25514 (87.25%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: SB135_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: SB135_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB135_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB135_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB135_forward.paired.headclipped.fastq.gz SB135_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB135 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB135_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB135_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB135.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB135.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB135.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB135.hist\n",
      "[FLASH]     ./SB135.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2587 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2587\n",
      "[FLASH]     Combined pairs:   277\n",
      "[FLASH]     Uncombined pairs: 2310\n",
      "[FLASH]     Percent combined: 10.71%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.015 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "277 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB135/SB135_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB135_centroids.fasta --uc SB135.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 277 sequences and identified 185 clusters (clustering threshold 1.00) - 3 clusters (representing 80 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB136 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB136_1.fastq /home/working/demultiplexed/SB136_2.fastq SB136_forward.paired.fastq.gz SB136_forward.singletons.fastq.gz SB136_reverse.paired.fastq.gz SB136_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB136_1.fastq /home/working/demultiplexed/SB136_2.fastq SB136_forward.paired.fastq.gz SB136_forward.singletons.fastq.gz SB136_reverse.paired.fastq.gz SB136_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:13:18 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB136/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 43996 Both Surviving: 15162 (34.46%) Forward Only Surviving: 3126 (7.11%) Reverse Only Surviving: 704 (1.60%) Dropped: 25004 (56.83%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: SB136_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB136_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB136_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB136_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB136_forward.paired.headclipped.fastq.gz SB136_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB136 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB136_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB136_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB136.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB136.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB136.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB136.hist\n",
      "[FLASH]     ./SB136.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 15162 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      15162\n",
      "[FLASH]     Combined pairs:   8678\n",
      "[FLASH]     Uncombined pairs: 6484\n",
      "[FLASH]     Percent combined: 57.24%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.237 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "8678 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB136/SB136_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB136_centroids.fasta --uc SB136.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 8678 sequences and identified 3769 clusters (clustering threshold 1.00) - 160 clusters (representing 4524 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB138 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB138_1.fastq /home/working/demultiplexed/SB138_2.fastq SB138_forward.paired.fastq.gz SB138_forward.singletons.fastq.gz SB138_reverse.paired.fastq.gz SB138_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB138_1.fastq /home/working/demultiplexed/SB138_2.fastq SB138_forward.paired.fastq.gz SB138_forward.singletons.fastq.gz SB138_reverse.paired.fastq.gz SB138_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:13:37 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB138/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 35038 Both Surviving: 11925 (34.03%) Forward Only Surviving: 4745 (13.54%) Reverse Only Surviving: 416 (1.19%) Dropped: 17952 (51.24%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SB138_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SB138_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB138_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB138_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB138_forward.paired.headclipped.fastq.gz SB138_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB138 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB138_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB138_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB138.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB138.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB138.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB138.hist\n",
      "[FLASH]     ./SB138.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 11925 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      11925\n",
      "[FLASH]     Combined pairs:   7192\n",
      "[FLASH]     Uncombined pairs: 4733\n",
      "[FLASH]     Percent combined: 60.31%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.205 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "7192 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB138/SB138_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB138_centroids.fasta --uc SB138.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 7192 sequences and identified 2542 clusters (clustering threshold 1.00) - 182 clusters (representing 4448 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB139 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB139_1.fastq /home/working/demultiplexed/SB139_2.fastq SB139_forward.paired.fastq.gz SB139_forward.singletons.fastq.gz SB139_reverse.paired.fastq.gz SB139_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB139_1.fastq /home/working/demultiplexed/SB139_2.fastq SB139_forward.paired.fastq.gz SB139_forward.singletons.fastq.gz SB139_reverse.paired.fastq.gz SB139_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:13:52 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB139/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 32952 Both Surviving: 6139 (18.63%) Forward Only Surviving: 1682 (5.10%) Reverse Only Surviving: 480 (1.46%) Dropped: 24651 (74.81%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SB139_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB139_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB139_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB139_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB139_forward.paired.headclipped.fastq.gz SB139_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB139 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB139_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB139_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB139.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB139.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB139.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB139.hist\n",
      "[FLASH]     ./SB139.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 6139 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      6139\n",
      "[FLASH]     Combined pairs:   3251\n",
      "[FLASH]     Uncombined pairs: 2888\n",
      "[FLASH]     Percent combined: 52.96%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.101 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "3251 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB139/SB139_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB139_centroids.fasta --uc SB139.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 3251 sequences and identified 1707 clusters (clustering threshold 1.00) - 31 clusters (representing 1401 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB140 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB140_1.fastq /home/working/demultiplexed/SB140_2.fastq SB140_forward.paired.fastq.gz SB140_forward.singletons.fastq.gz SB140_reverse.paired.fastq.gz SB140_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB140_1.fastq /home/working/demultiplexed/SB140_2.fastq SB140_forward.paired.fastq.gz SB140_forward.singletons.fastq.gz SB140_reverse.paired.fastq.gz SB140_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:14:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB140/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 11730 Both Surviving: 4648 (39.62%) Forward Only Surviving: 718 (6.12%) Reverse Only Surviving: 205 (1.75%) Dropped: 6159 (52.51%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SB140_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SB140_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB140_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB140_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB140_forward.paired.headclipped.fastq.gz SB140_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB140 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB140_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB140_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB140.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB140.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB140.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB140.hist\n",
      "[FLASH]     ./SB140.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 4648 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      4648\n",
      "[FLASH]     Combined pairs:   4230\n",
      "[FLASH]     Uncombined pairs: 418\n",
      "[FLASH]     Percent combined: 91.01%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.112 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "4230 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB140/SB140_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB140_centroids.fasta --uc SB140.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 4230 sequences and identified 1791 clusters (clustering threshold 1.00) - 140 clusters (representing 2275 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB141 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB141_1.fastq /home/working/demultiplexed/SB141_2.fastq SB141_forward.paired.fastq.gz SB141_forward.singletons.fastq.gz SB141_reverse.paired.fastq.gz SB141_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB141_1.fastq /home/working/demultiplexed/SB141_2.fastq SB141_forward.paired.fastq.gz SB141_forward.singletons.fastq.gz SB141_reverse.paired.fastq.gz SB141_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:14:10 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB141/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 9355 Both Surviving: 173 (1.85%) Forward Only Surviving: 141 (1.51%) Reverse Only Surviving: 88 (0.94%) Dropped: 8953 (95.70%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: SB141_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB141_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SB141_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SB141_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB141_forward.paired.headclipped.fastq.gz SB141_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB141 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB141_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB141_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB141.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB141.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB141.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB141.hist\n",
      "[FLASH]     ./SB141.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 173 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      173\n",
      "[FLASH]     Combined pairs:   44\n",
      "[FLASH]     Uncombined pairs: 129\n",
      "[FLASH]     Percent combined: 25.43%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.003 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "44 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB141/SB141_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB141_centroids.fasta --uc SB141.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 44 sequences and identified 39 clusters (clustering threshold 1.00) - 1 clusters (representing 5 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB142 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB142_1.fastq /home/working/demultiplexed/SB142_2.fastq SB142_forward.paired.fastq.gz SB142_forward.singletons.fastq.gz SB142_reverse.paired.fastq.gz SB142_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB142_1.fastq /home/working/demultiplexed/SB142_2.fastq SB142_forward.paired.fastq.gz SB142_forward.singletons.fastq.gz SB142_reverse.paired.fastq.gz SB142_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:14:12 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB142/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 22361 Both Surviving: 5354 (23.94%) Forward Only Surviving: 1256 (5.62%) Reverse Only Surviving: 779 (3.48%) Dropped: 14972 (66.96%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: SB142_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB142_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB142_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB142_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB142_forward.paired.headclipped.fastq.gz SB142_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB142 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB142_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB142_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB142.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB142.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB142.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB142.hist\n",
      "[FLASH]     ./SB142.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5354 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5354\n",
      "[FLASH]     Combined pairs:   293\n",
      "[FLASH]     Uncombined pairs: 5061\n",
      "[FLASH]     Percent combined: 5.47%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.041 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "293 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB142/SB142_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB142_centroids.fasta --uc SB142.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 293 sequences and identified 239 clusters (clustering threshold 1.00) - 5 clusters (representing 36 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB146 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB146_1.fastq /home/working/demultiplexed/SB146_2.fastq SB146_forward.paired.fastq.gz SB146_forward.singletons.fastq.gz SB146_reverse.paired.fastq.gz SB146_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB146_1.fastq /home/working/demultiplexed/SB146_2.fastq SB146_forward.paired.fastq.gz SB146_forward.singletons.fastq.gz SB146_reverse.paired.fastq.gz SB146_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:14:17 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB146/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 31692 Both Surviving: 5584 (17.62%) Forward Only Surviving: 1183 (3.73%) Reverse Only Surviving: 530 (1.67%) Dropped: 24395 (76.98%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: SB146_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB146_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB146_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB146_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB146_forward.paired.headclipped.fastq.gz SB146_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB146 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB146_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB146_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB146.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB146.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB146.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB146.hist\n",
      "[FLASH]     ./SB146.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5584 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5584\n",
      "[FLASH]     Combined pairs:   208\n",
      "[FLASH]     Uncombined pairs: 5376\n",
      "[FLASH]     Percent combined: 3.72%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.045 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "208 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB146/SB146_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB146_centroids.fasta --uc SB146.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 208 sequences and identified 163 clusters (clustering threshold 1.00) - 1 clusters (representing 32 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB147 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB147_1.fastq /home/working/demultiplexed/SB147_2.fastq SB147_forward.paired.fastq.gz SB147_forward.singletons.fastq.gz SB147_reverse.paired.fastq.gz SB147_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB147_1.fastq /home/working/demultiplexed/SB147_2.fastq SB147_forward.paired.fastq.gz SB147_forward.singletons.fastq.gz SB147_reverse.paired.fastq.gz SB147_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:14:23 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB147/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 32021 Both Surviving: 8758 (27.35%) Forward Only Surviving: 2026 (6.33%) Reverse Only Surviving: 496 (1.55%) Dropped: 20741 (64.77%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SB147_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB147_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB147_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB147_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB147_forward.paired.headclipped.fastq.gz SB147_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB147 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB147_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB147_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB147.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB147.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB147.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB147.hist\n",
      "[FLASH]     ./SB147.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 8758 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      8758\n",
      "[FLASH]     Combined pairs:   3053\n",
      "[FLASH]     Uncombined pairs: 5705\n",
      "[FLASH]     Percent combined: 34.86%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.117 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "3053 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB147/SB147_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB147_centroids.fasta --uc SB147.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 3053 sequences and identified 1676 clusters (clustering threshold 1.00) - 32 clusters (representing 1242 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB15 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB15_1.fastq /home/working/demultiplexed/SB15_2.fastq SB15_forward.paired.fastq.gz SB15_forward.singletons.fastq.gz SB15_reverse.paired.fastq.gz SB15_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB15_1.fastq /home/working/demultiplexed/SB15_2.fastq SB15_forward.paired.fastq.gz SB15_forward.singletons.fastq.gz SB15_reverse.paired.fastq.gz SB15_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:14:35 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB15/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 65483 Both Surviving: 3843 (5.87%) Forward Only Surviving: 1745 (2.66%) Reverse Only Surviving: 903 (1.38%) Dropped: 58992 (90.09%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: SB15_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: SB15_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB15_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB15_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB15_forward.paired.headclipped.fastq.gz SB15_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB15 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB15_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB15_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB15.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB15.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB15.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB15.hist\n",
      "[FLASH]     ./SB15.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3843 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3843\n",
      "[FLASH]     Combined pairs:   500\n",
      "[FLASH]     Uncombined pairs: 3343\n",
      "[FLASH]     Percent combined: 13.01%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.049 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "500 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB15/SB15_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB15_centroids.fasta --uc SB15.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 500 sequences and identified 264 clusters (clustering threshold 1.00) - 8 clusters (representing 214 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB150 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB150_1.fastq /home/working/demultiplexed/SB150_2.fastq SB150_forward.paired.fastq.gz SB150_forward.singletons.fastq.gz SB150_reverse.paired.fastq.gz SB150_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB150_1.fastq /home/working/demultiplexed/SB150_2.fastq SB150_forward.paired.fastq.gz SB150_forward.singletons.fastq.gz SB150_reverse.paired.fastq.gz SB150_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:14:42 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB150/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 36268 Both Surviving: 8292 (22.86%) Forward Only Surviving: 2052 (5.66%) Reverse Only Surviving: 552 (1.52%) Dropped: 25372 (69.96%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SB150_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB150_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB150_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB150_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB150_forward.paired.headclipped.fastq.gz SB150_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB150 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB150_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB150_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB150.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB150.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB150.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB150.hist\n",
      "[FLASH]     ./SB150.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 8292 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      8292\n",
      "[FLASH]     Combined pairs:   3195\n",
      "[FLASH]     Uncombined pairs: 5097\n",
      "[FLASH]     Percent combined: 38.53%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.123 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "3195 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB150/SB150_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB150_centroids.fasta --uc SB150.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 3195 sequences and identified 1886 clusters (clustering threshold 1.00) - 37 clusters (representing 1107 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB153 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB153_1.fastq /home/working/demultiplexed/SB153_2.fastq SB153_forward.paired.fastq.gz SB153_forward.singletons.fastq.gz SB153_reverse.paired.fastq.gz SB153_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB153_1.fastq /home/working/demultiplexed/SB153_2.fastq SB153_forward.paired.fastq.gz SB153_forward.singletons.fastq.gz SB153_reverse.paired.fastq.gz SB153_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:14:54 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB153/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 41820 Both Surviving: 11773 (28.15%) Forward Only Surviving: 3169 (7.58%) Reverse Only Surviving: 853 (2.04%) Dropped: 26025 (62.23%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SB153_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB153_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB153_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB153_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB153_forward.paired.headclipped.fastq.gz SB153_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB153 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB153_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB153_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB153.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB153.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB153.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB153.hist\n",
      "[FLASH]     ./SB153.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 11773 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      11773\n",
      "[FLASH]     Combined pairs:   1450\n",
      "[FLASH]     Uncombined pairs: 10323\n",
      "[FLASH]     Percent combined: 12.32%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.114 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1450 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB153/SB153_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB153_centroids.fasta --uc SB153.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1450 sequences and identified 839 clusters (clustering threshold 1.00) - 18 clusters (representing 558 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB159 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB159_1.fastq /home/working/demultiplexed/SB159_2.fastq SB159_forward.paired.fastq.gz SB159_forward.singletons.fastq.gz SB159_reverse.paired.fastq.gz SB159_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB159_1.fastq /home/working/demultiplexed/SB159_2.fastq SB159_forward.paired.fastq.gz SB159_forward.singletons.fastq.gz SB159_reverse.paired.fastq.gz SB159_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:15:05 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB159/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 37399 Both Surviving: 7205 (19.27%) Forward Only Surviving: 1797 (4.80%) Reverse Only Surviving: 683 (1.83%) Dropped: 27714 (74.10%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SB159_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB159_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB159_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB159_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB159_forward.paired.headclipped.fastq.gz SB159_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB159 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB159_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB159_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB159.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB159.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB159.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB159.hist\n",
      "[FLASH]     ./SB159.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 7205 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      7205\n",
      "[FLASH]     Combined pairs:   60\n",
      "[FLASH]     Uncombined pairs: 7145\n",
      "[FLASH]     Percent combined: 0.83%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.057 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "60 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB159/SB159_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB159_centroids.fasta --uc SB159.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 60 sequences and identified 51 clusters (clustering threshold 1.00) - 1 clusters (representing 9 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB163 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB163_1.fastq /home/working/demultiplexed/SB163_2.fastq SB163_forward.paired.fastq.gz SB163_forward.singletons.fastq.gz SB163_reverse.paired.fastq.gz SB163_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB163_1.fastq /home/working/demultiplexed/SB163_2.fastq SB163_forward.paired.fastq.gz SB163_forward.singletons.fastq.gz SB163_reverse.paired.fastq.gz SB163_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:15:15 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB163/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 80933 Both Surviving: 14908 (18.42%) Forward Only Surviving: 5632 (6.96%) Reverse Only Surviving: 1070 (1.32%) Dropped: 59323 (73.30%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: SB163_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: SB163_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB163_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB163_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB163_forward.paired.headclipped.fastq.gz SB163_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB163 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB163_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB163_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB163.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB163.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB163.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB163.hist\n",
      "[FLASH]     ./SB163.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 14908 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      14908\n",
      "[FLASH]     Combined pairs:   8755\n",
      "[FLASH]     Uncombined pairs: 6153\n",
      "[FLASH]     Percent combined: 58.73%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.346 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "8755 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB163/SB163_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB163_centroids.fasta --uc SB163.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 8755 sequences and identified 2594 clusters (clustering threshold 1.00) - 208 clusters (representing 5810 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB166 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB166_1.fastq /home/working/demultiplexed/SB166_2.fastq SB166_forward.paired.fastq.gz SB166_forward.singletons.fastq.gz SB166_reverse.paired.fastq.gz SB166_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB166_1.fastq /home/working/demultiplexed/SB166_2.fastq SB166_forward.paired.fastq.gz SB166_forward.singletons.fastq.gz SB166_reverse.paired.fastq.gz SB166_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:15:40 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB166/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 24390 Both Surviving: 5441 (22.31%) Forward Only Surviving: 1502 (6.16%) Reverse Only Surviving: 319 (1.31%) Dropped: 17128 (70.23%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SB166_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB166_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB166_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB166_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB166_forward.paired.headclipped.fastq.gz SB166_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB166 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB166_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB166_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB166.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB166.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB166.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB166.hist\n",
      "[FLASH]     ./SB166.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5441 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5441\n",
      "[FLASH]     Combined pairs:   2584\n",
      "[FLASH]     Uncombined pairs: 2857\n",
      "[FLASH]     Percent combined: 47.49%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.092 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2584 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB166/SB166_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB166_centroids.fasta --uc SB166.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2584 sequences and identified 1272 clusters (clustering threshold 1.00) - 24 clusters (representing 1096 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB170 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB170_1.fastq /home/working/demultiplexed/SB170_2.fastq SB170_forward.paired.fastq.gz SB170_forward.singletons.fastq.gz SB170_reverse.paired.fastq.gz SB170_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB170_1.fastq /home/working/demultiplexed/SB170_2.fastq SB170_forward.paired.fastq.gz SB170_forward.singletons.fastq.gz SB170_reverse.paired.fastq.gz SB170_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:15:49 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB170/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 40667 Both Surviving: 9713 (23.88%) Forward Only Surviving: 2448 (6.02%) Reverse Only Surviving: 653 (1.61%) Dropped: 27853 (68.49%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SB170_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB170_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB170_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB170_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB170_forward.paired.headclipped.fastq.gz SB170_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB170 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB170_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB170_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB170.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB170.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB170.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB170.hist\n",
      "[FLASH]     ./SB170.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 9713 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      9713\n",
      "[FLASH]     Combined pairs:   2788\n",
      "[FLASH]     Uncombined pairs: 6925\n",
      "[FLASH]     Percent combined: 28.70%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.119 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2788 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB170/SB170_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB170_centroids.fasta --uc SB170.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2788 sequences and identified 1482 clusters (clustering threshold 1.00) - 26 clusters (representing 1114 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB171 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB171_1.fastq /home/working/demultiplexed/SB171_2.fastq SB171_forward.paired.fastq.gz SB171_forward.singletons.fastq.gz SB171_reverse.paired.fastq.gz SB171_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB171_1.fastq /home/working/demultiplexed/SB171_2.fastq SB171_forward.paired.fastq.gz SB171_forward.singletons.fastq.gz SB171_reverse.paired.fastq.gz SB171_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:16:00 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB171/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 14032 Both Surviving: 3915 (27.90%) Forward Only Surviving: 1302 (9.28%) Reverse Only Surviving: 152 (1.08%) Dropped: 8663 (61.74%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SB171_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB171_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SB171_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SB171_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB171_forward.paired.headclipped.fastq.gz SB171_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB171 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB171_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB171_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB171.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB171.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB171.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB171.hist\n",
      "[FLASH]     ./SB171.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3915 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3915\n",
      "[FLASH]     Combined pairs:   265\n",
      "[FLASH]     Uncombined pairs: 3650\n",
      "[FLASH]     Percent combined: 6.77%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.046 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "265 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB171/SB171_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB171_centroids.fasta --uc SB171.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 265 sequences and identified 233 clusters (clustering threshold 1.00) - 2 clusters (representing 17 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB172 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB172_1.fastq /home/working/demultiplexed/SB172_2.fastq SB172_forward.paired.fastq.gz SB172_forward.singletons.fastq.gz SB172_reverse.paired.fastq.gz SB172_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB172_1.fastq /home/working/demultiplexed/SB172_2.fastq SB172_forward.paired.fastq.gz SB172_forward.singletons.fastq.gz SB172_reverse.paired.fastq.gz SB172_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:16:05 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB172/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 43254 Both Surviving: 10559 (24.41%) Forward Only Surviving: 2599 (6.01%) Reverse Only Surviving: 1370 (3.17%) Dropped: 28726 (66.41%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: SB172_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB172_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB172_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB172_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB172_forward.paired.headclipped.fastq.gz SB172_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB172 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB172_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB172_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB172.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB172.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB172.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB172.hist\n",
      "[FLASH]     ./SB172.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 10559 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      10559\n",
      "[FLASH]     Combined pairs:   737\n",
      "[FLASH]     Uncombined pairs: 9822\n",
      "[FLASH]     Percent combined: 6.98%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.100 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "737 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB172/SB172_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB172_centroids.fasta --uc SB172.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 737 sequences and identified 480 clusters (clustering threshold 1.00) - 11 clusters (representing 227 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB180 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB180_1.fastq /home/working/demultiplexed/SB180_2.fastq SB180_forward.paired.fastq.gz SB180_forward.singletons.fastq.gz SB180_reverse.paired.fastq.gz SB180_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB180_1.fastq /home/working/demultiplexed/SB180_2.fastq SB180_forward.paired.fastq.gz SB180_forward.singletons.fastq.gz SB180_reverse.paired.fastq.gz SB180_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:16:14 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB180/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 34105 Both Surviving: 6549 (19.20%) Forward Only Surviving: 1906 (5.59%) Reverse Only Surviving: 465 (1.36%) Dropped: 25185 (73.85%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SB180_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB180_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB180_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB180_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB180_forward.paired.headclipped.fastq.gz SB180_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB180 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB180_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB180_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB180.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB180.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB180.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB180.hist\n",
      "[FLASH]     ./SB180.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 6549 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      6549\n",
      "[FLASH]     Combined pairs:   2670\n",
      "[FLASH]     Uncombined pairs: 3879\n",
      "[FLASH]     Percent combined: 40.77%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.091 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2670 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB180/SB180_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB180_centroids.fasta --uc SB180.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2670 sequences and identified 1420 clusters (clustering threshold 1.00) - 24 clusters (representing 1121 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB182 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB182_1.fastq /home/working/demultiplexed/SB182_2.fastq SB182_forward.paired.fastq.gz SB182_forward.singletons.fastq.gz SB182_reverse.paired.fastq.gz SB182_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB182_1.fastq /home/working/demultiplexed/SB182_2.fastq SB182_forward.paired.fastq.gz SB182_forward.singletons.fastq.gz SB182_reverse.paired.fastq.gz SB182_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:16:26 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB182/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 49576 Both Surviving: 154 (0.31%) Forward Only Surviving: 517 (1.04%) Reverse Only Surviving: 627 (1.26%) Dropped: 48278 (97.38%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SB182_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SB182_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB182_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB182_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB182_forward.paired.headclipped.fastq.gz SB182_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB182 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB182_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB182_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB182.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB182.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB182.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB182.hist\n",
      "[FLASH]     ./SB182.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 154 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      154\n",
      "[FLASH]     Combined pairs:   54\n",
      "[FLASH]     Uncombined pairs: 100\n",
      "[FLASH]     Percent combined: 35.06%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.003 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "54 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB182/SB182_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB182_centroids.fasta --uc SB182.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 54 sequences and identified 45 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB183 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB183_1.fastq /home/working/demultiplexed/SB183_2.fastq SB183_forward.paired.fastq.gz SB183_forward.singletons.fastq.gz SB183_reverse.paired.fastq.gz SB183_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB183_1.fastq /home/working/demultiplexed/SB183_2.fastq SB183_forward.paired.fastq.gz SB183_forward.singletons.fastq.gz SB183_reverse.paired.fastq.gz SB183_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:16:30 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB183/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 14364 Both Surviving: 316 (2.20%) Forward Only Surviving: 343 (2.39%) Reverse Only Surviving: 130 (0.91%) Dropped: 13575 (94.51%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SB183_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SB183_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB183_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB183_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB183_forward.paired.headclipped.fastq.gz SB183_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB183 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB183_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB183_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB183.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB183.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB183.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB183.hist\n",
      "[FLASH]     ./SB183.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 316 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      316\n",
      "[FLASH]     Combined pairs:   237\n",
      "[FLASH]     Uncombined pairs: 79\n",
      "[FLASH]     Percent combined: 75.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.008 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "237 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB183/SB183_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB183_centroids.fasta --uc SB183.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 237 sequences and identified 140 clusters (clustering threshold 1.00) - 7 clusters (representing 91 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB186 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB186_1.fastq /home/working/demultiplexed/SB186_2.fastq SB186_forward.paired.fastq.gz SB186_forward.singletons.fastq.gz SB186_reverse.paired.fastq.gz SB186_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB186_1.fastq /home/working/demultiplexed/SB186_2.fastq SB186_forward.paired.fastq.gz SB186_forward.singletons.fastq.gz SB186_reverse.paired.fastq.gz SB186_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:16:35 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB186/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 72012 Both Surviving: 3107 (4.31%) Forward Only Surviving: 2444 (3.39%) Reverse Only Surviving: 755 (1.05%) Dropped: 65706 (91.24%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SB186_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SB186_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB186_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB186_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB186_forward.paired.headclipped.fastq.gz SB186_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB186 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB186_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB186_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB186.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB186.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB186.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB186.hist\n",
      "[FLASH]     ./SB186.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3107 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3107\n",
      "[FLASH]     Combined pairs:   2217\n",
      "[FLASH]     Uncombined pairs: 890\n",
      "[FLASH]     Percent combined: 71.36%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.059 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2217 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB186/SB186_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB186_centroids.fasta --uc SB186.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2217 sequences and identified 829 clusters (clustering threshold 1.00) - 35 clusters (representing 1212 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB194 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB194_1.fastq /home/working/demultiplexed/SB194_2.fastq SB194_forward.paired.fastq.gz SB194_forward.singletons.fastq.gz SB194_reverse.paired.fastq.gz SB194_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB194_1.fastq /home/working/demultiplexed/SB194_2.fastq SB194_forward.paired.fastq.gz SB194_forward.singletons.fastq.gz SB194_reverse.paired.fastq.gz SB194_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:16:47 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB194/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 97194 Both Surviving: 24529 (25.24%) Forward Only Surviving: 8080 (8.31%) Reverse Only Surviving: 1643 (1.69%) Dropped: 62942 (64.76%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: SB194_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB194_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB194_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB194_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB194_forward.paired.headclipped.fastq.gz SB194_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB194 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB194_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB194_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB194.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB194.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB194.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB194.hist\n",
      "[FLASH]     ./SB194.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 24529 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      24529\n",
      "[FLASH]     Combined pairs:   2260\n",
      "[FLASH]     Uncombined pairs: 22269\n",
      "[FLASH]     Percent combined: 9.21%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.183 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2260 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB194/SB194_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB194_centroids.fasta --uc SB194.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2260 sequences and identified 1427 clusters (clustering threshold 1.00) - 32 clusters (representing 629 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB195 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB195_1.fastq /home/working/demultiplexed/SB195_2.fastq SB195_forward.paired.fastq.gz SB195_forward.singletons.fastq.gz SB195_reverse.paired.fastq.gz SB195_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB195_1.fastq /home/working/demultiplexed/SB195_2.fastq SB195_forward.paired.fastq.gz SB195_forward.singletons.fastq.gz SB195_reverse.paired.fastq.gz SB195_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:17:04 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB195/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 25672 Both Surviving: 6139 (23.91%) Forward Only Surviving: 920 (3.58%) Reverse Only Surviving: 954 (3.72%) Dropped: 17659 (68.79%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: SB195_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB195_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB195_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB195_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB195_forward.paired.headclipped.fastq.gz SB195_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB195 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB195_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB195_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB195.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB195.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB195.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB195.hist\n",
      "[FLASH]     ./SB195.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 6139 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      6139\n",
      "[FLASH]     Combined pairs:   198\n",
      "[FLASH]     Uncombined pairs: 5941\n",
      "[FLASH]     Percent combined: 3.23%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.057 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "198 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB195/SB195_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB195_centroids.fasta --uc SB195.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 198 sequences and identified 155 clusters (clustering threshold 1.00) - 3 clusters (representing 27 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB197 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB197_1.fastq /home/working/demultiplexed/SB197_2.fastq SB197_forward.paired.fastq.gz SB197_forward.singletons.fastq.gz SB197_reverse.paired.fastq.gz SB197_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB197_1.fastq /home/working/demultiplexed/SB197_2.fastq SB197_forward.paired.fastq.gz SB197_forward.singletons.fastq.gz SB197_reverse.paired.fastq.gz SB197_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:17:11 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB197/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 53032 Both Surviving: 13391 (25.25%) Forward Only Surviving: 3997 (7.54%) Reverse Only Surviving: 831 (1.57%) Dropped: 34813 (65.65%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SB197_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB197_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB197_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB197_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB197_forward.paired.headclipped.fastq.gz SB197_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB197 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB197_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB197_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB197.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB197.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB197.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB197.hist\n",
      "[FLASH]     ./SB197.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 13391 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      13391\n",
      "[FLASH]     Combined pairs:   461\n",
      "[FLASH]     Uncombined pairs: 12930\n",
      "[FLASH]     Percent combined: 3.44%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.092 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "461 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB197/SB197_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB197_centroids.fasta --uc SB197.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 461 sequences and identified 319 clusters (clustering threshold 1.00) - 9 clusters (representing 105 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB199 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB199_1.fastq /home/working/demultiplexed/SB199_2.fastq SB199_forward.paired.fastq.gz SB199_forward.singletons.fastq.gz SB199_reverse.paired.fastq.gz SB199_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB199_1.fastq /home/working/demultiplexed/SB199_2.fastq SB199_forward.paired.fastq.gz SB199_forward.singletons.fastq.gz SB199_reverse.paired.fastq.gz SB199_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:17:19 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB199/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 20753 Both Surviving: 2795 (13.47%) Forward Only Surviving: 931 (4.49%) Reverse Only Surviving: 313 (1.51%) Dropped: 16714 (80.54%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SB199_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SB199_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB199_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB199_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB199_forward.paired.headclipped.fastq.gz SB199_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB199 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB199_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB199_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB199.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB199.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB199.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB199.hist\n",
      "[FLASH]     ./SB199.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2795 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2795\n",
      "[FLASH]     Combined pairs:   56\n",
      "[FLASH]     Uncombined pairs: 2739\n",
      "[FLASH]     Percent combined: 2.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.025 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "56 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB199/SB199_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB199_centroids.fasta --uc SB199.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 56 sequences and identified 49 clusters (clustering threshold 1.00) - 1 clusters (representing 5 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB200 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB200_1.fastq /home/working/demultiplexed/SB200_2.fastq SB200_forward.paired.fastq.gz SB200_forward.singletons.fastq.gz SB200_reverse.paired.fastq.gz SB200_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB200_1.fastq /home/working/demultiplexed/SB200_2.fastq SB200_forward.paired.fastq.gz SB200_forward.singletons.fastq.gz SB200_reverse.paired.fastq.gz SB200_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:17:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB200/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 22 bases off reads in file: SB200_forward.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB200_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB200_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB200_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB200_forward.paired.headclipped.fastq.gz SB200_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB200 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB200_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB200_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB200.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB200.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB200.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB200.hist\n",
      "[FLASH]     ./SB200.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      0\n",
      "[FLASH]     Combined pairs:   0\n",
      "[FLASH]     Uncombined pairs: 0\n",
      "[FLASH]     Percent combined: 0.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "0 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB200/SB200_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB200_centroids.fasta --uc SB200.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB201 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB201_1.fastq /home/working/demultiplexed/SB201_2.fastq SB201_forward.paired.fastq.gz SB201_forward.singletons.fastq.gz SB201_reverse.paired.fastq.gz SB201_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB201_1.fastq /home/working/demultiplexed/SB201_2.fastq SB201_forward.paired.fastq.gz SB201_forward.singletons.fastq.gz SB201_reverse.paired.fastq.gz SB201_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:17:23 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB201/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 33178 Both Surviving: 1958 (5.90%) Forward Only Surviving: 1408 (4.24%) Reverse Only Surviving: 279 (0.84%) Dropped: 29533 (89.01%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SB201_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SB201_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB201_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB201_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB201_forward.paired.headclipped.fastq.gz SB201_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB201 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB201_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB201_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB201.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB201.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB201.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB201.hist\n",
      "[FLASH]     ./SB201.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1958 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1958\n",
      "[FLASH]     Combined pairs:   826\n",
      "[FLASH]     Uncombined pairs: 1132\n",
      "[FLASH]     Percent combined: 42.19%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.029 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "826 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB201/SB201_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB201_centroids.fasta --uc SB201.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 826 sequences and identified 569 clusters (clustering threshold 1.00) - 19 clusters (representing 228 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB204 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB204_1.fastq /home/working/demultiplexed/SB204_2.fastq SB204_forward.paired.fastq.gz SB204_forward.singletons.fastq.gz SB204_reverse.paired.fastq.gz SB204_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB204_1.fastq /home/working/demultiplexed/SB204_2.fastq SB204_forward.paired.fastq.gz SB204_forward.singletons.fastq.gz SB204_reverse.paired.fastq.gz SB204_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:17:32 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB204/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 99793 Both Surviving: 5432 (5.44%) Forward Only Surviving: 2232 (2.24%) Reverse Only Surviving: 1231 (1.23%) Dropped: 90898 (91.09%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SB204_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SB204_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB204_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB204_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB204_forward.paired.headclipped.fastq.gz SB204_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB204 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB204_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB204_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB204.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB204.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB204.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB204.hist\n",
      "[FLASH]     ./SB204.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5432 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5432\n",
      "[FLASH]     Combined pairs:   4767\n",
      "[FLASH]     Uncombined pairs: 665\n",
      "[FLASH]     Percent combined: 87.76%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.131 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "4767 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB204/SB204_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB204_centroids.fasta --uc SB204.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 4767 sequences and identified 1938 clusters (clustering threshold 1.00) - 70 clusters (representing 2414 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB21 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB21_1.fastq /home/working/demultiplexed/SB21_2.fastq SB21_forward.paired.fastq.gz SB21_forward.singletons.fastq.gz SB21_reverse.paired.fastq.gz SB21_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB21_1.fastq /home/working/demultiplexed/SB21_2.fastq SB21_forward.paired.fastq.gz SB21_forward.singletons.fastq.gz SB21_reverse.paired.fastq.gz SB21_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:17:47 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB21/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 25058 Both Surviving: 5378 (21.46%) Forward Only Surviving: 1236 (4.93%) Reverse Only Surviving: 413 (1.65%) Dropped: 18031 (71.96%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SB21_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SB21_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB21_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB21_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB21_forward.paired.headclipped.fastq.gz SB21_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB21 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB21_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB21_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB21.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB21.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB21.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB21.hist\n",
      "[FLASH]     ./SB21.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5378 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5378\n",
      "[FLASH]     Combined pairs:   3058\n",
      "[FLASH]     Uncombined pairs: 2320\n",
      "[FLASH]     Percent combined: 56.86%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.114 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "3058 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB21/SB21_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB21_centroids.fasta --uc SB21.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 3058 sequences and identified 1907 clusters (clustering threshold 1.00) - 45 clusters (representing 1006 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB22 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB22_1.fastq /home/working/demultiplexed/SB22_2.fastq SB22_forward.paired.fastq.gz SB22_forward.singletons.fastq.gz SB22_reverse.paired.fastq.gz SB22_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB22_1.fastq /home/working/demultiplexed/SB22_2.fastq SB22_forward.paired.fastq.gz SB22_forward.singletons.fastq.gz SB22_reverse.paired.fastq.gz SB22_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:17:57 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB22/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 37803 Both Surviving: 9518 (25.18%) Forward Only Surviving: 2446 (6.47%) Reverse Only Surviving: 592 (1.57%) Dropped: 25247 (66.79%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SB22_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SB22_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB22_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB22_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB22_forward.paired.headclipped.fastq.gz SB22_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB22 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB22_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB22_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB22.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB22.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB22.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB22.hist\n",
      "[FLASH]     ./SB22.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 9518 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      9518\n",
      "[FLASH]     Combined pairs:   4105\n",
      "[FLASH]     Uncombined pairs: 5413\n",
      "[FLASH]     Percent combined: 43.13%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.143 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "4105 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB22/SB22_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB22_centroids.fasta --uc SB22.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 4105 sequences and identified 1997 clusters (clustering threshold 1.00) - 40 clusters (representing 1786 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB23 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB23_1.fastq /home/working/demultiplexed/SB23_2.fastq SB23_forward.paired.fastq.gz SB23_forward.singletons.fastq.gz SB23_reverse.paired.fastq.gz SB23_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB23_1.fastq /home/working/demultiplexed/SB23_2.fastq SB23_forward.paired.fastq.gz SB23_forward.singletons.fastq.gz SB23_reverse.paired.fastq.gz SB23_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:18:08 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB23/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 22 bases off reads in file: SB23_forward.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB23_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB23_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB23_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB23_forward.paired.headclipped.fastq.gz SB23_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB23 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB23_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB23_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB23.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB23.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB23.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB23.hist\n",
      "[FLASH]     ./SB23.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      0\n",
      "[FLASH]     Combined pairs:   0\n",
      "[FLASH]     Uncombined pairs: 0\n",
      "[FLASH]     Percent combined: 0.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "0 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB23/SB23_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB23_centroids.fasta --uc SB23.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB24 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB24_1.fastq /home/working/demultiplexed/SB24_2.fastq SB24_forward.paired.fastq.gz SB24_forward.singletons.fastq.gz SB24_reverse.paired.fastq.gz SB24_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB24_1.fastq /home/working/demultiplexed/SB24_2.fastq SB24_forward.paired.fastq.gz SB24_forward.singletons.fastq.gz SB24_reverse.paired.fastq.gz SB24_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:18:10 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB24/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 28519 Both Surviving: 11971 (41.98%) Forward Only Surviving: 2534 (8.89%) Reverse Only Surviving: 494 (1.73%) Dropped: 13520 (47.41%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SB24_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB24_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB24_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB24_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB24_forward.paired.headclipped.fastq.gz SB24_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB24 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB24_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB24_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB24.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB24.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB24.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB24.hist\n",
      "[FLASH]     ./SB24.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 11971 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      11971\n",
      "[FLASH]     Combined pairs:   6050\n",
      "[FLASH]     Uncombined pairs: 5921\n",
      "[FLASH]     Percent combined: 50.54%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.168 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "6050 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB24/SB24_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB24_centroids.fasta --uc SB24.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 6050 sequences and identified 3589 clusters (clustering threshold 1.00) - 138 clusters (representing 2114 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB27 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB27_1.fastq /home/working/demultiplexed/SB27_2.fastq SB27_forward.paired.fastq.gz SB27_forward.singletons.fastq.gz SB27_reverse.paired.fastq.gz SB27_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB27_1.fastq /home/working/demultiplexed/SB27_2.fastq SB27_forward.paired.fastq.gz SB27_forward.singletons.fastq.gz SB27_reverse.paired.fastq.gz SB27_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:18:24 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB27/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 15596 Both Surviving: 2718 (17.43%) Forward Only Surviving: 745 (4.78%) Reverse Only Surviving: 204 (1.31%) Dropped: 11929 (76.49%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SB27_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB27_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB27_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB27_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB27_forward.paired.headclipped.fastq.gz SB27_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB27 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB27_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB27_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB27.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB27.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB27.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB27.hist\n",
      "[FLASH]     ./SB27.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2718 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2718\n",
      "[FLASH]     Combined pairs:   55\n",
      "[FLASH]     Uncombined pairs: 2663\n",
      "[FLASH]     Percent combined: 2.02%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.025 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "55 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB27/SB27_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB27_centroids.fasta --uc SB27.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 55 sequences and identified 46 clusters (clustering threshold 1.00) - 1 clusters (representing 5 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB29 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB29_1.fastq /home/working/demultiplexed/SB29_2.fastq SB29_forward.paired.fastq.gz SB29_forward.singletons.fastq.gz SB29_reverse.paired.fastq.gz SB29_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB29_1.fastq /home/working/demultiplexed/SB29_2.fastq SB29_forward.paired.fastq.gz SB29_forward.singletons.fastq.gz SB29_reverse.paired.fastq.gz SB29_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:18:27 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB29/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 37392 Both Surviving: 1224 (3.27%) Forward Only Surviving: 976 (2.61%) Reverse Only Surviving: 315 (0.84%) Dropped: 34877 (93.27%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: SB29_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB29_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB29_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB29_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB29_forward.paired.headclipped.fastq.gz SB29_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB29 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB29_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB29_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB29.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB29.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB29.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB29.hist\n",
      "[FLASH]     ./SB29.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1224 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1224\n",
      "[FLASH]     Combined pairs:   867\n",
      "[FLASH]     Uncombined pairs: 357\n",
      "[FLASH]     Percent combined: 70.83%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.022 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "867 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB29/SB29_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB29_centroids.fasta --uc SB29.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 867 sequences and identified 475 clusters (clustering threshold 1.00) - 12 clusters (representing 341 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB3 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB3_1.fastq /home/working/demultiplexed/SB3_2.fastq SB3_forward.paired.fastq.gz SB3_forward.singletons.fastq.gz SB3_reverse.paired.fastq.gz SB3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB3_1.fastq /home/working/demultiplexed/SB3_2.fastq SB3_forward.paired.fastq.gz SB3_forward.singletons.fastq.gz SB3_reverse.paired.fastq.gz SB3_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:18:36 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB3/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 107968 Both Surviving: 4015 (3.72%) Forward Only Surviving: 5331 (4.94%) Reverse Only Surviving: 1056 (0.98%) Dropped: 97566 (90.37%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SB3_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SB3_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB3_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB3_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB3_forward.paired.headclipped.fastq.gz SB3_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB3 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB3_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB3_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB3.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB3.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB3.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB3.hist\n",
      "[FLASH]     ./SB3.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 4015 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      4015\n",
      "[FLASH]     Combined pairs:   1099\n",
      "[FLASH]     Uncombined pairs: 2916\n",
      "[FLASH]     Percent combined: 27.37%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.033 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1099 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB3/SB3_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB3_centroids.fasta --uc SB3.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1099 sequences and identified 670 clusters (clustering threshold 1.00) - 31 clusters (representing 390 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB33 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB33_1.fastq /home/working/demultiplexed/SB33_2.fastq SB33_forward.paired.fastq.gz SB33_forward.singletons.fastq.gz SB33_reverse.paired.fastq.gz SB33_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB33_1.fastq /home/working/demultiplexed/SB33_2.fastq SB33_forward.paired.fastq.gz SB33_forward.singletons.fastq.gz SB33_reverse.paired.fastq.gz SB33_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:18:59 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB33/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 250333 Both Surviving: 102563 (40.97%) Forward Only Surviving: 23564 (9.41%) Reverse Only Surviving: 4893 (1.95%) Dropped: 119313 (47.66%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SB33_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB33_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB33_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB33_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB33_forward.paired.headclipped.fastq.gz SB33_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB33 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB33_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB33_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB33.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB33.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB33.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB33.hist\n",
      "[FLASH]     ./SB33.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 25000 read pairs\n",
      "[FLASH] Processed 50000 read pairs\n",
      "[FLASH] Processed 75000 read pairs\n",
      "[FLASH] Processed 100000 read pairs\n",
      "[FLASH] Processed 102563 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      102563\n",
      "[FLASH]     Combined pairs:   60103\n",
      "[FLASH]     Uncombined pairs: 42460\n",
      "[FLASH]     Percent combined: 58.60%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 1.585 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "60103 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB33/SB33_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB33_centroids.fasta --uc SB33.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 60103 sequences and identified 22427 clusters (clustering threshold 1.00) - 747 clusters (representing 35660 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB34 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB34_1.fastq /home/working/demultiplexed/SB34_2.fastq SB34_forward.paired.fastq.gz SB34_forward.singletons.fastq.gz SB34_reverse.paired.fastq.gz SB34_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB34_1.fastq /home/working/demultiplexed/SB34_2.fastq SB34_forward.paired.fastq.gz SB34_forward.singletons.fastq.gz SB34_reverse.paired.fastq.gz SB34_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:21:23 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB34/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 5099 Both Surviving: 1755 (34.42%) Forward Only Surviving: 415 (8.14%) Reverse Only Surviving: 67 (1.31%) Dropped: 2862 (56.13%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SB34_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SB34_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB34_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB34_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB34_forward.paired.headclipped.fastq.gz SB34_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB34 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB34_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB34_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB34.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB34.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB34.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB34.hist\n",
      "[FLASH]     ./SB34.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1755 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1755\n",
      "[FLASH]     Combined pairs:   721\n",
      "[FLASH]     Uncombined pairs: 1034\n",
      "[FLASH]     Percent combined: 41.08%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.023 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "721 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB34/SB34_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB34_centroids.fasta --uc SB34.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 721 sequences and identified 509 clusters (clustering threshold 1.00) - 13 clusters (representing 166 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB37 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB37_1.fastq /home/working/demultiplexed/SB37_2.fastq SB37_forward.paired.fastq.gz SB37_forward.singletons.fastq.gz SB37_reverse.paired.fastq.gz SB37_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB37_1.fastq /home/working/demultiplexed/SB37_2.fastq SB37_forward.paired.fastq.gz SB37_forward.singletons.fastq.gz SB37_reverse.paired.fastq.gz SB37_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:21:25 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB37/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 15290 Both Surviving: 2458 (16.08%) Forward Only Surviving: 686 (4.49%) Reverse Only Surviving: 189 (1.24%) Dropped: 11957 (78.20%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SB37_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SB37_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB37_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB37_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB37_forward.paired.headclipped.fastq.gz SB37_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB37 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB37_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB37_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB37.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB37.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB37.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB37.hist\n",
      "[FLASH]     ./SB37.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2458 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2458\n",
      "[FLASH]     Combined pairs:   62\n",
      "[FLASH]     Uncombined pairs: 2396\n",
      "[FLASH]     Percent combined: 2.52%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.024 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "62 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB37/SB37_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB37_centroids.fasta --uc SB37.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 62 sequences and identified 44 clusters (clustering threshold 1.00) - 1 clusters (representing 7 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB4 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB4_1.fastq /home/working/demultiplexed/SB4_2.fastq SB4_forward.paired.fastq.gz SB4_forward.singletons.fastq.gz SB4_reverse.paired.fastq.gz SB4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB4_1.fastq /home/working/demultiplexed/SB4_2.fastq SB4_forward.paired.fastq.gz SB4_forward.singletons.fastq.gz SB4_reverse.paired.fastq.gz SB4_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:21:28 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB4/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 17746 Both Surviving: 2361 (13.30%) Forward Only Surviving: 616 (3.47%) Reverse Only Surviving: 545 (3.07%) Dropped: 14224 (80.15%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: SB4_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB4_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB4_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB4_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB4_forward.paired.headclipped.fastq.gz SB4_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB4 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB4_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB4_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB4.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB4.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB4.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB4.hist\n",
      "[FLASH]     ./SB4.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2361 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2361\n",
      "[FLASH]     Combined pairs:   110\n",
      "[FLASH]     Uncombined pairs: 2251\n",
      "[FLASH]     Percent combined: 4.66%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.024 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "110 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB4/SB4_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB4_centroids.fasta --uc SB4.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 110 sequences and identified 104 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB45 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB45_1.fastq /home/working/demultiplexed/SB45_2.fastq SB45_forward.paired.fastq.gz SB45_forward.singletons.fastq.gz SB45_reverse.paired.fastq.gz SB45_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB45_1.fastq /home/working/demultiplexed/SB45_2.fastq SB45_forward.paired.fastq.gz SB45_forward.singletons.fastq.gz SB45_reverse.paired.fastq.gz SB45_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:21:37 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB45/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 114934 Both Surviving: 33820 (29.43%) Forward Only Surviving: 8649 (7.53%) Reverse Only Surviving: 1860 (1.62%) Dropped: 70605 (61.43%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SB45_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB45_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB45_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB45_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB45_forward.paired.headclipped.fastq.gz SB45_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB45 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB45_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB45_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB45.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB45.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB45.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB45.hist\n",
      "[FLASH]     ./SB45.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 25000 read pairs\n",
      "[FLASH] Processed 33820 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      33820\n",
      "[FLASH]     Combined pairs:   16731\n",
      "[FLASH]     Uncombined pairs: 17089\n",
      "[FLASH]     Percent combined: 49.47%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.544 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "16731 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB45/SB45_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB45_centroids.fasta --uc SB45.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 16731 sequences and identified 5203 clusters (clustering threshold 1.00) - 315 clusters (representing 10908 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB48 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB48_1.fastq /home/working/demultiplexed/SB48_2.fastq SB48_forward.paired.fastq.gz SB48_forward.singletons.fastq.gz SB48_reverse.paired.fastq.gz SB48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB48_1.fastq /home/working/demultiplexed/SB48_2.fastq SB48_forward.paired.fastq.gz SB48_forward.singletons.fastq.gz SB48_reverse.paired.fastq.gz SB48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:22:16 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB48/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 38763 Both Surviving: 7572 (19.53%) Forward Only Surviving: 1864 (4.81%) Reverse Only Surviving: 658 (1.70%) Dropped: 28669 (73.96%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SB48_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB48_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB48_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB48_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB48_forward.paired.headclipped.fastq.gz SB48_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB48 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB48_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB48_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB48.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB48.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB48.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB48.hist\n",
      "[FLASH]     ./SB48.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 7572 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      7572\n",
      "[FLASH]     Combined pairs:   837\n",
      "[FLASH]     Uncombined pairs: 6735\n",
      "[FLASH]     Percent combined: 11.05%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.066 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "837 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB48/SB48_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB48_centroids.fasta --uc SB48.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 837 sequences and identified 474 clusters (clustering threshold 1.00) - 20 clusters (representing 329 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB51 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB51_1.fastq /home/working/demultiplexed/SB51_2.fastq SB51_forward.paired.fastq.gz SB51_forward.singletons.fastq.gz SB51_reverse.paired.fastq.gz SB51_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB51_1.fastq /home/working/demultiplexed/SB51_2.fastq SB51_forward.paired.fastq.gz SB51_forward.singletons.fastq.gz SB51_reverse.paired.fastq.gz SB51_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:22:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB51/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 2 Both Surviving: 2 (100.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 0 (0.00%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 22 bases off reads in file: SB51_forward.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB51_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB51_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB51_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB51_forward.paired.headclipped.fastq.gz SB51_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB51 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB51_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB51_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB51.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB51.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB51.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB51.hist\n",
      "[FLASH]     ./SB51.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2\n",
      "[FLASH]     Combined pairs:   1\n",
      "[FLASH]     Uncombined pairs: 1\n",
      "[FLASH]     Percent combined: 50.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB51/SB51_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB51_centroids.fasta --uc SB51.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1 sequences and identified 1 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB53 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB53_1.fastq /home/working/demultiplexed/SB53_2.fastq SB53_forward.paired.fastq.gz SB53_forward.singletons.fastq.gz SB53_reverse.paired.fastq.gz SB53_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB53_1.fastq /home/working/demultiplexed/SB53_2.fastq SB53_forward.paired.fastq.gz SB53_forward.singletons.fastq.gz SB53_reverse.paired.fastq.gz SB53_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:22:24 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB53/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 34274 Both Surviving: 8781 (25.62%) Forward Only Surviving: 2596 (7.57%) Reverse Only Surviving: 500 (1.46%) Dropped: 22397 (65.35%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SB53_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB53_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB53_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB53_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB53_forward.paired.headclipped.fastq.gz SB53_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB53 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB53_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB53_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB53.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB53.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB53.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB53.hist\n",
      "[FLASH]     ./SB53.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 8781 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      8781\n",
      "[FLASH]     Combined pairs:   2543\n",
      "[FLASH]     Uncombined pairs: 6238\n",
      "[FLASH]     Percent combined: 28.96%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.093 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2543 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB53/SB53_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB53_centroids.fasta --uc SB53.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2543 sequences and identified 1393 clusters (clustering threshold 1.00) - 20 clusters (representing 1010 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB59 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB59_1.fastq /home/working/demultiplexed/SB59_2.fastq SB59_forward.paired.fastq.gz SB59_forward.singletons.fastq.gz SB59_reverse.paired.fastq.gz SB59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB59_1.fastq /home/working/demultiplexed/SB59_2.fastq SB59_forward.paired.fastq.gz SB59_forward.singletons.fastq.gz SB59_reverse.paired.fastq.gz SB59_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:22:34 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB59/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 37151 Both Surviving: 7895 (21.25%) Forward Only Surviving: 2714 (7.31%) Reverse Only Surviving: 494 (1.33%) Dropped: 26048 (70.11%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SB59_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB59_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB59_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB59_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB59_forward.paired.headclipped.fastq.gz SB59_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB59 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB59_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB59_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB59.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB59.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB59.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB59.hist\n",
      "[FLASH]     ./SB59.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 7895 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      7895\n",
      "[FLASH]     Combined pairs:   2360\n",
      "[FLASH]     Uncombined pairs: 5535\n",
      "[FLASH]     Percent combined: 29.89%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.101 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2360 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB59/SB59_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB59_centroids.fasta --uc SB59.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2360 sequences and identified 1147 clusters (clustering threshold 1.00) - 32 clusters (representing 1081 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB60 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB60_1.fastq /home/working/demultiplexed/SB60_2.fastq SB60_forward.paired.fastq.gz SB60_forward.singletons.fastq.gz SB60_reverse.paired.fastq.gz SB60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB60_1.fastq /home/working/demultiplexed/SB60_2.fastq SB60_forward.paired.fastq.gz SB60_forward.singletons.fastq.gz SB60_reverse.paired.fastq.gz SB60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:22:42 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB60/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 26851 Both Surviving: 9687 (36.08%) Forward Only Surviving: 2603 (9.69%) Reverse Only Surviving: 322 (1.20%) Dropped: 14239 (53.03%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SB60_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB60_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SB60_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SB60_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB60_forward.paired.headclipped.fastq.gz SB60_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB60 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB60_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB60_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB60.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB60.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB60.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB60.hist\n",
      "[FLASH]     ./SB60.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 9687 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      9687\n",
      "[FLASH]     Combined pairs:   4979\n",
      "[FLASH]     Uncombined pairs: 4708\n",
      "[FLASH]     Percent combined: 51.40%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.161 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "4979 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB60/SB60_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB60_centroids.fasta --uc SB60.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 4979 sequences and identified 2895 clusters (clustering threshold 1.00) - 123 clusters (representing 1906 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB63 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB63_1.fastq /home/working/demultiplexed/SB63_2.fastq SB63_forward.paired.fastq.gz SB63_forward.singletons.fastq.gz SB63_reverse.paired.fastq.gz SB63_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB63_1.fastq /home/working/demultiplexed/SB63_2.fastq SB63_forward.paired.fastq.gz SB63_forward.singletons.fastq.gz SB63_reverse.paired.fastq.gz SB63_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:22:54 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB63/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 15119 Both Surviving: 3736 (24.71%) Forward Only Surviving: 845 (5.59%) Reverse Only Surviving: 227 (1.50%) Dropped: 10311 (68.20%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SB63_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB63_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SB63_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SB63_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB63_forward.paired.headclipped.fastq.gz SB63_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB63 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB63_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB63_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB63.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB63.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB63.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB63.hist\n",
      "[FLASH]     ./SB63.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3736 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3736\n",
      "[FLASH]     Combined pairs:   1303\n",
      "[FLASH]     Uncombined pairs: 2433\n",
      "[FLASH]     Percent combined: 34.88%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.040 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1303 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB63/SB63_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB63_centroids.fasta --uc SB63.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1303 sequences and identified 675 clusters (clustering threshold 1.00) - 25 clusters (representing 552 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB64 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB64_1.fastq /home/working/demultiplexed/SB64_2.fastq SB64_forward.paired.fastq.gz SB64_forward.singletons.fastq.gz SB64_reverse.paired.fastq.gz SB64_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB64_1.fastq /home/working/demultiplexed/SB64_2.fastq SB64_forward.paired.fastq.gz SB64_forward.singletons.fastq.gz SB64_reverse.paired.fastq.gz SB64_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:22:58 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB64/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 20694 Both Surviving: 1095 (5.29%) Forward Only Surviving: 979 (4.73%) Reverse Only Surviving: 253 (1.22%) Dropped: 18367 (88.76%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SB64_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SB64_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB64_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB64_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB64_forward.paired.headclipped.fastq.gz SB64_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB64 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB64_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB64_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB64.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB64.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB64.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB64.hist\n",
      "[FLASH]     ./SB64.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1095 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1095\n",
      "[FLASH]     Combined pairs:   180\n",
      "[FLASH]     Uncombined pairs: 915\n",
      "[FLASH]     Percent combined: 16.44%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.011 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "180 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB64/SB64_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB64_centroids.fasta --uc SB64.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 180 sequences and identified 149 clusters (clustering threshold 1.00) - 2 clusters (representing 12 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB67 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB67_1.fastq /home/working/demultiplexed/SB67_2.fastq SB67_forward.paired.fastq.gz SB67_forward.singletons.fastq.gz SB67_reverse.paired.fastq.gz SB67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB67_1.fastq /home/working/demultiplexed/SB67_2.fastq SB67_forward.paired.fastq.gz SB67_forward.singletons.fastq.gz SB67_reverse.paired.fastq.gz SB67_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:23:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB67/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 25560 Both Surviving: 302 (1.18%) Forward Only Surviving: 566 (2.21%) Reverse Only Surviving: 267 (1.04%) Dropped: 24425 (95.56%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: SB67_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: SB67_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB67_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB67_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB67_forward.paired.headclipped.fastq.gz SB67_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB67 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB67_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB67_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB67.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB67.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB67.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB67.hist\n",
      "[FLASH]     ./SB67.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 302 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      302\n",
      "[FLASH]     Combined pairs:   211\n",
      "[FLASH]     Uncombined pairs: 91\n",
      "[FLASH]     Percent combined: 69.87%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.007 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "211 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB67/SB67_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB67_centroids.fasta --uc SB67.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 211 sequences and identified 136 clusters (clustering threshold 1.00) - 4 clusters (representing 77 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB71 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB71_1.fastq /home/working/demultiplexed/SB71_2.fastq SB71_forward.paired.fastq.gz SB71_forward.singletons.fastq.gz SB71_reverse.paired.fastq.gz SB71_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB71_1.fastq /home/working/demultiplexed/SB71_2.fastq SB71_forward.paired.fastq.gz SB71_forward.singletons.fastq.gz SB71_reverse.paired.fastq.gz SB71_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:23:05 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB71/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 29730 Both Surviving: 5425 (18.25%) Forward Only Surviving: 1311 (4.41%) Reverse Only Surviving: 389 (1.31%) Dropped: 22605 (76.03%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: SB71_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB71_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB71_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB71_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB71_forward.paired.headclipped.fastq.gz SB71_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB71 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB71_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB71_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB71.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB71.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB71.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB71.hist\n",
      "[FLASH]     ./SB71.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5425 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5425\n",
      "[FLASH]     Combined pairs:   5019\n",
      "[FLASH]     Uncombined pairs: 406\n",
      "[FLASH]     Percent combined: 92.52%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.129 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "5019 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB71/SB71_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB71_centroids.fasta --uc SB71.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 5019 sequences and identified 2229 clusters (clustering threshold 1.00) - 89 clusters (representing 2479 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB73 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB73_1.fastq /home/working/demultiplexed/SB73_2.fastq SB73_forward.paired.fastq.gz SB73_forward.singletons.fastq.gz SB73_reverse.paired.fastq.gz SB73_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB73_1.fastq /home/working/demultiplexed/SB73_2.fastq SB73_forward.paired.fastq.gz SB73_forward.singletons.fastq.gz SB73_reverse.paired.fastq.gz SB73_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:23:15 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB73/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 30141 Both Surviving: 1194 (3.96%) Forward Only Surviving: 905 (3.00%) Reverse Only Surviving: 309 (1.03%) Dropped: 27733 (92.01%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SB73_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SB73_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB73_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB73_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB73_forward.paired.headclipped.fastq.gz SB73_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB73 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB73_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB73_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB73.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB73.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB73.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB73.hist\n",
      "[FLASH]     ./SB73.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1194 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1194\n",
      "[FLASH]     Combined pairs:   944\n",
      "[FLASH]     Uncombined pairs: 250\n",
      "[FLASH]     Percent combined: 79.06%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.025 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "944 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB73/SB73_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB73_centroids.fasta --uc SB73.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 944 sequences and identified 464 clusters (clustering threshold 1.00) - 8 clusters (representing 438 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB76 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB76_1.fastq /home/working/demultiplexed/SB76_2.fastq SB76_forward.paired.fastq.gz SB76_forward.singletons.fastq.gz SB76_reverse.paired.fastq.gz SB76_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB76_1.fastq /home/working/demultiplexed/SB76_2.fastq SB76_forward.paired.fastq.gz SB76_forward.singletons.fastq.gz SB76_reverse.paired.fastq.gz SB76_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:23:26 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB76/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 118828 Both Surviving: 47500 (39.97%) Forward Only Surviving: 12180 (10.25%) Reverse Only Surviving: 2009 (1.69%) Dropped: 57139 (48.09%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SB76_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB76_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB76_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB76_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB76_forward.paired.headclipped.fastq.gz SB76_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB76 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB76_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB76_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB76.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB76.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB76.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB76.hist\n",
      "[FLASH]     ./SB76.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 25000 read pairs\n",
      "[FLASH] Processed 47500 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      47500\n",
      "[FLASH]     Combined pairs:   24052\n",
      "[FLASH]     Uncombined pairs: 23448\n",
      "[FLASH]     Percent combined: 50.64%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.721 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "24052 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB76/SB76_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB76_centroids.fasta --uc SB76.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 24052 sequences and identified 9072 clusters (clustering threshold 1.00) - 456 clusters (representing 14503 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB77 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB77_1.fastq /home/working/demultiplexed/SB77_2.fastq SB77_forward.paired.fastq.gz SB77_forward.singletons.fastq.gz SB77_reverse.paired.fastq.gz SB77_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB77_1.fastq /home/working/demultiplexed/SB77_2.fastq SB77_forward.paired.fastq.gz SB77_forward.singletons.fastq.gz SB77_reverse.paired.fastq.gz SB77_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:24:18 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB77/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 24892 Both Surviving: 4667 (18.75%) Forward Only Surviving: 1168 (4.69%) Reverse Only Surviving: 427 (1.72%) Dropped: 18630 (74.84%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SB77_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB77_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB77_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB77_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB77_forward.paired.headclipped.fastq.gz SB77_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB77 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB77_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB77_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB77.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB77.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB77.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB77.hist\n",
      "[FLASH]     ./SB77.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 4667 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      4667\n",
      "[FLASH]     Combined pairs:   53\n",
      "[FLASH]     Uncombined pairs: 4614\n",
      "[FLASH]     Percent combined: 1.14%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.041 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "53 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB77/SB77_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB77_centroids.fasta --uc SB77.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 53 sequences and identified 48 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB8 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB8_1.fastq /home/working/demultiplexed/SB8_2.fastq SB8_forward.paired.fastq.gz SB8_forward.singletons.fastq.gz SB8_reverse.paired.fastq.gz SB8_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB8_1.fastq /home/working/demultiplexed/SB8_2.fastq SB8_forward.paired.fastq.gz SB8_forward.singletons.fastq.gz SB8_reverse.paired.fastq.gz SB8_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:24:23 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB8/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 26459 Both Surviving: 2673 (10.10%) Forward Only Surviving: 944 (3.57%) Reverse Only Surviving: 335 (1.27%) Dropped: 22507 (85.06%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SB8_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SB8_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB8_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB8_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB8_forward.paired.headclipped.fastq.gz SB8_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB8 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB8_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB8_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB8.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB8.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB8.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB8.hist\n",
      "[FLASH]     ./SB8.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2673 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2673\n",
      "[FLASH]     Combined pairs:   2308\n",
      "[FLASH]     Uncombined pairs: 365\n",
      "[FLASH]     Percent combined: 86.34%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.062 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2308 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB8/SB8_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB8_centroids.fasta --uc SB8.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2308 sequences and identified 1361 clusters (clustering threshold 1.00) - 24 clusters (representing 806 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB80 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB80_1.fastq /home/working/demultiplexed/SB80_2.fastq SB80_forward.paired.fastq.gz SB80_forward.singletons.fastq.gz SB80_reverse.paired.fastq.gz SB80_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB80_1.fastq /home/working/demultiplexed/SB80_2.fastq SB80_forward.paired.fastq.gz SB80_forward.singletons.fastq.gz SB80_reverse.paired.fastq.gz SB80_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:24:30 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB80/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 34212 Both Surviving: 398 (1.16%) Forward Only Surviving: 465 (1.36%) Reverse Only Surviving: 237 (0.69%) Dropped: 33112 (96.78%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SB80_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB80_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB80_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB80_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB80_forward.paired.headclipped.fastq.gz SB80_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB80 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB80_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB80_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB80.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB80.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB80.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB80.hist\n",
      "[FLASH]     ./SB80.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 398 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      398\n",
      "[FLASH]     Combined pairs:   15\n",
      "[FLASH]     Uncombined pairs: 383\n",
      "[FLASH]     Percent combined: 3.77%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.004 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "15 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB80/SB80_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB80_centroids.fasta --uc SB80.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 15 sequences and identified 14 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB82 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB82_1.fastq /home/working/demultiplexed/SB82_2.fastq SB82_forward.paired.fastq.gz SB82_forward.singletons.fastq.gz SB82_reverse.paired.fastq.gz SB82_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB82_1.fastq /home/working/demultiplexed/SB82_2.fastq SB82_forward.paired.fastq.gz SB82_forward.singletons.fastq.gz SB82_reverse.paired.fastq.gz SB82_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:24:35 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB82/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 56602 Both Surviving: 19825 (35.03%) Forward Only Surviving: 3783 (6.68%) Reverse Only Surviving: 937 (1.66%) Dropped: 32057 (56.64%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: SB82_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB82_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB82_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SB82_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB82_forward.paired.headclipped.fastq.gz SB82_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB82 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB82_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB82_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB82.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB82.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB82.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB82.hist\n",
      "[FLASH]     ./SB82.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 19825 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      19825\n",
      "[FLASH]     Combined pairs:   10734\n",
      "[FLASH]     Uncombined pairs: 9091\n",
      "[FLASH]     Percent combined: 54.14%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.307 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "10734 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB82/SB82_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB82_centroids.fasta --uc SB82.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 10734 sequences and identified 5324 clusters (clustering threshold 1.00) - 215 clusters (representing 4986 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB83 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB83_1.fastq /home/working/demultiplexed/SB83_2.fastq SB83_forward.paired.fastq.gz SB83_forward.singletons.fastq.gz SB83_reverse.paired.fastq.gz SB83_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB83_1.fastq /home/working/demultiplexed/SB83_2.fastq SB83_forward.paired.fastq.gz SB83_forward.singletons.fastq.gz SB83_reverse.paired.fastq.gz SB83_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:24:57 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB83/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 13296 Both Surviving: 2639 (19.85%) Forward Only Surviving: 502 (3.78%) Reverse Only Surviving: 223 (1.68%) Dropped: 9932 (74.70%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: SB83_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SB83_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB83_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB83_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB83_forward.paired.headclipped.fastq.gz SB83_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB83 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB83_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB83_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB83.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB83.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB83.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB83.hist\n",
      "[FLASH]     ./SB83.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2639 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2639\n",
      "[FLASH]     Combined pairs:   195\n",
      "[FLASH]     Uncombined pairs: 2444\n",
      "[FLASH]     Percent combined: 7.39%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.018 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "195 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB83/SB83_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB83_centroids.fasta --uc SB83.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 195 sequences and identified 88 clusters (clustering threshold 1.00) - 2 clusters (representing 98 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB85 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB85_1.fastq /home/working/demultiplexed/SB85_2.fastq SB85_forward.paired.fastq.gz SB85_forward.singletons.fastq.gz SB85_reverse.paired.fastq.gz SB85_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB85_1.fastq /home/working/demultiplexed/SB85_2.fastq SB85_forward.paired.fastq.gz SB85_forward.singletons.fastq.gz SB85_reverse.paired.fastq.gz SB85_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:25:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB85/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 28986 Both Surviving: 5672 (19.57%) Forward Only Surviving: 1435 (4.95%) Reverse Only Surviving: 464 (1.60%) Dropped: 21415 (73.88%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SB85_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB85_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB85_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB85_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB85_forward.paired.headclipped.fastq.gz SB85_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB85 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB85_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB85_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB85.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB85.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB85.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB85.hist\n",
      "[FLASH]     ./SB85.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5672 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5672\n",
      "[FLASH]     Combined pairs:   702\n",
      "[FLASH]     Uncombined pairs: 4970\n",
      "[FLASH]     Percent combined: 12.38%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.038 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "702 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB85/SB85_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB85_centroids.fasta --uc SB85.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 702 sequences and identified 426 clusters (clustering threshold 1.00) - 13 clusters (representing 266 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB86 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB86_1.fastq /home/working/demultiplexed/SB86_2.fastq SB86_forward.paired.fastq.gz SB86_forward.singletons.fastq.gz SB86_reverse.paired.fastq.gz SB86_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB86_1.fastq /home/working/demultiplexed/SB86_2.fastq SB86_forward.paired.fastq.gz SB86_forward.singletons.fastq.gz SB86_reverse.paired.fastq.gz SB86_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:25:05 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB86/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 10822 Both Surviving: 3987 (36.84%) Forward Only Surviving: 1134 (10.48%) Reverse Only Surviving: 161 (1.49%) Dropped: 5540 (51.19%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SB86_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB86_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB86_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB86_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB86_forward.paired.headclipped.fastq.gz SB86_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB86 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB86_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB86_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB86.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB86.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB86.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB86.hist\n",
      "[FLASH]     ./SB86.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3987 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3987\n",
      "[FLASH]     Combined pairs:   1868\n",
      "[FLASH]     Uncombined pairs: 2119\n",
      "[FLASH]     Percent combined: 46.85%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.056 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1868 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB86/SB86_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB86_centroids.fasta --uc SB86.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1868 sequences and identified 1174 clusters (clustering threshold 1.00) - 27 clusters (representing 600 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB87 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB87_1.fastq /home/working/demultiplexed/SB87_2.fastq SB87_forward.paired.fastq.gz SB87_forward.singletons.fastq.gz SB87_reverse.paired.fastq.gz SB87_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB87_1.fastq /home/working/demultiplexed/SB87_2.fastq SB87_forward.paired.fastq.gz SB87_forward.singletons.fastq.gz SB87_reverse.paired.fastq.gz SB87_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:25:10 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB87/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 22 bases off reads in file: SB87_forward.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB87_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB87_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB87_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB87_forward.paired.headclipped.fastq.gz SB87_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB87 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB87_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB87_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB87.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB87.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB87.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB87.hist\n",
      "[FLASH]     ./SB87.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      0\n",
      "[FLASH]     Combined pairs:   0\n",
      "[FLASH]     Uncombined pairs: 0\n",
      "[FLASH]     Percent combined: 0.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "0 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB87/SB87_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB87_centroids.fasta --uc SB87.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB91 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB91_1.fastq /home/working/demultiplexed/SB91_2.fastq SB91_forward.paired.fastq.gz SB91_forward.singletons.fastq.gz SB91_reverse.paired.fastq.gz SB91_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB91_1.fastq /home/working/demultiplexed/SB91_2.fastq SB91_forward.paired.fastq.gz SB91_forward.singletons.fastq.gz SB91_reverse.paired.fastq.gz SB91_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:25:12 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB91/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 35326 Both Surviving: 2322 (6.57%) Forward Only Surviving: 1198 (3.39%) Reverse Only Surviving: 322 (0.91%) Dropped: 31484 (89.12%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SB91_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SB91_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB91_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SB91_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB91_forward.paired.headclipped.fastq.gz SB91_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB91 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB91_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB91_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB91.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB91.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB91.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB91.hist\n",
      "[FLASH]     ./SB91.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2322 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2322\n",
      "[FLASH]     Combined pairs:   2028\n",
      "[FLASH]     Uncombined pairs: 294\n",
      "[FLASH]     Percent combined: 87.34%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.055 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2028 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB91/SB91_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB91_centroids.fasta --uc SB91.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2028 sequences and identified 1108 clusters (clustering threshold 1.00) - 24 clusters (representing 820 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB93 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB93_1.fastq /home/working/demultiplexed/SB93_2.fastq SB93_forward.paired.fastq.gz SB93_forward.singletons.fastq.gz SB93_reverse.paired.fastq.gz SB93_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB93_1.fastq /home/working/demultiplexed/SB93_2.fastq SB93_forward.paired.fastq.gz SB93_forward.singletons.fastq.gz SB93_reverse.paired.fastq.gz SB93_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:25:18 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB93/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 20814 Both Surviving: 2455 (11.79%) Forward Only Surviving: 977 (4.69%) Reverse Only Surviving: 388 (1.86%) Dropped: 16994 (81.65%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SB93_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SB93_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB93_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SB93_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB93_forward.paired.headclipped.fastq.gz SB93_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB93 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB93_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB93_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB93.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB93.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB93.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB93.hist\n",
      "[FLASH]     ./SB93.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2455 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2455\n",
      "[FLASH]     Combined pairs:   365\n",
      "[FLASH]     Uncombined pairs: 2090\n",
      "[FLASH]     Percent combined: 14.87%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.019 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "365 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB93/SB93_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB93_centroids.fasta --uc SB93.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 365 sequences and identified 188 clusters (clustering threshold 1.00) - 4 clusters (representing 165 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SB99 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB99_1.fastq /home/working/demultiplexed/SB99_2.fastq SB99_forward.paired.fastq.gz SB99_forward.singletons.fastq.gz SB99_reverse.paired.fastq.gz SB99_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SB99_1.fastq /home/working/demultiplexed/SB99_2.fastq SB99_forward.paired.fastq.gz SB99_forward.singletons.fastq.gz SB99_reverse.paired.fastq.gz SB99_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:25:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SB99/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 22690 Both Surviving: 2824 (12.45%) Forward Only Surviving: 846 (3.73%) Reverse Only Surviving: 417 (1.84%) Dropped: 18603 (81.99%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SB99_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SB99_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB99_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SB99_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SB99_forward.paired.headclipped.fastq.gz SB99_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SB99 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SB99_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SB99_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SB99.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SB99.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SB99.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SB99.hist\n",
      "[FLASH]     ./SB99.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2824 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2824\n",
      "[FLASH]     Combined pairs:   746\n",
      "[FLASH]     Uncombined pairs: 2078\n",
      "[FLASH]     Percent combined: 26.42%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.028 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "746 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SB99/SB99_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SB99_centroids.fasta --uc SB99.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 746 sequences and identified 394 clusters (clustering threshold 1.00) - 8 clusters (representing 317 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC10 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC10_1.fastq /home/working/demultiplexed/SC10_2.fastq SC10_forward.paired.fastq.gz SC10_forward.singletons.fastq.gz SC10_reverse.paired.fastq.gz SC10_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC10_1.fastq /home/working/demultiplexed/SC10_2.fastq SC10_forward.paired.fastq.gz SC10_forward.singletons.fastq.gz SC10_reverse.paired.fastq.gz SC10_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:25:27 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC10/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 39166 Both Surviving: 16252 (41.50%) Forward Only Surviving: 3651 (9.32%) Reverse Only Surviving: 596 (1.52%) Dropped: 18667 (47.66%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SC10_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SC10_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SC10_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SC10_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC10_forward.paired.headclipped.fastq.gz SC10_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC10 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC10_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC10_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC10.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC10.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC10.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC10.hist\n",
      "[FLASH]     ./SC10.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 16252 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      16252\n",
      "[FLASH]     Combined pairs:   9440\n",
      "[FLASH]     Uncombined pairs: 6812\n",
      "[FLASH]     Percent combined: 58.09%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.263 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "9440 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC10/SC10_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC10_centroids.fasta --uc SC10.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 9440 sequences and identified 4841 clusters (clustering threshold 1.00) - 261 clusters (representing 4381 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC101 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC101_1.fastq /home/working/demultiplexed/SC101_2.fastq SC101_forward.paired.fastq.gz SC101_forward.singletons.fastq.gz SC101_reverse.paired.fastq.gz SC101_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC101_1.fastq /home/working/demultiplexed/SC101_2.fastq SC101_forward.paired.fastq.gz SC101_forward.singletons.fastq.gz SC101_reverse.paired.fastq.gz SC101_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:25:47 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC101/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 29995 Both Surviving: 6087 (20.29%) Forward Only Surviving: 1131 (3.77%) Reverse Only Surviving: 494 (1.65%) Dropped: 22283 (74.29%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SC101_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC101_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC101_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC101_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC101_forward.paired.headclipped.fastq.gz SC101_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC101 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC101_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC101_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC101.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC101.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC101.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC101.hist\n",
      "[FLASH]     ./SC101.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 6087 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      6087\n",
      "[FLASH]     Combined pairs:   11\n",
      "[FLASH]     Uncombined pairs: 6076\n",
      "[FLASH]     Percent combined: 0.18%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.048 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "11 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC101/SC101_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC101_centroids.fasta --uc SC101.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 11 sequences and identified 11 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC106 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC106_1.fastq /home/working/demultiplexed/SC106_2.fastq SC106_forward.paired.fastq.gz SC106_forward.singletons.fastq.gz SC106_reverse.paired.fastq.gz SC106_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC106_1.fastq /home/working/demultiplexed/SC106_2.fastq SC106_forward.paired.fastq.gz SC106_forward.singletons.fastq.gz SC106_reverse.paired.fastq.gz SC106_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:25:50 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC106/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 1 Both Surviving: 0 (0.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 0 (0.00%) Dropped: 1 (100.00%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 22 bases off reads in file: SC106_forward.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC106_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC106_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC106_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC106_forward.paired.headclipped.fastq.gz SC106_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC106 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC106_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC106_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC106.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC106.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC106.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC106.hist\n",
      "[FLASH]     ./SC106.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      0\n",
      "[FLASH]     Combined pairs:   0\n",
      "[FLASH]     Uncombined pairs: 0\n",
      "[FLASH]     Percent combined: 0.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "0 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC106/SC106_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC106_centroids.fasta --uc SC106.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC107 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC107_1.fastq /home/working/demultiplexed/SC107_2.fastq SC107_forward.paired.fastq.gz SC107_forward.singletons.fastq.gz SC107_reverse.paired.fastq.gz SC107_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC107_1.fastq /home/working/demultiplexed/SC107_2.fastq SC107_forward.paired.fastq.gz SC107_forward.singletons.fastq.gz SC107_reverse.paired.fastq.gz SC107_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:25:51 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC107/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 16670 Both Surviving: 3240 (19.44%) Forward Only Surviving: 944 (5.66%) Reverse Only Surviving: 191 (1.15%) Dropped: 12295 (73.76%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SC107_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SC107_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SC107_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SC107_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC107_forward.paired.headclipped.fastq.gz SC107_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC107 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC107_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC107_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC107.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC107.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC107.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC107.hist\n",
      "[FLASH]     ./SC107.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3240 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3240\n",
      "[FLASH]     Combined pairs:   268\n",
      "[FLASH]     Uncombined pairs: 2972\n",
      "[FLASH]     Percent combined: 8.27%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.020 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "268 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC107/SC107_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC107_centroids.fasta --uc SC107.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 268 sequences and identified 195 clusters (clustering threshold 1.00) - 1 clusters (representing 51 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC110 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC110_1.fastq /home/working/demultiplexed/SC110_2.fastq SC110_forward.paired.fastq.gz SC110_forward.singletons.fastq.gz SC110_reverse.paired.fastq.gz SC110_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC110_1.fastq /home/working/demultiplexed/SC110_2.fastq SC110_forward.paired.fastq.gz SC110_forward.singletons.fastq.gz SC110_reverse.paired.fastq.gz SC110_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:25:55 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC110/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 19004 Both Surviving: 3316 (17.45%) Forward Only Surviving: 1648 (8.67%) Reverse Only Surviving: 250 (1.32%) Dropped: 13790 (72.56%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SC110_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SC110_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC110_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC110_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC110_forward.paired.headclipped.fastq.gz SC110_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC110 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC110_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC110_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC110.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC110.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC110.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC110.hist\n",
      "[FLASH]     ./SC110.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3316 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3316\n",
      "[FLASH]     Combined pairs:   2849\n",
      "[FLASH]     Uncombined pairs: 467\n",
      "[FLASH]     Percent combined: 85.92%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.092 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2849 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC110/SC110_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC110_centroids.fasta --uc SC110.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2849 sequences and identified 1498 clusters (clustering threshold 1.00) - 27 clusters (representing 1109 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC113 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC113_1.fastq /home/working/demultiplexed/SC113_2.fastq SC113_forward.paired.fastq.gz SC113_forward.singletons.fastq.gz SC113_reverse.paired.fastq.gz SC113_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC113_1.fastq /home/working/demultiplexed/SC113_2.fastq SC113_forward.paired.fastq.gz SC113_forward.singletons.fastq.gz SC113_reverse.paired.fastq.gz SC113_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:26:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC113/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 14610 Both Surviving: 351 (2.40%) Forward Only Surviving: 423 (2.90%) Reverse Only Surviving: 117 (0.80%) Dropped: 13719 (93.90%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SC113_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SC113_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC113_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC113_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC113_forward.paired.headclipped.fastq.gz SC113_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC113 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC113_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC113_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC113.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC113.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC113.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC113.hist\n",
      "[FLASH]     ./SC113.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 351 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      351\n",
      "[FLASH]     Combined pairs:   263\n",
      "[FLASH]     Uncombined pairs: 88\n",
      "[FLASH]     Percent combined: 74.93%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.012 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "263 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC113/SC113_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC113_centroids.fasta --uc SC113.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 263 sequences and identified 176 clusters (clustering threshold 1.00) - 4 clusters (representing 68 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC117 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC117_1.fastq /home/working/demultiplexed/SC117_2.fastq SC117_forward.paired.fastq.gz SC117_forward.singletons.fastq.gz SC117_reverse.paired.fastq.gz SC117_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC117_1.fastq /home/working/demultiplexed/SC117_2.fastq SC117_forward.paired.fastq.gz SC117_forward.singletons.fastq.gz SC117_reverse.paired.fastq.gz SC117_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:26:05 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC117/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 34241 Both Surviving: 7288 (21.28%) Forward Only Surviving: 1972 (5.76%) Reverse Only Surviving: 492 (1.44%) Dropped: 24489 (71.52%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SC117_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC117_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC117_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC117_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC117_forward.paired.headclipped.fastq.gz SC117_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC117 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC117_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC117_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC117.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC117.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC117.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC117.hist\n",
      "[FLASH]     ./SC117.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 7288 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      7288\n",
      "[FLASH]     Combined pairs:   423\n",
      "[FLASH]     Uncombined pairs: 6865\n",
      "[FLASH]     Percent combined: 5.80%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.046 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "423 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC117/SC117_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC117_centroids.fasta --uc SC117.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 423 sequences and identified 260 clusters (clustering threshold 1.00) - 2 clusters (representing 138 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC119 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC119_1.fastq /home/working/demultiplexed/SC119_2.fastq SC119_forward.paired.fastq.gz SC119_forward.singletons.fastq.gz SC119_reverse.paired.fastq.gz SC119_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC119_1.fastq /home/working/demultiplexed/SC119_2.fastq SC119_forward.paired.fastq.gz SC119_forward.singletons.fastq.gz SC119_reverse.paired.fastq.gz SC119_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:26:10 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC119/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 16249 Both Surviving: 4595 (28.28%) Forward Only Surviving: 1155 (7.11%) Reverse Only Surviving: 242 (1.49%) Dropped: 10257 (63.12%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SC119_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SC119_forward.singletons.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SC119_reverse.paired.fastq.gz\n",
      "Cropping the first 29 bases off reads in file: SC119_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC119_forward.paired.headclipped.fastq.gz SC119_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC119 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC119_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC119_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC119.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC119.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC119.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC119.hist\n",
      "[FLASH]     ./SC119.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 4595 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      4595\n",
      "[FLASH]     Combined pairs:   827\n",
      "[FLASH]     Uncombined pairs: 3768\n",
      "[FLASH]     Percent combined: 18.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.038 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "827 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC119/SC119_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC119_centroids.fasta --uc SC119.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 827 sequences and identified 414 clusters (clustering threshold 1.00) - 7 clusters (representing 376 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC122 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC122_1.fastq /home/working/demultiplexed/SC122_2.fastq SC122_forward.paired.fastq.gz SC122_forward.singletons.fastq.gz SC122_reverse.paired.fastq.gz SC122_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC122_1.fastq /home/working/demultiplexed/SC122_2.fastq SC122_forward.paired.fastq.gz SC122_forward.singletons.fastq.gz SC122_reverse.paired.fastq.gz SC122_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:26:15 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC122/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 32336 Both Surviving: 1273 (3.94%) Forward Only Surviving: 724 (2.24%) Reverse Only Surviving: 364 (1.13%) Dropped: 29975 (92.70%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: SC122_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: SC122_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SC122_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SC122_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC122_forward.paired.headclipped.fastq.gz SC122_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC122 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC122_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC122_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC122.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC122.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC122.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC122.hist\n",
      "[FLASH]     ./SC122.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1273 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1273\n",
      "[FLASH]     Combined pairs:   1029\n",
      "[FLASH]     Uncombined pairs: 244\n",
      "[FLASH]     Percent combined: 80.83%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.025 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1029 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC122/SC122_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC122_centroids.fasta --uc SC122.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1029 sequences and identified 653 clusters (clustering threshold 1.00) - 17 clusters (representing 338 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC124 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC124_1.fastq /home/working/demultiplexed/SC124_2.fastq SC124_forward.paired.fastq.gz SC124_forward.singletons.fastq.gz SC124_reverse.paired.fastq.gz SC124_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC124_1.fastq /home/working/demultiplexed/SC124_2.fastq SC124_forward.paired.fastq.gz SC124_forward.singletons.fastq.gz SC124_reverse.paired.fastq.gz SC124_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:26:19 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC124/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 16653 Both Surviving: 2962 (17.79%) Forward Only Surviving: 779 (4.68%) Reverse Only Surviving: 233 (1.40%) Dropped: 12679 (76.14%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SC124_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC124_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC124_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC124_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC124_forward.paired.headclipped.fastq.gz SC124_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC124 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC124_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC124_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC124.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC124.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC124.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC124.hist\n",
      "[FLASH]     ./SC124.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2962 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2962\n",
      "[FLASH]     Combined pairs:   56\n",
      "[FLASH]     Uncombined pairs: 2906\n",
      "[FLASH]     Percent combined: 1.89%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.021 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "56 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC124/SC124_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC124_centroids.fasta --uc SC124.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 56 sequences and identified 54 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC125 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC125_1.fastq /home/working/demultiplexed/SC125_2.fastq SC125_forward.paired.fastq.gz SC125_forward.singletons.fastq.gz SC125_reverse.paired.fastq.gz SC125_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC125_1.fastq /home/working/demultiplexed/SC125_2.fastq SC125_forward.paired.fastq.gz SC125_forward.singletons.fastq.gz SC125_reverse.paired.fastq.gz SC125_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:26:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC125/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 18973 Both Surviving: 2226 (11.73%) Forward Only Surviving: 628 (3.31%) Reverse Only Surviving: 189 (1.00%) Dropped: 15930 (83.96%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 23 bases off reads in file: SC125_forward.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SC125_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC125_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC125_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC125_forward.paired.headclipped.fastq.gz SC125_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC125 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC125_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC125_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC125.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC125.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC125.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC125.hist\n",
      "[FLASH]     ./SC125.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2226 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2226\n",
      "[FLASH]     Combined pairs:   37\n",
      "[FLASH]     Uncombined pairs: 2189\n",
      "[FLASH]     Percent combined: 1.66%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.030 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "37 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC125/SC125_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC125_centroids.fasta --uc SC125.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 37 sequences and identified 30 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC141 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC141_1.fastq /home/working/demultiplexed/SC141_2.fastq SC141_forward.paired.fastq.gz SC141_forward.singletons.fastq.gz SC141_reverse.paired.fastq.gz SC141_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC141_1.fastq /home/working/demultiplexed/SC141_2.fastq SC141_forward.paired.fastq.gz SC141_forward.singletons.fastq.gz SC141_reverse.paired.fastq.gz SC141_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:26:25 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC141/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 12531 Both Surviving: 2206 (17.60%) Forward Only Surviving: 549 (4.38%) Reverse Only Surviving: 337 (2.69%) Dropped: 9439 (75.33%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 27 bases off reads in file: SC141_forward.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC141_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SC141_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SC141_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC141_forward.paired.headclipped.fastq.gz SC141_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC141 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC141_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC141_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC141.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC141.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC141.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC141.hist\n",
      "[FLASH]     ./SC141.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2206 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2206\n",
      "[FLASH]     Combined pairs:   57\n",
      "[FLASH]     Uncombined pairs: 2149\n",
      "[FLASH]     Percent combined: 2.58%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.017 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "57 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC141/SC141_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC141_centroids.fasta --uc SC141.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 57 sequences and identified 45 clusters (clustering threshold 1.00) - 1 clusters (representing 6 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC142 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC142_1.fastq /home/working/demultiplexed/SC142_2.fastq SC142_forward.paired.fastq.gz SC142_forward.singletons.fastq.gz SC142_reverse.paired.fastq.gz SC142_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC142_1.fastq /home/working/demultiplexed/SC142_2.fastq SC142_forward.paired.fastq.gz SC142_forward.singletons.fastq.gz SC142_reverse.paired.fastq.gz SC142_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:26:27 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC142/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 17791 Both Surviving: 3480 (19.56%) Forward Only Surviving: 964 (5.42%) Reverse Only Surviving: 223 (1.25%) Dropped: 13124 (73.77%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SC142_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SC142_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SC142_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SC142_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC142_forward.paired.headclipped.fastq.gz SC142_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC142 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC142_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC142_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC142.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC142.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC142.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC142.hist\n",
      "[FLASH]     ./SC142.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3480 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3480\n",
      "[FLASH]     Combined pairs:   55\n",
      "[FLASH]     Uncombined pairs: 3425\n",
      "[FLASH]     Percent combined: 1.58%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.023 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "55 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC142/SC142_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC142_centroids.fasta --uc SC142.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 55 sequences and identified 50 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC143 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC143_1.fastq /home/working/demultiplexed/SC143_2.fastq SC143_forward.paired.fastq.gz SC143_forward.singletons.fastq.gz SC143_reverse.paired.fastq.gz SC143_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC143_1.fastq /home/working/demultiplexed/SC143_2.fastq SC143_forward.paired.fastq.gz SC143_forward.singletons.fastq.gz SC143_reverse.paired.fastq.gz SC143_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:26:31 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC143/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 27590 Both Surviving: 2582 (9.36%) Forward Only Surviving: 805 (2.92%) Reverse Only Surviving: 426 (1.54%) Dropped: 23777 (86.18%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SC143_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC143_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC143_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC143_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC143_forward.paired.headclipped.fastq.gz SC143_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC143 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC143_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC143_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC143.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC143.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC143.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC143.hist\n",
      "[FLASH]     ./SC143.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2582 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2582\n",
      "[FLASH]     Combined pairs:   122\n",
      "[FLASH]     Uncombined pairs: 2460\n",
      "[FLASH]     Percent combined: 4.73%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.032 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "122 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC143/SC143_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC143_centroids.fasta --uc SC143.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 122 sequences and identified 91 clusters (clustering threshold 1.00) - 2 clusters (representing 20 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC157 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC157_1.fastq /home/working/demultiplexed/SC157_2.fastq SC157_forward.paired.fastq.gz SC157_forward.singletons.fastq.gz SC157_reverse.paired.fastq.gz SC157_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC157_1.fastq /home/working/demultiplexed/SC157_2.fastq SC157_forward.paired.fastq.gz SC157_forward.singletons.fastq.gz SC157_reverse.paired.fastq.gz SC157_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:26:34 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC157/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 4782 Both Surviving: 123 (2.57%) Forward Only Surviving: 143 (2.99%) Reverse Only Surviving: 56 (1.17%) Dropped: 4460 (93.27%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SC157_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SC157_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC157_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC157_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC157_forward.paired.headclipped.fastq.gz SC157_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC157 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC157_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC157_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC157.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC157.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC157.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC157.hist\n",
      "[FLASH]     ./SC157.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 123 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      123\n",
      "[FLASH]     Combined pairs:   34\n",
      "[FLASH]     Uncombined pairs: 89\n",
      "[FLASH]     Percent combined: 27.64%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.002 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "34 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC157/SC157_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC157_centroids.fasta --uc SC157.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 34 sequences and identified 30 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC16 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC16_1.fastq /home/working/demultiplexed/SC16_2.fastq SC16_forward.paired.fastq.gz SC16_forward.singletons.fastq.gz SC16_reverse.paired.fastq.gz SC16_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC16_1.fastq /home/working/demultiplexed/SC16_2.fastq SC16_forward.paired.fastq.gz SC16_forward.singletons.fastq.gz SC16_reverse.paired.fastq.gz SC16_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:26:36 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC16/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 33389 Both Surviving: 9086 (27.21%) Forward Only Surviving: 3296 (9.87%) Reverse Only Surviving: 356 (1.07%) Dropped: 20651 (61.85%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SC16_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC16_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC16_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC16_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC16_forward.paired.headclipped.fastq.gz SC16_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC16 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC16_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC16_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC16.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC16.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC16.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC16.hist\n",
      "[FLASH]     ./SC16.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 9086 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      9086\n",
      "[FLASH]     Combined pairs:   8024\n",
      "[FLASH]     Uncombined pairs: 1062\n",
      "[FLASH]     Percent combined: 88.31%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.246 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "8024 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC16/SC16_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC16_centroids.fasta --uc SC16.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 8024 sequences and identified 3299 clusters (clustering threshold 1.00) - 194 clusters (representing 4308 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC168 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC168_1.fastq /home/working/demultiplexed/SC168_2.fastq SC168_forward.paired.fastq.gz SC168_forward.singletons.fastq.gz SC168_reverse.paired.fastq.gz SC168_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC168_1.fastq /home/working/demultiplexed/SC168_2.fastq SC168_forward.paired.fastq.gz SC168_forward.singletons.fastq.gz SC168_reverse.paired.fastq.gz SC168_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:26:54 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC168/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 37551 Both Surviving: 5209 (13.87%) Forward Only Surviving: 1233 (3.28%) Reverse Only Surviving: 602 (1.60%) Dropped: 30507 (81.24%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SC168_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC168_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC168_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC168_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC168_forward.paired.headclipped.fastq.gz SC168_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC168 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC168_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC168_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC168.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC168.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC168.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC168.hist\n",
      "[FLASH]     ./SC168.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5209 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5209\n",
      "[FLASH]     Combined pairs:   19\n",
      "[FLASH]     Uncombined pairs: 5190\n",
      "[FLASH]     Percent combined: 0.36%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.039 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "19 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC168/SC168_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC168_centroids.fasta --uc SC168.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 19 sequences and identified 19 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC178 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC178_1.fastq /home/working/demultiplexed/SC178_2.fastq SC178_forward.paired.fastq.gz SC178_forward.singletons.fastq.gz SC178_reverse.paired.fastq.gz SC178_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC178_1.fastq /home/working/demultiplexed/SC178_2.fastq SC178_forward.paired.fastq.gz SC178_forward.singletons.fastq.gz SC178_reverse.paired.fastq.gz SC178_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:26:58 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC178/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 0 Both Surviving: 0 (%) Forward Only Surviving: 0 (%) Reverse Only Surviving: 0 (%) Dropped: 0 (%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 22 bases off reads in file: SC178_forward.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC178_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC178_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC178_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC178_forward.paired.headclipped.fastq.gz SC178_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC178 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC178_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC178_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC178.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC178.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC178.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC178.hist\n",
      "[FLASH]     ./SC178.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      0\n",
      "[FLASH]     Combined pairs:   0\n",
      "[FLASH]     Uncombined pairs: 0\n",
      "[FLASH]     Percent combined: 0.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "0 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC178/SC178_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC178_centroids.fasta --uc SC178.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC187 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC187_1.fastq /home/working/demultiplexed/SC187_2.fastq SC187_forward.paired.fastq.gz SC187_forward.singletons.fastq.gz SC187_reverse.paired.fastq.gz SC187_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC187_1.fastq /home/working/demultiplexed/SC187_2.fastq SC187_forward.paired.fastq.gz SC187_forward.singletons.fastq.gz SC187_reverse.paired.fastq.gz SC187_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:26:59 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC187/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 20627 Both Surviving: 1801 (8.73%) Forward Only Surviving: 617 (2.99%) Reverse Only Surviving: 312 (1.51%) Dropped: 17897 (86.76%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SC187_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC187_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC187_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC187_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC187_forward.paired.headclipped.fastq.gz SC187_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC187 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC187_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC187_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC187.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC187.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC187.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC187.hist\n",
      "[FLASH]     ./SC187.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1801 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1801\n",
      "[FLASH]     Combined pairs:   22\n",
      "[FLASH]     Uncombined pairs: 1779\n",
      "[FLASH]     Percent combined: 1.22%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.014 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "22 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC187/SC187_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC187_centroids.fasta --uc SC187.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 22 sequences and identified 17 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC198 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC198_1.fastq /home/working/demultiplexed/SC198_2.fastq SC198_forward.paired.fastq.gz SC198_forward.singletons.fastq.gz SC198_reverse.paired.fastq.gz SC198_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC198_1.fastq /home/working/demultiplexed/SC198_2.fastq SC198_forward.paired.fastq.gz SC198_forward.singletons.fastq.gz SC198_reverse.paired.fastq.gz SC198_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:27:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC198/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 17339 Both Surviving: 3301 (19.04%) Forward Only Surviving: 903 (5.21%) Reverse Only Surviving: 242 (1.40%) Dropped: 12893 (74.36%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SC198_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SC198_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SC198_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SC198_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC198_forward.paired.headclipped.fastq.gz SC198_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC198 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC198_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC198_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC198.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC198.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC198.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC198.hist\n",
      "[FLASH]     ./SC198.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3301 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3301\n",
      "[FLASH]     Combined pairs:   51\n",
      "[FLASH]     Uncombined pairs: 3250\n",
      "[FLASH]     Percent combined: 1.54%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.019 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "51 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC198/SC198_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC198_centroids.fasta --uc SC198.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 51 sequences and identified 44 clusters (clustering threshold 1.00) - 1 clusters (representing 5 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC19A #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC19A_1.fastq /home/working/demultiplexed/SC19A_2.fastq SC19A_forward.paired.fastq.gz SC19A_forward.singletons.fastq.gz SC19A_reverse.paired.fastq.gz SC19A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC19A_1.fastq /home/working/demultiplexed/SC19A_2.fastq SC19A_forward.paired.fastq.gz SC19A_forward.singletons.fastq.gz SC19A_reverse.paired.fastq.gz SC19A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:27:05 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC19A/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 31576 Both Surviving: 2907 (9.21%) Forward Only Surviving: 964 (3.05%) Reverse Only Surviving: 422 (1.34%) Dropped: 27283 (86.40%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: SC19A_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: SC19A_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SC19A_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SC19A_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC19A_forward.paired.headclipped.fastq.gz SC19A_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC19A -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC19A_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC19A_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC19A.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC19A.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC19A.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC19A.hist\n",
      "[FLASH]     ./SC19A.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2907 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2907\n",
      "[FLASH]     Combined pairs:   2641\n",
      "[FLASH]     Uncombined pairs: 266\n",
      "[FLASH]     Percent combined: 90.85%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.104 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2641 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC19A/SC19A_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC19A_centroids.fasta --uc SC19A.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2641 sequences and identified 1242 clusters (clustering threshold 1.00) - 23 clusters (representing 1210 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC202 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC202_1.fastq /home/working/demultiplexed/SC202_2.fastq SC202_forward.paired.fastq.gz SC202_forward.singletons.fastq.gz SC202_reverse.paired.fastq.gz SC202_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC202_1.fastq /home/working/demultiplexed/SC202_2.fastq SC202_forward.paired.fastq.gz SC202_forward.singletons.fastq.gz SC202_reverse.paired.fastq.gz SC202_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:27:12 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC202/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 20207 Both Surviving: 377 (1.87%) Forward Only Surviving: 334 (1.65%) Reverse Only Surviving: 179 (0.89%) Dropped: 19317 (95.60%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SC202_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC202_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC202_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC202_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC202_forward.paired.headclipped.fastq.gz SC202_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC202 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC202_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC202_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC202.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC202.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC202.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC202.hist\n",
      "[FLASH]     ./SC202.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 377 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      377\n",
      "[FLASH]     Combined pairs:   7\n",
      "[FLASH]     Uncombined pairs: 370\n",
      "[FLASH]     Percent combined: 1.86%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.004 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "7 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC202/SC202_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC202_centroids.fasta --uc SC202.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 7 sequences and identified 6 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC26 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC26_1.fastq /home/working/demultiplexed/SC26_2.fastq SC26_forward.paired.fastq.gz SC26_forward.singletons.fastq.gz SC26_reverse.paired.fastq.gz SC26_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC26_1.fastq /home/working/demultiplexed/SC26_2.fastq SC26_forward.paired.fastq.gz SC26_forward.singletons.fastq.gz SC26_reverse.paired.fastq.gz SC26_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:27:16 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC26/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 38586 Both Surviving: 14376 (37.26%) Forward Only Surviving: 3099 (8.03%) Reverse Only Surviving: 648 (1.68%) Dropped: 20463 (53.03%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SC26_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SC26_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SC26_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SC26_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC26_forward.paired.headclipped.fastq.gz SC26_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC26 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC26_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC26_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC26.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC26.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC26.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC26.hist\n",
      "[FLASH]     ./SC26.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 14376 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      14376\n",
      "[FLASH]     Combined pairs:   7773\n",
      "[FLASH]     Uncombined pairs: 6603\n",
      "[FLASH]     Percent combined: 54.07%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.213 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "7773 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC26/SC26_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC26_centroids.fasta --uc SC26.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 7773 sequences and identified 3818 clusters (clustering threshold 1.00) - 209 clusters (representing 3667 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC28 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC28_1.fastq /home/working/demultiplexed/SC28_2.fastq SC28_forward.paired.fastq.gz SC28_forward.singletons.fastq.gz SC28_reverse.paired.fastq.gz SC28_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC28_1.fastq /home/working/demultiplexed/SC28_2.fastq SC28_forward.paired.fastq.gz SC28_forward.singletons.fastq.gz SC28_reverse.paired.fastq.gz SC28_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:27:31 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC28/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 25262 Both Surviving: 3707 (14.67%) Forward Only Surviving: 1212 (4.80%) Reverse Only Surviving: 310 (1.23%) Dropped: 20033 (79.30%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SC28_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC28_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC28_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC28_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC28_forward.paired.headclipped.fastq.gz SC28_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC28 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC28_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC28_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC28.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC28.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC28.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC28.hist\n",
      "[FLASH]     ./SC28.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3707 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3707\n",
      "[FLASH]     Combined pairs:   410\n",
      "[FLASH]     Uncombined pairs: 3297\n",
      "[FLASH]     Percent combined: 11.06%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.024 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "410 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC28/SC28_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC28_centroids.fasta --uc SC28.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 410 sequences and identified 243 clusters (clustering threshold 1.00) - 8 clusters (representing 159 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC34 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC34_1.fastq /home/working/demultiplexed/SC34_2.fastq SC34_forward.paired.fastq.gz SC34_forward.singletons.fastq.gz SC34_reverse.paired.fastq.gz SC34_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC34_1.fastq /home/working/demultiplexed/SC34_2.fastq SC34_forward.paired.fastq.gz SC34_forward.singletons.fastq.gz SC34_reverse.paired.fastq.gz SC34_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:27:36 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC34/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 28305 Both Surviving: 36 (0.13%) Forward Only Surviving: 490 (1.73%) Reverse Only Surviving: 366 (1.29%) Dropped: 27413 (96.85%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: SC34_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: SC34_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC34_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC34_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC34_forward.paired.headclipped.fastq.gz SC34_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC34 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC34_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC34_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC34.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC34.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC34.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC34.hist\n",
      "[FLASH]     ./SC34.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 36 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      36\n",
      "[FLASH]     Combined pairs:   12\n",
      "[FLASH]     Uncombined pairs: 24\n",
      "[FLASH]     Percent combined: 33.33%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "12 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC34/SC34_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC34_centroids.fasta --uc SC34.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 12 sequences and identified 10 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC35 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC35_1.fastq /home/working/demultiplexed/SC35_2.fastq SC35_forward.paired.fastq.gz SC35_forward.singletons.fastq.gz SC35_reverse.paired.fastq.gz SC35_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC35_1.fastq /home/working/demultiplexed/SC35_2.fastq SC35_forward.paired.fastq.gz SC35_forward.singletons.fastq.gz SC35_reverse.paired.fastq.gz SC35_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:27:39 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC35/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 22422 Both Surviving: 6879 (30.68%) Forward Only Surviving: 2026 (9.04%) Reverse Only Surviving: 280 (1.25%) Dropped: 13237 (59.04%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SC35_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SC35_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SC35_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SC35_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC35_forward.paired.headclipped.fastq.gz SC35_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC35 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC35_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC35_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC35.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC35.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC35.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC35.hist\n",
      "[FLASH]     ./SC35.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 6879 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      6879\n",
      "[FLASH]     Combined pairs:   1981\n",
      "[FLASH]     Uncombined pairs: 4898\n",
      "[FLASH]     Percent combined: 28.80%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.063 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1981 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC35/SC35_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC35_centroids.fasta --uc SC35.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1981 sequences and identified 1341 clusters (clustering threshold 1.00) - 41 clusters (representing 505 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC43 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC43_1.fastq /home/working/demultiplexed/SC43_2.fastq SC43_forward.paired.fastq.gz SC43_forward.singletons.fastq.gz SC43_reverse.paired.fastq.gz SC43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC43_1.fastq /home/working/demultiplexed/SC43_2.fastq SC43_forward.paired.fastq.gz SC43_forward.singletons.fastq.gz SC43_reverse.paired.fastq.gz SC43_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:27:46 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC43/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 16887 Both Surviving: 490 (2.90%) Forward Only Surviving: 524 (3.10%) Reverse Only Surviving: 123 (0.73%) Dropped: 15750 (93.27%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SC43_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC43_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC43_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC43_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC43_forward.paired.headclipped.fastq.gz SC43_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC43 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC43_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC43_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC43.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC43.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC43.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC43.hist\n",
      "[FLASH]     ./SC43.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 490 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      490\n",
      "[FLASH]     Combined pairs:   330\n",
      "[FLASH]     Uncombined pairs: 160\n",
      "[FLASH]     Percent combined: 67.35%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.013 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "330 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC43/SC43_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC43_centroids.fasta --uc SC43.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 330 sequences and identified 253 clusters (clustering threshold 1.00) - 10 clusters (representing 65 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC44 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC44_1.fastq /home/working/demultiplexed/SC44_2.fastq SC44_forward.paired.fastq.gz SC44_forward.singletons.fastq.gz SC44_reverse.paired.fastq.gz SC44_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC44_1.fastq /home/working/demultiplexed/SC44_2.fastq SC44_forward.paired.fastq.gz SC44_forward.singletons.fastq.gz SC44_reverse.paired.fastq.gz SC44_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:27:49 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC44/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 18958 Both Surviving: 4960 (26.16%) Forward Only Surviving: 1556 (8.21%) Reverse Only Surviving: 276 (1.46%) Dropped: 12166 (64.17%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SC44_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC44_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC44_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC44_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC44_forward.paired.headclipped.fastq.gz SC44_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC44 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC44_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC44_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC44.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC44.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC44.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC44.hist\n",
      "[FLASH]     ./SC44.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 4960 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      4960\n",
      "[FLASH]     Combined pairs:   146\n",
      "[FLASH]     Uncombined pairs: 4814\n",
      "[FLASH]     Percent combined: 2.94%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.044 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "146 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC44/SC44_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC44_centroids.fasta --uc SC44.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 146 sequences and identified 137 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC48 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC48_1.fastq /home/working/demultiplexed/SC48_2.fastq SC48_forward.paired.fastq.gz SC48_forward.singletons.fastq.gz SC48_reverse.paired.fastq.gz SC48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC48_1.fastq /home/working/demultiplexed/SC48_2.fastq SC48_forward.paired.fastq.gz SC48_forward.singletons.fastq.gz SC48_reverse.paired.fastq.gz SC48_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:27:53 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC48/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 20800 Both Surviving: 72 (0.35%) Forward Only Surviving: 318 (1.53%) Reverse Only Surviving: 200 (0.96%) Dropped: 20210 (97.16%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: SC48_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: SC48_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC48_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC48_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC48_forward.paired.headclipped.fastq.gz SC48_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC48 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC48_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC48_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC48.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC48.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC48.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC48.hist\n",
      "[FLASH]     ./SC48.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 72 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      72\n",
      "[FLASH]     Combined pairs:   24\n",
      "[FLASH]     Uncombined pairs: 48\n",
      "[FLASH]     Percent combined: 33.33%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.002 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "24 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC48/SC48_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC48_centroids.fasta --uc SC48.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 24 sequences and identified 20 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC49 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC49_1.fastq /home/working/demultiplexed/SC49_2.fastq SC49_forward.paired.fastq.gz SC49_forward.singletons.fastq.gz SC49_reverse.paired.fastq.gz SC49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC49_1.fastq /home/working/demultiplexed/SC49_2.fastq SC49_forward.paired.fastq.gz SC49_forward.singletons.fastq.gz SC49_reverse.paired.fastq.gz SC49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:27:55 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC49/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 31250 Both Surviving: 6674 (21.36%) Forward Only Surviving: 2063 (6.60%) Reverse Only Surviving: 491 (1.57%) Dropped: 22022 (70.47%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SC49_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC49_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC49_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC49_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC49_forward.paired.headclipped.fastq.gz SC49_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC49 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC49_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC49_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC49.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC49.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC49.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC49.hist\n",
      "[FLASH]     ./SC49.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 6674 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      6674\n",
      "[FLASH]     Combined pairs:   899\n",
      "[FLASH]     Uncombined pairs: 5775\n",
      "[FLASH]     Percent combined: 13.47%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.046 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "899 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC49/SC49_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC49_centroids.fasta --uc SC49.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 899 sequences and identified 406 clusters (clustering threshold 1.00) - 9 clusters (representing 426 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC51 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC51_1.fastq /home/working/demultiplexed/SC51_2.fastq SC51_forward.paired.fastq.gz SC51_forward.singletons.fastq.gz SC51_reverse.paired.fastq.gz SC51_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC51_1.fastq /home/working/demultiplexed/SC51_2.fastq SC51_forward.paired.fastq.gz SC51_forward.singletons.fastq.gz SC51_reverse.paired.fastq.gz SC51_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:28:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC51/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 23723 Both Surviving: 886 (3.73%) Forward Only Surviving: 537 (2.26%) Reverse Only Surviving: 282 (1.19%) Dropped: 22018 (92.81%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: SC51_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SC51_forward.singletons.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SC51_reverse.paired.fastq.gz\n",
      "Cropping the first 23 bases off reads in file: SC51_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC51_forward.paired.headclipped.fastq.gz SC51_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC51 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC51_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC51_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC51.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC51.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC51.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC51.hist\n",
      "[FLASH]     ./SC51.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 886 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      886\n",
      "[FLASH]     Combined pairs:   396\n",
      "[FLASH]     Uncombined pairs: 490\n",
      "[FLASH]     Percent combined: 44.70%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.017 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "396 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC51/SC51_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC51_centroids.fasta --uc SC51.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 396 sequences and identified 187 clusters (clustering threshold 1.00) - 2 clusters (representing 184 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC51A #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC51A_1.fastq /home/working/demultiplexed/SC51A_2.fastq SC51A_forward.paired.fastq.gz SC51A_forward.singletons.fastq.gz SC51A_reverse.paired.fastq.gz SC51A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC51A_1.fastq /home/working/demultiplexed/SC51A_2.fastq SC51A_forward.paired.fastq.gz SC51A_forward.singletons.fastq.gz SC51A_reverse.paired.fastq.gz SC51A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:28:04 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC51A/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 14869 Both Surviving: 83 (0.56%) Forward Only Surviving: 247 (1.66%) Reverse Only Surviving: 92 (0.62%) Dropped: 14447 (97.16%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SC51A_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SC51A_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC51A_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC51A_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC51A_forward.paired.headclipped.fastq.gz SC51A_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC51A -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC51A_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC51A_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC51A.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC51A.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC51A.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC51A.hist\n",
      "[FLASH]     ./SC51A.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 83 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      83\n",
      "[FLASH]     Combined pairs:   40\n",
      "[FLASH]     Uncombined pairs: 43\n",
      "[FLASH]     Percent combined: 48.19%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.003 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "40 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC51A/SC51A_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC51A_centroids.fasta --uc SC51A.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 40 sequences and identified 31 clusters (clustering threshold 1.00) - 2 clusters (representing 10 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC54A #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC54A_1.fastq /home/working/demultiplexed/SC54A_2.fastq SC54A_forward.paired.fastq.gz SC54A_forward.singletons.fastq.gz SC54A_reverse.paired.fastq.gz SC54A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC54A_1.fastq /home/working/demultiplexed/SC54A_2.fastq SC54A_forward.paired.fastq.gz SC54A_forward.singletons.fastq.gz SC54A_reverse.paired.fastq.gz SC54A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:28:07 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC54A/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 21305 Both Surviving: 2339 (10.98%) Forward Only Surviving: 692 (3.25%) Reverse Only Surviving: 326 (1.53%) Dropped: 17948 (84.24%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SC54A_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC54A_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC54A_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC54A_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC54A_forward.paired.headclipped.fastq.gz SC54A_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC54A -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC54A_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC54A_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC54A.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC54A.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC54A.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC54A.hist\n",
      "[FLASH]     ./SC54A.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2339 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2339\n",
      "[FLASH]     Combined pairs:   270\n",
      "[FLASH]     Uncombined pairs: 2069\n",
      "[FLASH]     Percent combined: 11.54%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.020 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "270 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC54A/SC54A_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC54A_centroids.fasta --uc SC54A.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 270 sequences and identified 148 clusters (clustering threshold 1.00) - 3 clusters (representing 113 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC55 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC55_1.fastq /home/working/demultiplexed/SC55_2.fastq SC55_forward.paired.fastq.gz SC55_forward.singletons.fastq.gz SC55_reverse.paired.fastq.gz SC55_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC55_1.fastq /home/working/demultiplexed/SC55_2.fastq SC55_forward.paired.fastq.gz SC55_forward.singletons.fastq.gz SC55_reverse.paired.fastq.gz SC55_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:28:10 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC55/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 22557 Both Surviving: 87 (0.39%) Forward Only Surviving: 392 (1.74%) Reverse Only Surviving: 196 (0.87%) Dropped: 21882 (97.01%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SC55_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SC55_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC55_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC55_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC55_forward.paired.headclipped.fastq.gz SC55_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC55 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC55_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC55_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC55.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC55.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC55.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC55.hist\n",
      "[FLASH]     ./SC55.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 87 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      87\n",
      "[FLASH]     Combined pairs:   27\n",
      "[FLASH]     Uncombined pairs: 60\n",
      "[FLASH]     Percent combined: 31.03%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.002 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "27 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC55/SC55_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC55_centroids.fasta --uc SC55.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 27 sequences and identified 22 clusters (clustering threshold 1.00) - 1 clusters (representing 5 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC57 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC57_1.fastq /home/working/demultiplexed/SC57_2.fastq SC57_forward.paired.fastq.gz SC57_forward.singletons.fastq.gz SC57_reverse.paired.fastq.gz SC57_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC57_1.fastq /home/working/demultiplexed/SC57_2.fastq SC57_forward.paired.fastq.gz SC57_forward.singletons.fastq.gz SC57_reverse.paired.fastq.gz SC57_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:28:14 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC57/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 41048 Both Surviving: 3355 (8.17%) Forward Only Surviving: 1317 (3.21%) Reverse Only Surviving: 345 (0.84%) Dropped: 36031 (87.78%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SC57_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SC57_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC57_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC57_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC57_forward.paired.headclipped.fastq.gz SC57_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC57 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC57_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC57_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC57.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC57.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC57.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC57.hist\n",
      "[FLASH]     ./SC57.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3355 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3355\n",
      "[FLASH]     Combined pairs:   2894\n",
      "[FLASH]     Uncombined pairs: 461\n",
      "[FLASH]     Percent combined: 86.26%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.086 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2894 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC57/SC57_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC57_centroids.fasta --uc SC57.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2894 sequences and identified 1559 clusters (clustering threshold 1.00) - 25 clusters (representing 1111 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC60 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC60_1.fastq /home/working/demultiplexed/SC60_2.fastq SC60_forward.paired.fastq.gz SC60_forward.singletons.fastq.gz SC60_reverse.paired.fastq.gz SC60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC60_1.fastq /home/working/demultiplexed/SC60_2.fastq SC60_forward.paired.fastq.gz SC60_forward.singletons.fastq.gz SC60_reverse.paired.fastq.gz SC60_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:28:22 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC60/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 37624 Both Surviving: 4932 (13.11%) Forward Only Surviving: 1535 (4.08%) Reverse Only Surviving: 322 (0.86%) Dropped: 30835 (81.96%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SC60_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SC60_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC60_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC60_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC60_forward.paired.headclipped.fastq.gz SC60_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC60 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC60_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC60_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC60.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC60.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC60.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC60.hist\n",
      "[FLASH]     ./SC60.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 4932 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      4932\n",
      "[FLASH]     Combined pairs:   4428\n",
      "[FLASH]     Uncombined pairs: 504\n",
      "[FLASH]     Percent combined: 89.78%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.121 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "4428 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC60/SC60_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC60_centroids.fasta --uc SC60.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 4428 sequences and identified 2621 clusters (clustering threshold 1.00) - 51 clusters (representing 1597 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC61 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC61_1.fastq /home/working/demultiplexed/SC61_2.fastq SC61_forward.paired.fastq.gz SC61_forward.singletons.fastq.gz SC61_reverse.paired.fastq.gz SC61_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC61_1.fastq /home/working/demultiplexed/SC61_2.fastq SC61_forward.paired.fastq.gz SC61_forward.singletons.fastq.gz SC61_reverse.paired.fastq.gz SC61_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:28:32 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC61/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 18563 Both Surviving: 312 (1.68%) Forward Only Surviving: 310 (1.67%) Reverse Only Surviving: 307 (1.65%) Dropped: 17634 (95.00%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SC61_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC61_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC61_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC61_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC61_forward.paired.headclipped.fastq.gz SC61_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC61 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC61_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC61_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC61.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC61.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC61.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC61.hist\n",
      "[FLASH]     ./SC61.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 312 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      312\n",
      "[FLASH]     Combined pairs:   4\n",
      "[FLASH]     Uncombined pairs: 308\n",
      "[FLASH]     Percent combined: 1.28%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.003 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "4 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC61/SC61_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC61_centroids.fasta --uc SC61.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 4 sequences and identified 4 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC66 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC66_1.fastq /home/working/demultiplexed/SC66_2.fastq SC66_forward.paired.fastq.gz SC66_forward.singletons.fastq.gz SC66_reverse.paired.fastq.gz SC66_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC66_1.fastq /home/working/demultiplexed/SC66_2.fastq SC66_forward.paired.fastq.gz SC66_forward.singletons.fastq.gz SC66_reverse.paired.fastq.gz SC66_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:28:34 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC66/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 8462 Both Surviving: 1017 (12.02%) Forward Only Surviving: 497 (5.87%) Reverse Only Surviving: 76 (0.90%) Dropped: 6872 (81.21%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SC66_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SC66_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SC66_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SC66_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC66_forward.paired.headclipped.fastq.gz SC66_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC66 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC66_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC66_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC66.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC66.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC66.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC66.hist\n",
      "[FLASH]     ./SC66.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1017 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1017\n",
      "[FLASH]     Combined pairs:   811\n",
      "[FLASH]     Uncombined pairs: 206\n",
      "[FLASH]     Percent combined: 79.74%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.022 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "811 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC66/SC66_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC66_centroids.fasta --uc SC66.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 811 sequences and identified 360 clusters (clustering threshold 1.00) - 7 clusters (representing 410 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC7 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC7_1.fastq /home/working/demultiplexed/SC7_2.fastq SC7_forward.paired.fastq.gz SC7_forward.singletons.fastq.gz SC7_reverse.paired.fastq.gz SC7_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC7_1.fastq /home/working/demultiplexed/SC7_2.fastq SC7_forward.paired.fastq.gz SC7_forward.singletons.fastq.gz SC7_reverse.paired.fastq.gz SC7_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:28:36 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC7/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 4 Both Surviving: 0 (0.00%) Forward Only Surviving: 0 (0.00%) Reverse Only Surviving: 1 (25.00%) Dropped: 3 (75.00%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 22 bases off reads in file: SC7_forward.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC7_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC7_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC7_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC7_forward.paired.headclipped.fastq.gz SC7_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC7 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC7_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC7_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC7.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC7.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC7.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC7.hist\n",
      "[FLASH]     ./SC7.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      0\n",
      "[FLASH]     Combined pairs:   0\n",
      "[FLASH]     Uncombined pairs: 0\n",
      "[FLASH]     Percent combined: 0.00%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "0 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC7/SC7_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC7_centroids.fasta --uc SC7.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 0 sequences and identified 0 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC72 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC72_1.fastq /home/working/demultiplexed/SC72_2.fastq SC72_forward.paired.fastq.gz SC72_forward.singletons.fastq.gz SC72_reverse.paired.fastq.gz SC72_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC72_1.fastq /home/working/demultiplexed/SC72_2.fastq SC72_forward.paired.fastq.gz SC72_forward.singletons.fastq.gz SC72_reverse.paired.fastq.gz SC72_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:28:37 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC72/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 9942 Both Surviving: 120 (1.21%) Forward Only Surviving: 158 (1.59%) Reverse Only Surviving: 139 (1.40%) Dropped: 9525 (95.81%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SC72_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SC72_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC72_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC72_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC72_forward.paired.headclipped.fastq.gz SC72_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC72 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC72_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC72_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC72.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC72.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC72.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC72.hist\n",
      "[FLASH]     ./SC72.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 120 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      120\n",
      "[FLASH]     Combined pairs:   9\n",
      "[FLASH]     Uncombined pairs: 111\n",
      "[FLASH]     Percent combined: 7.50%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.002 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "9 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC72/SC72_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC72_centroids.fasta --uc SC72.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 9 sequences and identified 9 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC74 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC74_1.fastq /home/working/demultiplexed/SC74_2.fastq SC74_forward.paired.fastq.gz SC74_forward.singletons.fastq.gz SC74_reverse.paired.fastq.gz SC74_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC74_1.fastq /home/working/demultiplexed/SC74_2.fastq SC74_forward.paired.fastq.gz SC74_forward.singletons.fastq.gz SC74_reverse.paired.fastq.gz SC74_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:28:38 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC74/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 14010 Both Surviving: 31 (0.22%) Forward Only Surviving: 250 (1.78%) Reverse Only Surviving: 64 (0.46%) Dropped: 13665 (97.54%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SC74_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SC74_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC74_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC74_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC74_forward.paired.headclipped.fastq.gz SC74_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC74 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC74_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC74_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC74.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC74.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC74.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC74.hist\n",
      "[FLASH]     ./SC74.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 31 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      31\n",
      "[FLASH]     Combined pairs:   20\n",
      "[FLASH]     Uncombined pairs: 11\n",
      "[FLASH]     Percent combined: 64.52%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "20 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC74/SC74_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC74_centroids.fasta --uc SC74.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 20 sequences and identified 19 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC77 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC77_1.fastq /home/working/demultiplexed/SC77_2.fastq SC77_forward.paired.fastq.gz SC77_forward.singletons.fastq.gz SC77_reverse.paired.fastq.gz SC77_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC77_1.fastq /home/working/demultiplexed/SC77_2.fastq SC77_forward.paired.fastq.gz SC77_forward.singletons.fastq.gz SC77_reverse.paired.fastq.gz SC77_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:28:41 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC77/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 28064 Both Surviving: 1951 (6.95%) Forward Only Surviving: 902 (3.21%) Reverse Only Surviving: 222 (0.79%) Dropped: 24989 (89.04%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SC77_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SC77_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC77_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC77_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC77_forward.paired.headclipped.fastq.gz SC77_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC77 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC77_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC77_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC77.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC77.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC77.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC77.hist\n",
      "[FLASH]     ./SC77.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1951 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1951\n",
      "[FLASH]     Combined pairs:   1605\n",
      "[FLASH]     Uncombined pairs: 346\n",
      "[FLASH]     Percent combined: 82.27%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.042 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1605 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC77/SC77_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC77_centroids.fasta --uc SC77.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1605 sequences and identified 885 clusters (clustering threshold 1.00) - 18 clusters (representing 637 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC78 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC78_1.fastq /home/working/demultiplexed/SC78_2.fastq SC78_forward.paired.fastq.gz SC78_forward.singletons.fastq.gz SC78_reverse.paired.fastq.gz SC78_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC78_1.fastq /home/working/demultiplexed/SC78_2.fastq SC78_forward.paired.fastq.gz SC78_forward.singletons.fastq.gz SC78_reverse.paired.fastq.gz SC78_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:28:46 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC78/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 14701 Both Surviving: 2518 (17.13%) Forward Only Surviving: 803 (5.46%) Reverse Only Surviving: 238 (1.62%) Dropped: 11142 (75.79%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SC78_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC78_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC78_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: SC78_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC78_forward.paired.headclipped.fastq.gz SC78_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC78 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC78_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC78_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC78.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC78.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC78.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC78.hist\n",
      "[FLASH]     ./SC78.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 2518 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      2518\n",
      "[FLASH]     Combined pairs:   102\n",
      "[FLASH]     Uncombined pairs: 2416\n",
      "[FLASH]     Percent combined: 4.05%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.016 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "102 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC78/SC78_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC78_centroids.fasta --uc SC78.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 102 sequences and identified 82 clusters (clustering threshold 1.00) - 1 clusters (representing 12 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC81 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC81_1.fastq /home/working/demultiplexed/SC81_2.fastq SC81_forward.paired.fastq.gz SC81_forward.singletons.fastq.gz SC81_reverse.paired.fastq.gz SC81_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC81_1.fastq /home/working/demultiplexed/SC81_2.fastq SC81_forward.paired.fastq.gz SC81_forward.singletons.fastq.gz SC81_reverse.paired.fastq.gz SC81_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:28:49 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC81/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 24476 Both Surviving: 4241 (17.33%) Forward Only Surviving: 1081 (4.42%) Reverse Only Surviving: 382 (1.56%) Dropped: 18772 (76.70%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: SC81_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: SC81_forward.singletons.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SC81_reverse.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: SC81_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC81_forward.paired.headclipped.fastq.gz SC81_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC81 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC81_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC81_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC81.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC81.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC81.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC81.hist\n",
      "[FLASH]     ./SC81.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 4241 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      4241\n",
      "[FLASH]     Combined pairs:   3838\n",
      "[FLASH]     Uncombined pairs: 403\n",
      "[FLASH]     Percent combined: 90.50%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.122 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "3838 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC81/SC81_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC81_centroids.fasta --uc SC81.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 3838 sequences and identified 2236 clusters (clustering threshold 1.00) - 47 clusters (representing 1401 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC82 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC82_1.fastq /home/working/demultiplexed/SC82_2.fastq SC82_forward.paired.fastq.gz SC82_forward.singletons.fastq.gz SC82_reverse.paired.fastq.gz SC82_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC82_1.fastq /home/working/demultiplexed/SC82_2.fastq SC82_forward.paired.fastq.gz SC82_forward.singletons.fastq.gz SC82_reverse.paired.fastq.gz SC82_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:28:57 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC82/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 11078 Both Surviving: 1658 (14.97%) Forward Only Surviving: 712 (6.43%) Reverse Only Surviving: 156 (1.41%) Dropped: 8552 (77.20%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SC82_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC82_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC82_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC82_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC82_forward.paired.headclipped.fastq.gz SC82_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC82 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC82_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC82_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC82.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC82.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC82.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC82.hist\n",
      "[FLASH]     ./SC82.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1658 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1658\n",
      "[FLASH]     Combined pairs:   228\n",
      "[FLASH]     Uncombined pairs: 1430\n",
      "[FLASH]     Percent combined: 13.75%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.014 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "228 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC82/SC82_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC82_centroids.fasta --uc SC82.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 228 sequences and identified 168 clusters (clustering threshold 1.00) - 6 clusters (representing 56 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC86A #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC86A_1.fastq /home/working/demultiplexed/SC86A_2.fastq SC86A_forward.paired.fastq.gz SC86A_forward.singletons.fastq.gz SC86A_reverse.paired.fastq.gz SC86A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC86A_1.fastq /home/working/demultiplexed/SC86A_2.fastq SC86A_forward.paired.fastq.gz SC86A_forward.singletons.fastq.gz SC86A_reverse.paired.fastq.gz SC86A_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:29:01 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC86A/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 33212 Both Surviving: 5816 (17.51%) Forward Only Surviving: 1743 (5.25%) Reverse Only Surviving: 506 (1.52%) Dropped: 25147 (75.72%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SC86A_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC86A_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC86A_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC86A_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC86A_forward.paired.headclipped.fastq.gz SC86A_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC86A -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC86A_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC86A_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC86A.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC86A.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC86A.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC86A.hist\n",
      "[FLASH]     ./SC86A.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 5816 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      5816\n",
      "[FLASH]     Combined pairs:   1295\n",
      "[FLASH]     Uncombined pairs: 4521\n",
      "[FLASH]     Percent combined: 22.27%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.046 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1295 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC86A/SC86A_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC86A_centroids.fasta --uc SC86A.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1295 sequences and identified 585 clusters (clustering threshold 1.00) - 9 clusters (representing 583 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC88 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC88_1.fastq /home/working/demultiplexed/SC88_2.fastq SC88_forward.paired.fastq.gz SC88_forward.singletons.fastq.gz SC88_reverse.paired.fastq.gz SC88_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC88_1.fastq /home/working/demultiplexed/SC88_2.fastq SC88_forward.paired.fastq.gz SC88_forward.singletons.fastq.gz SC88_reverse.paired.fastq.gz SC88_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:29:07 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC88/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 22460 Both Surviving: 3099 (13.80%) Forward Only Surviving: 1398 (6.22%) Reverse Only Surviving: 318 (1.42%) Dropped: 17645 (78.56%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: SC88_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: SC88_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC88_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC88_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC88_forward.paired.headclipped.fastq.gz SC88_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC88 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC88_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC88_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC88.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC88.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC88.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC88.hist\n",
      "[FLASH]     ./SC88.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3099 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3099\n",
      "[FLASH]     Combined pairs:   1576\n",
      "[FLASH]     Uncombined pairs: 1523\n",
      "[FLASH]     Percent combined: 50.86%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.046 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "1576 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC88/SC88_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC88_centroids.fasta --uc SC88.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 1576 sequences and identified 873 clusters (clustering threshold 1.00) - 22 clusters (representing 660 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC9 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC9_1.fastq /home/working/demultiplexed/SC9_2.fastq SC9_forward.paired.fastq.gz SC9_forward.singletons.fastq.gz SC9_reverse.paired.fastq.gz SC9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC9_1.fastq /home/working/demultiplexed/SC9_2.fastq SC9_forward.paired.fastq.gz SC9_forward.singletons.fastq.gz SC9_reverse.paired.fastq.gz SC9_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:29:12 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC9/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 19434 Both Surviving: 882 (4.54%) Forward Only Surviving: 689 (3.55%) Reverse Only Surviving: 234 (1.20%) Dropped: 17629 (90.71%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 24 bases off reads in file: SC9_forward.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: SC9_forward.singletons.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC9_reverse.paired.fastq.gz\n",
      "Cropping the first 22 bases off reads in file: SC9_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC9_forward.paired.headclipped.fastq.gz SC9_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC9 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC9_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC9_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC9.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC9.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC9.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC9.hist\n",
      "[FLASH]     ./SC9.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 882 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      882\n",
      "[FLASH]     Combined pairs:   665\n",
      "[FLASH]     Uncombined pairs: 217\n",
      "[FLASH]     Percent combined: 75.40%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.018 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "665 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC9/SC9_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC9_centroids.fasta --uc SC9.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 665 sequences and identified 445 clusters (clustering threshold 1.00) - 15 clusters (representing 211 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: SC93 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC93_1.fastq /home/working/demultiplexed/SC93_2.fastq SC93_forward.paired.fastq.gz SC93_forward.singletons.fastq.gz SC93_reverse.paired.fastq.gz SC93_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/SC93_1.fastq /home/working/demultiplexed/SC93_2.fastq SC93_forward.paired.fastq.gz SC93_forward.singletons.fastq.gz SC93_reverse.paired.fastq.gz SC93_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:29:15 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/SC93/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 13069 Both Surviving: 496 (3.80%) Forward Only Surviving: 247 (1.89%) Reverse Only Surviving: 106 (0.81%) Dropped: 12220 (93.50%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 21 bases off reads in file: SC93_forward.paired.fastq.gz\n",
      "Cropping the first 21 bases off reads in file: SC93_forward.singletons.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SC93_reverse.paired.fastq.gz\n",
      "Cropping the first 28 bases off reads in file: SC93_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash SC93_forward.paired.headclipped.fastq.gz SC93_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o SC93 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     SC93_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     SC93_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./SC93.extendedFrags.fastq.gz\n",
      "[FLASH]     ./SC93.notCombined_1.fastq.gz\n",
      "[FLASH]     ./SC93.notCombined_2.fastq.gz\n",
      "[FLASH]     ./SC93.hist\n",
      "[FLASH]     ./SC93.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 496 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      496\n",
      "[FLASH]     Combined pairs:   37\n",
      "[FLASH]     Uncombined pairs: 459\n",
      "[FLASH]     Percent combined: 7.46%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.005 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "37 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/SC93/SC93_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids SC93_centroids.fasta --uc SC93.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 37 sequences and identified 31 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: T19 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T19_1.fastq /home/working/demultiplexed/T19_2.fastq T19_forward.paired.fastq.gz T19_forward.singletons.fastq.gz T19_reverse.paired.fastq.gz T19_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T19_1.fastq /home/working/demultiplexed/T19_2.fastq T19_forward.paired.fastq.gz T19_forward.singletons.fastq.gz T19_reverse.paired.fastq.gz T19_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:29:18 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/T19/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 29655 Both Surviving: 29 (0.10%) Forward Only Surviving: 265 (0.89%) Reverse Only Surviving: 90 (0.30%) Dropped: 29271 (98.71%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: T19_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: T19_forward.singletons.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: T19_reverse.paired.fastq.gz\n",
      "Cropping the first 24 bases off reads in file: T19_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash T19_forward.paired.headclipped.fastq.gz T19_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o T19 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     T19_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     T19_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./T19.extendedFrags.fastq.gz\n",
      "[FLASH]     ./T19.notCombined_1.fastq.gz\n",
      "[FLASH]     ./T19.notCombined_2.fastq.gz\n",
      "[FLASH]     ./T19.hist\n",
      "[FLASH]     ./T19.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 29 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      29\n",
      "[FLASH]     Combined pairs:   18\n",
      "[FLASH]     Uncombined pairs: 11\n",
      "[FLASH]     Percent combined: 62.07%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.001 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "18 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/T19/T19_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids T19_centroids.fasta --uc T19.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 18 sequences and identified 16 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: T21 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T21_1.fastq /home/working/demultiplexed/T21_2.fastq T21_forward.paired.fastq.gz T21_forward.singletons.fastq.gz T21_reverse.paired.fastq.gz T21_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T21_1.fastq /home/working/demultiplexed/T21_2.fastq T21_forward.paired.fastq.gz T21_forward.singletons.fastq.gz T21_reverse.paired.fastq.gz T21_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:29:20 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/T21/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 8196 Both Surviving: 288 (3.51%) Forward Only Surviving: 162 (1.98%) Reverse Only Surviving: 140 (1.71%) Dropped: 7606 (92.80%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: T21_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: T21_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: T21_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: T21_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash T21_forward.paired.headclipped.fastq.gz T21_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o T21 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     T21_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     T21_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./T21.extendedFrags.fastq.gz\n",
      "[FLASH]     ./T21.notCombined_1.fastq.gz\n",
      "[FLASH]     ./T21.notCombined_2.fastq.gz\n",
      "[FLASH]     ./T21.hist\n",
      "[FLASH]     ./T21.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 288 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      288\n",
      "[FLASH]     Combined pairs:   28\n",
      "[FLASH]     Uncombined pairs: 260\n",
      "[FLASH]     Percent combined: 9.72%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.003 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "28 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/T21/T21_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids T21_centroids.fasta --uc T21.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 28 sequences and identified 24 clusters (clustering threshold 1.00) - 0 clusters (representing 0 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: T32 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T32_1.fastq /home/working/demultiplexed/T32_2.fastq T32_forward.paired.fastq.gz T32_forward.singletons.fastq.gz T32_reverse.paired.fastq.gz T32_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T32_1.fastq /home/working/demultiplexed/T32_2.fastq T32_forward.paired.fastq.gz T32_forward.singletons.fastq.gz T32_reverse.paired.fastq.gz T32_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:29:21 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/T32/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 13172 Both Surviving: 1614 (12.25%) Forward Only Surviving: 386 (2.93%) Reverse Only Surviving: 163 (1.24%) Dropped: 11009 (83.58%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: T32_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: T32_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: T32_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: T32_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash T32_forward.paired.headclipped.fastq.gz T32_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o T32 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     T32_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     T32_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./T32.extendedFrags.fastq.gz\n",
      "[FLASH]     ./T32.notCombined_1.fastq.gz\n",
      "[FLASH]     ./T32.notCombined_2.fastq.gz\n",
      "[FLASH]     ./T32.hist\n",
      "[FLASH]     ./T32.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 1614 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      1614\n",
      "[FLASH]     Combined pairs:   285\n",
      "[FLASH]     Uncombined pairs: 1329\n",
      "[FLASH]     Percent combined: 17.66%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.013 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "285 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/T32/T32_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids T32_centroids.fasta --uc T32.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 285 sequences and identified 187 clusters (clustering threshold 1.00) - 3 clusters (representing 87 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: T49 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T49_1.fastq /home/working/demultiplexed/T49_2.fastq T49_forward.paired.fastq.gz T49_forward.singletons.fastq.gz T49_reverse.paired.fastq.gz T49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T49_1.fastq /home/working/demultiplexed/T49_2.fastq T49_forward.paired.fastq.gz T49_forward.singletons.fastq.gz T49_reverse.paired.fastq.gz T49_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:29:24 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/T49/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 9868 Both Surviving: 421 (4.27%) Forward Only Surviving: 275 (2.79%) Reverse Only Surviving: 113 (1.15%) Dropped: 9059 (91.80%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 20 bases off reads in file: T49_forward.paired.fastq.gz\n",
      "Cropping the first 20 bases off reads in file: T49_forward.singletons.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: T49_reverse.paired.fastq.gz\n",
      "Cropping the first 27 bases off reads in file: T49_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash T49_forward.paired.headclipped.fastq.gz T49_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o T49 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     T49_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     T49_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./T49.extendedFrags.fastq.gz\n",
      "[FLASH]     ./T49.notCombined_1.fastq.gz\n",
      "[FLASH]     ./T49.notCombined_2.fastq.gz\n",
      "[FLASH]     ./T49.hist\n",
      "[FLASH]     ./T49.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 421 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      421\n",
      "[FLASH]     Combined pairs:   339\n",
      "[FLASH]     Uncombined pairs: 82\n",
      "[FLASH]     Percent combined: 80.52%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.009 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "339 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/T49/T49_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids T49_centroids.fasta --uc T49.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 339 sequences and identified 210 clusters (clustering threshold 1.00) - 10 clusters (representing 127 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: T68 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T68_1.fastq /home/working/demultiplexed/T68_2.fastq T68_forward.paired.fastq.gz T68_forward.singletons.fastq.gz T68_reverse.paired.fastq.gz T68_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T68_1.fastq /home/working/demultiplexed/T68_2.fastq T68_forward.paired.fastq.gz T68_forward.singletons.fastq.gz T68_reverse.paired.fastq.gz T68_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:29:26 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/T68/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 23250 Both Surviving: 3693 (15.88%) Forward Only Surviving: 778 (3.35%) Reverse Only Surviving: 388 (1.67%) Dropped: 18391 (79.10%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 25 bases off reads in file: T68_forward.paired.fastq.gz\n",
      "Cropping the first 25 bases off reads in file: T68_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: T68_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: T68_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash T68_forward.paired.headclipped.fastq.gz T68_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o T68 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     T68_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     T68_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./T68.extendedFrags.fastq.gz\n",
      "[FLASH]     ./T68.notCombined_1.fastq.gz\n",
      "[FLASH]     ./T68.notCombined_2.fastq.gz\n",
      "[FLASH]     ./T68.hist\n",
      "[FLASH]     ./T68.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 3693 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      3693\n",
      "[FLASH]     Combined pairs:   151\n",
      "[FLASH]     Uncombined pairs: 3542\n",
      "[FLASH]     Percent combined: 4.09%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.022 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "151 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/T68/T68_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids T68_centroids.fasta --uc T68.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 151 sequences and identified 99 clusters (clustering threshold 1.00) - 2 clusters (representing 38 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "##### processing query ID: T78 #####\n",
      "\n",
      "\n",
      "### READ QUALITY TRIMMING ###\n",
      "\n",
      "\n",
      "trimming PE reads with trimmomatic\n",
      "java -jar /usr/bin/trimmomatic-0.32.jar PE -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T78_1.fastq /home/working/demultiplexed/T78_2.fastq T78_forward.paired.fastq.gz T78_forward.singletons.fastq.gz T78_reverse.paired.fastq.gz T78_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "TrimmomaticPE: Started with arguments: -threads 1 -phred33 -trimlog trimmomatic.log /home/working/demultiplexed/T78_1.fastq /home/working/demultiplexed/T78_2.fastq T78_forward.paired.fastq.gz T78_forward.singletons.fastq.gz T78_reverse.paired.fastq.gz T78_reverse.singletons.fastq.gz ILLUMINACLIP:None:5:5:5 TRAILING:30 LEADING:30 SLIDINGWINDOW:5:30 MINLEN:90\n",
      "Oct 19, 2016 5:29:30 PM org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer makeIlluminaClippingTrimmer\n",
      "SEVERE: null\n",
      "java.io.FileNotFoundException: /home/working/T78/None (No such file or directory)\n",
      "\tat java.io.FileInputStream.open(Native Method)\n",
      "\tat java.io.FileInputStream.<init>(FileInputStream.java:146)\n",
      "\tat org.usadellab.trimmomatic.fasta.FastaParser.parse(FastaParser.java:54)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.loadSequences(IlluminaClippingTrimmer.java:107)\n",
      "\tat org.usadellab.trimmomatic.trim.IlluminaClippingTrimmer.makeIlluminaClippingTrimmer(IlluminaClippingTrimmer.java:70)\n",
      "\tat org.usadellab.trimmomatic.trim.TrimmerFactory.makeTrimmer(TrimmerFactory.java:27)\n",
      "\tat org.usadellab.trimmomatic.TrimmomaticPE.run(TrimmomaticPE.java:495)\n",
      "\tat org.usadellab.trimmomatic.Trimmomatic.main(Trimmomatic.java:35)\n",
      "\n",
      "Input Read Pairs: 23555 Both Surviving: 6269 (26.61%) Forward Only Surviving: 1325 (5.63%) Reverse Only Surviving: 408 (1.73%) Dropped: 15553 (66.03%)\n",
      "TrimmomaticPE: Completed successfully\n",
      "\n",
      "\n",
      "### READ CLIPPING ###\n",
      "\n",
      "Cropping the first 26 bases off reads in file: T78_forward.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: T78_forward.singletons.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: T78_reverse.paired.fastq.gz\n",
      "Cropping the first 26 bases off reads in file: T78_reverse.singletons.fastq.gz\n",
      "\n",
      "### MERGING READ PAIRS ###\n",
      "\n",
      "merging paired-end reads with flash\n",
      "\n",
      "flash T78_forward.paired.headclipped.fastq.gz T78_reverse.paired.headclipped.fastq.gz -M 350 -t 1 -p 33 -o T78 -z\n",
      "[FLASH] Starting FLASH v1.2.11\n",
      "[FLASH] Fast Length Adjustment of SHort reads\n",
      "[FLASH]  \n",
      "[FLASH] Input files:\n",
      "[FLASH]     T78_forward.paired.headclipped.fastq.gz\n",
      "[FLASH]     T78_reverse.paired.headclipped.fastq.gz\n",
      "[FLASH]  \n",
      "[FLASH] Output files:\n",
      "[FLASH]     ./T78.extendedFrags.fastq.gz\n",
      "[FLASH]     ./T78.notCombined_1.fastq.gz\n",
      "[FLASH]     ./T78.notCombined_2.fastq.gz\n",
      "[FLASH]     ./T78.hist\n",
      "[FLASH]     ./T78.histogram\n",
      "[FLASH]  \n",
      "[FLASH] Parameters:\n",
      "[FLASH]     Min overlap:           10\n",
      "[FLASH]     Max overlap:           350\n",
      "[FLASH]     Max mismatch density:  0.250000\n",
      "[FLASH]     Allow \"outie\" pairs:   false\n",
      "[FLASH]     Cap mismatch quals:    false\n",
      "[FLASH]     Combiner threads:      1\n",
      "[FLASH]     Input format:          FASTQ, phred_offset=33\n",
      "[FLASH]     Output format:         FASTQ, phred_offset=33, gzip\n",
      "[FLASH]  \n",
      "[FLASH] Starting reader and writer threads\n",
      "[FLASH] Starting 1 combiner threads\n",
      "[FLASH] Processed 6269 read pairs\n",
      "[FLASH]  \n",
      "[FLASH] Read combination statistics:\n",
      "[FLASH]     Total pairs:      6269\n",
      "[FLASH]     Combined pairs:   2603\n",
      "[FLASH]     Uncombined pairs: 3666\n",
      "[FLASH]     Percent combined: 41.52%\n",
      "[FLASH]  \n",
      "[FLASH] Writing histogram files.\n",
      "[FLASH]  \n",
      "[FLASH] FLASH v1.2.11 complete!\n",
      "[FLASH] 0.082 seconds elapsed\n",
      "\n",
      "\n",
      "\n",
      "### Fastq to fasta (plus filtering for length, merged-only, etc. if applicable) ###\n",
      "\n",
      "\n",
      "keeping only merged reads for subsequent analyses\n",
      "\n",
      "2603 sequences survived filtering\n",
      "\n",
      "\n",
      "### CLUSTERING ###\n",
      "\n",
      "\n",
      "clustering using vsearch\n",
      "vsearch --cluster_fast /home/working/T78/T78_trimmed.fasta --id 1.00 --strand both --threads 1 --centroids T78_centroids.fasta --uc T78.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "reduce cluster files - minimum coverage: 5\n",
      "\n",
      "vsearch processed 2603 sequences and identified 1486 clusters (clustering threshold 1.00) - 60 clusters (representing 972 sequences) passed the filter criteria (minimum of 5 sequences per cluster) and will be used in subsequent analyses\n",
      "\n",
      "WRITING BASIC READ STATS TO FILE\n",
      "\n",
      "\n",
      "### GLOBAL CLUSTERING ###\n",
      "\n",
      "\n",
      "Concatenating for global clustering\n",
      "\n",
      "\n",
      "Performing global clustering\n",
      "\n",
      "vsearch --cluster_fast /home/working/GLOBAL/global_queries.fasta --id 1.00 --strand both --threads 1 --centroids global_centroids.fasta --uc global.uc\n",
      "vsearch v1.1.0_linux_x86_64, 31.4GB RAM, 12 cores\n",
      "https://github.com/torognes/vsearch\n",
      "\n",
      "\n",
      "\n",
      "parse vsearch uc file\n",
      "\n",
      "\n",
      "writing denovo OTU table\n",
      "\n",
      "\n",
      "Please select a method for taxonomic assignment:\n",
      "--blast\n",
      "--pplace\n",
      "--kraken\n",
      "\n",
      "\n",
      "##### DONE! #####\n",
      "\n",
      "\n",
      "Wed Oct 19 17:29:49 2016\n",
      "\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "metaBEAT_global.py \\\n",
    "-Q Querymap_global.txt \\\n",
    "--trim_qual 30 \\\n",
    "--trim_minlength 90 \\\n",
    "--merge \\\n",
    "--product_length 350 \\\n",
    "--merged_only \\\n",
    "-R REFmap.txt \\\n",
    "--cluster \\\n",
    "--clust_match 1 \\\n",
    "--clust_cov 5 \\\n",
    "--min_ident 0.90 \\\n",
    "-o MothPollenRestart \\\n",
    "-@ callumjmacgregor@gmail.com\n",
    "-n 5 -v &> log_restart"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "collapsed": true
   },
   "source": [
    "Trimming, pairing and clustering (within samples) has been successful. Next step is to try __identification using BLAST__ and __clustering across samples__."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Produce new querymap based on the one from the previous runs. Specify the clustering results from the last run as input."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SA177\tfasta\tSA177/SA177_trimmed.fasta\n",
      "SB140\tfasta\tSB140/SB140_trimmed.fasta\n",
      "T19\tfasta\tT19/T19_trimmed.fasta\n",
      "Lg45\tfasta\tLg45/Lg45_trimmed.fasta\n",
      "Q69\tfasta\tQ69/Q69_trimmed.fasta\n",
      "S72\tfasta\tS72/S72_trimmed.fasta\n",
      "SC66\tfasta\tSC66/SC66_trimmed.fasta\n",
      "S31\tfasta\tS31/S31_trimmed.fasta\n",
      "SC74\tfasta\tSC74/SC74_trimmed.fasta\n",
      "SA68\tfasta\tSA68/SA68_trimmed.fasta\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "for sample in $(cat Querymap_global.txt | cut -f 1)\n",
    "do\n",
    "    fasta=$(ls -1 $sample/$sample\\_trimmed.fasta)\n",
    "    echo -e \"$sample\\tfasta\\t$fasta\"\n",
    "done > Querymap.txt\n",
    "\n",
    "head Querymap.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now try doing a BLAST against the entirety of Genbank, using a recently downloaded local version"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "N.b. sometimes this will fail due to a dodgy GI/TaxID combination. In this case you need to: (1) figure out the GI of the search that failed; (2) go to NCBI and find out the TaxID for that GI; (3) create a file called gi_to_taxid.csv; (4) write the first line of that file as gi,taxid. If this still fails you may not have chosen the right GI, or there may be several dodgy ones."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "metaBEAT_global.py \\\n",
    "-Q Querymap.txt \\\n",
    "--cluster \\\n",
    "--clust_match 1 \\\n",
    "--clust_cov 5 \\\n",
    "--blast \\\n",
    "--blast_db Final/nt/nt \\\n",
    "--min_ident 0.95 \\\n",
    "-n 5 \\\n",
    "-v \\\n",
    "-@ callumjmacgregor@gmail.com \\\n",
    "-o MothPollenGenbankRestart > log"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We want to inspect the outputs of this on a case by case basis. The file we need for this is a .xml file and it would be easier to have it as a .txt file, so we partially rerun the above query to convert it."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "blastn -query /home/working/GLOBAL/global_centroids.fasta \\\n",
    "-db /home/working/Final/nt/nt \\\n",
    "-out GLOBAL/BLAST_0.95/global_blastn.out.txt"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now we want to download our curated reference list (with help from inspecting the outputs of the above, and a list of East Yorkshire flora from Dick Middleton).\n",
    "\n",
    "The list of binomial species names was prepared in a simple text file: FinalRefList.txt."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Abies alba\r",
      "\r\n",
      "Abies cephalonica\r",
      "\r\n",
      "Abies cilicica\r",
      "\r\n",
      "Abies concolor\r",
      "\r\n",
      "Abies delavayi\r",
      "\r\n",
      "Abies firma\r",
      "\r\n",
      "Abies grandis\r",
      "\r\n",
      "Abies homolepis\r",
      "\r\n",
      "Abies lasiocarpa\r",
      "\r\n",
      "Abies nordmanniana\r",
      "\r\n"
     ]
    }
   ],
   "source": [
    "!head FinalRefList.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "metaBEAT may be querying NCBI's Entrez databases to fetch/verify taxonomic ids. Entrez User requirements state that you need to identify yourself by providing an email address so that NCBI can contact you in case there is a problem.\n",
      "\n",
      "You have specified: 'callumjmacgregor@gmail.com'\n",
      "\n",
      "\n",
      "QUERYING GENBANK\n",
      "\n",
      "check for synonyms for \"rbcl\" (this is relevant only for Genbank searches)\n",
      "\n",
      "fetching accessions ..\n",
      "\n",
      "#\tAbies alba\t8\n",
      "#\tAbies cephalonica\t2\n",
      "#\tAbies cilicica\t1\n",
      "#\tAbies concolor\t6\n",
      "#\tAbies delavayi\t6\n",
      "#\tAbies firma\t6\n",
      "#\tAbies grandis\t4\n",
      "#\tAbies homolepis\t4\n",
      "#\tAbies lasiocarpa\t5\n",
      "#\tAbies nordmanniana\t4\n",
      "#\tAbies pinsapo\t5\n",
      "#\tAbies procera\t1\n",
      "#\tAbies veitchii\t4\n",
      "#\tAcaena novae-zelandiae\t3\n",
      "#\tAcer campestre\t17\n",
      "#\tAcer platanoides\t6\n",
      "#\tAcer pseudoplatanus\t16\n",
      "#\tAchillea alpina\t0\n",
      "#\tAchillea distans\t0\n",
      "#\tAchillea ligustica\t0\n",
      "#\tAchillea millefolium\t39\n",
      "#\tAchillea ptarmica\t4\n",
      "#\tAconitum napellus\t5\n",
      "#\tAcorus calamus\t33\n",
      "#\tActaea spicata\t5\n",
      "#\tAdonis annua\t2\n",
      "#\tAdoxa moschatellina\t7\n",
      "#\tAegopodium podagraria\t7\n",
      "#\tAesculus carnea\t0\n",
      "#\tAesculus hippocastanum\t11\n",
      "#\tAethusa cynapium\t5\n",
      "#\tAgrimonia eupatoria\t5\n",
      "#\tAgrimonia procera\t5\n",
      "#\tAgrostemma githago\t7\n",
      "#\tAgrostis canina\t3\n",
      "#\tAgrostis capillaris\t5\n",
      "#\tAgrostis castellana\t0\n",
      "#\tAgrostis gigantea\t11\n",
      "#\tAgrostis scabra\t6\n",
      "#\tAgrostis stolonifera\t20\n",
      "#\tAgrostis vinealis\t3\n",
      "#\tAira caryophyllea\t6\n",
      "#\tAira praecox\t6\n",
      "#\tAjuga reptans\t10\n",
      "#\tAlcea rosea\t2\n",
      "#\tAlchemilla glabra\t5\n",
      "#\tAlchemilla mollis\t2\n",
      "#\tAlchemilla vulgaris\t2\n",
      "#\tAlchemilla xanthochlora\t5\n",
      "#\tAlisma lanceolatum\t4\n",
      "#\tAlisma plantago-aquatica\t17\n",
      "#\tAlliaria petiolata\t21\n",
      "#\tAllium carinatum\t1\n",
      "#\tAllium moly\t3\n",
      "#\tAllium neapolitanum\t9\n",
      "#\tAllium oleraceum\t3\n",
      "#\tAllium schoenoprasum\t3\n",
      "#\tAllium scorodoprasum\t2\n",
      "#\tAllium triquetrum\t6\n",
      "#\tAllium ursinum\t10\n",
      "#\tAllium vineale\t3\n",
      "#\tAlnus cordata\t1\n",
      "#\tAlnus glutinosa\t17\n",
      "#\tAlnus incana\t11\n",
      "#\tAlopecurus aequalis\t16\n",
      "#\tAlopecurus geniculatus\t4\n",
      "#\tAlopecurus myosuroides\t4\n",
      "#\tAlopecurus pratensis\t9\n",
      "#\tAlyssum alyssoides\t0\n",
      "#\tAmaranthus albus\t5\n",
      "#\tAmaranthus hybridus\t18\n",
      "#\tAmaranthus retroflexus\t16\n",
      "#\tAmaranthus thunbergii\t0\n",
      "#\tAmbrosia artemisiifolia\t19\n",
      "#\tAmmi majus\t1\n",
      "#\tAmmi visnaga\t1\n",
      "#\tAmsinckia lycopsoides\t0\n",
      "#\tAmsinckia micrantha\t1\n",
      "#\tAnacamptis morio\t1\n",
      "#\tAnacamptis pyramidalis\t2\n",
      "#\tAnagallis arvensis\t9\n",
      "#\tAnagallis tenella\t5\n",
      "#\tAnaphalis margaritacea\t7\n",
      "#\tAnchusa arvensis\t4\n",
      "#\tAndromeda polifolia\t15\n",
      "#\tAnemone apennina\t0\n",
      "#\tAnemone nemorosa\t6\n",
      "#\tAnemone ranunculoides\t1\n",
      "#\tAnethum graveolens\t6\n",
      "#\tAngelica sylvestris\t4\n",
      "#\tAnisantha diandra\t0\n",
      "#\tAnisantha madritensis\t0\n",
      "#\tAnisantha rigida\t7\n",
      "#\tAnisantha rubens\t0\n",
      "#\tAnisantha sterilis\t7\n",
      "#\tAnisantha tectorum\t0\n",
      "#\tAntennaria dioica\t4\n",
      "#\tAnthemis arvensis\t2\n",
      "#\tAnthemis cotula\t2\n",
      "#\tAnthemis tinctoria\t1\n",
      "#\tAnthoxanthum aristatum\t0\n",
      "#\tAnthoxanthum odoratum\t14\n",
      "#\tAnthriscus caucalis\t2\n",
      "#\tAnthriscus sylvestris\t9\n",
      "#\tAnthyllis vulneraria\t8\n",
      "#\tAntirrhinum majus\t7\n",
      "#\tApera interrupta\t1\n",
      "#\tApera spica-venti\t1\n",
      "#\tAphanes arvensis\t4\n",
      "#\tAphanes australis\t2\n",
      "#\tApium graveolens\t10\n",
      "#\tApium inundatum\t2\n",
      "#\tApium nodiflorum\t6\n",
      "#\tApium repens\t0\n",
      "#\tAquilegia vulgaris\t7\n",
      "#\tArabidopsis thaliana\t187\n",
      "#\tArabis caucasica\t1\n",
      "#\tArabis hirsuta\t7\n",
      "#\tAraucaria araucana\t5\n",
      "#\tArbutus unedo\t3\n",
      "#\tArctium lappa\t11\n",
      "#\tArctium minus\t15\n",
      "#\tArctium nemorosum\t0\n",
      "#\tArenaria leptoclados\t0\n",
      "#\tArenaria serpyllifolia\t15\n",
      "#\tArmeria maritima\t6\n",
      "#\tArmoracia rusticana\t5\n",
      "#\tArnoseris minima\t0\n",
      "#\tArrhenatherum elatius\t11\n",
      "#\tArtemisia absinthium\t8\n",
      "#\tArtemisia campestris\t3\n",
      "#\tArtemisia maritima\t2\n",
      "#\tArtemisia verlotiorum\t0\n",
      "#\tArtemisia vulgaris\t21\n",
      "#\tArum italicum\t9\n",
      "#\tArum maculatum\t7\n",
      "#\tAsparagus officinalis\t17\n",
      "#\tAsperugo procumbens\t1\n",
      "#\tAsperula arvensis\t0\n",
      "#\tAsperula cynanchica\t6\n",
      "#\tAsplenium adiantum-nigrum\t15\n",
      "#\tAsplenium ceterach\t9\n",
      "#\tAsplenium marinum\t2\n",
      "#\tAsplenium ruta-muraria\t23\n",
      "#\tAsplenium scolopendrium\t8\n",
      "#\tAsplenium trichomanes\t20\n",
      "#\tAster lanceolatus\t7\n",
      "#\tAster novi-belgii\t1\n",
      "#\tAster tripolium\t3\n",
      "#\tAstragalus danicus\t1\n",
      "#\tAstragalus glycyphyllos\t3\n",
      "#\tAstrantia major\t3\n",
      "#\tAthyrium filix-femina\t17\n",
      "#\tAtriplex littoralis\t4\n",
      "#\tAtriplex patula\t13\n",
      "#\tAtriplex prostrata\t5\n",
      "#\tAtropa belladonna\t13\n",
      "#\tAubrieta deltoidea\t1\n",
      "#\tAvena fatua\t15\n",
      "#\tAvena sativa\t10\n",
      "#\tAvenula pratensis\t2\n",
      "#\tAvenula pubescens\t5\n",
      "#\tAzolla filiculoides\t6\n",
      "#\tBaldellia ranunculoides\t4\n",
      "#\tBallota nigra\t6\n",
      "#\tBarbarea intermedia\t1\n",
      "#\tBarbarea stricta\t0\n",
      "#\tBarbarea verna\t4\n",
      "#\tBarbarea vulgaris\t16\n",
      "#\tBassia scoparia\t4\n",
      "#\tBellis perennis\t12\n",
      "#\tBerberis darwinii\t0\n",
      "#\tBerberis vulgaris\t3\n",
      "#\tBerteroa incana\t3\n",
      "#\tBerula erecta\t5\n",
      "#\tBeta trigyna\t1\n",
      "#\tBeta vulgaris\t121\n",
      "#\tBetonica officinalis\t8\n",
      "#\tBetula pendula\t18\n",
      "#\tBetula pubescens\t8\n",
      "#\tBidens cernua\t12\n",
      "#\tBidens tripartita\t7\n",
      "#\tBlackstonia perfoliata\t5\n",
      "#\tBlechnum spicant\t4\n",
      "#\tBlysmus compressus\t8\n",
      "#\tBolboschoenus maritimus\t7\n",
      "#\tBorago officinalis\t9\n",
      "#\tBotrychium lunaria\t11\n",
      "#\tBrachypodium pinnatum\t7\n",
      "#\tBrachypodium rupestre\t3\n",
      "#\tBrachypodium sylvaticum\t9\n",
      "#\tBrassica juncea\t16\n",
      "#\tBrassica napus\t290\n",
      "#\tBrassica nigra\t11\n",
      "#\tBrassica oleracea\t102\n",
      "#\tBrassica rapa\t82\n",
      "#\tBriza maxima\t5\n",
      "#\tBriza media\t7\n",
      "#\tBromopsis benekenii\t2\n",
      "#\tBromopsis erecta\t9\n",
      "#\tBromopsis inermis\t22\n",
      "#\tBromopsis ramosa\t3\n",
      "#\tBromus arvensis\t1\n",
      "#\tBromus commutatus\t4\n",
      "#\tBromus hordeaceus\t8\n",
      "#\tBromus lanceolatus\t0\n",
      "#\tBromus lepidus\t0\n",
      "#\tBromus racemosus\t3\n",
      "#\tBromus secalinus\t2\n",
      "#\tBryonia dioica\t5\n",
      "#\tBuddleja davidii\t6\n",
      "#\tBuddleja globosa\t0\n",
      "#\tBuddleja officinalis\t1\n",
      "#\tBunias orientalis\t1\n",
      "#\tBupleurum baldense\t0\n",
      "#\tBupleurum rotundifolium\t0\n",
      "#\tBupleurum subovatum\t0\n",
      "#\tBupleurum tenuissimum\t2\n",
      "#\tButomus umbellatus\t7\n",
      "#\tBuxus sempervirens\t13\n",
      "#\tCakile maritima\t4\n",
      "#\tCalamagrostis canescens\t5\n",
      "#\tCalamagrostis epigejos\t13\n",
      "#\tCalamagrostis gracilescens\t0\n",
      "#\tCalendula officinalis\t5\n",
      "#\tCalla palustris\t5\n",
      "#\tCallitriche hermaphroditica\t6\n",
      "#\tCallitriche obtusangula\t3\n",
      "#\tCallitriche platycarpa\t5\n",
      "#\tCallitriche stagnalis\t7\n",
      "#\tCalluna vulgaris\t11\n",
      "#\tCaltha palustris\t32\n",
      "#\tCalystegia pulchra\t2\n",
      "#\tCalystegia sepium\t18\n",
      "#\tCalystegia silvatica\t4\n",
      "#\tCalystegia soldanella\t6\n",
      "#\tCamelina sativa\t156\n",
      "#\tCampanula glomerata\t11\n",
      "#\tCampanula latifolia\t6\n",
      "#\tCampanula medium\t2\n",
      "#\tCampanula patula\t10\n",
      "#\tCampanula persicifolia\t2\n",
      "#\tCampanula portenschlagiana\t2\n",
      "#\tCampanula poscharskyana\t1\n",
      "#\tCampanula rapunculoides\t10\n",
      "#\tCampanula rapunculus\t5\n",
      "#\tCampanula rotundifolia\t22\n",
      "#\tCampanula trachelium\t9\n",
      "#\tCannabis sativa\t26\n",
      "#\tCapsella bursa-pastoris\t24\n",
      "#\tCapsicum annuum\t137\n",
      "#\tCardamine amara\t2\n",
      "#\tCardamine flexuosa\t5\n",
      "#\tCardamine hirsuta\t13\n",
      "#\tCardamine impatiens\t10\n",
      "#\tCardamine pratensis\t24\n",
      "#\tCarduus crispus\t3\n",
      "#\tCarduus nutans\t6\n",
      "#\tCarduus pycnocephalus\t0\n",
      "#\tCarduus tenuiflorus\t4\n",
      "#\tCarex acuta\t3\n",
      "#\tCarex acutiformis\t3\n",
      "#\tCarex appropinquata\t0\n",
      "#\tCarex arenaria\t5\n",
      "#\tCarex binervis\t3\n",
      "#\tCarex canescens\t8\n",
      "#\tCarex caryophyllea\t4\n",
      "#\tCarex demissa\t0\n",
      "#\tCarex diandra\t5\n",
      "#\tCarex dioica\t4\n",
      "#\tCarex distans\t4\n",
      "#\tCarex disticha\t3\n",
      "#\tCarex divisa\t3\n",
      "#\tCarex divulsa\t5\n",
      "#\tCarex echinata\t3\n",
      "#\tCarex elata\t3\n",
      "#\tCarex elongata\t2\n",
      "#\tCarex extensa\t5\n",
      "#\tCarex flacca\t8\n",
      "#\tCarex hirta\t4\n",
      "#\tCarex hostiana\t5\n",
      "#\tCarex lasiocarpa\t4\n",
      "#\tCarex lepidocarpa\t0\n",
      "#\tCarex leporina\t0\n",
      "#\tCarex muricata\t4\n",
      "#\tCarex nigra\t7\n",
      "#\tCarex oederi\t0\n",
      "#\tCarex otrubae\t5\n",
      "#\tCarex pallescens\t5\n",
      "#\tCarex panicea\t7\n",
      "#\tCarex paniculata\t4\n",
      "#\tCarex pendula\t9\n",
      "#\tCarex pilulifera\t5\n",
      "#\tCarex pseudocyperus\t6\n",
      "#\tCarex pulicaris\t5\n",
      "#\tCarex remota\t4\n",
      "#\tCarex riparia\t3\n",
      "#\tCarex rostrata\t4\n",
      "#\tCarex spicata\t4\n",
      "#\tCarex sylvatica\t5\n",
      "#\tCarex vesicaria\t3\n",
      "#\tCarex viridula\t4\n",
      "#\tCarlina vulgaris\t5\n",
      "#\tCarpinus betulus\t8\n",
      "#\tCarthamus tinctorius\t18\n",
      "#\tCarum carvi\t6\n",
      "#\tCastanea sativa\t12\n",
      "#\tCatabrosa aquatica\t6\n",
      "#\tCatapodium marinum\t3\n",
      "#\tCatapodium rigidum\t6\n",
      "#\tCedrus atlantica\t6\n",
      "#\tCedrus deodara\t17\n",
      "#\tCedrus libani\t4\n",
      "#\tCentaurea aspera\t0\n",
      "#\tCentaurea calcitrapa\t0\n",
      "#\tCentaurea cyanus\t5\n",
      "#\tCentaurea diluta\t0\n",
      "#\tCentaurea montana\t1\n",
      "#\tCentaurea nigra\t6\n",
      "#\tCentaurea scabiosa\t5\n",
      "#\tCentaurea solstitialis\t0\n",
      "#\tCentaurium erythraea\t7\n",
      "#\tCentaurium littorale\t1\n",
      "#\tCentaurium pulchellum\t5\n",
      "#\tCentranthus ruber\t7\n",
      "#\tCentunculus minimus\t0\n",
      "#\tCephalanthera damasonium\t1\n",
      "#\tCephalaria gigantea\t1\n",
      "#\tCerastium arvense\t7\n",
      "#\tCerastium diffusum\t4\n",
      "#\tCerastium fontanum\t15\n",
      "#\tCerastium glomeratum\t6\n",
      "#\tCerastium semidecandrum\t2\n",
      "#\tCerastium tomentosum\t2\n",
      "#\tCeratocapnos claviculata\t4\n",
      "#\tCeratochloa carinata\t2\n",
      "#\tCeratochloa cathartica\t12\n",
      "#\tCeratophyllum demersum\t15\n",
      "#\tCeratophyllum submersum\t4\n",
      "#\tChaenorhinum minus\t7\n",
      "#\tChaerophyllum temulum\t9\n",
      "#\tChamaecyparis lawsoniana\t8\n",
      "#\tChamaecyparis obtusa\t7\n",
      "#\tChamaecyparis pisifera\t7\n",
      "#\tChamaemelum nobile\t5\n",
      "#\tChamerion angustifolium\t29\n",
      "#\tChara vulgaris\t8\n",
      "#\tChelidonium majus\t14\n",
      "#\tChenopodium album\t46\n",
      "#\tChenopodium bonus-henricus\t6\n",
      "#\tChenopodium ficifolium\t5\n",
      "#\tChenopodium glaucum\t7\n",
      "#\tChenopodium hybridum\t0\n",
      "#\tChenopodium opulifolium\t2\n",
      "#\tChenopodium polyspermum\t4\n",
      "#\tChenopodium pratericola\t0\n",
      "#\tChenopodium quinoa\t13\n",
      "#\tChenopodium rubrum\t4\n",
      "#\tChenopodium urbicum\t4\n",
      "#\tChenopodium vulvaria\t4\n",
      "#\tChrysosplenium alternifolium\t4\n",
      "#\tChrysosplenium oppositifolium\t4\n",
      "#\tCicerbita macrophylla\t3\n",
      "#\tCichorium intybus\t17\n",
      "#\tCicuta virosa\t7\n",
      "#\tCircaea lutetiana\t15\n",
      "#\tCirsium acaule\t4\n",
      "#\tCirsium arvense\t14\n",
      "#\tCirsium dissectum\t3\n",
      "#\tCirsium eriophorum\t3\n",
      "#\tCirsium palustre\t8\n",
      "#\tCirsium vulgare\t18\n",
      "#\tCladium mariscus\t7\n",
      "#\tClaytonia perfoliata\t1\n",
      "#\tClaytonia sibirica\t2\n",
      "#\tClematis montana\t2\n",
      "#\tClematis tangutica\t1\n",
      "#\tClematis vitalba\t8\n",
      "#\tClinopodium acinos\t5\n",
      "#\tClinopodium ascendens\t1\n",
      "#\tClinopodium vulgare\t17\n",
      "#\tCochlearia anglica\t4\n",
      "#\tCochlearia danica\t4\n",
      "#\tCochlearia officinalis\t3\n",
      "#\tCoeloglossum viride\t6\n",
      "#\tCoincya monensis\t2\n",
      "#\tColchicum autumnale\t14\n",
      "#\tComarum palustre\t8\n",
      "#\tConium maculatum\t8\n",
      "#\tConopodium majus\t5\n",
      "#\tConringia orientalis\t0\n",
      "#\tConsolida ajacis\t3\n",
      "#\tConvallaria majalis\t15\n",
      "#\tConvolvulus arvensis\t20\n",
      "#\tConyza canadensis\t9\n",
      "#\tConyza floribunda\t0\n",
      "#\tCoriandrum sativum\t7\n",
      "#\tCornus alba\t1\n",
      "#\tCornus sanguinea\t11\n",
      "#\tCoronilla scorpioides\t0\n",
      "#\tCoronilla valentina\t0\n",
      "#\tCorrigiola litoralis\t1\n",
      "#\tCortaderia selloana\t3\n",
      "#\tCorylus avellana\t22\n",
      "#\tCotoneaster dielsianus\t3\n",
      "#\tCotoneaster horizontalis\t5\n",
      "#\tCotoneaster integrifolius\t0\n",
      "#\tCotoneaster lacteus\t2\n",
      "#\tCotoneaster rehderi\t2\n",
      "#\tCotoneaster simonsii\t1\n",
      "#\tCotoneaster splendens\t0\n",
      "#\tCrambe hispanica\t0\n",
      "#\tCrambe maritima\t3\n",
      "#\tCrassula helmsii\t4\n",
      "#\tCrataegus laevigata\t9\n",
      "#\tCrataegus monogyna\t36\n",
      "#\tCrepis biennis\t2\n",
      "#\tCrepis capillaris\t6\n",
      "#\tCrepis nicaeensis\t0\n",
      "#\tCrepis paludosa\t3\n",
      "#\tCrepis vesicaria\t2\n",
      "#\tCrocus vernus\t2\n",
      "#\tCruciata laevipes\t6\n",
      "#\tCupressus duclouxiana\t8\n",
      "#\tCupressus goveniana\t4\n",
      "#\tCupressus macrocarpa\t8\n",
      "#\tCupressus sempervirens\t10\n",
      "#\tCuscuta campestris\t5\n",
      "#\tCuscuta epithymum\t7\n",
      "#\tCuscuta europaea\t5\n",
      "#\tCuscuta suaveolens\t2\n",
      "#\tCymbalaria muralis\t2\n",
      "#\tCymbalaria pallida\t0\n",
      "#\tCynoglossum officinale\t7\n",
      "#\tCynosurus cristatus\t8\n",
      "#\tCynosurus echinatus\t2\n",
      "#\tCyperus eragrostis\t4\n",
      "#\tCyperus longus\t7\n",
      "#\tCyrtomium falcatum\t13\n",
      "#\tCyrtomium fortunei\t21\n",
      "#\tCystopteris fragilis\t19\n",
      "#\tCytisus scoparius\t8\n",
      "#\tDactylis glomerata\t32\n",
      "#\tDactylorhiza fuchsii\t4\n",
      "#\tDactylorhiza incarnata\t4\n",
      "#\tDactylorhiza maculata\t5\n",
      "#\tDactylorhiza praetermissa\t3\n",
      "#\tDactylorhiza purpurella\t4\n",
      "#\tDamasonium alisma\t1\n",
      "#\tDanthonia decumbens\t6\n",
      "#\tDaphne laureola\t4\n",
      "#\tDatura stramonium\t31\n",
      "#\tDaucus carota\t42\n",
      "#\tDeschampsia cespitosa\t53\n",
      "#\tDeschampsia flexuosa\t5\n",
      "#\tDeschampsia setacea\t2\n",
      "#\tDescurainia sophia\t5\n",
      "#\tDianthus barbatus\t1\n",
      "#\tDianthus deltoides\t8\n",
      "#\tDigitalis purpurea\t14\n",
      "#\tDigitaria sanguinalis\t9\n",
      "#\tDiplotaxis muralis\t1\n",
      "#\tDiplotaxis tenuifolia\t9\n",
      "#\tDipsacus fullonum\t13\n",
      "#\tDipsacus pilosus\t4\n",
      "#\tDipsacus sativus\t1\n",
      "#\tDoronicum pardalianches\t1\n",
      "#\tDrosera intermedia\t1\n",
      "#\tDrosera rotundifolia\t10\n",
      "#\tDryopteris affinis\t4\n",
      "#\tDryopteris borreri\t4\n",
      "#\tDryopteris cambrensis\t0\n",
      "#\tDryopteris carthusiana\t22\n",
      "#\tDryopteris cristata\t20\n",
      "#\tDryopteris dilatata\t11\n",
      "#\tDryopteris filix-mas\t15\n",
      "#\tDysphania pumilio\t0\n",
      "#\tEchinochloa colona\t9\n",
      "#\tEchinochloa crus-galli\t33\n",
      "#\tEchium plantagineum\t3\n",
      "#\tEchium vulgare\t5\n",
      "#\tEleocharis acicularis\t11\n",
      "#\tEleocharis multicaulis\t3\n",
      "#\tEleocharis palustris\t13\n",
      "#\tEleocharis quinqueflora\t5\n",
      "#\tEleocharis uniglumis\t4\n",
      "#\tEleogiton fluitans\t9\n",
      "#\tEleusine indica\t18\n",
      "#\tElodea canadensis\t7\n",
      "#\tElodea nuttallii\t8\n",
      "#\tElymus caninus\t9\n",
      "#\tElytrigia atherica\t2\n",
      "#\tElytrigia juncea\t3\n",
      "#\tElytrigia repens\t25\n",
      "#\tEnsete ventricosum\t4\n",
      "#\tEpilobium ciliatum\t4\n",
      "#\tEpilobium hirsutum\t8\n",
      "#\tEpilobium komarovianum\t0\n",
      "#\tEpilobium montanum\t7\n",
      "#\tEpilobium obscurum\t3\n",
      "#\tEpilobium palustre\t18\n",
      "#\tEpilobium parviflorum\t13\n",
      "#\tEpilobium roseum\t3\n",
      "#\tEpilobium tetragonum\t6\n",
      "#\tEpipactis dunensis\t1\n",
      "#\tEpipactis helleborine\t21\n",
      "#\tEpipactis leptochila\t3\n",
      "#\tEpipactis palustris\t5\n",
      "#\tEpipactis phyllanthes\t2\n",
      "#\tEquisetum arvense\t44\n",
      "#\tEquisetum fluviatile\t10\n",
      "#\tEquisetum hyemale\t23\n",
      "#\tEquisetum palustre\t7\n",
      "#\tEquisetum telmateia\t3\n",
      "#\tEragrostis cilianensis\t2\n",
      "#\tEragrostis pectinacea\t13\n",
      "#\tEranthis hyemalis\t2\n",
      "#\tErica cinerea\t5\n",
      "#\tErica tetralix\t5\n",
      "#\tErigeron acris\t4\n",
      "#\tErigeron karvinskianus\t2\n",
      "#\tEriophorum angustifolium\t23\n",
      "#\tEriophorum latifolium\t4\n",
      "#\tEriophorum vaginatum\t19\n",
      "#\tErodium cicutarium\t6\n",
      "#\tErodium moschatum\t5\n",
      "#\tErophila verna\t10\n",
      "#\tEryngium maritimum\t6\n",
      "#\tErysimum cheiranthoides\t11\n",
      "#\tErysimum cheiri\t3\n",
      "#\tEscallonia macrantha\t1\n",
      "#\tEschscholzia californica\t7\n",
      "#\tEuonymus europaeus\t8\n",
      "#\tEupatorium cannabinum\t3\n",
      "#\tEuphorbia cyparissias\t13\n",
      "#\tEuphorbia esula\t8\n",
      "#\tEuphorbia exigua\t6\n",
      "#\tEuphorbia helioscopia\t10\n",
      "#\tEuphorbia lathyris\t5\n",
      "#\tEuphorbia peplus\t13\n",
      "#\tEuphorbia platyphyllos\t3\n",
      "#\tEuphrasia micrantha\t3\n",
      "#\tEuphrasia nemorosa\t3\n",
      "#\tEuphrasia officinalis\t2\n",
      "#\tFagopyrum esculentum\t12\n",
      "#\tFagus sylvatica\t14\n",
      "#\tFalcaria vulgaris\t1\n",
      "#\tFallopia baldschuanica\t2\n",
      "#\tFallopia convolvulus\t13\n",
      "#\tFallopia japonica\t114\n",
      "#\tFallopia sachalinensis\t191\n",
      "#\tFestuca arenaria\t3\n",
      "#\tFestuca brevipila\t0\n",
      "#\tFestuca filiformis\t2\n",
      "#\tFestuca ovina\t10\n",
      "#\tFestuca rubra\t52\n",
      "#\tFicaria verna\t6\n",
      "#\tFicus carica\t12\n",
      "#\tFilago lutescens\t0\n",
      "#\tFilago minima\t2\n",
      "#\tFilago pyramidata\t1\n",
      "#\tFilago vulgaris\t3\n",
      "#\tFilipendula ulmaria\t9\n",
      "#\tFilipendula vulgaris\t8\n",
      "#\tFoeniculum vulgare\t17\n",
      "#\tFragaria ananassa\t5\n",
      "#\tFragaria moschata\t2\n",
      "#\tFragaria vesca\t69\n",
      "#\tFrangula alnus\t12\n",
      "#\tFraxinus excelsior\t21\n",
      "#\tFritillaria meleagris\t4\n",
      "#\tFumaria densiflora\t2\n",
      "#\tFumaria muralis\t5\n",
      "#\tFumaria officinalis\t4\n",
      "#\tFumaria parviflora\t0\n",
      "#\tFumaria purpurea\t2\n",
      "#\tFumaria vaillantii\t0\n",
      "#\tGagea bohemica\t3\n",
      "#\tGagea lutea\t7\n",
      "#\tGalanthus nivalis\t2\n",
      "#\tGalanthus plicatus\t1\n",
      "#\tGalanthus woronowii\t0\n",
      "#\tGalega officinalis\t2\n",
      "#\tGaleopsis angustifolia\t3\n",
      "#\tGaleopsis bifida\t5\n",
      "#\tGaleopsis speciosa\t3\n",
      "#\tGaleopsis tetrahit\t8\n",
      "#\tGalinsoga parviflora\t3\n",
      "#\tGalinsoga quadriradiata\t11\n",
      "#\tGalium aparine\t25\n",
      "#\tGalium odoratum\t5\n",
      "#\tGalium palustre\t14\n",
      "#\tGalium saxatile\t5\n",
      "#\tGalium uliginosum\t3\n",
      "#\tGalium verum\t14\n",
      "#\tGentiana pneumonanthe\t2\n",
      "#\tGentianella amarella\t7\n",
      "#\tGentianella campestris\t4\n",
      "#\tGeranium columbinum\t4\n",
      "#\tGeranium dissectum\t7\n",
      "#\tGeranium endressii\t3\n",
      "#\tGeranium lucidum\t4\n",
      "#\tGeranium molle\t9\n",
      "#\tGeranium nodosum\t3\n",
      "#\tGeranium phaeum\t6\n",
      "#\tGeranium pratense\t4\n",
      "#\tGeranium pusillum\t3\n",
      "#\tGeranium pyrenaicum\t2\n",
      "#\tGeranium robertianum\t13\n",
      "#\tGeranium rotundifolium\t6\n",
      "#\tGeranium sanguineum\t10\n",
      "#\tGeranium sylvaticum\t6\n",
      "#\tGeranium versicolor\t0\n",
      "#\tGeum rivale\t5\n",
      "#\tGeum urbanum\t8\n",
      "#\tGinkgo biloba\t30\n",
      "#\tGladiolus communis\t1\n",
      "#\tGlaucium flavum\t6\n",
      "#\tGlaux maritima\t5\n",
      "#\tGlebionis segetum\t6\n",
      "#\tGlechoma hederacea\t19\n",
      "#\tGlyceria declinata\t5\n",
      "#\tGlyceria fluitans\t7\n",
      "#\tGlyceria maxima\t3\n",
      "#\tGlyceria notata\t5\n",
      "#\tGlycine max\t159\n",
      "#\tGnaphalium luteoalbum\t3\n",
      "#\tGnaphalium sylvaticum\t1\n",
      "#\tGnaphalium uliginosum\t6\n",
      "#\tGoodyera repens\t9\n",
      "#\tGroenlandia densa\t6\n",
      "#\tGuizotia abyssinica\t3\n",
      "#\tGymnadenia conopsea\t9\n",
      "#\tGymnadenia densiflora\t0\n",
      "#\tGymnocarpium dryopteris\t17\n",
      "#\tGymnocarpium robertianum\t3\n",
      "#\tHedera helix\t26\n",
      "#\tHedera hibernica\t2\n",
      "#\tHelianthemum nummularium\t9\n",
      "#\tHelianthus annuus\t14\n",
      "#\tHelianthus decapetalus\t7\n",
      "#\tHelianthus tuberosus\t11\n",
      "#\tHelleborus foetidus\t3\n",
      "#\tHelleborus viridis\t4\n",
      "#\tHelminthotheca echioides\t6\n",
      "#\tHelosciadium nodiflorum\t6\n",
      "#\tHemizonia pungens\t0\n",
      "#\tHepatica nobilis\t8\n",
      "#\tHeracleum mantegazzianum\t0\n",
      "#\tHeracleum sphondylium\t6\n",
      "#\tHerniaria glabra\t2\n",
      "#\tHesperis matronalis\t7\n",
      "#\tHieracium acuminatum\t0\n",
      "#\tHieracium aggregatum\t0\n",
      "#\tHieracium calcaricola\t0\n",
      "#\tHieracium decolor\t0\n",
      "#\tHieracium diaphanoides\t0\n",
      "#\tHieracium diaphanum\t0\n",
      "#\tHieracium grandidens\t0\n",
      "#\tHieracium lepidulum\t1\n",
      "#\tHieracium sabaudum\t0\n",
      "#\tHieracium umbellatum\t6\n",
      "#\tHieracium vagum\t0\n",
      "#\tHieracium vulgatum\t0\n",
      "#\tHippocrepis comosa\t7\n",
      "#\tHippophae rhamnoides\t14\n",
      "#\tHippuris vulgaris\t21\n",
      "#\tHirschfeldia incana\t2\n",
      "#\tHolcus lanatus\t9\n",
      "#\tHolcus mollis\t3\n",
      "#\tHonckenya peploides\t21\n",
      "#\tHordeum distichon\t0\n",
      "#\tHordeum jubatum\t13\n",
      "#\tHordeum marinum\t22\n",
      "#\tHordeum murinum\t19\n",
      "#\tHordeum secalinum\t7\n",
      "#\tHordeum vulgare\t34\n",
      "#\tHottonia palustris\t3\n",
      "#\tHumulus lupulus\t20\n",
      "#\tHyacinthoides hispanica\t28\n",
      "#\tHyacinthoides non-scripta\t15\n",
      "#\tHydrangea anomala\t5\n",
      "#\tHydrangea arborescens\t9\n",
      "#\tHydrangea aspera\t2\n",
      "#\tHydrocharis morsus-ranae\t4\n",
      "#\tHydrocotyle ranunculoides\t1\n",
      "#\tHydrocotyle vulgaris\t5\n",
      "#\tHyoscyamus niger\t7\n",
      "#\tHypericum androsaemum\t5\n",
      "#\tHypericum calycinum\t1\n",
      "#\tHypericum elodes\t7\n",
      "#\tHypericum hirsutum\t3\n",
      "#\tHypericum humifusum\t4\n",
      "#\tHypericum montanum\t2\n",
      "#\tHypericum perforatum\t23\n",
      "#\tHypericum pulchrum\t4\n",
      "#\tHypericum tetrapterum\t3\n",
      "#\tHypochaeris glabra\t5\n",
      "#\tHypochaeris maculata\t1\n",
      "#\tHypochaeris radicata\t8\n",
      "#\tHypopitys monotropa\t1\n",
      "#\tIberis amara\t3\n",
      "#\tIlex aquifolium\t19\n",
      "#\tImpatiens capensis\t17\n",
      "#\tImpatiens glandulifera\t2\n",
      "#\tImpatiens noli-tangere\t5\n",
      "#\tImpatiens parviflora\t2\n",
      "#\tInula conyzae\t3\n",
      "#\tInula helenium\t11\n",
      "#\tIris foetidissima\t3\n",
      "#\tIris germanica\t6\n",
      "#\tIris pseudacorus\t12\n",
      "#\tIsoetes lacustris\t2\n",
      "#\tIsolepis setacea\t11\n",
      "#\tIva xanthiifolia\t0\n",
      "#\tJacobaea aquatica\t5\n",
      "#\tJacobaea vulgaris\t7\n",
      "#\tJasione montana\t6\n",
      "#\tJuglans nigra\t10\n",
      "#\tJuglans regia\t8\n",
      "#\tJuncus acutiflorus\t3\n",
      "#\tJuncus articulatus\t8\n",
      "#\tJuncus bufonius\t12\n",
      "#\tJuncus bulbosus\t7\n",
      "#\tJuncus compressus\t5\n",
      "#\tJuncus conglomeratus\t6\n",
      "#\tJuncus effusus\t20\n",
      "#\tJuncus gerardii\t6\n",
      "#\tJuncus inflexus\t5\n",
      "#\tJuncus maritimus\t7\n",
      "#\tJuncus ranarius\t1\n",
      "#\tJuncus squarrosus\t5\n",
      "#\tJuncus subnodulosus\t3\n",
      "#\tJuncus tenuis\t7\n",
      "#\tKickxia elatine\t4\n",
      "#\tKnautia arvensis\t8\n",
      "#\tKoeleria macrantha\t7\n",
      "#\tLaburnum anagyroides\t5\n",
      "#\tLactuca sativa\t12\n",
      "#\tLactuca serriola\t14\n",
      "#\tLactuca virosa\t3\n",
      "#\tLagarosiphon major\t14\n",
      "#\tLamiastrum galeobdolon\t7\n",
      "#\tLamium album\t9\n",
      "#\tLamium amplexicaule\t10\n",
      "#\tLamium confertum\t4\n",
      "#\tLamium hybridum\t3\n",
      "#\tLamium maculatum\t5\n",
      "#\tLamium purpureum\t14\n",
      "#\tLappula squarrosa\t2\n",
      "#\tLapsana communis\t17\n",
      "#\tLarix decidua\t9\n",
      "#\tLarix kaempferi\t11\n",
      "#\tLathraea clandestina\t12\n",
      "#\tLathraea squamaria\t5\n",
      "#\tLathyrus aphaca\t14\n",
      "#\tLathyrus grandiflorus\t0\n",
      "#\tLathyrus hirsutus\t2\n",
      "#\tLathyrus latifolius\t3\n",
      "#\tLathyrus linifolius\t3\n",
      "#\tLathyrus nissolia\t2\n",
      "#\tLathyrus palustris\t6\n",
      "#\tLathyrus pratensis\t12\n",
      "#\tLathyrus sativus\t4\n",
      "#\tLathyrus sylvestris\t10\n",
      "#\tLathyrus tuberosus\t11\n",
      "#\tLegousia hybrida\t3\n",
      "#\tLemna gibba\t15\n",
      "#\tLemna minor\t21\n",
      "#\tLemna minuta\t6\n",
      "#\tLemna trisulca\t11\n",
      "#\tLeontodon hispidus\t6\n",
      "#\tLeontodon saxatilis\t6\n",
      "#\tLeonurus cardiaca\t8\n",
      "#\tLepidium campestre\t12\n",
      "#\tLepidium coronopus\t4\n",
      "#\tLepidium didymum\t2\n",
      "#\tLepidium draba\t3\n",
      "#\tLepidium heterophyllum\t4\n",
      "#\tLepidium latifolium\t3\n",
      "#\tLepidium ruderale\t6\n",
      "#\tLepidium sativum\t1\n",
      "#\tLepidium virginicum\t4\n",
      "#\tLeucanthemum vulgare\t17\n",
      "#\tLeucojum aestivum\t2\n",
      "#\tLeycesteria formosa\t4\n",
      "#\tLeymus arenarius\t10\n",
      "#\tLigularia dentata\t1\n",
      "#\tLigustrum ovalifolium\t13\n",
      "#\tLigustrum vulgare\t15\n",
      "#\tLilium martagon\t5\n",
      "#\tLilium pyrenaicum\t2\n",
      "#\tLimnanthes douglasii\t1\n",
      "#\tLimonium vulgare\t5\n",
      "#\tLimosella aquatica\t6\n",
      "#\tLinaria purpurea\t1\n",
      "#\tLinaria repens\t3\n",
      "#\tLinaria vulgaris\t17\n",
      "#\tLinum bienne\t7\n",
      "#\tLinum catharticum\t5\n",
      "#\tLinum perenne\t8\n",
      "#\tLinum usitatissimum\t2\n",
      "#\tLithospermum arvense\t4\n",
      "#\tLithospermum officinale\t7\n",
      "#\tLittorella uniflora\t4\n",
      "#\tLobelia erinus\t2\n",
      "#\tLobularia maritima\t4\n",
      "#\tLolium multiflorum\t17\n",
      "#\tLolium perenne\t28\n",
      "#\tLolium rigidum\t9\n",
      "#\tLolium temulentum\t8\n",
      "#\tLonicera caprifolium\t0\n",
      "#\tLonicera involucrata\t0\n",
      "#\tLonicera periclymenum\t6\n",
      "#\tLonicera xylosteum\t1\n",
      "#\tLotus corniculatus\t21\n",
      "#\tLotus pedunculatus\t6\n",
      "#\tLotus tenuis\t3\n",
      "#\tLunaria annua\t1\n",
      "#\tLupinus arboreus\t1\n",
      "#\tLupinus polyphyllus\t2\n",
      "#\tLuzula campestris\t9\n",
      "#\tLuzula luzuloides\t0\n",
      "#\tLuzula multiflora\t9\n",
      "#\tLuzula pilosa\t6\n",
      "#\tLuzula sylvatica\t6\n",
      "#\tLycium barbarum\t6\n",
      "#\tLycium chinense\t10\n",
      "#\tLycopodiella inundata\t2\n",
      "#\tLycopodium clavatum\t13\n",
      "#\tLycopus europaeus\t3\n",
      "#\tLysimachia nemorum\t3\n",
      "#\tLysimachia nummularia\t6\n",
      "#\tLysimachia punctata\t1\n",
      "#\tLysimachia thyrsiflora\t0\n",
      "#\tLysimachia vulgaris\t5\n",
      "#\tLythrum hyssopifolia\t2\n",
      "#\tLythrum portula\t4\n",
      "#\tLythrum salicaria\t16\n",
      "#\tMahonia aquifolium\t2\n",
      "#\tMaianthemum bifolium\t15\n",
      "#\tMalus pumila\t16\n",
      "#\tMalus sylvestris\t5\n",
      "#\tMalva arborea\t3\n",
      "#\tMalva moschata\t5\n",
      "#\tMalva neglecta\t8\n",
      "#\tMalva parviflora\t6\n",
      "#\tMalva pseudolavatera\t1\n",
      "#\tMalva pusilla\t3\n",
      "#\tMalva setigera\t0\n",
      "#\tMalva sylvestris\t14\n",
      "#\tMarrubium vulgare\t8\n",
      "#\tMatricaria chamomilla\t34\n",
      "#\tMatricaria discoidea\t6\n",
      "#\tMatthiola sinuata\t3\n",
      "#\tMeconopsis cambrica\t6\n",
      "#\tMedicago arabica\t5\n",
      "#\tMedicago lupulina\t20\n",
      "#\tMedicago polymorpha\t7\n",
      "#\tMedicago sativa\t18\n",
      "#\tMelampyrum pratense\t7\n",
      "#\tMelica uniflora\t9\n",
      "#\tMelilotus albus\t15\n",
      "#\tMelilotus altissimus\t6\n",
      "#\tMelilotus indicus\t11\n",
      "#\tMelilotus officinalis\t15\n",
      "#\tMelissa officinalis\t10\n",
      "#\tMentha aquatica\t6\n",
      "#\tMentha arvensis\t7\n",
      "#\tMentha pulegium\t10\n",
      "#\tMentha spicata\t16\n",
      "#\tMenyanthes trifoliata\t17\n",
      "#\tMercurialis annua\t9\n",
      "#\tMercurialis perennis\t10\n",
      "#\tMetasequoia glyptostroboides\t13\n",
      "#\tMilium effusum\t8\n",
      "#\tMimulus guttatus\t89\n",
      "#\tMimulus moschatus\t3\n",
      "#\tMinuartia hybrida\t3\n",
      "#\tMisopates orontium\t4\n",
      "#\tMoehringia trinervia\t6\n",
      "#\tMolinia caerulea\t9\n",
      "#\tMontia fontana\t10\n",
      "#\tMusa acuminata\t160\n",
      "#\tMusa balbisiana\t29\n",
      "#\tMusa ornata\t9\n",
      "#\tMusa velutina\t6\n",
      "#\tMuscari armeniacum\t1\n",
      "#\tMyagrum perfoliatum\t0\n",
      "#\tMycelis muralis\t6\n",
      "#\tMyosotis arvensis\t5\n",
      "#\tMyosotis discolor\t5\n",
      "#\tMyosotis laxa\t8\n",
      "#\tMyosotis ramosissima\t4\n",
      "#\tMyosotis scorpioides\t4\n",
      "#\tMyosotis secunda\t5\n",
      "#\tMyosotis sylvatica\t6\n",
      "#\tMyosoton aquaticum\t4\n",
      "#\tMyosurus minimus\t3\n",
      "#\tMyrica gale\t12\n",
      "#\tMyriophyllum alterniflorum\t2\n",
      "#\tMyriophyllum aquaticum\t10\n",
      "#\tMyriophyllum spicatum\t11\n",
      "#\tMyriophyllum verticillatum\t8\n",
      "#\tMyrrhis odorata\t2\n",
      "#\tNarcissus poeticus\t0\n",
      "#\tNarcissus pseudonarcissus\t1\n",
      "#\tNardus stricta\t7\n",
      "#\tNarthecium ossifragum\t9\n",
      "#\tNasturtium microphyllum\t7\n",
      "#\tNasturtium officinale\t13\n",
      "#\tNeotinea ustulata\t1\n",
      "#\tNeottia cordata\t2\n",
      "#\tNeottia nidus-avis\t7\n",
      "#\tNeottia ovata\t7\n",
      "#\tNepeta cataria\t12\n",
      "#\tNeslia paniculata\t2\n",
      "#\tNicandra physalodes\t4\n",
      "#\tNicotiana tabacum\t127\n",
      "#\tNigella damascena\t3\n",
      "#\tNuphar lutea\t5\n",
      "#\tNymphaea alba\t12\n",
      "#\tNymphoides peltata\t2\n",
      "#\tOdontites vernus\t6\n",
      "#\tOenanthe aquatica\t3\n",
      "#\tOenanthe crocata\t3\n",
      "#\tOenanthe fistulosa\t2\n",
      "#\tOenanthe fluviatilis\t0\n",
      "#\tOenanthe lachenalii\t4\n",
      "#\tOenanthe pimpinelloides\t3\n",
      "#\tOenanthe silaifolia\t0\n",
      "#\tOenothera biennis\t17\n",
      "#\tOenothera glazioviana\t7\n",
      "#\tOmphalodes verna\t1\n",
      "#\tOnobrychis viciifolia\t3\n",
      "#\tOnoclea sensibilis\t18\n",
      "#\tOnonis repens\t6\n",
      "#\tOnonis spinosa\t7\n",
      "#\tOnopordum acanthium\t4\n",
      "#\tOphioglossum vulgatum\t8\n",
      "#\tOphrys apifera\t8\n",
      "#\tOphrys insectifera\t3\n",
      "#\tOrchis mascula\t4\n",
      "#\tOrchis simia\t1\n",
      "#\tOreopteris limbosperma\t3\n",
      "#\tOriganum vulgare\t20\n",
      "#\tOrnithogalum nutans\t1\n",
      "#\tOrnithopus perpusillus\t5\n",
      "#\tOrobanche elatior\t1\n",
      "#\tOrobanche hederae\t11\n",
      "#\tOrobanche minor\t46\n",
      "#\tOrobanche rapum-genistae\t8\n",
      "#\tOrobanche reticulata\t0\n",
      "#\tOsmunda regalis\t41\n",
      "#\tOxalis acetosella\t18\n",
      "#\tOxalis articulata\t3\n",
      "#\tOxalis corniculata\t13\n",
      "#\tOxalis exilis\t1\n",
      "#\tOxalis incarnata\t1\n",
      "#\tOxalis pes-caprae\t3\n",
      "#\tOxalis stricta\t13\n",
      "#\tPanicum capillare\t13\n",
      "#\tPanicum miliaceum\t8\n",
      "#\tPapaver argemone\t2\n",
      "#\tPapaver dubium\t5\n",
      "#\tPapaver hybridum\t3\n",
      "#\tPapaver lecoqii\t0\n",
      "#\tPapaver rhoeas\t15\n",
      "#\tPapaver somniferum\t10\n",
      "#\tParapholis incurva\t5\n",
      "#\tParapholis strigosa\t4\n",
      "#\tParentucellia viscosa\t7\n",
      "#\tParietaria judaica\t8\n",
      "#\tParis quadrifolia\t8\n",
      "#\tParnassia palustris\t29\n",
      "#\tPastinaca sativa\t7\n",
      "#\tPedicularis palustris\t3\n",
      "#\tPedicularis sylvatica\t5\n",
      "#\tPentaglottis sempervirens\t3\n",
      "#\tPersicaria amphibia\t14\n",
      "#\tPersicaria bistorta\t6\n",
      "#\tPersicaria capitata\t5\n",
      "#\tPersicaria hydropiper\t28\n",
      "#\tPersicaria lapathifolia\t17\n",
      "#\tPersicaria maculosa\t13\n",
      "#\tPersicaria minor\t4\n",
      "#\tPersicaria pensylvanica\t10\n",
      "#\tPersicaria wallichii\t6\n",
      "#\tPetasites albus\t2\n",
      "#\tPetasites fragrans\t1\n",
      "#\tPetasites hybridus\t5\n",
      "#\tPetroselinum crispum\t8\n",
      "#\tPetroselinum segetum\t2\n",
      "#\tPhacelia ciliata\t0\n",
      "#\tPhacelia parviflora\t0\n",
      "#\tPhacelia tanacetifolia\t1\n",
      "#\tPhalaris aquatica\t0\n",
      "#\tPhalaris arundinacea\t26\n",
      "#\tPhalaris canariensis\t1\n",
      "#\tPhalaris paradoxa\t1\n",
      "#\tPhegopteris connectilis\t12\n",
      "#\tPhiladelphus coronarius\t2\n",
      "#\tPhleum arenarium\t3\n",
      "#\tPhleum bertolonii\t4\n",
      "#\tPhleum pratense\t23\n",
      "#\tPhragmites australis\t45\n",
      "#\tPhysalis alkekengi\t3\n",
      "#\tPhytolacca acinosa\t0\n",
      "#\tPicea abies\t33\n",
      "#\tPicea sitchensis\t10\n",
      "#\tPicris hieracioides\t11\n",
      "#\tPilosella aurantiaca\t8\n",
      "#\tPilosella officinarum\t10\n",
      "#\tPilularia globulifera\t5\n",
      "#\tPimpinella major\t1\n",
      "#\tPimpinella saxifraga\t8\n",
      "#\tPinguicula vulgaris\t21\n",
      "#\tPinus contorta\t9\n",
      "#\tPinus nigra\t15\n",
      "#\tPinus sylvestris\t27\n",
      "#\tPisum sativum\t31\n",
      "#\tPlantago arenaria\t0\n",
      "#\tPlantago coronopus\t12\n",
      "#\tPlantago lanceolata\t18\n",
      "#\tPlantago major\t24\n",
      "#\tPlantago maritima\t9\n",
      "#\tPlantago media\t10\n",
      "#\tPlatanthera bifolia\t2\n",
      "#\tPlatanthera chlorantha\t8\n",
      "#\tPoa angustifolia\t4\n",
      "#\tPoa annua\t27\n",
      "#\tPoa compressa\t6\n",
      "#\tPoa humilis\t5\n",
      "#\tPoa nemoralis\t6\n",
      "#\tPoa pratensis\t89\n",
      "#\tPoa trivialis\t11\n",
      "#\tPolemonium caeruleum\t3\n",
      "#\tPolygala calcarea\t1\n",
      "#\tPolygala serpyllifolia\t7\n",
      "#\tPolygala vulgaris\t7\n",
      "#\tPolygonatum multiflorum\t6\n",
      "#\tPolygonum arenarium\t0\n",
      "#\tPolygonum arenastrum\t4\n",
      "#\tPolygonum aviculare\t21\n",
      "#\tPolygonum rurivagum\t4\n",
      "#\tPolypodium interjectum\t3\n",
      "#\tPolypodium vulgare\t6\n",
      "#\tPolypogon monspeliensis\t4\n",
      "#\tPolystichum aculeatum\t2\n",
      "#\tPolystichum setiferum\t3\n",
      "#\tPontederia cordata\t4\n",
      "#\tPopulus alba\t22\n",
      "#\tPopulus nigra\t164\n",
      "#\tPopulus tremula\t17\n",
      "#\tPopulus trichocarpa\t85\n",
      "#\tPortulaca oleracea\t17\n",
      "#\tPotamogeton acutifolius\t0\n",
      "#\tPotamogeton alpinus\t9\n",
      "#\tPotamogeton berchtoldii\t4\n",
      "#\tPotamogeton coloratus\t3\n",
      "#\tPotamogeton compressus\t6\n",
      "#\tPotamogeton crispus\t18\n",
      "#\tPotamogeton friesii\t2\n",
      "#\tPotamogeton gramineus\t11\n",
      "#\tPotamogeton lucens\t8\n",
      "#\tPotamogeton natans\t16\n",
      "#\tPotamogeton obtusifolius\t6\n",
      "#\tPotamogeton pectinatus\t13\n",
      "#\tPotamogeton perfoliatus\t22\n",
      "#\tPotamogeton polygonifolius\t4\n",
      "#\tPotamogeton pusillus\t17\n",
      "#\tPotamogeton trichoides\t3\n",
      "#\tPotentilla anglica\t4\n",
      "#\tPotentilla anserina\t8\n",
      "#\tPotentilla argentea\t4\n",
      "#\tPotentilla erecta\t5\n",
      "#\tPotentilla intermedia\t0\n",
      "#\tPotentilla norvegica\t4\n",
      "#\tPotentilla recta\t4\n",
      "#\tPotentilla reptans\t9\n",
      "#\tPotentilla sterilis\t5\n",
      "#\tPotentilla tabernaemontani\t0\n",
      "#\tPoterium sanguisorba\t9\n",
      "#\tPrimula auricula\t0\n",
      "#\tPrimula pulverulenta\t3\n",
      "#\tPrimula veris\t10\n",
      "#\tPrimula vulgaris\t7\n",
      "#\tPrunella vulgaris\t33\n",
      "#\tPrunus avium\t11\n",
      "#\tPrunus cerasifera\t13\n",
      "#\tPrunus cerasus\t11\n",
      "#\tPrunus domestica\t17\n",
      "#\tPrunus dulcis\t8\n",
      "#\tPrunus laurocerasus\t4\n",
      "#\tPrunus padus\t15\n",
      "#\tPrunus spinosa\t22\n",
      "#\tPseudofumaria lutea\t4\n",
      "#\tPseudotsuga menziesii\t9\n",
      "#\tPteridium aquilinum\t20\n",
      "#\tPteris cretica\t21\n",
      "#\tPteris incompleta\t1\n",
      "#\tPteris tremula\t5\n",
      "#\tPteris vittata\t47\n",
      "#\tPuccinellia distans\t11\n",
      "#\tPuccinellia maritima\t4\n",
      "#\tPuccinellia rupestris\t2\n",
      "#\tPulicaria dysenterica\t4\n",
      "#\tPulmonaria officinalis\t2\n",
      "#\tPyracantha coccinea\t10\n",
      "#\tPyrola minor\t4\n",
      "#\tPyrus communis\t17\n",
      "#\tPyrus pyraster\t2\n",
      "#\tQuercus cerris\t4\n",
      "#\tQuercus ilex\t63\n",
      "#\tQuercus petraea\t11\n",
      "#\tQuercus robur\t19\n",
      "#\tQuercus rubra\t19\n",
      "#\tRadiola linoides\t6\n",
      "#\tRanunculus acris\t18\n",
      "#\tRanunculus aquatilis\t9\n",
      "#\tRanunculus arvensis\t1\n",
      "#\tRanunculus auricomus\t4\n",
      "#\tRanunculus baudotii\t2\n",
      "#\tRanunculus bulbosus\t10\n",
      "#\tRanunculus circinatus\t3\n",
      "#\tRanunculus flammula\t7\n",
      "#\tRanunculus fluitans\t1\n",
      "#\tRanunculus hederaceus\t1\n",
      "#\tRanunculus lingua\t5\n",
      "#\tRanunculus parviflorus\t3\n",
      "#\tRanunculus peltatus\t3\n",
      "#\tRanunculus penicillatus\t2\n",
      "#\tRanunculus repens\t10\n",
      "#\tRanunculus sardous\t3\n",
      "#\tRanunculus sceleratus\t8\n",
      "#\tRanunculus trichophyllus\t5\n",
      "#\tRanunculus water-crowfoot\t0\n",
      "#\tRaphanus raphanistrum\t12\n",
      "#\tRaphanus sativus\t35\n",
      "#\tRapistrum rugosum\t4\n",
      "#\tReseda alba\t3\n",
      "#\tReseda lutea\t8\n",
      "#\tReseda luteola\t5\n",
      "#\tRhamnus cathartica\t23\n",
      "#\tRhinanthus angustifolius\t0\n",
      "#\tRhinanthus minor\t8\n",
      "#\tRhododendron ponticum\t4\n",
      "#\tRhus typhina\t8\n",
      "#\tRibes alpinum\t5\n",
      "#\tRibes nigrum\t1\n",
      "#\tRibes rubrum\t8\n",
      "#\tRibes sanguineum\t0\n",
      "#\tRibes uva-crispa\t1\n",
      "#\tRicinus communis\t111\n",
      "#\tRobinia pseudoacacia\t14\n",
      "#\tRorippa amphibia\t3\n",
      "#\tRorippa palustris\t5\n",
      "#\tRorippa sylvestris\t5\n",
      "#\tRosa arvensis\t5\n",
      "#\tRosa caesia\t5\n",
      "#\tRosa canina\t10\n",
      "#\tRosa micrantha\t4\n",
      "#\tRosa mollis\t2\n",
      "#\tRosa multiflora\t12\n",
      "#\tRosa obtusifolia\t3\n",
      "#\tRosa rubiginosa\t4\n",
      "#\tRosa rugosa\t6\n",
      "#\tRosa sherardii\t4\n",
      "#\tRosa spinosissima\t3\n",
      "#\tRosmarinus officinalis\t12\n",
      "#\tRubus armeniacus\t0\n",
      "#\tRubus caesius\t5\n",
      "#\tRubus dasyphyllus\t0\n",
      "#\tRubus eboracensis\t0\n",
      "#\tRubus echinatoides\t0\n",
      "#\tRubus fruticosus\t4\n",
      "#\tRubus halsteadensis\t0\n",
      "#\tRubus idaeus\t20\n",
      "#\tRubus nemoralis\t0\n",
      "#\tRubus polyanthemus\t0\n",
      "#\tRubus saxatilis\t1\n",
      "#\tRubus spectabilis\t1\n",
      "#\tRubus sprengelii\t0\n",
      "#\tRubus ulmifolius\t4\n",
      "#\tRubus warrenii\t0\n",
      "#\tRudbeckia laciniata\t5\n",
      "#\tRumex acetosa\t20\n",
      "#\tRumex acetosella\t12\n",
      "#\tRumex conglomeratus\t6\n",
      "#\tRumex crispus\t22\n",
      "#\tRumex dentatus\t0\n",
      "#\tRumex hydrolapathum\t5\n",
      "#\tRumex longifolius\t1\n",
      "#\tRumex maritimus\t3\n",
      "#\tRumex obtusifolius\t18\n",
      "#\tRumex palustris\t3\n",
      "#\tRumex patientia\t9\n",
      "#\tRumex salicifolius\t1\n",
      "#\tRumex sanguineus\t4\n",
      "#\tRuppia cirrhosa\t8\n",
      "#\tRuppia maritima\t48\n",
      "#\tRuscus aculeatus\t12\n",
      "#\tSagina apetala\t5\n",
      "#\tSagina filicaulis\t0\n",
      "#\tSagina maritima\t3\n",
      "#\tSagina nodosa\t3\n",
      "#\tSagina procumbens\t4\n",
      "#\tSagittaria sagittifolia\t2\n",
      "#\tSalicornia dolichostachya\t4\n",
      "#\tSalicornia europaea\t8\n",
      "#\tSalicornia fragilis\t1\n",
      "#\tSalicornia ramosissima\t3\n",
      "#\tSalix alba\t19\n",
      "#\tSalix aurita\t6\n",
      "#\tSalix caprea\t13\n",
      "#\tSalix cinerea\t10\n",
      "#\tSalix daphnoides\t3\n",
      "#\tSalix pentandra\t10\n",
      "#\tSalix purpurea\t12\n",
      "#\tSalix repens\t6\n",
      "#\tSalix triandra\t11\n",
      "#\tSalix viminalis\t8\n",
      "#\tSalsola kali\t8\n",
      "#\tSalvia pratensis\t8\n",
      "#\tSalvia reflexa\t0\n",
      "#\tSalvia verbenaca\t5\n",
      "#\tSalvia verticillata\t1\n",
      "#\tSambucus nigra\t35\n",
      "#\tSamolus valerandi\t6\n",
      "#\tSanguisorba officinalis\t15\n",
      "#\tSanicula europaea\t5\n",
      "#\tSaponaria officinalis\t14\n",
      "#\tSaxifraga granulata\t2\n",
      "#\tSaxifraga hirsuta\t2\n",
      "#\tSaxifraga hypnoides\t3\n",
      "#\tSaxifraga tridactylites\t3\n",
      "#\tScabiosa atropurpurea\t1\n",
      "#\tScabiosa columbaria\t7\n",
      "#\tScandix pecten-veneris\t3\n",
      "#\tSchedonorus arundinaceus\t30\n",
      "#\tSchedonorus giganteus\t6\n",
      "#\tSchedonorus pratensis\t16\n",
      "#\tSchoenoplectus lacustris\t6\n",
      "#\tSchoenoplectus tabernaemontani\t12\n",
      "#\tSchoenus nigricans\t7\n",
      "#\tScilla bifolia\t1\n",
      "#\tScilla forbesii\t1\n",
      "#\tScirpus sylvaticus\t8\n",
      "#\tScleranthus annuus\t4\n",
      "#\tScolymus hispanicus\t1\n",
      "#\tScorzoneroides autumnalis\t6\n",
      "#\tScrophularia auriculata\t5\n",
      "#\tScrophularia nodosa\t2\n",
      "#\tScutellaria altissima\t1\n",
      "#\tScutellaria galericulata\t6\n",
      "#\tScutellaria minor\t4\n",
      "#\tSecurigera varia\t5\n",
      "#\tSedum acre\t9\n",
      "#\tSedum album\t5\n",
      "#\tSedum anglicum\t3\n",
      "#\tSedum forsterianum\t4\n",
      "#\tSedum rupestre\t5\n",
      "#\tSedum telephium\t4\n",
      "#\tSelaginella selaginoides\t3\n",
      "#\tSempervivum tectorum\t2\n",
      "#\tSenecio erucifolius\t5\n",
      "#\tSenecio inaequidens\t3\n",
      "#\tSenecio squalidus\t0\n",
      "#\tSenecio sylvaticus\t3\n",
      "#\tSenecio viscosus\t0\n",
      "#\tSenecio vulgaris\t11\n",
      "#\tSerratula tinctoria\t3\n",
      "#\tSesleria caerulea\t3\n",
      "#\tSetaria italica\t83\n",
      "#\tSetaria pumila\t12\n",
      "#\tSetaria verticillata\t9\n",
      "#\tSetaria viridis\t16\n",
      "#\tSherardia arvensis\t9\n",
      "#\tSidalcea candida\t0\n",
      "#\tSidalcea malviflora\t0\n",
      "#\tSilaum silaus\t3\n",
      "#\tSilene dioica\t22\n",
      "#\tSilene flos-cuculi\t7\n",
      "#\tSilene gallica\t5\n",
      "#\tSilene latifolia\t73\n",
      "#\tSilene noctiflora\t5\n",
      "#\tSilene vulgaris\t17\n",
      "#\tSilybum marianum\t10\n",
      "#\tSinapis alba\t6\n",
      "#\tSinapis arvensis\t10\n",
      "#\tSison amomum\t5\n",
      "#\tSisymbrium altissimum\t2\n",
      "#\tSisymbrium irio\t1\n",
      "#\tSisymbrium loeselii\t1\n",
      "#\tSisymbrium officinale\t12\n",
      "#\tSisymbrium orientale\t3\n",
      "#\tSisymbrium strictissimum\t0\n",
      "#\tSisymbrium volgense\t0\n",
      "#\tSisyrinchium montanum\t10\n",
      "#\tSium latifolium\t3\n",
      "#\tSmyrnium olusatrum\t2\n",
      "#\tSolanum chenopodioides\t3\n",
      "#\tSolanum dulcamara\t16\n",
      "#\tSolanum lycopersicum\t138\n",
      "#\tSolanum nigrum\t31\n",
      "#\tSolanum physalifolium\t2\n",
      "#\tSolanum rostratum\t1\n",
      "#\tSolanum sarachoides\t0\n",
      "#\tSolanum sisymbriifolium\t1\n",
      "#\tSolanum tuberosum\t89\n",
      "#\tSoleirolia soleirolii\t5\n",
      "#\tSolidago canadensis\t18\n",
      "#\tSolidago gigantea\t7\n",
      "#\tSolidago graminifolia\t7\n",
      "#\tSolidago virgaurea\t6\n",
      "#\tSonchus arvensis\t11\n",
      "#\tSonchus asper\t13\n",
      "#\tSonchus oleraceus\t15\n",
      "#\tSonchus palustris\t1\n",
      "#\tSorbus aria\t8\n",
      "#\tSorbus aucuparia\t16\n",
      "#\tSorbus intermedia\t7\n",
      "#\tSorbus latifolia\t0\n",
      "#\tSorbus torminalis\t7\n",
      "#\tSparganium emersum\t12\n",
      "#\tSparganium erectum\t6\n",
      "#\tSparganium natans\t7\n",
      "#\tSpartina anglica\t6\n",
      "#\tSpergula arvensis\t5\n",
      "#\tSpergularia marina\t5\n",
      "#\tSpergularia media\t3\n",
      "#\tSpergularia rubra\t4\n",
      "#\tSpilanthes decumbens\t0\n",
      "#\tSpiraea alba\t2\n",
      "#\tSpiraea douglasii\t0\n",
      "#\tSpiraea salicifolia\t5\n",
      "#\tSpiranthes spiralis\t3\n",
      "#\tSpirodela polyrhiza\t21\n",
      "#\tStachys annua\t0\n",
      "#\tStachys arvensis\t5\n",
      "#\tStachys byzantina\t4\n",
      "#\tStachys palustris\t5\n",
      "#\tStachys sylvatica\t11\n",
      "#\tStellaria alsine\t3\n",
      "#\tStellaria graminea\t6\n",
      "#\tStellaria holostea\t5\n",
      "#\tStellaria media\t16\n",
      "#\tStellaria neglecta\t2\n",
      "#\tStellaria nemorum\t3\n",
      "#\tStellaria pallida\t3\n",
      "#\tStellaria palustris\t4\n",
      "#\tStratiotes aloides\t5\n",
      "#\tSuaeda maritima\t13\n",
      "#\tSuccisa pratensis\t4\n",
      "#\tSymphoricarpos albus\t1\n",
      "#\tSymphoricarpos orbiculatus\t2\n",
      "#\tSymphytum bulbosum\t0\n",
      "#\tSymphytum officinale\t10\n",
      "#\tSymphytum orientale\t1\n",
      "#\tSyringa vulgaris\t9\n",
      "#\tTagetes minuta\t4\n",
      "#\tTamarix gallica\t2\n",
      "#\tTamus communis\t15\n",
      "#\tTanacetum parthenium\t4\n",
      "#\tTanacetum vulgare\t7\n",
      "#\tTaraxacum aequilobum\t0\n",
      "#\tTaraxacum alatum\t0\n",
      "#\tTaraxacum ancistrolobum\t0\n",
      "#\tTaraxacum arenastrum\t0\n",
      "#\tTaraxacum argutum\t0\n",
      "#\tTaraxacum aurosulum\t0\n",
      "#\tTaraxacum boekmanii\t0\n",
      "#\tTaraxacum brachyglossum\t0\n",
      "#\tTaraxacum bracteatum\t0\n",
      "#\tTaraxacum cordatum\t0\n",
      "#\tTaraxacum croceiflorum\t0\n",
      "#\tTaraxacum dilatatum\t0\n",
      "#\tTaraxacum duplidentifrons\t0\n",
      "#\tTaraxacum ekmanii\t1\n",
      "#\tTaraxacum euryphyllum\t0\n",
      "#\tTaraxacum exacutum\t0\n",
      "#\tTaraxacum expallidiforme\t0\n",
      "#\tTaraxacum faeroense\t0\n",
      "#\tTaraxacum fasciatum\t0\n",
      "#\tTaraxacum fulgidum\t0\n",
      "#\tTaraxacum fulviforme\t0\n",
      "#\tTaraxacum haematicum\t0\n",
      "#\tTaraxacum hamatiforme\t0\n",
      "#\tTaraxacum hamatum\t0\n",
      "#\tTaraxacum huelphersianum\t0\n",
      "#\tTaraxacum insigne\t0\n",
      "#\tTaraxacum interveniens\t0\n",
      "#\tTaraxacum kernianum\t0\n",
      "#\tTaraxacum laciniosifrons\t0\n",
      "#\tTaraxacum lacistophyllum\t0\n",
      "#\tTaraxacum lamprophyllum\t0\n",
      "#\tTaraxacum latisectum\t0\n",
      "#\tTaraxacum lingulatum\t0\n",
      "#\tTaraxacum maculosum\t0\n",
      "#\tTaraxacum marklundii\t0\n",
      "#\tTaraxacum nordstedtii\t0\n",
      "#\tTaraxacum ochrochlorum\t0\n",
      "#\tTaraxacum ostenfeldii\t0\n",
      "#\tTaraxacum oxoniense\t0\n",
      "#\tTaraxacum pachymerum\t0\n",
      "#\tTaraxacum pallescens\t0\n",
      "#\tTaraxacum palustre\t0\n",
      "#\tTaraxacum pannucium\t0\n",
      "#\tTaraxacum polyodon\t0\n",
      "#\tTaraxacum proximum\t0\n",
      "#\tTaraxacum pseudohamatum\t0\n",
      "#\tTaraxacum pseudolarssonii\t0\n",
      "#\tTaraxacum quadrans\t0\n",
      "#\tTaraxacum richardsianum\t0\n",
      "#\tTaraxacum rubicundum\t0\n",
      "#\tTaraxacum sagittipotens\t0\n",
      "#\tTaraxacum sellandii\t0\n",
      "#\tTaraxacum sinuatum\t0\n",
      "#\tTaraxacum stenacrum\t0\n",
      "#\tTaraxacum stictophyllum\t0\n",
      "#\tTaraxacum subcyanolepis\t0\n",
      "#\tTaraxacum subhamatum\t0\n",
      "#\tTaraxacum sublaeticolor\t0\n",
      "#\tTaraxacum subundulatum\t0\n",
      "#\tTaraxacum trilobatum\t0\n",
      "#\tTaraxacum undulatiflorum\t0\n",
      "#\tTaraxacum xanthostigma\t0\n",
      "#\tTaxus baccata\t24\n",
      "#\tTeesdalia nudicaulis\t1\n",
      "#\tTellima grandiflora\t2\n",
      "#\tTeucrium scorodonia\t7\n",
      "#\tThalictrum flavum\t3\n",
      "#\tThelypteris palustris\t14\n",
      "#\tThlaspi arvense\t13\n",
      "#\tThuja plicata\t4\n",
      "#\tThymus polytrichus\t8\n",
      "#\tThymus pulegioides\t10\n",
      "#\tThyselium palustre\t0\n",
      "#\tTilia cordata\t6\n",
      "#\tTilia platyphyllos\t6\n",
      "#\tTolmiea menziesii\t1\n",
      "#\tTorilis arvensis\t7\n",
      "#\tTorilis japonica\t8\n",
      "#\tTorilis nodosa\t4\n",
      "#\tTragopogon porrifolius\t2\n",
      "#\tTragopogon pratensis\t11\n",
      "#\tTrichophorum germanicum\t0\n",
      "#\tTrifolium arvense\t8\n",
      "#\tTrifolium campestre\t11\n",
      "#\tTrifolium dubium\t9\n",
      "#\tTrifolium fragiferum\t4\n",
      "#\tTrifolium hybridum\t7\n",
      "#\tTrifolium medium\t4\n",
      "#\tTrifolium micranthum\t3\n",
      "#\tTrifolium pratense\t30\n",
      "#\tTrifolium repens\t24\n",
      "#\tTriglochin maritima\t21\n",
      "#\tTriglochin palustris\t14\n",
      "#\tTripleurospermum inodorum\t4\n",
      "#\tTripleurospermum maritimum\t26\n",
      "#\tTrisetum flavescens\t7\n",
      "#\tTriticum aestivum\t70\n",
      "#\tTriticum spelta\t9\n",
      "#\tTropaeolum majus\t6\n",
      "#\tTsuga heterophylla\t1\n",
      "#\tTulipa gesneriana\t1\n",
      "#\tTurgenia latifolia\t1\n",
      "#\tTussilago farfara\t15\n",
      "#\tTypha angustifolia\t12\n",
      "#\tTypha latifolia\t14\n",
      "#\tUlex europaeus\t5\n",
      "#\tUlmus glabra\t12\n",
      "#\tUlmus minor\t8\n",
      "#\tUlmus plotii\t0\n",
      "#\tUlmus procera\t6\n",
      "#\tUrtica dioica\t16\n",
      "#\tUrtica urens\t8\n",
      "#\tUtricularia minor\t5\n",
      "#\tUtricularia vulgaris\t2\n",
      "#\tVaccaria hispanica\t3\n",
      "#\tVaccinium myrtillus\t11\n",
      "#\tVaccinium oxycoccos\t4\n",
      "#\tVaccinium vitis-idaea\t24\n",
      "#\tValeriana dioica\t7\n",
      "#\tValeriana officinalis\t16\n",
      "#\tValeriana pyrenaica\t3\n",
      "#\tValerianella carinata\t3\n",
      "#\tValerianella dentata\t4\n",
      "#\tValerianella eriocarpa\t3\n",
      "#\tValerianella locusta\t7\n",
      "#\tValerianella rimosa\t1\n",
      "#\tVerbascum blattaria\t0\n",
      "#\tVerbascum nigrum\t5\n",
      "#\tVerbascum phlomoides\t1\n",
      "#\tVerbascum thapsus\t14\n",
      "#\tVerbascum virgatum\t3\n",
      "#\tVerbena bonariensis\t7\n",
      "#\tVerbena hastata\t7\n",
      "#\tVerbena officinalis\t13\n",
      "#\tVeronica agrestis\t5\n",
      "#\tVeronica anagallis-aquatica\t6\n",
      "#\tVeronica arvensis\t9\n",
      "#\tVeronica beccabunga\t4\n",
      "#\tVeronica catenata\t4\n",
      "#\tVeronica chamaedrys\t8\n",
      "#\tVeronica filiformis\t1\n",
      "#\tVeronica hederifolia\t3\n",
      "#\tVeronica longifolia\t0\n",
      "#\tVeronica montana\t4\n",
      "#\tVeronica officinalis\t13\n",
      "#\tVeronica persica\t9\n",
      "#\tVeronica polita\t2\n",
      "#\tVeronica scutellata\t4\n",
      "#\tVeronica serpyllifolia\t14\n",
      "#\tVeronica spicata\t5\n",
      "#\tViburnum lantana\t9\n",
      "#\tViburnum opulus\t16\n",
      "#\tVicia cracca\t27\n",
      "#\tVicia faba\t18\n",
      "#\tVicia hirsuta\t8\n",
      "#\tVicia lathyroides\t2\n",
      "#\tVicia lutea\t4\n",
      "#\tVicia sativa\t35\n",
      "#\tVicia sepium\t6\n",
      "#\tVicia sylvatica\t5\n",
      "#\tVicia tetrasperma\t17\n",
      "#\tVinca major\t1\n",
      "#\tVinca minor\t12\n",
      "#\tViola arvensis\t9\n",
      "#\tViola canina\t5\n",
      "#\tViola hirta\t4\n",
      "#\tViola odorata\t6\n",
      "#\tViola palustris\t5\n",
      "#\tViola reichenbachiana\t5\n",
      "#\tViola riviniana\t10\n",
      "#\tViola tricolor\t4\n",
      "#\tViscum album\t8\n",
      "#\tVitis vinifera\t129\n",
      "#\tVulpia bromoides\t7\n",
      "#\tVulpia myuros\t14\n",
      "#\tXanthium spinosum\t1\n",
      "#\tXanthium strumarium\t3\n",
      "#\tZannichellia palustris\t13\n",
      "#\tZantedeschia aethiopica\t13\n",
      "#\tZea mays\t113\n",
      "#\tZostera marina\t33\n",
      "#\tZostera noltei\t5\n",
      "\n",
      "total number of accessions fetched: 13583\n",
      "\n",
      "\n",
      "downloading 13583 records .. processing 1000 accessions per batch\n",
      "\n",
      "[Wed Oct 19 2016 20:58:24]\tbatch: 0000001\t1000 unique records downloaded\t-> eyorks_flora_curated.0000001.gb\t(total: 1000 (7.36 %); remaining: 12583 (92.64 %) -> eyorks_flora_curated.accessions.txt)\n",
      "[Wed Oct 19 2016 20:59:16]\tbatch: 0000002\t1000 unique records downloaded\t-> eyorks_flora_curated.0000002.gb\t(total: 2000 (14.72 %); remaining: 11583 (85.28 %) -> eyorks_flora_curated.accessions.txt)\n",
      "[Wed Oct 19 2016 21:00:58]\tbatch: 0000003\t1000 unique records downloaded\t-> eyorks_flora_curated.0000003.gb\t(total: 3000 (22.09 %); remaining: 10583 (77.91 %) -> eyorks_flora_curated.accessions.txt)\n",
      "[Wed Oct 19 2016 21:02:08]\tbatch: 0000004\t1000 unique records downloaded\t-> eyorks_flora_curated.0000004.gb\t(total: 4000 (29.45 %); remaining: 9583 (70.55 %) -> eyorks_flora_curated.accessions.txt)\n",
      "[Wed Oct 19 2016 21:02:51]\tbatch: 0000005\t1000 unique records downloaded\t-> eyorks_flora_curated.0000005.gb\t(total: 5000 (36.81 %); remaining: 8583 (63.19 %) -> eyorks_flora_curated.accessions.txt)\n",
      "[Wed Oct 19 2016 21:03:56]\tbatch: 0000006\t1000 unique records downloaded\t-> eyorks_flora_curated.0000006.gb\t(total: 6000 (44.17 %); remaining: 7583 (55.83 %) -> eyorks_flora_curated.accessions.txt)\n",
      "[Wed Oct 19 2016 21:04:52]\tbatch: 0000007\t1000 unique records downloaded\t-> eyorks_flora_curated.0000007.gb\t(total: 7000 (51.54 %); remaining: 6583 (48.46 %) -> eyorks_flora_curated.accessions.txt)\n",
      "[Wed Oct 19 2016 21:05:58]\tbatch: 0000008\t1000 unique records downloaded\t-> eyorks_flora_curated.0000008.gb\t(total: 8000 (58.90 %); remaining: 5583 (41.10 %) -> eyorks_flora_curated.accessions.txt)\n",
      "[Wed Oct 19 2016 21:07:18]\tbatch: 0000009\t1000 unique records downloaded\t-> eyorks_flora_curated.0000009.gb\t(total: 9000 (66.26 %); remaining: 4583 (33.74 %) -> eyorks_flora_curated.accessions.txt)\n",
      "[Wed Oct 19 2016 21:08:51]\tbatch: 0000010\t1000 unique records downloaded\t-> eyorks_flora_curated.0000010.gb\t(total: 10000 (73.62 %); remaining: 3583 (26.38 %) -> eyorks_flora_curated.accessions.txt)\n",
      "[Wed Oct 19 2016 21:10:24]\tbatch: 0000011\t1000 unique records downloaded\t-> eyorks_flora_curated.0000011.gb\t(total: 11000 (80.98 %); remaining: 2583 (19.02 %) -> eyorks_flora_curated.accessions.txt)\n",
      "[Wed Oct 19 2016 21:11:15]\tbatch: 0000012\t1000 unique records downloaded\t-> eyorks_flora_curated.0000012.gb\t(total: 12000 (88.35 %); remaining: 1583 (11.65 %) -> eyorks_flora_curated.accessions.txt)\n",
      "[Wed Oct 19 2016 21:12:14]\tbatch: 0000013\t1000 unique records downloaded\t-> eyorks_flora_curated.0000013.gb\t(total: 13000 (95.71 %); remaining: 583 (4.29 %) -> eyorks_flora_curated.accessions.txt)\n",
      "[Wed Oct 19 2016 21:39:22]\tbatch: 0000014\t583 unique records downloaded\t-> eyorks_flora_curated.0000014.gb\t(total: 13583 (100.00 %); remaining: 0 -> DONE!)\n",
      "\n",
      "Concatenate all files\t-> eyorks_flora_curated.gb\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!fetch_from_db.py -t FinalRefList.txt -m rbcl -o eyorks_flora_curated -@ callumjmacgregor@gmail.com"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now we want an additional reference file containing the positive controls (unfortunately this may be less successful as several are unexpectedly missing - so I've included a number of congenerics for those that I know are missing, in the hope of a hit)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": false,
    "scrolled": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "metaBEAT may be querying NCBI's Entrez databases to fetch/verify taxonomic ids. Entrez User requirements state that you need to identify yourself by providing an email address so that NCBI can contact you in case there is a problem.\n",
      "\n",
      "You have specified: 'callumjmacgregor@gmail.com'\n",
      "\n",
      "\n",
      "QUERYING GENBANK\n",
      "\n",
      "check for synonyms for \"rbcl\" (this is relevant only for Genbank searches)\n",
      "\n",
      "fetching accessions ..\n",
      "\n",
      "#\tBryophyllum pinnatum\t0\n",
      "#\tCrassula capitella\t0\n",
      "#\tCrassula columnaris\t0\n",
      "#\tCrassula deceptor\t0\n",
      "#\tCrassula lactea\t0\n",
      "#\tCrassula nudicaulis\t1\n",
      "#\tCrassula ovata\t0\n",
      "#\tCrassula perforata\t1\n",
      "#\tCrassula socialis\t0\n",
      "#\tDelosperma cooperi\t0\n",
      "#\tDelosperma echinatum\t3\n",
      "#\tDelosperma jansei\t0\n",
      "#\tDelosperma sutherlandii\t0\n",
      "#\tDelosperma tradescantioides\t0\n",
      "#\tDracaena aletriformis\t2\n",
      "#\tDracaena draco\t3\n",
      "#\tDracaena fragrans\t1\n",
      "#\tDracaena mannii\t1\n",
      "#\tDracaena marginata\t0\n",
      "#\tDracaena transvaalensis\t1\n",
      "#\tKalanchoe pinnata\t7\n",
      "\n",
      "total number of accessions fetched: 20\n",
      "\n",
      "\n",
      "downloading 20 records .. processing 1000 accessions per batch\n",
      "\n",
      "[Thu Oct 20 2016 08:09:54]\tbatch: 0000001\t20 unique records downloaded\t-> positives_curated.0000001.gb\t(total: 20 (100.00 %); remaining: 0 -> DONE!)\n",
      "\n",
      "Concatenate all files\t-> positives_curated.gb\n",
      "\n"
     ]
    }
   ],
   "source": [
    "!fetch_from_db.py -t PosList.txt -m rbcl -o positives_curated -@ callumjmacgregor@gmail.com"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now run another BLAST with the same parameters as before, but the curated reference database instead of the local Genbank copy"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "/home/working/AgainstRefDB\n"
     ]
    }
   ],
   "source": [
    "cd AgainstRefDB/"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Create a new Querymap.txt, telling it to go up one directory first (in order to find the trimmed, tag-assigned sequences)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SA177\tfasta\t../SA177/SA177_trimmed.fasta\n",
      "SB140\tfasta\t../SB140/SB140_trimmed.fasta\n",
      "T19\tfasta\t../T19/T19_trimmed.fasta\n",
      "Lg45\tfasta\t../Lg45/Lg45_trimmed.fasta\n",
      "Q69\tfasta\t../Q69/Q69_trimmed.fasta\n",
      "S72\tfasta\t../S72/S72_trimmed.fasta\n",
      "SC66\tfasta\t../SC66/SC66_trimmed.fasta\n",
      "S31\tfasta\t../S31/S31_trimmed.fasta\n",
      "SC74\tfasta\t../SC74/SC74_trimmed.fasta\n",
      "SA68\tfasta\t../SA68/SA68_trimmed.fasta\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "for sample in $(cat ../Querymap_global.txt | cut -f 1)\n",
    "do\n",
    "    fasta=$(ls -1 ../$sample/$sample\\_trimmed.fasta)\n",
    "    echo -e \"$sample\\tfasta\\t$fasta\"\n",
    "done > Querymap.txt\n",
    "\n",
    "head Querymap.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "metaBEAT_global.py \\\n",
    "-Q Querymap.txt \\\n",
    "--cluster \\\n",
    "--clust_match 1 \\\n",
    "--clust_cov 5 \\\n",
    "--blast \\\n",
    "--REFlist REFmap.txt \\\n",
    "--min_ident 0.95 \\\n",
    "-n 5 \\\n",
    "-v \\\n",
    "-@ callumjmacgregor@gmail.com \\\n",
    "-o MothPollenReflistRestart > log_reflist"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Again we want a .txt file to inspect"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "blastn -query /home/working/AgainstRefDB/GLOBAL/global_centroids.fasta \\\n",
    "-db /home/working/AgainstRefDB/GLOBAL/BLAST_0.95/marker_blast_db \\\n",
    "-out GLOBAL/BLAST_0.95/global_blastn.out.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 50,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "USAGE\r\n",
      "  blastn [-h] [-help] [-import_search_strategy filename]\r\n",
      "    [-export_search_strategy filename] [-task task_name] [-db database_name]\r\n",
      "    [-dbsize num_letters] [-gilist filename] [-seqidlist filename]\r\n",
      "    [-negative_gilist filename] [-entrez_query entrez_query]\r\n",
      "    [-db_soft_mask filtering_algorithm] [-db_hard_mask filtering_algorithm]\r\n",
      "    [-subject subject_input_file] [-subject_loc range] [-query input_file]\r\n",
      "    [-out output_file] [-evalue evalue] [-word_size int_value]\r\n",
      "    [-gapopen open_penalty] [-gapextend extend_penalty]\r\n",
      "    [-perc_identity float_value] [-xdrop_ungap float_value]\r\n",
      "    [-xdrop_gap float_value] [-xdrop_gap_final float_value]\r\n",
      "    [-searchsp int_value] [-max_hsps_per_subject int_value] [-penalty penalty]\r\n",
      "    [-reward reward] [-no_greedy] [-min_raw_gapped_score int_value]\r\n",
      "    [-template_type type] [-template_length int_value] [-dust DUST_options]\r\n",
      "    [-filtering_db filtering_database]\r\n",
      "    [-window_masker_taxid window_masker_taxid]\r\n",
      "    [-window_masker_db window_masker_db] [-soft_masking soft_masking]\r\n",
      "    [-ungapped] [-culling_limit int_value] [-best_hit_overhang float_value]\r\n",
      "    [-best_hit_score_edge float_value] [-window_size int_value]\r\n",
      "    [-off_diagonal_range int_value] [-use_index boolean] [-index_name string]\r\n",
      "    [-lcase_masking] [-query_loc range] [-strand strand] [-parse_deflines]\r\n",
      "    [-outfmt format] [-show_gis] [-num_descriptions int_value]\r\n",
      "    [-num_alignments int_value] [-html] [-max_target_seqs num_sequences]\r\n",
      "    [-num_threads int_value] [-remote] [-version]\r\n",
      "\r\n",
      "DESCRIPTION\r\n",
      "   Nucleotide-Nucleotide BLAST 2.2.28+\r\n",
      "\r\n",
      "OPTIONAL ARGUMENTS\r\n",
      " -h\r\n",
      "   Print USAGE and DESCRIPTION;  ignore all other parameters\r\n",
      " -help\r\n",
      "   Print USAGE, DESCRIPTION and ARGUMENTS; ignore all other parameters\r\n",
      " -version\r\n",
      "   Print version number;  ignore other arguments\r\n",
      "\r\n",
      " *** Input query options\r\n",
      " -query <File_In>\r\n",
      "   Input file name\r\n",
      "   Default = `-'\r\n",
      " -query_loc <String>\r\n",
      "   Location on the query sequence in 1-based offsets (Format: start-stop)\r\n",
      " -strand <String, `both', `minus', `plus'>\r\n",
      "   Query strand(s) to search against database/subject\r\n",
      "   Default = `both'\r\n",
      "\r\n",
      " *** General search options\r\n",
      " -task <String, Permissible values: 'blastn' 'blastn-short' 'dc-megablast'\r\n",
      "                'megablast' 'rmblastn' >\r\n",
      "   Task to execute\r\n",
      "   Default = `megablast'\r\n",
      " -db <String>\r\n",
      "   BLAST database name\r\n",
      "    * Incompatible with:  subject, subject_loc\r\n",
      " -out <File_Out>\r\n",
      "   Output file name\r\n",
      "   Default = `-'\r\n",
      " -evalue <Real>\r\n",
      "   Expectation value (E) threshold for saving hits \r\n",
      "   Default = `10'\r\n",
      " -word_size <Integer, >=4>\r\n",
      "   Word size for wordfinder algorithm (length of best perfect match)\r\n",
      " -gapopen <Integer>\r\n",
      "   Cost to open a gap\r\n",
      " -gapextend <Integer>\r\n",
      "   Cost to extend a gap\r\n",
      " -penalty <Integer, <=0>\r\n",
      "   Penalty for a nucleotide mismatch\r\n",
      " -reward <Integer, >=0>\r\n",
      "   Reward for a nucleotide match\r\n",
      " -use_index <Boolean>\r\n",
      "   Use MegaBLAST database index\r\n",
      " -index_name <String>\r\n",
      "   MegaBLAST database index name\r\n",
      "\r\n",
      " *** BLAST-2-Sequences options\r\n",
      " -subject <File_In>\r\n",
      "   Subject sequence(s) to search\r\n",
      "    * Incompatible with:  db, gilist, seqidlist, negative_gilist,\r\n",
      "   db_soft_mask, db_hard_mask\r\n",
      " -subject_loc <String>\r\n",
      "   Location on the subject sequence in 1-based offsets (Format: start-stop)\r\n",
      "    * Incompatible with:  db, gilist, seqidlist, negative_gilist,\r\n",
      "   db_soft_mask, db_hard_mask, remote\r\n",
      "\r\n",
      " *** Formatting options\r\n",
      " -outfmt <String>\r\n",
      "   alignment view options:\r\n",
      "     0 = pairwise,\r\n",
      "     1 = query-anchored showing identities,\r\n",
      "     2 = query-anchored no identities,\r\n",
      "     3 = flat query-anchored, show identities,\r\n",
      "     4 = flat query-anchored, no identities,\r\n",
      "     5 = XML Blast output,\r\n",
      "     6 = tabular,\r\n",
      "     7 = tabular with comment lines,\r\n",
      "     8 = Text ASN.1,\r\n",
      "     9 = Binary ASN.1,\r\n",
      "    10 = Comma-separated values,\r\n",
      "    11 = BLAST archive format (ASN.1) \r\n",
      "   \r\n",
      "   Options 6, 7, and 10 can be additionally configured to produce\r\n",
      "   a custom format specified by space delimited format specifiers.\r\n",
      "   The supported format specifiers are:\r\n",
      "   \t    qseqid means Query Seq-id\r\n",
      "   \t       qgi means Query GI\r\n",
      "   \t      qacc means Query accesion\r\n",
      "   \t   qaccver means Query accesion.version\r\n",
      "   \t      qlen means Query sequence length\r\n",
      "   \t    sseqid means Subject Seq-id\r\n",
      "   \t sallseqid means All subject Seq-id(s), separated by a ';'\r\n",
      "   \t       sgi means Subject GI\r\n",
      "   \t    sallgi means All subject GIs\r\n",
      "   \t      sacc means Subject accession\r\n",
      "   \t   saccver means Subject accession.version\r\n",
      "   \t   sallacc means All subject accessions\r\n",
      "   \t      slen means Subject sequence length\r\n",
      "   \t    qstart means Start of alignment in query\r\n",
      "   \t      qend means End of alignment in query\r\n",
      "   \t    sstart means Start of alignment in subject\r\n",
      "   \t      send means End of alignment in subject\r\n",
      "   \t      qseq means Aligned part of query sequence\r\n",
      "   \t      sseq means Aligned part of subject sequence\r\n",
      "   \t    evalue means Expect value\r\n",
      "   \t  bitscore means Bit score\r\n",
      "   \t     score means Raw score\r\n",
      "   \t    length means Alignment length\r\n",
      "   \t    pident means Percentage of identical matches\r\n",
      "   \t    nident means Number of identical matches\r\n",
      "   \t  mismatch means Number of mismatches\r\n",
      "   \t  positive means Number of positive-scoring matches\r\n",
      "   \t   gapopen means Number of gap openings\r\n",
      "   \t      gaps means Total number of gaps\r\n",
      "   \t      ppos means Percentage of positive-scoring matches\r\n",
      "   \t    frames means Query and subject frames separated by a '/'\r\n",
      "   \t    qframe means Query frame\r\n",
      "   \t    sframe means Subject frame\r\n",
      "   \t      btop means Blast traceback operations (BTOP)\r\n",
      "   \t   staxids means Subject Taxonomy ID(s), separated by a ';'\r\n",
      "   \t sscinames means Subject Scientific Name(s), separated by a ';'\r\n",
      "   \t scomnames means Subject Common Name(s), separated by a ';'\r\n",
      "   \tsblastnames means Subject Blast Name(s), separated by a ';'\r\n",
      "   \t\t\t (in alphabetical order)\r\n",
      "   \tsskingdoms means Subject Super Kingdom(s), separated by a ';'\r\n",
      "   \t\t\t (in alphabetical order) \r\n",
      "   \t    stitle means Subject Title\r\n",
      "   \tsalltitles means All Subject Title(s), separated by a '<>'\r\n",
      "   \t   sstrand means Subject Strand\r\n",
      "   \t     qcovs means Query Coverage Per Subject\r\n",
      "   \t   qcovhsp means Query Coverage Per HSP\r\n",
      "   When not provided, the default value is:\r\n",
      "   'qseqid sseqid pident length mismatch gapopen qstart qend sstart send\r\n",
      "   evalue bitscore', which is equivalent to the keyword 'std'\r\n",
      "   Default = `0'\r\n",
      " -show_gis\r\n",
      "   Show NCBI GIs in deflines?\r\n",
      " -num_descriptions <Integer, >=0>\r\n",
      "   Number of database sequences to show one-line descriptions for\r\n",
      "   Not applicable for outfmt > 4\r\n",
      "   Default = `500'\r\n",
      "    * Incompatible with:  max_target_seqs\r\n",
      " -num_alignments <Integer, >=0>\r\n",
      "   Number of database sequences to show alignments for\r\n",
      "   Default = `250'\r\n",
      "    * Incompatible with:  max_target_seqs\r\n",
      " -html\r\n",
      "   Produce HTML output?\r\n",
      "\r\n",
      " *** Query filtering options\r\n",
      " -dust <String>\r\n",
      "   Filter query sequence with DUST (Format: 'yes', 'level window linker', or\r\n",
      "   'no' to disable)\r\n",
      "   Default = `20 64 1'\r\n",
      " -filtering_db <String>\r\n",
      "   BLAST database containing filtering elements (i.e.: repeats)\r\n",
      " -window_masker_taxid <Integer>\r\n",
      "   Enable WindowMasker filtering using a Taxonomic ID\r\n",
      " -window_masker_db <String>\r\n",
      "   Enable WindowMasker filtering using this repeats database.\r\n",
      " -soft_masking <Boolean>\r\n",
      "   Apply filtering locations as soft masks\r\n",
      "   Default = `true'\r\n",
      " -lcase_masking\r\n",
      "   Use lower case filtering in query and subject sequence(s)?\r\n",
      "\r\n",
      " *** Restrict search or results\r\n",
      " -gilist <String>\r\n",
      "   Restrict search of database to list of GI's\r\n",
      "    * Incompatible with:  negative_gilist, seqidlist, remote, subject,\r\n",
      "   subject_loc\r\n",
      " -seqidlist <String>\r\n",
      "   Restrict search of database to list of SeqId's\r\n",
      "    * Incompatible with:  gilist, negative_gilist, remote, subject,\r\n",
      "   subject_loc\r\n",
      " -negative_gilist <String>\r\n",
      "   Restrict search of database to everything except the listed GIs\r\n",
      "    * Incompatible with:  gilist, seqidlist, remote, subject, subject_loc\r\n",
      " -entrez_query <String>\r\n",
      "   Restrict search with the given Entrez query\r\n",
      "    * Requires:  remote\r\n",
      " -db_soft_mask <String>\r\n",
      "   Filtering algorithm ID to apply to the BLAST database as soft masking\r\n",
      "    * Incompatible with:  db_hard_mask, subject, subject_loc\r\n",
      " -db_hard_mask <String>\r\n",
      "   Filtering algorithm ID to apply to the BLAST database as hard masking\r\n",
      "    * Incompatible with:  db_soft_mask, subject, subject_loc\r\n",
      " -perc_identity <Real, 0..100>\r\n",
      "   Percent identity\r\n",
      " -culling_limit <Integer, >=0>\r\n",
      "   If the query range of a hit is enveloped by that of at least this many\r\n",
      "   higher-scoring hits, delete the hit\r\n",
      "    * Incompatible with:  best_hit_overhang, best_hit_score_edge\r\n",
      " -best_hit_overhang <Real, (>=0 and =<0.5)>\r\n",
      "   Best Hit algorithm overhang value (recommended value: 0.1)\r\n",
      "    * Incompatible with:  culling_limit\r\n",
      " -best_hit_score_edge <Real, (>=0 and =<0.5)>\r\n",
      "   Best Hit algorithm score edge value (recommended value: 0.1)\r\n",
      "    * Incompatible with:  culling_limit\r\n",
      " -max_target_seqs <Integer, >=1>\r\n",
      "   Maximum number of aligned sequences to keep \r\n",
      "   Not applicable for outfmt <= 4\r\n",
      "   Default = `500'\r\n",
      "    * Incompatible with:  num_descriptions, num_alignments\r\n",
      "\r\n",
      " *** Discontiguous MegaBLAST options\r\n",
      " -template_type <String, `coding', `coding_and_optimal', `optimal'>\r\n",
      "   Discontiguous MegaBLAST template type\r\n",
      "    * Requires:  template_length\r\n",
      " -template_length <Integer, Permissible values: '16' '18' '21' >\r\n",
      "   Discontiguous MegaBLAST template length\r\n",
      "    * Requires:  template_type\r\n",
      "\r\n",
      " *** Statistical options\r\n",
      " -dbsize <Int8>\r\n",
      "   Effective length of the database \r\n",
      " -searchsp <Int8, >=0>\r\n",
      "   Effective length of the search space\r\n",
      " -max_hsps_per_subject <Integer, >=0>\r\n",
      "   Override maximum number of HSPs per subject to save for ungapped searches\r\n",
      "   (0 means do not override)\r\n",
      "   Default = `0'\r\n",
      "\r\n",
      " *** Search strategy options\r\n",
      " -import_search_strategy <File_In>\r\n",
      "   Search strategy to use\r\n",
      "    * Incompatible with:  export_search_strategy\r\n",
      " -export_search_strategy <File_Out>\r\n",
      "   File name to record the search strategy used\r\n",
      "    * Incompatible with:  import_search_strategy\r\n",
      "\r\n",
      " *** Extension options\r\n",
      " -xdrop_ungap <Real>\r\n",
      "   X-dropoff value (in bits) for ungapped extensions\r\n",
      " -xdrop_gap <Real>\r\n",
      "   X-dropoff value (in bits) for preliminary gapped extensions\r\n",
      " -xdrop_gap_final <Real>\r\n",
      "   X-dropoff value (in bits) for final gapped alignment\r\n",
      " -no_greedy\r\n",
      "   Use non-greedy dynamic programming extension\r\n",
      " -min_raw_gapped_score <Integer>\r\n",
      "   Minimum raw gapped score to keep an alignment in the preliminary gapped and\r\n",
      "   traceback stages\r\n",
      " -ungapped\r\n",
      "   Perform ungapped alignment only?\r\n",
      " -window_size <Integer, >=0>\r\n",
      "   Multiple hits window size, use 0 to specify 1-hit algorithm\r\n",
      " -off_diagonal_range <Integer, >=0>\r\n",
      "   Number of off-diagonals to search for the 2nd hit, use 0 to turn off\r\n",
      "   Default = `0'\r\n",
      "\r\n",
      " *** Miscellaneous options\r\n",
      " -parse_deflines\r\n",
      "   Should the query and subject defline(s) be parsed?\r\n",
      " -num_threads <Integer, >=1>\r\n",
      "   Number of threads (CPUs) to use in the BLAST search\r\n",
      "   Default = `1'\r\n",
      "    * Incompatible with:  remote\r\n",
      " -remote\r\n",
      "   Execute search remotely?\r\n",
      "    * Incompatible with:  gilist, seqidlist, negative_gilist, subject_loc,\r\n",
      "   num_threads\r\n",
      "\r\n"
     ]
    }
   ],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "metaBEAT_global.py \\\n",
    "-Q Querymap.txt \\\n",
    "--cluster \\\n",
    "--clust_match 1 \\\n",
    "--clust_cov 5 \\\n",
    "--blast \\\n",
    "--blast_db Final/nt/nt \\\n",
    "--min_ident 0.95 \\\n",
    "-n 5 \\\n",
    "-v \\\n",
    "-@ callumjmacgregor@gmail.com \\\n",
    "-o MothPollenGenbankRestart > log"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "metaBEAT_global.py \\\n",
    "-B ../GLOBAL/MothPollenGenbankRestart-OTU-denovo.biom \\\n",
    "--REFlist REFmap.txt \\\n",
    "--min_ident 0.95 \\\n",
    "-n 5 \\\n",
    "-v \\\n",
    "-@ callumjmacgregor@gmail.com \\\n",
    "-o MothPollenReflistRestart > log_reflist.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "blastn -query /home/working/AgainstRefDB/GLOBAL/global_centroids.fasta \\\n",
    "-db /home/working/Final/nt/nt \\\n",
    "-out GLOBAL/BLAST_0.95/global_blastn.out.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "%%bash\n",
    "\n",
    "metaBEAT_global.py \\\n",
    "-B GLOBAL/MothPollenGenbank-OTU-denovo.biom \\\n",
    "--blast_xml GLOBAL/BLAST_0.95/global_blastn.out.xml \\\n",
    "--blast --min_ident 0.95 \\\n",
    "-n 5 -v \\\n",
    "-@ callumjmacgregor@gmail.com \\\n",
    "-o MothPollenGenbank > log-assignment_restart.txt"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import metaBEAT_global_misc_functions as mb"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Specified BIOM input format 'json' - ok!\n"
     ]
    }
   ],
   "source": [
    "table = mb.load_BIOM('GLOBAL/BLAST_0.95/MothPollenGenbank-by-taxonomy-readcounts.blast.biom')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "S79.blast\t(0.0191 %)\n",
      "SA98.blast\t(0.1277 %)\n",
      "SB163.blast\t(0.0508 %)\n"
     ]
    }
   ],
   "source": [
    "mb.find_target(target='Ensete_ventricosum', BIOM=table)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
